<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 An Introduction To Viruses | Text For SCI204 Microbiology at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Microbiology." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="5 An Introduction To Viruses | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/microbiology_text_cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Microbiology." />
  <meta name="github-repo" content="nikolaussucher/microbiology-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 An Introduction To Viruses | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Microbiology." />
  <meta name="twitter:image" content="/microbiology_text_cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="eukaryotic-microorganisms.html"/>
<link rel="next" href="microbial-nutrition-ecology-and-growth.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />




<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">SCI204 Microbiology at RCC</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html"><i class="fa fa-check"></i><b>1</b> The Main Themes Of Microbiology</a></li>
<li class="chapter" data-level="2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html"><i class="fa fa-check"></i><b>2</b> Methods Of Studying Microorganisms</a>
<ul>
<li class="chapter" data-level="2.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microscopy"><i class="fa fa-check"></i><b>2.1</b> Microscopy</a></li>
<li class="chapter" data-level="2.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#light-microscopy"><i class="fa fa-check"></i><b>2.2</b> Light Microscopy</a></li>
<li class="chapter" data-level="2.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#the-optical-light-microscope"><i class="fa fa-check"></i><b>2.3</b> The Optical (Light) Microscope</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#simple-microscopeedit"><i class="fa fa-check"></i><b>2.3.1</b> Simple microscope[edit]</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#compound-microscope"><i class="fa fa-check"></i><b>2.3.2</b> Compound microscope</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#main-components-of-the-light-microscope"><i class="fa fa-check"></i><b>2.3.3</b> Main Components of The Light Microscope</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microscope-slide"><i class="fa fa-check"></i><b>2.4</b> Microscope Slide</a></li>
<li class="chapter" data-level="2.5" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#gram-stain"><i class="fa fa-check"></i><b>2.5</b> Gram Stain</a></li>
<li class="chapter" data-level="2.6" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#electron-microscopy"><i class="fa fa-check"></i><b>2.6</b> Electron microscopy</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#transmission-electron-microscope-tem"><i class="fa fa-check"></i><b>2.6.1</b> Transmission electron microscope (TEM)</a></li>
<li class="chapter" data-level="2.6.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#scanning-electron-microscope-sem"><i class="fa fa-check"></i><b>2.6.2</b> Scanning electron microscope (SEM)</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microbiological-culture"><i class="fa fa-check"></i><b>2.7</b> Microbiological Culture</a>
<ul>
<li class="chapter" data-level="2.7.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#growth-medium"><i class="fa fa-check"></i><b>2.7.1</b> Growth Medium</a></li>
<li class="chapter" data-level="2.7.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#minimal-mediaedit"><i class="fa fa-check"></i><b>2.7.2</b> Minimal Media[edit]</a></li>
<li class="chapter" data-level="2.7.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#selective-media"><i class="fa fa-check"></i><b>2.7.3</b> Selective media</a></li>
<li class="chapter" data-level="2.7.4" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#differential-media"><i class="fa fa-check"></i><b>2.7.4</b> Differential Media</a></li>
<li class="chapter" data-level="2.7.5" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#agar-plate"><i class="fa fa-check"></i><b>2.7.5</b> Agar Plate</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html"><i class="fa fa-check"></i><b>3</b> Prokaryotic Microorganisms</a>
<ul>
<li class="chapter" data-level="3.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#bacteria"><i class="fa fa-check"></i><b>3.1</b> Bacteria</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#intracellular-structure"><i class="fa fa-check"></i><b>3.1.1</b> Intracellular Structure</a></li>
<li class="chapter" data-level="3.1.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#extracellular-structures"><i class="fa fa-check"></i><b>3.1.2</b> Extracellular Structures</a></li>
<li class="chapter" data-level="3.1.3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#endospores"><i class="fa fa-check"></i><b>3.1.3</b> Endospores</a></li>
<li class="chapter" data-level="3.1.4" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#metabolism"><i class="fa fa-check"></i><b>3.1.4</b> Metabolism</a></li>
<li class="chapter" data-level="3.1.5" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#growth-and-reproduction"><i class="fa fa-check"></i><b>3.1.5</b> Growth and reproduction</a></li>
<li class="chapter" data-level="3.1.6" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#genetics"><i class="fa fa-check"></i><b>3.1.6</b> Genetics</a></li>
<li class="chapter" data-level="3.1.7" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#movement"><i class="fa fa-check"></i><b>3.1.7</b> Movement</a></li>
<li class="chapter" data-level="3.1.8" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#communication"><i class="fa fa-check"></i><b>3.1.8</b> Communication</a></li>
<li class="chapter" data-level="3.1.9" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#classification-and-identification"><i class="fa fa-check"></i><b>3.1.9</b> Classification and identification</a></li>
<li class="chapter" data-level="3.1.10" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#interactions-with-other-organisms"><i class="fa fa-check"></i><b>3.1.10</b> Interactions With Other Organisms</a></li>
<li class="chapter" data-level="3.1.11" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#bacteria-as-pathogens"><i class="fa fa-check"></i><b>3.1.11</b> Bacteria as Pathogens</a></li>
<li class="chapter" data-level="3.1.12" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#significance-in-technology-and-industry"><i class="fa fa-check"></i><b>3.1.12</b> Significance in Technology And Industry</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#archaea"><i class="fa fa-check"></i><b>3.2</b> Archaea</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#origin-and-evolution"><i class="fa fa-check"></i><b>3.2.1</b> Origin and evolution</a></li>
<li class="chapter" data-level="3.2.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#relationship-to-bacteria"><i class="fa fa-check"></i><b>3.2.2</b> Relationship to Bacteria</a></li>
<li class="chapter" data-level="3.2.3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#relation-to-eukaryotes"><i class="fa fa-check"></i><b>3.2.3</b> Relation to Eukaryotes</a></li>
<li class="chapter" data-level="3.2.4" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#morphology"><i class="fa fa-check"></i><b>3.2.4</b> Morphology</a></li>
<li class="chapter" data-level="3.2.5" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#structure-composition-development-and-operation"><i class="fa fa-check"></i><b>3.2.5</b> Structure, Composition Development, And Operation</a></li>
<li class="chapter" data-level="3.2.6" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#cell-wall-and-flagella"><i class="fa fa-check"></i><b>3.2.6</b> Cell Wall And Flagella</a></li>
<li class="chapter" data-level="3.2.7" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#membranes"><i class="fa fa-check"></i><b>3.2.7</b> Membranes</a></li>
<li class="chapter" data-level="3.2.8" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#metabolism-1"><i class="fa fa-check"></i><b>3.2.8</b> Metabolism</a></li>
<li class="chapter" data-level="3.2.9" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#genetics-1"><i class="fa fa-check"></i><b>3.2.9</b> Genetics</a></li>
<li class="chapter" data-level="3.2.10" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#ecology"><i class="fa fa-check"></i><b>3.2.10</b> Ecology</a></li>
<li class="chapter" data-level="3.2.11" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#significance-in-technology-and-industry-1"><i class="fa fa-check"></i><b>3.2.11</b> Significance in Technology And Industry</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html"><i class="fa fa-check"></i><b>4</b> Eukaryotic Microorganisms</a>
<ul>
<li class="chapter" data-level="4.1" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#protists"><i class="fa fa-check"></i><b>4.1</b> Protists</a></li>
<li class="chapter" data-level="4.2" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#protozoa"><i class="fa fa-check"></i><b>4.2</b> Protozoa</a></li>
<li class="chapter" data-level="4.3" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#flagellates"><i class="fa fa-check"></i><b>4.3</b> Flagellates</a></li>
<li class="chapter" data-level="4.4" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#ciliates"><i class="fa fa-check"></i><b>4.4</b> Ciliates</a></li>
<li class="chapter" data-level="4.5" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#amoebozoa"><i class="fa fa-check"></i><b>4.5</b> Amoebozoa</a></li>
<li class="chapter" data-level="4.6" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#algae"><i class="fa fa-check"></i><b>4.6</b> Algae</a></li>
<li class="chapter" data-level="4.7" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#slime-molds"><i class="fa fa-check"></i><b>4.7</b> Slime Molds</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#reproduction-of-protists"><i class="fa fa-check"></i><b>4.7.1</b> Reproduction of Protists</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html"><i class="fa fa-check"></i><b>5</b> An Introduction To Viruses</a>
<ul>
<li class="chapter" data-level="5.0.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#role-in-human-disease"><i class="fa fa-check"></i><b>5.0.1</b> Role in human disease</a></li>
<li class="chapter" data-level="5.0.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#animal-viruses"><i class="fa fa-check"></i><b>5.0.2</b> Animal viruses</a></li>
<li class="chapter" data-level="5.0.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#plant-viruses"><i class="fa fa-check"></i><b>5.0.3</b> Plant viruses</a></li>
<li class="chapter" data-level="5.0.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#bacterial-viruses"><i class="fa fa-check"></i><b>5.0.4</b> Bacterial viruses</a></li>
<li class="chapter" data-level="5.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#classification"><i class="fa fa-check"></i><b>5.1</b> Classification</a></li>
<li class="chapter" data-level="5.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#baltimore-classification"><i class="fa fa-check"></i><b>5.2</b> Baltimore classification</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html"><i class="fa fa-check"></i><b>6</b> Microbial Nutrition, Ecology And Growth</a>
<ul>
<li class="chapter" data-level="6.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#thermodynamics-of-living-organisms"><i class="fa fa-check"></i><b>6.1</b> Thermodynamics of Living Organisms</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html"><i class="fa fa-check"></i><b>7</b> Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="7.0.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#catabolism"><i class="fa fa-check"></i><b>7.0.1</b> Catabolism</a></li>
<li class="chapter" data-level="7.0.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#digestion"><i class="fa fa-check"></i><b>7.0.2</b> Digestion</a></li>
<li class="chapter" data-level="7.0.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-organic-compounds"><i class="fa fa-check"></i><b>7.0.3</b> Energy From Organic Compounds</a></li>
<li class="chapter" data-level="7.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-transformations"><i class="fa fa-check"></i><b>7.1</b> Energy Transformations</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-phosphorylation"><i class="fa fa-check"></i><b>7.1.1</b> Oxidative Phosphorylation</a></li>
<li class="chapter" data-level="7.1.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-inorganic-compounds"><i class="fa fa-check"></i><b>7.1.2</b> Energy From Inorganic Compounds</a></li>
<li class="chapter" data-level="7.1.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-light"><i class="fa fa-check"></i><b>7.1.3</b> Energy From Light</a></li>
<li class="chapter" data-level="7.1.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#anabolism"><i class="fa fa-check"></i><b>7.1.4</b> Anabolism</a></li>
<li class="chapter" data-level="7.1.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbon-fixation"><i class="fa fa-check"></i><b>7.1.5</b> Carbon Fixation</a></li>
<li class="chapter" data-level="7.1.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbohydrates-and-glycans"><i class="fa fa-check"></i><b>7.1.6</b> Carbohydrates And Glycans</a></li>
<li class="chapter" data-level="7.1.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fatty-acids-isoprenoids-and-sterol"><i class="fa fa-check"></i><b>7.1.7</b> Fatty Acids, Isoprenoids and Sterol</a></li>
<li class="chapter" data-level="7.1.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#proteins"><i class="fa fa-check"></i><b>7.1.8</b> Proteins</a></li>
<li class="chapter" data-level="7.1.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#nucleotide-synthesis-and-salvage"><i class="fa fa-check"></i><b>7.1.9</b> Nucleotide Synthesis And Salvage</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#heterotrophic-microbial-metabolism"><i class="fa fa-check"></i><b>7.2</b> Heterotrophic microbial metabolism</a></li>
<li class="chapter" data-level="7.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fermentation"><i class="fa fa-check"></i><b>7.3</b> Fermentation</a></li>
<li class="chapter" data-level="7.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#chemolithotrophy"><i class="fa fa-check"></i><b>7.4</b> Chemolithotrophy</a></li>
<li class="chapter" data-level="7.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#phototrophy"><i class="fa fa-check"></i><b>7.5</b> Phototrophy</a></li>
<li class="chapter" data-level="7.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#enzymes"><i class="fa fa-check"></i><b>7.6</b> Enzymes</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#allosteric-modulation"><i class="fa fa-check"></i><b>7.6.1</b> Allosteric Modulation</a></li>
<li class="chapter" data-level="7.6.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cofactors-of-enzymes-and-coenzymes"><i class="fa fa-check"></i><b>7.6.2</b> Cofactors of Enzymes and Coenzymes</a></li>
<li class="chapter" data-level="7.6.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#inhibition-of-enzymes"><i class="fa fa-check"></i><b>7.6.3</b> Inhibition of Enzymes</a></li>
<li class="chapter" data-level="7.6.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#factors-affecting-enzyme-activity"><i class="fa fa-check"></i><b>7.6.4</b> Factors Affecting Enzyme Activity</a></li>
<li class="chapter" data-level="7.6.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#control-of-activity"><i class="fa fa-check"></i><b>7.6.5</b> Control of activity</a></li>
<li class="chapter" data-level="7.6.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#post-translational-modification"><i class="fa fa-check"></i><b>7.6.6</b> Post-translational Modification</a></li>
<li class="chapter" data-level="7.6.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#quantity"><i class="fa fa-check"></i><b>7.6.7</b> Quantity</a></li>
<li class="chapter" data-level="7.6.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#subcellular-distribution"><i class="fa fa-check"></i><b>7.6.8</b> Subcellular Distribution</a></li>
<li class="chapter" data-level="7.6.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#organ-specialization"><i class="fa fa-check"></i><b>7.6.9</b> Organ Specialization</a></li>
<li class="chapter" data-level="7.6.10" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#coenzymes"><i class="fa fa-check"></i><b>7.6.10</b> Coenzymes</a></li>
<li class="chapter" data-level="7.6.11" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#mineral-cofactors"><i class="fa fa-check"></i><b>7.6.11</b> Mineral Cofactors</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthesis-and-cellular-respiration"><i class="fa fa-check"></i><b>7.7</b> Photosynthesis And Cellular Respiration</a></li>
<li class="chapter" data-level="7.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#the-electron-transport-chain-and-chemiosmosis"><i class="fa fa-check"></i><b>7.8</b> The Electron Transport Chain And Chemiosmosis</a></li>
<li class="chapter" data-level="7.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthesis"><i class="fa fa-check"></i><b>7.9</b> Photosynthesis</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthetic-membranes-and-organelles"><i class="fa fa-check"></i><b>7.9.1</b> Photosynthetic Membranes And Organelles</a></li>
<li class="chapter" data-level="7.9.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#z-scheme"><i class="fa fa-check"></i><b>7.9.2</b> Z Scheme</a></li>
<li class="chapter" data-level="7.9.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#water-photolysis"><i class="fa fa-check"></i><b>7.9.3</b> Water Photolysis</a></li>
<li class="chapter" data-level="7.9.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#calvin-cycle"><i class="fa fa-check"></i><b>7.9.4</b> Calvin Cycle</a></li>
<li class="chapter" data-level="7.9.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbon-dioxide-levels-and-photorespiration"><i class="fa fa-check"></i><b>7.9.5</b> Carbon Dioxide Levels And Photorespiration</a></li>
<li class="chapter" data-level="7.9.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#c4-carbon-fixation"><i class="fa fa-check"></i><b>7.9.6</b> C4 Carbon Fixation</a></li>
<li class="chapter" data-level="7.9.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cam-photosynthesis"><i class="fa fa-check"></i><b>7.9.7</b> CAM Photosynthesis</a></li>
</ul></li>
<li class="chapter" data-level="7.10" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#symbiosis-and-the-origin-of-chloroplasts"><i class="fa fa-check"></i><b>7.10</b> Symbiosis And The Origin of Chloroplasts</a></li>
<li class="chapter" data-level="7.11" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cyanobacteria-and-the-evolution-of-photosynthesis"><i class="fa fa-check"></i><b>7.11</b> Cyanobacteria And The Evolution of Photosynthesis</a></li>
<li class="chapter" data-level="7.12" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cellular-respiration"><i class="fa fa-check"></i><b>7.12</b> Cellular Respiration</a>
<ul>
<li class="chapter" data-level="7.12.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#aerobic-respiration"><i class="fa fa-check"></i><b>7.12.1</b> Aerobic Respiration</a></li>
<li class="chapter" data-level="7.12.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#glycolysis"><i class="fa fa-check"></i><b>7.12.2</b> Glycolysis</a></li>
<li class="chapter" data-level="7.12.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-decarboxylation-of-pyruvate"><i class="fa fa-check"></i><b>7.12.3</b> Oxidative Decarboxylation of Pyruvate</a></li>
<li class="chapter" data-level="7.12.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#the-citric-acid-cycle"><i class="fa fa-check"></i><b>7.12.4</b> The Citric Acid Cycle</a></li>
<li class="chapter" data-level="7.12.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-phosphorylation-1"><i class="fa fa-check"></i><b>7.12.5</b> Oxidative Phosphorylation</a></li>
</ul></li>
<li class="chapter" data-level="7.13" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fermentation-1"><i class="fa fa-check"></i><b>7.13</b> Fermentation</a></li>
<li class="chapter" data-level="7.14" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#anaerobic-respiration"><i class="fa fa-check"></i><b>7.14</b> Anaerobic Respiration</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="microbial-genetics.html"><a href="microbial-genetics.html"><i class="fa fa-check"></i><b>8</b> Microbial Genetics</a></li>
<li class="chapter" data-level="9" data-path="genetic-engineering.html"><a href="genetic-engineering.html"><i class="fa fa-check"></i><b>9</b> Genetic Engineering</a>
<ul>
<li class="chapter" data-level="9.1" data-path="genetic-engineering.html"><a href="genetic-engineering.html#creatin-a-genetically-modified-organism-gmo"><i class="fa fa-check"></i><b>9.1</b> Creatin A Genetically Modified Organism (GMO)</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html"><i class="fa fa-check"></i><b>10</b> Physical And Chemical Agents for Microbial Control</a>
<ul>
<li class="chapter" data-level="10.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#disinfectant"><i class="fa fa-check"></i><b>10.1</b> Disinfectant</a></li>
<li class="chapter" data-level="10.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antiseptic"><i class="fa fa-check"></i><b>10.2</b> Antiseptic</a></li>
<li class="chapter" data-level="10.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#sterilization"><i class="fa fa-check"></i><b>10.3</b> Sterilization</a></li>
<li class="chapter" data-level="10.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#pasteurization"><i class="fa fa-check"></i><b>10.4</b> Pasteurization</a></li>
<li class="chapter" data-level="10.5" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antibiotics"><i class="fa fa-check"></i><b>10.5</b> Antibiotics</a></li>
<li class="chapter" data-level="10.6" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#classes-of-antibiotics"><i class="fa fa-check"></i><b>10.6</b> Classes of Antibiotics</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html"><i class="fa fa-check"></i><b>11</b> Drugs, Microbes, And The Host</a>
<ul>
<li class="chapter" data-level="11.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antibacterials"><i class="fa fa-check"></i><b>11.1</b> Antibacterials</a></li>
<li class="chapter" data-level="11.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#pathogenic-bacteria"><i class="fa fa-check"></i><b>11.2</b> Pathogenic bacteria</a></li>
<li class="chapter" data-level="11.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#pathogenic-susceptibility"><i class="fa fa-check"></i><b>11.3</b> Pathogenic susceptibility</a></li>
<li class="chapter" data-level="11.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#infection"><i class="fa fa-check"></i><b>11.4</b> Infection</a></li>
<li class="chapter" data-level="11.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#historic-pandemics"><i class="fa fa-check"></i><b>11.5</b> Historic pandemics</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html"><i class="fa fa-check"></i><b>12</b> Host Defenses:The Innate Immune System</a>
<ul>
<li class="chapter" data-level="12.1" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#layered-defenseedit"><i class="fa fa-check"></i><b>12.1</b> Layered defense[edit]</a></li>
<li class="chapter" data-level="12.2" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#surface-barriers"><i class="fa fa-check"></i><b>12.2</b> Surface Barriers</a></li>
<li class="chapter" data-level="12.3" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#innate-immune-system"><i class="fa fa-check"></i><b>12.3</b> Innate immune system</a></li>
<li class="chapter" data-level="12.4" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#anatomical-barriers"><i class="fa fa-check"></i><b>12.4</b> Anatomical barriers</a></li>
<li class="chapter" data-level="12.5" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#inflammation"><i class="fa fa-check"></i><b>12.5</b> Inflammation</a></li>
<li class="chapter" data-level="12.6" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#complement-systemedit"><i class="fa fa-check"></i><b>12.6</b> Complement system[edit]</a></li>
<li class="chapter" data-level="12.7" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#white-blood-cells"><i class="fa fa-check"></i><b>12.7</b> White blood cells</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html"><i class="fa fa-check"></i><b>13</b> Host Defenses: The Adaptive Immune System</a>
<ul>
<li class="chapter" data-level="13.1" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#lymphocytes"><i class="fa fa-check"></i><b>13.1</b> Lymphocytes</a></li>
<li class="chapter" data-level="13.2" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#antigen-presentation"><i class="fa fa-check"></i><b>13.2</b> Antigen presentation</a></li>
<li class="chapter" data-level="13.3" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#exogenous-antigens"><i class="fa fa-check"></i><b>13.3</b> Exogenous antigens</a></li>
<li class="chapter" data-level="13.4" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#endogenous-antigensedit"><i class="fa fa-check"></i><b>13.4</b> Endogenous antigens[edit]</a></li>
<li class="chapter" data-level="13.5" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#t-lymphocytesedit"><i class="fa fa-check"></i><b>13.5</b> T lymphocytes[edit]</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For SCI204 Microbiology at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="an-introduction-to-viruses" class="section level1" number="5">
<h1><span class="header-section-number">5</span> An Introduction To Viruses</h1>
<p>A virus is a submicroscopic infectious agent that replicates only inside the living cells of an organism.[1] Viruses infect all life forms, from animals and plants to microorganisms, including bacteria and archaea.[2][3] Since Dmitri Ivanovsky’s 1892 article describing a non-bacterial pathogen infecting tobacco plants and the discovery of the tobacco mosaic virus by Martinus Beijerinck in 1898,[4] more than 9,000 virus species have been described in detail[5] of the millions of types of viruses in the environment.[6] Viruses are found in almost every ecosystem on Earth and are the most numerous type of biological entity.[7][8] The study of viruses is known as virology, a subspeciality of microbiology.</p>
<p>When infected, a host cell is forced to rapidly produce thousands of copies of the original virus. When not inside an infected cell or in the process of infecting a cell, viruses exist in the form of independent particles, or virions, consisting of (i) the genetic material, i.e., long molecules of DNA or RNA that encode the structure of the proteins by which the virus acts; (ii) a protein coat, the capsid, which surrounds and protects the genetic material; and in some cases (iii) an outside envelope of lipids. The shapes of these virus particles range from simple helical and icosahedral forms to more complex structures. Most virus species have virions too small to be seen with an optical microscope, as they are one-hundredth the size of most bacteria.</p>
<p>The origins of viruses in the evolutionary history of life are unclear: some may have evolved from plasmids—pieces of DNA that can move between cells—while others may have evolved from bacteria. In evolution, viruses are an important means of horizontal gene transfer, which increases genetic diversity in a way analogous to sexual reproduction.[9] Viruses are considered by some biologists to be a life form, because they carry genetic material, reproduce, and evolve through natural selection, although they lack the key characteristics, such as cell structure, that are generally considered necessary criteria for defining life. Because they possess some but not all such qualities, viruses have been described as “organisms at the edge of life”,[10] and as self-replicators.[11]</p>
<p>Viruses spread in many ways. One transmission pathway is through disease-bearing organisms known as vectors: for example, viruses are often transmitted from plant to plant by insects that feed on plant sap, such as aphids; and viruses in animals can be carried by blood-sucking insects. Influenza viruses spread in the air by coughing and sneezing. Norovirus and rotavirus, common causes of viral gastroenteritis, are transmitted by the faecal–oral route, passed by hand-to-mouth contact or in food or water. The infectious dose of norovirus required to produce infection in humans is less than 100 particles.[12] HIV is one of several viruses transmitted through sexual contact and by exposure to infected blood. The variety of host cells that a virus can infect is called its “host range”. This can be narrow, meaning a virus is capable of infecting few species, or broad, meaning it is capable of infecting many.[13]</p>
<p>Viral infections in animals provoke an immune response that usually eliminates the infecting virus. Immune responses can also be produced by vaccines, which confer an artificially acquired immunity to the specific viral infection. Some viruses, including those that cause AIDS, HPV infection, and viral hepatitis, evade these immune responses and result in chronic infections. Several antiviral drugs have been developed.</p>
<p>The word is from the Latin neuter vīrus referring to poison and other noxious liquids, from the same Indo-European base as Sanskrit viṣa, Avestan vīša, and ancient Greek ἰός (all meaning “poison”), first attested in English in 1398 in John Trevisa’s translation of Bartholomeus Anglicus’s De Proprietatibus Rerum.[14][15] Virulent, from Latin virulentus (poisonous), dates to c. 1400.[16][17] A meaning of “agent that causes infectious disease” is first recorded in 1728,[15] long before the discovery of viruses by Dmitri Ivanovsky in 1892. The English plural is viruses (sometimes also vira),[18] whereas the Latin word is a mass noun, which has no classically attested plural (vīra is used in Neo-Latin[19]). The adjective viral dates to 1948.[20] The term virion (plural virions), which dates from 1959,[21] is also used to refer to a single viral particle that is released from the cell and is capable of infecting other cells of the same type.[22]</p>
<p>Louis Pasteur was unable to find a causative agent for rabies and speculated about a pathogen too small to be detected by microscopes.[23] In 1884, the French microbiologist Charles Chamberland invented the Chamberland filter (or Pasteur-Chamberland filter) with pores small enough to remove all bacteria from a solution passed through it.[24] In 1892, the Russian biologist Dmitri Ivanovsky used this filter to study what is now known as the tobacco mosaic virus: crushed leaf extracts from infected tobacco plants remained infectious even after filtration to remove bacteria. Ivanovsky suggested the infection might be caused by a toxin produced by bacteria, but he did not pursue the idea.[25] At the time it was thought that all infectious agents could be retained by filters and grown on a nutrient medium—this was part of the germ theory of disease.[4] In 1898, the Dutch microbiologist Martinus Beijerinck repeated the experiments and became convinced that the filtered solution contained a new form of infectious agent.[26] He observed that the agent multiplied only in cells that were dividing, but as his experiments did not show that it was made of particles, he called it a contagium vivum fluidum (soluble living germ) and reintroduced the word virus. Beijerinck maintained that viruses were liquid in nature, a theory later discredited by Wendell Stanley, who proved they were particulate.[25] In the same year, Friedrich Loeffler and Paul Frosch passed the first animal virus, aphthovirus (the agent of foot-and-mouth disease), through a similar filter.[27]</p>
<p>In the early 20th century, the English bacteriologist Frederick Twort discovered a group of viruses that infect bacteria, now called bacteriophages[28] (or commonly ‘phages’), and the French-Canadian microbiologist Félix d’Herelle described viruses that, when added to bacteria on an agar plate, would produce areas of dead bacteria. He accurately diluted a suspension of these viruses and discovered that the highest dilutions (lowest virus concentrations), rather than killing all the bacteria, formed discrete areas of dead organisms. Counting these areas and multiplying by the dilution factor allowed him to calculate the number of viruses in the original suspension.[29] Phages were heralded as a potential treatment for diseases such as typhoid and cholera, but their promise was forgotten with the development of penicillin. The development of bacterial resistance to antibiotics has renewed interest in the therapeutic use of bacteriophages.[30]</p>
<p>By the end of the 19th century, viruses were defined in terms of their infectivity, their ability to pass filters, and their requirement for living hosts. Viruses had been grown only in plants and animals. In 1906 Ross Granville Harrison invented a method for growing tissue in lymph, and in 1913 E. Steinhardt, C. Israeli, and R.A. Lambert used this method to grow vaccinia virus in fragments of guinea pig corneal tissue.[31] In 1928, H. B. Maitland and M. C. Maitland grew vaccinia virus in suspensions of minced hens’ kidneys. Their method was not widely adopted until the 1950s when poliovirus was grown on a large scale for vaccine production.[32]</p>
<p>Another breakthrough came in 1931 when the American pathologist Ernest William Goodpasture and Alice Miles Woodruff grew influenza and several other viruses in fertilised chicken eggs.[33] In 1949, John Franklin Enders, Thomas Weller, and Frederick Robbins grew poliovirus in cultured cells from aborted human embryonic tissue,[34] the first virus to be grown without using solid animal tissue or eggs. This work enabled Hilary Koprowski, and then Jonas Salk, to make an effective polio vaccine.[35]</p>
<p>The first images of viruses were obtained upon the invention of electron microscopy in 1931 by the German engineers Ernst Ruska and Max Knoll.[36] In 1935, American biochemist and virologist Wendell Meredith Stanley examined the tobacco mosaic virus and found it was mostly made of protein.[37] A short time later, this virus was separated into protein and RNA parts.[38] The tobacco mosaic virus was the first to be crystallised and its structure could, therefore, be elucidated in detail. The first X-ray diffraction pictures of the crystallised virus were obtained by Bernal and Fankuchen in 1941. Based on her X-ray crystallographic pictures, Rosalind Franklin discovered the full structure of the virus in 1955.[39] In the same year, Heinz Fraenkel-Conrat and Robley Williams showed that purified tobacco mosaic virus RNA and its protein coat can assemble by themselves to form functional viruses, suggesting that this simple mechanism was probably the means through which viruses were created within their host cells.[40]</p>
<p>The second half of the 20th century was the golden age of virus discovery, and most of the documented species of animal, plant, and bacterial viruses were discovered during these years.[41] In 1957 equine arterivirus and the cause of Bovine virus diarrhoea (a pestivirus) were discovered. In 1963 the hepatitis B virus was discovered by Baruch Blumberg,[42] and in 1965 Howard Temin described the first retrovirus. Reverse transcriptase, the enzyme that retroviruses use to make DNA copies of their RNA, was first described in 1970 by Temin and David Baltimore independently.[43] In 1983 Luc Montagnier’s team at the Pasteur Institute in France, first isolated the retrovirus now called HIV.[44] In 1989 Michael Houghton’s team at Chiron Corporation discovered Hepatitis C.[45][46]</p>
<p>Viruses are found wherever there is life and have probably existed since living cells first evolved.[47] The origin of viruses is unclear because they do not form fossils, so molecular techniques are used to investigate how they arose.[48] In addition, viral genetic material occasionally integrates into the germline of the host organisms, by which they can be passed on vertically to the offspring of the host for many generations. This provides an invaluable source of information for paleovirologists to trace back ancient viruses that have existed up to millions of years ago. There are three main hypotheses that aim to explain the origins of viruses:[49][50]</p>
<p>Regressive hypothesis
Viruses may have once been small cells that parasitised larger cells. Over time, genes not required by their parasitism were lost. The bacteria rickettsia and chlamydia are living cells that, like viruses, can reproduce only inside host cells. They lend support to this hypothesis, as their dependence on parasitism is likely to have caused the loss of genes that enabled them to survive outside a cell. This is also called the ‘degeneracy hypothesis’,[51][52] or ‘reduction hypothesis’.[53]
Cellular origin hypothesis
Some viruses may have evolved from bits of DNA or RNA that “escaped” from the genes of a larger organism. The escaped DNA could have come from plasmids (pieces of naked DNA that can move between cells) or transposons (molecules of DNA that replicate and move around to different positions within the genes of the cell).[54] Once called “jumping genes”, transposons are examples of mobile genetic elements and could be the origin of some viruses. They were discovered in maize by Barbara McClintock in 1950.[55] This is sometimes called the ‘vagrancy hypothesis’,[51][56] or the ‘escape hypothesis’.[53]
Co-evolution hypothesis
This is also called the ‘virus-first hypothesis’[53] and proposes that viruses may have evolved from complex molecules of protein and nucleic acid at the same time that cells first appeared on Earth and would have been dependent on cellular life for billions of years. Viroids are molecules of RNA that are not classified as viruses because they lack a protein coat. They have characteristics that are common to several viruses and are often called subviral agents.[57] Viroids are important pathogens of plants.[58] They do not code for proteins but interact with the host cell and use the host machinery for their replication.[59] The hepatitis delta virus of humans has an RNA genome similar to viroids but has a protein coat derived from hepatitis B virus and cannot produce one of its own. It is, therefore, a defective virus. Although hepatitis delta virus genome may replicate independently once inside a host cell, it requires the help of hepatitis B virus to provide a protein coat so that it can be transmitted to new cells.[60] In similar manner, the sputnik virophage is dependent on mimivirus, which infects the protozoan Acanthamoeba castellanii.[61] These viruses, which are dependent on the presence of other virus species in the host cell, are called ‘satellites’ and may represent evolutionary intermediates of viroids and viruses.[62][63]
In the past, there were problems with all of these hypotheses: the regressive hypothesis did not explain why even the smallest of cellular parasites do not resemble viruses in any way. The escape hypothesis did not explain the complex capsids and other structures on virus particles. The virus-first hypothesis contravened the definition of viruses in that they require host cells.[53] Viruses are now recognised as ancient and as having origins that pre-date the divergence of life into the three domains.[64] This discovery has led modern virologists to reconsider and re-evaluate these three classical hypotheses.[64]</p>
<p>The evidence for an ancestral world of RNA cells[65] and computer analysis of viral and host DNA sequences are giving a better understanding of the evolutionary relationships between different viruses and may help identify the ancestors of modern viruses. To date, such analyses have not proved which of these hypotheses is correct.[65] It seems unlikely that all currently known viruses have a common ancestor, and viruses have probably arisen numerous times in the past by one or more mechanisms.</p>
<p>Viruses are found wherever there is life and have probably existed since living cells first evolved. The origin of viruses is unclear because they do not form fossils, so molecular techniques are used to investigate how they arose. In addition, viral genetic material occasionally integrates into the germline of the host organisms, by which they can be passed on vertically to the offspring of the host for many generations. This provides an invaluable source of information for paleovirologists to trace back ancient viruses that have existed up to millions of years ago. There are three main hypotheses that aim to explain the origins of viruses:</p>
<p>The word is from the Latin neuter vīrus referring to poison and other noxious liquids, from the same Indo-European base as Sanskrit viṣa, Avestan vīša, and ancient Greek ἰός (all meaning “poison”), first attested in English in 1398 in John Trevisa’s translation of Bartholomeus Anglicus’s De Proprietatibus Rerum. Virulent, from Latin virulentus (poisonous), dates to c. 1400. A meaning of “agent that causes infectious disease” is first recorded in 1728, long before the discovery of viruses by Dmitri Ivanovsky in 1892. The English plural is viruses (sometimes also vira) whereas the Latin word is a mass noun, which has no classically attested plural (vīra is used in Neo-Latin). The adjective viral dates to 1948. The term virion (plural virions), which dates from 1959, is also used to refer to a single viral particle that is released from the cell and is capable of infecting other cells of the same type.</p>
<p>Scientific opinions differ on whether viruses are a form of life, or organic structures that interact with living organisms. They have been described as “organisms at the edge of life”, since they resemble organisms in that they possess genes, evolve by natural selection, and reproduce by creating multiple copies of themselves through self-assembly. Although they have genes, they do not have a cellular structure, which is often seen as the basic unit of life. Viruses do not have their own metabolism, and require a host cell to make new products. They therefore cannot naturally reproduce outside a host cell—although bacterial species such as rickettsia and chlamydia are considered living organisms despite the same limitation. Accepted forms of life use cell division to reproduce, whereas viruses spontaneously assemble within cells. They differ from autonomous growth of crystals as they inherit genetic mutations while being subject to natural selection. Virus self-assembly within host cells has implications for the study of the origin of life, as it lends further credence to the hypothesis that life could have started as self-assembling organic molecules.</p>
<p>Viruses display a wide diversity of shapes and sizes, called ‘morphologies’. In general, viruses are much smaller than bacteria. Most viruses that have been studied have a diameter between 20 and 300 nanometres. Some filoviruses have a total length of up to 1400 nm; their diameters are only about 80 nm. Most viruses cannot be seen with an optical microscope, so scanning and transmission electron microscopes are used to visualise them. To increase the contrast between viruses and the background, electron-dense “stains” are used. These are solutions of salts of heavy metals, such as tungsten, that scatter the electrons from regions covered with the stain. When virions are coated with stain (positive staining), fine detail is obscured. Negative staining overcomes this problem by staining the background only.</p>
<p>A complete virus particle, known as a virion, consists of nucleic acid surrounded by a protective coat of protein called a capsid. These are formed from identical protein subunits called capsomeres. Viruses can have a lipid “envelope” derived from the host cell membrane. The capsid is made from proteins encoded by the viral genome and its shape serves as the basis for morphological distinction. Virally-coded protein subunits will self-assemble to form a capsid, in general requiring the presence of the virus genome. Complex viruses code for proteins that assist in the construction of their capsid. Proteins associated with nucleic acid are known as nucleoproteins, and the association of viral capsid proteins with viral nucleic acid is called a nucleocapsid. The capsid and entire virus structure can be mechanically (physically) probed through atomic force microscopy. In general, there are four main morphological virus types:</p>
<p>Helical viruses are composed of a single type of capsomere stacked around a central axis to form a helical structure, which may have a central cavity, or tube. This arrangement results in rod-shaped or filamentous virions which can be short and highly rigid, or long and very flexible. The genetic material (typically single-stranded RNA, but ssDNA in some cases) is bound into the protein helix by interactions between the negatively charged nucleic acid and positive charges on the protein. Overall, the length of a helical capsid is related to the length of the nucleic acid contained within it, and the diameter is dependent on the size and arrangement of capsomeres. The well-studied tobacco mosaic virus is an example of a helical virus.</p>

<div class="figure" style="text-align: center"><span id="fig:tmvirus"></span>
<img src="figures/bacteria/TMV_structure_simple.png" alt="Structure of tobacco mosaic virus: RNA coiled in a helix of repeating protein sub-units" width="70%" />
<p class="caption">
Figure 5.1: <a href="https://commons.wikimedia.org/wiki/File:TMV_structure_simple.png">Structure of tobacco mosaic virus: RNA coiled in a helix of repeating protein sub-units</a>
</p>
</div>
<p>Most animal viruses are icosahedral or near-spherical with chiral icosahedral symmetry. A regular icosahedron is the optimum way of forming a closed shell from identical sub-units. The minimum number of identical capsomeres required for each triangular face is 3, which gives 60 for the icosahedron. Many viruses, such as rotavirus, have more than 60 capsomers and appear spherical but they retain this symmetry. To achieve this, the capsomeres at the apices are surrounded by five other capsomeres and are called pentons. Capsomeres on the triangular faces are surrounded by six others and are called hexons. Hexons are in essence flat and pentons, which form the 12 vertices, are curved. The same protein may act as the subunit of both the pentamers and hexamers or they may be composed of different proteins.</p>

<div class="figure" style="text-align: center"><span id="fig:icosavirus"></span>
<img src="figures/bacteria/CowpeaMosaicVirus3D.png" alt="Structure of an icosahedral cowpea mosaic virus" width="70%" />
<p class="caption">
Figure 5.2: <a href="https://commons.wikimedia.org/wiki/File:CowpeaMosaicVirus3D.png">Structure of an icosahedral cowpea mosaic virus</a>
</p>
</div>
<p>Some species of virus envelop themselves in a modified form of one of the cell membranes, either the outer membrane surrounding an infected host cell or internal membranes such as nuclear membrane or endoplasmic reticulum, thus gaining an outer lipid bilayer known as a viral envelope. This membrane is studded with proteins coded for by the viral genome and host genome; the lipid membrane itself and any carbohydrates present originate entirely from the host. The influenza virus and HIV use this strategy. Most enveloped viruses are dependent on the envelope for their infectivity.</p>
<p>These viruses possess a capsid that is neither purely helical nor purely icosahedral, and that may possess extra structures such as protein tails or a complex outer wall. Some bacteriophages, such as Enterobacteria phage T4, have a complex structure consisting of an icosahedral head bound to a helical tail, which may have a hexagonal base plate with protruding protein tail fibres. This tail structure acts like a molecular syringe, attaching to the bacterial host and then injecting the viral genome into the cell.
The poxviruses are large, complex viruses that have an unusual morphology. The viral genome is associated with proteins within a central disc structure known as a nucleoid. The nucleoid is surrounded by a membrane and two lateral bodies of unknown function. The virus has an outer envelope with a thick layer of protein studded over its surface. The whole virion is slightly pleomorphic, ranging from ovoid to brick-shaped.</p>
<p>Mimivirus is one of the largest characterised viruses, with a capsid diameter of 400 nm. Protein filaments measuring 100 nm project from the surface. The capsid appears hexagonal under an electron microscope, therefore the capsid is probably icosahedral. In 2011, researchers discovered the largest then known virus in samples of water collected from the ocean floor off the coast of Las Cruces, Chile. Provisionally named Megavirus chilensis, it can be seen with a basic optical microscope. In 2013, the Pandoravirus genus was discovered in Chile and Australia, and has genomes about twice as large as Megavirus and Mimivirus. All giant viruses have dsDNA genomes and they are classified into several families: Mimiviridae, Pithoviridae, Pandoraviridae, Phycodnaviridae, and the Mollivirus genus.</p>
<p>Some viruses that infect Archaea have complex structures unrelated to any other form of virus, with a wide variety of unusual shapes, ranging from spindle-shaped structures to viruses that resemble hooked rods, teardrops or even bottles. Other archaeal viruses resemble the tailed bacteriophages, and can have multiple tail structures.</p>
<p>An enormous variety of genomic structures can be seen among viral species; as a group, they contain more structural genomic diversity than plants, animals, archaea, or bacteria. There are millions of different types of viruses, although fewer than 7,000 types have been described in detail. As of September 2015, the NCBI Virus genome database has more than 75,000 complete genome sequences, but there are doubtlessly many more to be discovered.</p>
<p>A virus has either a DNA or an RNA genome and is called a DNA virus or an RNA virus, respectively. The vast majority of viruses have RNA genomes. Plant viruses tend to have single-stranded RNA genomes and bacteriophages tend to have double-stranded DNA genomes.</p>
<p>Viral genomes are circular, as in the polyomaviruses, or linear, as in the adenoviruses. The type of nucleic acid is irrelevant to the shape of the genome. Among RNA viruses and certain DNA viruses, the genome is often divided up into separate parts, in which case it is called segmented. For RNA viruses, each segment often codes for only one protein and they are usually found together in one capsid. All segments are not required to be in the same virion for the virus to be infectious, as demonstrated by brome mosaic virus and several other plant viruses.</p>
<p>A viral genome, irrespective of nucleic acid type, is almost always either single-stranded or double-stranded. Single-stranded genomes consist of an unpaired nucleic acid, analogous to one-half of a ladder split down the middle. Double-stranded genomes consist of two complementary paired nucleic acids, analogous to a ladder. The virus particles of some virus families, such as those belonging to the Hepadnaviridae, contain a genome that is partially double-stranded and partially single-stranded.</p>
<p>For most viruses with RNA genomes and some with single-stranded DNA genomes, the single strands are said to be either positive-sense (called the ‘plus-strand’) or negative-sense (called the ‘minus-strand’), depending on if they are complementary to the viral messenger RNA (mRNA). Positive-sense viral RNA is in the same sense as viral mRNA and thus at least a part of it can be immediately translated by the host cell. Negative-sense viral RNA is complementary to mRNA and thus must be converted to positive-sense RNA by an RNA-dependent RNA polymerase before translation. DNA nomenclature for viruses with single-sense genomic ssDNA is similar to RNA nomenclature, in that positive-strand viral ssDNA is identical in sequence to the viral mRNA and is thus a coding strand, while negative-strand viral ssDNA is complementary to the viral mRNA and is thus a template strand. Several types of ssDNA and ssRNA viruses have genomes that are ambisense in that transcription can occur off both strands in a double-stranded replicative intermediate. Examples include geminiviruses, which are ssDNA plant viruses and arenaviruses, which are ssRNA viruses of animals.</p>

<div class="figure" style="text-align: center"><span id="fig:influenza"></span>
<img src="figures/bacteria/Influenza_virus_particle_8430_lores.jpg" alt="This negative-stained transmission electron micrograph (TEM) depicts the ultrastructural details of an influenza virus particle, or “virion”. A member of the taxonomic family Orthomyxoviridae, the influenza virus is an enveloped, single-stranded RNA virus. Eight helical capsis are surrounded by the viral envelope. Particle diameter: 80 -120 nm" width="70%" />
<p class="caption">
Figure 5.3: <a href="https://commons.wikimedia.org/wiki/File:Influenza_virus_particle_8430_lores.jpg">This negative-stained transmission electron micrograph (TEM) depicts the ultrastructural details of an influenza virus particle, or “virion”. A member of the taxonomic family Orthomyxoviridae, the influenza virus is an enveloped, single-stranded RNA virus. Eight helical capsis are surrounded by the viral envelope. Particle diameter: 80 -120 nm</a>
</p>
</div>
<p>Genome size varies greatly between species. The smallest—the ssDNA circoviruses, family Circoviridae—code for only two proteins and have a genome size of only two kilobases; the largest—the pandoraviruses—have genome sizes of around two megabases which code for about 2500 proteins. Virus genes rarely have introns and often are arranged in the genome so that they overlap.</p>
<p>In general, RNA viruses have smaller genome sizes than DNA viruses because of a higher error-rate when replicating, and have a maximum upper size limit. Beyond this, errors when replicating render the virus useless or uncompetitive. To compensate, RNA viruses often have segmented genomes—the genome is split into smaller molecules—thus reducing the chance that an error in a single-component genome will incapacitate the entire genome. In contrast, DNA viruses generally have larger genomes because of the high fidelity of their replication enzymes. Single-strand DNA viruses are an exception to this rule, as mutation rates for these genomes can approach the extreme of the ssRNA virus case.</p>
<p>Viruses undergo genetic change by several mechanisms. These include a process called antigenic drift where individual bases in the DNA or RNA mutate to other bases. Most of these point mutations are “silent”—they do not change the protein that the gene encodes—but others can confer evolutionary advantages such as resistance to antiviral drugs. Antigenic shift occurs when there is a major change in the genome of the virus. This can be a result of recombination or reassortment. When this happens with influenza viruses, pandemics might result. RNA viruses often exist as quasispecies or swarms of viruses of the same species but with slightly different genome nucleoside sequences. Such quasispecies are a prime target for natural selection.</p>
<p>Segmented genomes confer evolutionary advantages; different strains of a virus with a segmented genome can shuffle and combine genes and produce progeny viruses (or offspring) that have unique characteristics. This is called reassortment or ‘viral sex’.</p>
<p>Genetic recombination is the process by which a strand of DNA is broken and then joined to the end of a different DNA molecule. This can occur when viruses infect cells simultaneously and studies of viral evolution have shown that recombination has been rampant in the species studied. Recombination is common to both RNA and DNA viruses.</p>
<p>Viral populations do not grow through cell division, because they are acellular. Instead, they use the machinery and metabolism of a host cell to produce multiple copies of themselves, and they assemble in the cell. When infected, the host cell is forced to rapidly produce thousands of identical copies of the original virus.</p>

<div class="figure" style="text-align: center"><span id="fig:virusrepc"></span>
<img src="figures/bacteria/HepC_replication.png" alt="A typical virus replication cycle." width="70%" />
<p class="caption">
Figure 5.4: <a href="https://commons.wikimedia.org/wiki/File:HepC_replication.png">A typical virus replication cycle.</a>
</p>
</div>
<p>Their life cycle differs greatly between species, but there are six basic stages in their life cycle:</p>
<p>Attachment is a specific binding between viral capsid proteins and specific receptors on the host cellular surface. This specificity determines the host range and type of host cell of a virus. For example, HIV infects a limited range of human leucocytes. This is because its surface protein, gp120, specifically interacts with the CD4 molecule—a chemokine receptor—which is most commonly found on the surface of CD4+ T-Cells. This mechanism has evolved to favour those viruses that infect only cells in which they are capable of replication. Attachment to the receptor can induce the viral envelope protein to undergo changes that result in the fusion of viral and cellular membranes, or changes of non-enveloped virus surface proteins that allow the virus to enter.</p>
<p>Penetration follows attachment: Virions enter the host cell through receptor-mediated endocytosis or membrane fusion in a process often known as viral entry. The infection of plant and fungal cells is different from that of animal cells. Plants have a rigid cell wall made of cellulose, and fungi one of chitin, so most viruses can get inside these cells only after trauma to the cell wall. Nearly all plant viruses (such as tobacco mosaic virus) can also move directly from cell to cell, in the form of single-stranded nucleoprotein complexes, through pores called plasmodesmata. Bacteria, like plants, have strong cell walls that a virus must breach to infect the cell. Given that bacterial cell walls are much thinner than plant cell walls due to their much smaller size, some viruses have evolved mechanisms that inject their genome into the bacterial cell across the cell wall, while the viral capsid remains outside.</p>
<p>Uncoating is a process in which the viral capsid is removed: This may be by degradation by viral enzymes or host enzymes or by simple dissociation; the end-result is the releasing of the viral genomic nucleic acid.</p>
<p>Replication of viruses involves primarily multiplication of the genome. Replication involves synthesis of viral messenger RNA (mRNA) from “early” genes (with exceptions for positive sense RNA viruses), viral protein synthesis, possible assembly of viral proteins, then viral genome replication mediated by early or regulatory protein expression. This may be followed, for complex viruses with larger genomes, by one or more further rounds of mRNA synthesis: “late” gene expression is, in general, of structural or virion proteins.</p>
<p>Assembly – Following the structure-mediated self-assembly of the virus particles, some modification of the proteins often occurs. In viruses such as HIV, this modification (sometimes called maturation) occurs after the virus has been released from the host cell.</p>
<p>Release – Viruses can be released from the host cell by lysis, a process that kills the cell by bursting its membrane and cell wall if present: this is a feature of many bacterial and some animal viruses. Some viruses undergo a lysogenic cycle where the viral genome is incorporated by genetic recombination into a specific place in the host’s chromosome. The viral genome is then known as a “provirus” or, in the case of bacteriophages a “prophage”. Whenever the host divides, the viral genome is also replicated. The viral genome is mostly silent within the host. At some point, the provirus or prophage may give rise to active virus, which may lyse the host cells. Enveloped viruses (e.g., HIV) typically are released from the host cell by budding. During this process the virus acquires its envelope, which is a modified piece of the host’s plasma or other, internal membrane.</p>
<p>The genetic material within virus particles, and the method by which the material is replicated, varies considerably between different types of viruses.</p>
<p>The genome replication of most DNA viruses takes place in the cell’s nucleus. If the cell has the appropriate receptor on its surface, these viruses enter the cell either by direct fusion with the cell membrane (e.g., herpesviruses) or—more usually—by receptor-mediated endocytosis. Most DNA viruses are entirely dependent on the host cell’s DNA and RNA synthesising machinery, and RNA processing machinery. Viruses with larger genomes may encode much of this machinery themselves. In eukaryotes the viral genome must cross the cell’s nuclear membrane to access this machinery, while in bacteria it need only enter the cell.</p>
<p>Replication of RNA viruses usually takes place in the cytoplasm. RNA viruses can be placed into four different groups depending on their modes of replication. The polarity (whether or not it can be used directly by ribosomes to make proteins) of single-stranded RNA viruses largely determines the replicative mechanism; the other major criterion is whether the genetic material is single-stranded or double-stranded. All RNA viruses use their own RNA replicase enzymes to create copies of their genomes.</p>
<p>Reverse transcribing viruses have ssRNA (Retroviridae, Metaviridae, Pseudoviridae) or dsDNA (Caulimoviridae, and Hepadnaviridae) in their particles. Reverse transcribing viruses with RNA genomes (retroviruses) use a DNA intermediate to replicate, whereas those with DNA genomes (pararetroviruses) use an RNA intermediate during genome replication. Both types use a reverse transcriptase, or RNA-dependent DNA polymerase enzyme, to carry out the nucleic acid conversion. Retroviruses integrate the DNA produced by reverse transcription into the host genome as a provirus as a part of the replication process; pararetroviruses do not, although integrated genome copies of especially plant pararetroviruses can give rise to infectious virus. They are susceptible to antiviral drugs that inhibit the reverse transcriptase enzyme, e.g. zidovudine and lamivudine. An example of the first type is HIV, which is a retrovirus. Examples of the second type are the Hepadnaviridae, which includes Hepatitis B virus.</p>

<div class="figure" style="text-align: center"><span id="fig:rnavirus"></span>
<img src="figures/bacteria/rna_virus.png" alt="Taxonomy and replication strategies of RNA viruses. A) Simplified taxonomy of the genome architecture of the RNA viruses described in this review. See main text for used abbreviations. B) (+RNA virus) Infection with a +RNA virus—as exemplified here with a CoV-like virion—releases a single-stranded RNA genome into the cytoplasm (1). (2) Translation of the 5′-terminal open-reading frame of the genome produces the viral replicase. (3) This multi-enzyme complex includes RdRp activity (orange) and associates with intracellular membranes before −RNA synthesis commences. Newly synthesised −RNAs are subsequently used to produce new +RNAs (4), which are typically capped (yellow) and polyadenylated (polyA). (Retrovirus) HIV-1 genomes are packaged as ssRNA in virions. When the ssRNA is released (1) a cDNA copy is synthesised by the RT (2). The RNA is next degraded by the intrinsic RNase H activity in the RT (3) and the single stranded cDNA converted to dsDNA (4). The dsDNA is imported in the nucleus (5) for integration into the host’s genetic material. (−RNA virus) (1) As illustrated here with an IAV-like particle, infection with an −RNA virus releases a viral RNA genome that is associated with a viral polymerase (orange) and nucleoprotein (green). (2) In the case of non-segmented −RNA viruses, these complexes support transcription to produce viral mRNAs or cRNAs. (3) Viral mRNAs are next translated and new viral proteins complex with cRNAs to synthesise new vRNAs. (5) The vRNA-containing complexes of some segmented −RNA viruses are imported into the nucleus of the host cell, where (6) the RdRp produces mRNAs or cRNAs. (7) mRNAs are transported to the cytoplasm, while cRNAs are bound by new viral proteins to form cRNPs for −RNA synthesis. (dsRNA virus) Fully duplexed RNA genomes lack cap and polyA elements. (1) The RdRp (orange), therefore, transcribes the viral genome inside the capsid of the virion (blue and red), so viral mRNAs can be (2) released into the cytoplasm as illustrated here with a rotavirus-like virion. In the cytoplasm the mRNA is translated (3) or replicated by newly synthesised viral RdRps (4)]" width="70%" />
<p class="caption">
Figure 5.5: <a href="https://commons.wikimedia.org/wiki/File:18_2014_1695_Fig1_HTML.webp">Taxonomy and replication strategies of RNA viruses.</a> A) Simplified taxonomy of the genome architecture of the RNA viruses described in this review. See main text for used abbreviations. B) (+RNA virus) Infection with a +RNA virus—as exemplified here with a CoV-like virion—releases a single-stranded RNA genome into the cytoplasm (1). (2) Translation of the 5′-terminal open-reading frame of the genome produces the viral replicase. (3) This multi-enzyme complex includes RdRp activity (orange) and associates with intracellular membranes before −RNA synthesis commences. Newly synthesised −RNAs are subsequently used to produce new +RNAs (4), which are typically capped (yellow) and polyadenylated (polyA). (Retrovirus) HIV-1 genomes are packaged as ssRNA in virions. When the ssRNA is released (1) a cDNA copy is synthesised by the RT (2). The RNA is next degraded by the intrinsic RNase H activity in the RT (3) and the single stranded cDNA converted to dsDNA (4). The dsDNA is imported in the nucleus (5) for integration into the host’s genetic material. (−RNA virus) (1) As illustrated here with an IAV-like particle, infection with an −RNA virus releases a viral RNA genome that is associated with a viral polymerase (orange) and nucleoprotein (green). (2) In the case of non-segmented −RNA viruses, these complexes support transcription to produce viral mRNAs or cRNAs. (3) Viral mRNAs are next translated and new viral proteins complex with cRNAs to synthesise new vRNAs. (5) The vRNA-containing complexes of some segmented −RNA viruses are imported into the nucleus of the host cell, where (6) the RdRp produces mRNAs or cRNAs. (7) mRNAs are transported to the cytoplasm, while cRNAs are bound by new viral proteins to form cRNPs for −RNA synthesis. (dsRNA virus) Fully duplexed RNA genomes lack cap and polyA elements. (1) The RdRp (orange), therefore, transcribes the viral genome inside the capsid of the virion (blue and red), so viral mRNAs can be (2) released into the cytoplasm as illustrated here with a rotavirus-like virion. In the cytoplasm the mRNA is translated (3) or replicated by newly synthesised viral RdRps (4)]
</p>
</div>
<p>The range of structural and biochemical effects that viruses have on the host cell is extensive. These are called ‘cytopathic effects’. Most virus infections eventually result in the death of the host cell. The causes of death include cell lysis, alterations to the cell’s surface membrane and apoptosis. Often cell death is caused by cessation of its normal activities because of suppression by virus-specific proteins, not all of which are components of the virus particle. The distinction between cytopathic and harmless is gradual. Some viruses, such as Epstein–Barr virus, can cause cells to proliferate without causing malignancy, while others, such as papillomaviruses, are established causes of cancer.</p>
<p>Some viruses cause no apparent changes to the infected cell. Cells in which the virus is latent and inactive show few signs of infection and often function normally. This causes persistent infections and the virus is often dormant for many months or years. This is often the case with herpes viruses.</p>
<p>Viruses are by far the most abundant biological entities on Earth and they outnumber all the others put together. They infect all types of cellular life including animals, plants, bacteria and fungi. Different types of viruses can infect only a limited range of hosts and many are species-specific. Some, such as smallpox virus for example, can infect only one species—in this case humans, and are said to have a narrow host range. Other viruses, such as rabies virus, can infect different species of mammals and are said to have a broad range. The viruses that infect plants are harmless to animals, and most viruses that infect other animals are harmless to humans. The host range of some bacteriophages is limited to a single strain of bacteria and they can be used to trace the source of outbreaks of infections by a method called phage typing. The complete set of viruses in an organism or habitat is called the virome; for example, all human viruses constitute the human virome.</p>
<div id="role-in-human-disease" class="section level3" number="5.0.1">
<h3><span class="header-section-number">5.0.1</span> Role in human disease</h3>
<p>Examples of common human diseases caused by viruses include the common cold, influenza, chickenpox, and cold sores. Many serious diseases such as rabies, Ebola virus disease, AIDS (HIV), avian influenza, and SARS are caused by viruses. The relative ability of viruses to cause disease is described in terms of virulence. Other diseases are under investigation to discover if they have a virus as the causative agent, such as the possible connection between human herpesvirus 6 (HHV6) and neurological diseases such as multiple sclerosis and chronic fatigue syndrome. There is controversy over whether the bornavirus, previously thought to cause neurological diseases in horses, could be responsible for psychiatric illnesses in humans.</p>
<p>Viruses have different mechanisms by which they produce disease in an organism, which depends largely on the viral species. Mechanisms at the cellular level primarily include cell lysis, the breaking open and subsequent death of the cell. In multicellular organisms, if enough cells die, the whole organism will start to suffer the effects. Although viruses cause disruption of healthy homeostasis, resulting in disease, they may exist relatively harmlessly within an organism. An example would include the ability of the herpes simplex virus, which causes cold sores, to remain in a dormant state within the human body. This is called latency and is a characteristic of the herpes viruses, including Epstein–Barr virus, which causes glandular fever, and varicella zoster virus, which causes chickenpox and shingles. Most people have been infected with at least one of these types of herpes virus. These latent viruses might sometimes be beneficial, as the presence of the virus can increase immunity against bacterial pathogens, such as Yersinia pestis.</p>
<p>Some viruses can cause lifelong or chronic infections, where the viruses continue to replicate in the body despite the host’s defence mechanisms. This is common in hepatitis B virus and hepatitis C virus infections. People chronically infected are known as carriers, as they serve as reservoirs of infectious virus. In populations with a high proportion of carriers, the disease is said to be endemic.</p>
<p>Viral epidemiology is the branch of medical science that deals with the transmission and control of virus infections in humans. Transmission of viruses can be vertical, which means from mother to child, or horizontal, which means from person to person. Examples of vertical transmission include hepatitis B virus and HIV, where the baby is born already infected with the virus. Another, more rare, example is the varicella zoster virus, which, although causing relatively mild infections in children and adults, can be fatal to the foetus and newborn baby.</p>
<p>Horizontal transmission is the most common mechanism of spread of viruses in populations. Horizontal transmission can occur when body fluids are exchanged during sexual activity, by exchange of saliva or when contaminated food or water is ingested. It can also occur when aerosols containing viruses are inhaled or by insect vectors such as when infected mosquitoes penetrate the skin of a host. Most types of viruses are restricted to just one or two of these mechanisms and they are referred to as “respiratory viruses” or “enteric viruses” and so forth. The rate or speed of transmission of viral infections depends on factors that include population density, the number of susceptible individuals, (i.e., those not immune), the quality of healthcare and the weather.</p>
<p>Epidemiology is used to break the chain of infection in populations during outbreaks of viral diseases. Control measures are used that are based on knowledge of how the virus is transmitted. It is important to find the source, or sources, of the outbreak and to identify the virus. Once the virus has been identified, the chain of transmission can sometimes be broken by vaccines. When vaccines are not available, sanitation and disinfection can be effective. Often, infected people are isolated from the rest of the community, and those that have been exposed to the virus are placed in quarantine. To control the outbreak of foot-and-mouth disease in cattle in Britain in 2001, thousands of cattle were slaughtered. Most viral infections of humans and other animals have incubation periods during which the infection causes no signs or symptoms. Incubation periods for viral diseases range from a few days to weeks, but are known for most infections. Somewhat overlapping, but mainly following the incubation period, there is a period of communicability—a time when an infected individual or animal is contagious and can infect another person or animal. This, too, is known for many viral infections, and knowledge of the length of both periods is important in the control of outbreaks. When outbreaks cause an unusually high proportion of cases in a population, community, or region, they are called epidemics. If outbreaks spread worldwide, they are called pandemics.</p>
<p>A pandemic is a worldwide epidemic. The 1918 flu pandemic, which lasted until 1919, was a category 5 influenza pandemic caused by an unusually severe and deadly influenza A virus. The victims were often healthy young adults, in contrast to most influenza outbreaks, which predominantly affect juvenile, elderly, or otherwise-weakened patients. Older estimates say it killed 40–50 million people, while more recent research suggests that it may have killed as many as 100 million people, or 5% of the world’s population in 1918.</p>
<p>Although viral pandemics are rare events, HIV—which evolved from viruses found in monkeys and chimpanzees—has been pandemic since at least the 1980s. During the 20th century there were four pandemics caused by influenza virus and those that occurred in 1918, 1957 and 1968 were severe. Most researchers believe that HIV originated in sub-Saharan Africa during the 20th century; it is now a pandemic, with an estimated 37.9 million people now living with the disease worldwide. There were about 770,000 deaths from AIDS in 2018. The Joint United Nations Programme on HIV/AIDS (UNAIDS) and the World Health Organization (WHO) estimate that AIDS has killed more than 25 million people since it was first recognised on 5 June 1981, making it one of the most destructive epidemics in recorded history. In 2007 there were 2.7 million new HIV infections and 2 million HIV-related deaths.</p>
<p>Several highly lethal viral pathogens are members of the Filoviridae. Filoviruses are filament-like viruses that cause viral hemorrhagic fever, and include ebolaviruses and marburgviruses. Marburg virus, first discovered in 1967, attracted widespread press attention in April 2005 for an outbreak in Angola. Ebola virus disease has also caused intermittent outbreaks with high mortality rates since 1976 when it was first identified. The worst and most recent one is the 2013–2016 West Africa epidemic.</p>
<p>With the exception of smallpox, most pandemics are caused by newly evolved viruses. These “emergent” viruses are usually mutants of less harmful viruses that have circulated previously either in humans or other animals.</p>
<p>Severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) are caused by new types of coronaviruses. Other coronaviruses are known to cause mild infections in humans, so the virulence and rapid spread of SARS infections—that by July 2003 had caused around 8,000 cases and 800 deaths—was unexpected and most countries were not prepared.</p>
<p>A related coronavirus emerged in Wuhan, China in November 2019 and spread rapidly around the world. Thought to have originated in bats and subsequently named severe acute respiratory syndrome coronavirus 2, infections with the virus caused a pandemic in 2020. Unprecedented restrictions in peacetime have been placed on international travel, and curfews imposed in several major cities worldwide.</p>
<p>Viruses are an established cause of cancer in humans and other species. Viral cancers occur only in a minority of infected persons (or animals). Cancer viruses come from a range of virus families, including both RNA and DNA viruses, and so there is no single type of “oncovirus” (an obsolete term originally used for acutely transforming retroviruses). The development of cancer is determined by a variety of factors such as host immunity and mutations in the host. Viruses accepted to cause human cancers include some genotypes of human papillomavirus, hepatitis B virus, hepatitis C virus, Epstein–Barr virus, Kaposi’s sarcoma-associated herpesvirus and human T-lymphotropic virus. The most recently discovered human cancer virus is a polyomavirus (Merkel cell polyomavirus) that causes most cases of a rare form of skin cancer called Merkel cell carcinoma. Hepatitis viruses can develop into a chronic viral infection that leads to liver cancer. Infection by human T-lymphotropic virus can lead to tropical spastic paraparesis and adult T-cell leukaemia. Human papillomaviruses are an established cause of cancers of cervix, skin, anus, and penis. Within the Herpesviridae, Kaposi’s sarcoma-associated herpesvirus causes Kaposi’s sarcoma and body-cavity lymphoma, and Epstein–Barr virus causes Burkitt’s lymphoma, Hodgkin’s lymphoma, B lymphoproliferative disorder, and nasopharyngeal carcinoma. Merkel cell polyomavirus closely related to SV40 and mouse polyomaviruses that have been used as animal models for cancer viruses for over 50 years.</p>
<p>The body’s first line of defence against viruses is the innate immune system. This comprises cells and other mechanisms that defend the host from infection in a non-specific manner. This means that the cells of the innate system recognise, and respond to, pathogens in a generic way, but, unlike the adaptive immune system, it does not confer long-lasting or protective immunity to the host.</p>
<p>RNA interference is an important innate defence against viruses. Many viruses have a replication strategy that involves double-stranded RNA (dsRNA). When such a virus infects a cell, it releases its RNA molecule or molecules, which immediately bind to a protein complex called a dicer that cuts the RNA into smaller pieces. A biochemical pathway—the RISC complex—is activated, which ensures cell survival by degrading the viral mRNA. Rotaviruses have evolved to avoid this defence mechanism by not uncoating fully inside the cell, and releasing newly produced mRNA through pores in the particle’s inner capsid. Their genomic dsRNA remains protected inside the core of the virion.</p>
<p>When the adaptive immune system of a vertebrate encounters a virus, it produces specific antibodies that bind to the virus and often render it non-infectious. This is called humoral immunity. Two types of antibodies are important. The first, called IgM, is highly effective at neutralising viruses but is produced by the cells of the immune system only for a few weeks. The second, called IgG, is produced indefinitely. The presence of IgM in the blood of the host is used to test for acute infection, whereas IgG indicates an infection sometime in the past. IgG antibody is measured when tests for immunity are carried out.</p>
<p>Antibodies can continue to be an effective defence mechanism even after viruses have managed to gain entry to the host cell. A protein that is in cells, called TRIM21, can attach to the antibodies on the surface of the virus particle. This primes the subsequent destruction of the virus by the enzymes of the cell’s proteosome system.</p>
<p>A second defence of vertebrates against viruses is called cell-mediated immunity and involves immune cells known as T cells. The body’s cells constantly display short fragments of their proteins on the cell’s surface, and, if a T cell recognises a suspicious viral fragment there, the host cell is destroyed by ‘killer T’ cells and the virus-specific T-cells proliferate. Cells such as the macrophage are specialists at this antigen presentation. The production of interferon is an important host defence mechanism. This is a hormone produced by the body when viruses are present. Its role in immunity is complex; it eventually stops the viruses from reproducing by killing the infected cell and its close neighbours.</p>
<p>Not all virus infections produce a protective immune response in this way. HIV evades the immune system by constantly changing the amino acid sequence of the proteins on the surface of the virion. This is known as “escape mutation” as the viral epitopes escape recognition by the host immune response. These persistent viruses evade immune control by sequestration, blockade of antigen presentation, cytokine resistance, evasion of natural killer cell activities, escape from apoptosis, and antigenic shift. Other viruses, called ‘neurotropic viruses’, are disseminated by neural spread where the immune system may be unable to reach them.</p>
<p>Because viruses use vital metabolic pathways within host cells to replicate, they are difficult to eliminate without using drugs that cause toxic effects to host cells in general. The most effective medical approaches to viral diseases are vaccinations to provide immunity to infection, and antiviral drugs that selectively interfere with viral replication.</p>
<p>Vaccination is a cheap and effective way of preventing infections by viruses. Vaccines were used to prevent viral infections long before the discovery of the actual viruses. Their use has resulted in a dramatic decline in morbidity (illness) and mortality (death) associated with viral infections such as polio, measles, mumps and rubella. Smallpox infections have been eradicated. Vaccines are available to prevent over thirteen viral infections of humans, and more are used to prevent viral infections of animals. Vaccines can consist of live-attenuated or killed viruses, or viral proteins (antigens). Live vaccines contain weakened forms of the virus, which do not cause the disease but, nonetheless, confer immunity. Such viruses are called attenuated. Live vaccines can be dangerous when given to people with a weak immunity (who are described as immunocompromised), because in these people, the weakened virus can cause the original disease. Biotechnology and genetic engineering techniques are used to produce subunit vaccines. These vaccines use only the capsid proteins of the virus. Hepatitis B vaccine is an example of this type of vaccine. Subunit vaccines are safe for immunocompromised patients because they cannot cause the disease. The yellow fever virus vaccine, a live-attenuated strain called 17D, is probably the safest and most effective vaccine ever generated.</p>
<p>Antiviral drugs are often nucleoside analogues (fake DNA building-blocks), which viruses mistakenly incorporate into their genomes during replication. The life-cycle of the virus is then halted because the newly synthesised DNA is inactive. This is because these analogues lack the hydroxyl groups, which, along with phosphorus atoms, link together to form the strong “backbone” of the DNA molecule. This is called DNA chain termination. Examples of nucleoside analogues are aciclovir for Herpes simplex virus infections and lamivudine for HIV and hepatitis B virus infections. Aciclovir is one of the oldest and most frequently prescribed antiviral drugs. Other antiviral drugs in use target different stages of the viral life cycle. HIV is dependent on a proteolytic enzyme called the HIV-1 protease for it to become fully infectious. There is a large class of drugs called protease inhibitors that inactivate this enzyme.</p>
<p>Hepatitis C is caused by an RNA virus. In 80% of people infected, the disease is chronic, and without treatment, they are infected for the remainder of their lives. There is now an effective treatment that uses the nucleoside analogue drug ribavirin combined with interferon. The treatment of chronic carriers of the hepatitis B virus by using a similar strategy using lamivudine has been developed.</p>
</div>
<div id="animal-viruses" class="section level3" number="5.0.2">
<h3><span class="header-section-number">5.0.2</span> Animal viruses</h3>
<p>Viruses are important pathogens of livestock. Diseases such as foot-and-mouth disease and bluetongue are caused by viruses. Companion animals such as cats, dogs, and horses, if not vaccinated, are susceptible to serious viral infections. Canine parvovirus is caused by a small DNA virus and infections are often fatal in pups. Like all invertebrates, the honey bee is susceptible to many viral infections. Most viruses co-exist harmlessly in their host and cause no signs or symptoms of disease.</p>
</div>
<div id="plant-viruses" class="section level3" number="5.0.3">
<h3><span class="header-section-number">5.0.3</span> Plant viruses</h3>
<p>There are many types of plant virus, but often they cause only a loss of yield, and it is not economically viable to try to control them. Plant viruses are often spread from plant to plant by organisms, known as vectors. These are usually insects, but some fungi, nematode worms, and single-celled organisms have been shown to be vectors. When control of plant virus infections is considered economical, for perennial fruits, for example, efforts are concentrated on killing the vectors and removing alternate hosts such as weeds. Plant viruses cannot infect humans and other animals because they can reproduce only in living plant cells.</p>
<p>Originally from Peru, the potato has become a staple crop worldwide. The potato virus Y causes disease in potatoes and related species including tomatoes and peppers. In the 1980s, this virus acquired economical importance when it proved difficult to control in seed potato crops. Transmitted by aphids, this virus can reduce crop yields by up to 80 per cent, causing significant losses to potato yields.</p>
<p>Plants have elaborate and effective defence mechanisms against viruses. One of the most effective is the presence of so-called resistance (R) genes. Each R gene confers resistance to a particular virus by triggering localised areas of cell death around the infected cell, which can often be seen with the unaided eye as large spots. This stops the infection from spreading. RNA interference is also an effective defence in plants. When they are infected, plants often produce natural disinfectants that kill viruses, such as salicylic acid, nitric oxide, and reactive oxygen molecules.</p>
<p>Plant virus particles or virus-like particles (VLPs) have applications in both biotechnology and nanotechnology. The capsids of most plant viruses are simple and robust structures and can be produced in large quantities either by the infection of plants or by expression in a variety of heterologous systems. Plant virus particles can be modified genetically and chemically to encapsulate foreign material and can be incorporated into supramolecular structures for use in biotechnology.</p>
</div>
<div id="bacterial-viruses" class="section level3" number="5.0.4">
<h3><span class="header-section-number">5.0.4</span> Bacterial viruses</h3>
<p>A bacteriophage, also known informally as a phage, is a virus that infects and replicates within bacteria and archaea. The term was derived from “bacteria” and the Greek φαγεῖν (phagein), meaning “to devour”. Bacteriophages are composed of proteins that encapsulate a DNA or RNA genome, and may have structures that are either simple or elaborate. Their genomes may encode as few as four genes and as many as hundreds of genes.</p>

<div class="figure" style="text-align: center"><span id="fig:phage"></span>
<img src="figures/bacteria/Tevenphage.svg" alt="Structural overview of T2 phage" width="70%" />
<p class="caption">
Figure 5.6: <a href="https://en.wikipedia.org/wiki/Hershey–Chase_experiment#/media/File:Tevenphage.svg">Structural overview of T2 phage</a>
</p>
</div>
<p>These viruses infect specific bacteria by binding to surface receptor molecules and then entering the cell. Within a short amount of time, in some cases just minutes, bacterial polymerase starts translating viral mRNA into protein. These proteins go on to become either new virions within the cell, helper proteins, which help assembly of new virions, or proteins involved in cell lysis. Viral enzymes aid in the breakdown of the cell membrane, and, in the case of the T4 phage, in just over twenty minutes after injection over three hundred phages could be released.</p>

<div class="figure" style="text-align: center"><span id="fig:bacphage"></span>
<img src="figures/bacteria/Phage_injecting_its_genome_into_bacteria.svg" alt="Diagram showing bacteriophages infecting a bacterial cell (not to scale; bacteriophages are much smaller than bacteria)" width="70%" />
<p class="caption">
Figure 5.7: <a href="https://commons.wikimedia.org/wiki/File:Phage_injecting_its_genome_into_bacteria.svg">Diagram showing bacteriophages infecting a bacterial cell (not to scale; bacteriophages are much smaller than bacteria)</a>
</p>
</div>
<p>The major way bacteria defend themselves from bacteriophages is by producing enzymes that destroy foreign DNA. These enzymes, called restriction endonucleases, cut up the viral DNA that bacteriophages inject into bacterial cells. Bacteria also contain a system that uses CRISPR sequences to retain fragments of the genomes of viruses that the bacteria have come into contact with in the past, which allows them to block the virus’s replication through a form of RNA interference. This genetic system provides bacteria with acquired immunity to infection.</p>
<p>Bacteriophages are among the most common and diverse entities in the biosphere. Bacteriophages are ubiquitous viruses, found wherever bacteria exist. It is estimated there are more than 1031 bacteriophages on the planet, more than every other organism on Earth, including bacteria, combined. One of the densest natural sources for phages and other viruses is seawater, where up to 9x10<sup>8</sup> virions per millilitre have been found in microbial mats at the surface, and up to 70% of marine bacteria may be infected by phages.</p>
<p>Phages have been used since the late 20th century as an alternative to antibiotics in the former Soviet Union and Central Europe, as well as in France. They are seen as a possible therapy against multi-drug-resistant strains of many bacteria (see phage therapy). On the other hand, phages of Inoviridae have been shown to complicate biofilms involved in pneumonia and cystic fibrosis and to shelter the bacteria from drugs meant to eradicate disease, thus promoting persistent infection.</p>
</div>
<div id="classification" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Classification</h2>
<p>Classification seeks to describe the diversity of viruses by naming and grouping them on the basis of similarities. In 1962, André Lwoff, Robert Horne, and Paul Tournier were the first to develop a means of virus classification, based on the Linnaean hierarchical system.[143] This system based classification on phylum, class, order, family, genus, and species. Viruses were grouped according to their shared properties (not those of their hosts) and the type of nucleic acid forming their genomes.[144] In 1966, the International Committee on Taxonomy of Viruses (ICTV) was formed. The system proposed by Lwoff, Horne and Tournier was initially not accepted by the ICTV because the small genome size of viruses and their high rate of mutation made it difficult to determine their ancestry beyond order. As such, the Baltimore classification system has come to be used to supplement the more traditional hierarchy.[145] Starting in 2018, the ICTV began to acknowledge deeper evolutionary relationships between viruses that have been discovered over time and adopted a 15-rank classification system ranging from realm to species.[146]</p>
<p>ICTV classification
The ICTV developed the current classification system and wrote guidelines that put a greater weight on certain virus properties to maintain family uniformity. A unified taxonomy (a universal system for classifying viruses) has been established. Only a small part of the total diversity of viruses has been studied.[147] As of 2020, 6 realms, 10 kingdoms, 17 phyla, 2 subphyla, 39 classes, 59 orders, 8 suborders, 189 families, 136 subfamilies, 2,224 genera, 70 subgenera, and 9,110 species of viruses have been defined by the ICTV.[5]</p>
</div>
<div id="baltimore-classification" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> Baltimore classification</h2>
<p>The Nobel Prize-winning biologist David Baltimore devised the Baltimore classification system.[43][148] The ICTV classification system is used in conjunction with the Baltimore classification system in modern virus classification.[149][150][151]</p>
<p>The Baltimore classification of viruses is based on the mechanism of mRNA production. Viruses must generate mRNAs from their genomes to produce proteins and replicate themselves, but different mechanisms are used to achieve this in each virus family. Viral genomes may be single-stranded (ss) or double-stranded (ds), RNA or DNA, and may or may not use reverse transcriptase (RT). In addition, ssRNA viruses may be either sense (+) or antisense (−). This classification places viruses into seven groups:</p>
<ul>
<li>I: dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses)</li>
<li>II: ssDNA viruses (+ strand or “sense”) DNA (e.g. Parvoviruses)</li>
<li>III: dsRNA viruses (e.g. Reoviruses)</li>
<li>IV: (+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses)</li>
<li>V: (−)ssRNA viruses (− strand or antisense) RNA (e.g. Orthomyxoviruses, Rhabdoviruses)</li>
<li>VI: ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses)</li>
<li>VII: dsDNA-RT viruses DNA with RNA intermediate in life-cycle (e.g. Hepadnaviruses)</li>
</ul>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="eukaryotic-microorganisms.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="microbial-nutrition-ecology-and-growth.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/microbiology-text/edit/master/viruses.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
