<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>12 Drugs, Microbes, And The Host | Text For SCI204 Microbiology at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Microbiology." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="12 Drugs, Microbes, And The Host | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/microbiology_text_cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Microbiology." />
  <meta name="github-repo" content="nikolaussucher/microbiology-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="12 Drugs, Microbes, And The Host | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Microbiology." />
  <meta name="twitter:image" content="/microbiology_text_cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="physical-and-chemical-agents-for-microbial-control.html"/>
<link rel="next" href="infection-and-disease.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />




<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">SCI204 Microbiology at RCC</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html"><i class="fa fa-check"></i><b>1</b> The Main Themes Of Microbiology</a>
<ul>
<li class="chapter" data-level="1.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#general-characteristics-and-evolution-of-microorganisms"><i class="fa fa-check"></i><b>1.1</b> General Characteristics And Evolution Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#archaea"><i class="fa fa-check"></i><b>1.1.1</b> Archaea</a></li>
<li class="chapter" data-level="1.1.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#bacteria"><i class="fa fa-check"></i><b>1.1.2</b> Bacteria</a></li>
<li class="chapter" data-level="1.1.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#eukaryotes"><i class="fa fa-check"></i><b>1.1.3</b> Eukaryotes</a></li>
<li class="chapter" data-level="1.1.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#protists"><i class="fa fa-check"></i><b>1.1.4</b> Protists</a></li>
<li class="chapter" data-level="1.1.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#fungi"><i class="fa fa-check"></i><b>1.1.5</b> Fungi</a></li>
<li class="chapter" data-level="1.1.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#plants"><i class="fa fa-check"></i><b>1.1.6</b> Plants</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#ecology-of-microorganisms"><i class="fa fa-check"></i><b>1.2</b> Ecology Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbiome"><i class="fa fa-check"></i><b>1.2.1</b> Microbiome</a></li>
<li class="chapter" data-level="1.2.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbiota-members-of-the-microbiome"><i class="fa fa-check"></i><b>1.2.2</b> Microbiota – members of the microbiome</a></li>
<li class="chapter" data-level="1.2.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbial-networks-and-interaction"><i class="fa fa-check"></i><b>1.2.3</b> Microbial Networks And Interaction</a></li>
<li class="chapter" data-level="1.2.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-human-microbiome"><i class="fa fa-check"></i><b>1.2.4</b> The Human Microbiome</a></li>
<li class="chapter" data-level="1.2.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#extremophiles"><i class="fa fa-check"></i><b>1.2.5</b> Extremophiles</a></li>
<li class="chapter" data-level="1.2.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#soil-microorganisms"><i class="fa fa-check"></i><b>1.2.6</b> Soil Microorganisms</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#human-interactions-with-microbes"><i class="fa fa-check"></i><b>1.3</b> Human Interactions With Microbes</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#food-production"><i class="fa fa-check"></i><b>1.3.1</b> Food production</a></li>
<li class="chapter" data-level="1.3.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#water-treatment"><i class="fa fa-check"></i><b>1.3.2</b> Water Treatment</a></li>
<li class="chapter" data-level="1.3.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#fuel-production"><i class="fa fa-check"></i><b>1.3.3</b> Fuel Production</a></li>
<li class="chapter" data-level="1.3.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#production-of-chemicals-and-enzymes"><i class="fa fa-check"></i><b>1.3.4</b> Production Of Chemicals And Enzymes</a></li>
<li class="chapter" data-level="1.3.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#use-of-microorganisms-in-science"><i class="fa fa-check"></i><b>1.3.5</b> Use Of Microorganisms In Science</a></li>
<li class="chapter" data-level="1.3.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#use-of-microorganisms-for-warfare"><i class="fa fa-check"></i><b>1.3.6</b> Use of Microorganisms For Warfare</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#role-of-microorganisms-in-disease"><i class="fa fa-check"></i><b>1.4</b> Role Of Microorganisms In Disease</a></li>
<li class="chapter" data-level="1.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#role-of-microorganisms-in-agriculture-and-horticulture"><i class="fa fa-check"></i><b>1.5</b> Role Of Microorganisms In Agriculture And Horticulture</a></li>
<li class="chapter" data-level="1.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#history-of-microbiology"><i class="fa fa-check"></i><b>1.6</b> History Of Microbiology</a></li>
<li class="chapter" data-level="1.7" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#taxonomy-classification-and-naming-of-microorganisms"><i class="fa fa-check"></i><b>1.7</b> Taxonomy: Classification And Naming Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-concept-of-species-in-biology"><i class="fa fa-check"></i><b>1.7.1</b> The Concept of Species in Biology</a></li>
<li class="chapter" data-level="1.7.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#binomial-nomenclature"><i class="fa fa-check"></i><b>1.7.2</b> Binomial Nomenclature</a></li>
<li class="chapter" data-level="1.7.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#taxonomy-and-systematics"><i class="fa fa-check"></i><b>1.7.3</b> Taxonomy And Systematics</a></li>
<li class="chapter" data-level="1.7.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#kingdoms-and-domains"><i class="fa fa-check"></i><b>1.7.4</b> Kingdoms And Domains</a></li>
<li class="chapter" data-level="1.7.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-three-domain-system"><i class="fa fa-check"></i><b>1.7.5</b> The Three-Domain System</a></li>
<li class="chapter" data-level="1.7.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#bergeys-manual-of-systematic-bacteriology"><i class="fa fa-check"></i><b>1.7.6</b> Bergey’s Manual of Systematic Bacteriology</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html"><i class="fa fa-check"></i><b>2</b> The Chemistry Of Biology</a>
<ul>
<li class="chapter" data-level="2.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#matter"><i class="fa fa-check"></i><b>2.1</b> Matter</a></li>
<li class="chapter" data-level="2.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-atom"><i class="fa fa-check"></i><b>2.2</b> The Atom</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structure-of-the-atom"><i class="fa fa-check"></i><b>2.2.1</b> Structure of The Atom</a></li>
<li class="chapter" data-level="2.2.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-atomic-nucleus"><i class="fa fa-check"></i><b>2.2.2</b> The Atomic Nucleus</a></li>
<li class="chapter" data-level="2.2.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-electron-cloud"><i class="fa fa-check"></i><b>2.2.3</b> The Electron Cloud</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-periodic-table-of-the-elements"><i class="fa fa-check"></i><b>2.3</b> The Periodic Table of The Elements</a></li>
<li class="chapter" data-level="2.4" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#chemical-bonds-molecules-and-compounds"><i class="fa fa-check"></i><b>2.4</b> Chemical Bonds, Molecules, And Compounds</a></li>
<li class="chapter" data-level="2.5" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#energy"><i class="fa fa-check"></i><b>2.5</b> Energy</a></li>
<li class="chapter" data-level="2.6" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#chemical-reactions"><i class="fa fa-check"></i><b>2.6</b> Chemical Reactions</a></li>
<li class="chapter" data-level="2.7" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#radioactive-decay-of-atoms"><i class="fa fa-check"></i><b>2.7</b> Radioactive Decay of Atoms</a></li>
<li class="chapter" data-level="2.8" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#ions-and-salts"><i class="fa fa-check"></i><b>2.8</b> Ions And Salts</a></li>
<li class="chapter" data-level="2.9" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#solutions"><i class="fa fa-check"></i><b>2.9</b> Solutions</a></li>
<li class="chapter" data-level="2.10" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#acids-and-bases"><i class="fa fa-check"></i><b>2.10</b> Acids And Bases</a></li>
<li class="chapter" data-level="2.11" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#reduction-oxidation-redox-reactions"><i class="fa fa-check"></i><b>2.11</b> Reduction-oxidation (Redox) Reactions</a></li>
<li class="chapter" data-level="2.12" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-chemistry-of-water"><i class="fa fa-check"></i><b>2.12</b> The Chemistry Of Water</a></li>
<li class="chapter" data-level="2.13" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#basic-organic-chemistry-for-biology"><i class="fa fa-check"></i><b>2.13</b> Basic Organic Chemistry For Biology</a>
<ul>
<li class="chapter" data-level="2.13.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#functional-groups"><i class="fa fa-check"></i><b>2.13.1</b> Functional Groups</a></li>
<li class="chapter" data-level="2.13.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#biomolecules"><i class="fa fa-check"></i><b>2.13.2</b> Biomolecules</a></li>
<li class="chapter" data-level="2.13.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#proteins"><i class="fa fa-check"></i><b>2.13.3</b> Proteins</a></li>
<li class="chapter" data-level="2.13.4" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structure"><i class="fa fa-check"></i><b>2.13.4</b> Structure</a></li>
<li class="chapter" data-level="2.13.5" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#cellular-functions-of-proteins"><i class="fa fa-check"></i><b>2.13.5</b> Cellular Functions of Proteins</a></li>
<li class="chapter" data-level="2.13.6" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#cell-signaling-and-ligand-binding"><i class="fa fa-check"></i><b>2.13.6</b> Cell Signaling And Ligand Binding</a></li>
<li class="chapter" data-level="2.13.7" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structural-proteins"><i class="fa fa-check"></i><b>2.13.7</b> Structural Proteins</a></li>
<li class="chapter" data-level="2.13.8" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#carbohydrates"><i class="fa fa-check"></i><b>2.13.8</b> Carbohydrates</a></li>
<li class="chapter" data-level="2.13.9" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#lipids"><i class="fa fa-check"></i><b>2.13.9</b> Lipids</a></li>
<li class="chapter" data-level="2.13.10" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#nucleic-acids"><i class="fa fa-check"></i><b>2.13.10</b> Nucleic Acids</a></li>
<li class="chapter" data-level="2.13.11" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#deoxyribonucleic-acid-dna"><i class="fa fa-check"></i><b>2.13.11</b> Deoxyribonucleic Acid (DNA)</a></li>
<li class="chapter" data-level="2.13.12" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#ribonucleic-acid-rna"><i class="fa fa-check"></i><b>2.13.12</b> Ribonucleic Acid (RNA)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html"><i class="fa fa-check"></i><b>3</b> An Introduction To Methods Of Studying Microorganisms</a>
<ul>
<li class="chapter" data-level="3.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#microscopy"><i class="fa fa-check"></i><b>3.1</b> Microscopy</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#light-microscopy"><i class="fa fa-check"></i><b>3.1.1</b> Light Microscopy</a></li>
<li class="chapter" data-level="3.1.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-optical-light-microscope"><i class="fa fa-check"></i><b>3.1.2</b> The Optical (Light) Microscope</a></li>
<li class="chapter" data-level="3.1.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#simple-microscope"><i class="fa fa-check"></i><b>3.1.3</b> Simple Microscope</a></li>
<li class="chapter" data-level="3.1.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#compound-microscope"><i class="fa fa-check"></i><b>3.1.4</b> Compound Microscope</a></li>
<li class="chapter" data-level="3.1.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-main-components-of-the-light-microscope"><i class="fa fa-check"></i><b>3.1.5</b> The Main Components Of The Light Microscope</a></li>
<li class="chapter" data-level="3.1.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#eyepiece-ocular-lens"><i class="fa fa-check"></i><b>3.1.6</b> Eyepiece (Ocular Lens)</a></li>
<li class="chapter" data-level="3.1.7" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#objective-lens"><i class="fa fa-check"></i><b>3.1.7</b> Objective Lens</a></li>
<li class="chapter" data-level="3.1.8" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#oil-immersion-objective"><i class="fa fa-check"></i><b>3.1.8</b> Oil Immersion Objective</a></li>
<li class="chapter" data-level="3.1.9" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#focus-knobs"><i class="fa fa-check"></i><b>3.1.9</b> Focus Knobs</a></li>
<li class="chapter" data-level="3.1.10" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-frame"><i class="fa fa-check"></i><b>3.1.10</b> The Frame</a></li>
<li class="chapter" data-level="3.1.11" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-stage"><i class="fa fa-check"></i><b>3.1.11</b> The Stage</a></li>
<li class="chapter" data-level="3.1.12" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-light-source"><i class="fa fa-check"></i><b>3.1.12</b> The Light Source</a></li>
<li class="chapter" data-level="3.1.13" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-condenser"><i class="fa fa-check"></i><b>3.1.13</b> The Condenser</a></li>
<li class="chapter" data-level="3.1.14" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#magnification"><i class="fa fa-check"></i><b>3.1.14</b> Magnification</a></li>
<li class="chapter" data-level="3.1.15" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-microscope-slide"><i class="fa fa-check"></i><b>3.1.15</b> The Microscope Slide</a></li>
<li class="chapter" data-level="3.1.16" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dry-mount"><i class="fa fa-check"></i><b>3.1.16</b> Dry Mount</a></li>
<li class="chapter" data-level="3.1.17" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#wet-mount-or-temporary-mount"><i class="fa fa-check"></i><b>3.1.17</b> Wet Mount Or Temporary Mount</a></li>
<li class="chapter" data-level="3.1.18" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#prepared-mount-or-permanent-mount"><i class="fa fa-check"></i><b>3.1.18</b> Prepared Mount Or Permanent Mount</a></li>
<li class="chapter" data-level="3.1.19" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#techniques-of-optical-microscopy"><i class="fa fa-check"></i><b>3.1.19</b> Techniques Of Optical Microscopy</a></li>
<li class="chapter" data-level="3.1.20" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#bright-field"><i class="fa fa-check"></i><b>3.1.20</b> Bright Field</a></li>
<li class="chapter" data-level="3.1.21" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#oblique-illumination"><i class="fa fa-check"></i><b>3.1.21</b> Oblique Illumination</a></li>
<li class="chapter" data-level="3.1.22" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dark-field"><i class="fa fa-check"></i><b>3.1.22</b> Dark Field</a></li>
<li class="chapter" data-level="3.1.23" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dispersion-staining"><i class="fa fa-check"></i><b>3.1.23</b> Dispersion Staining</a></li>
<li class="chapter" data-level="3.1.24" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#phase-contrast"><i class="fa fa-check"></i><b>3.1.24</b> Phase Contrast</a></li>
<li class="chapter" data-level="3.1.25" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#differential-interference-contrast"><i class="fa fa-check"></i><b>3.1.25</b> Differential Interference Contrast</a></li>
<li class="chapter" data-level="3.1.26" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#interference-reflection"><i class="fa fa-check"></i><b>3.1.26</b> Interference Reflection</a></li>
<li class="chapter" data-level="3.1.27" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#fluorescence-microscopy"><i class="fa fa-check"></i><b>3.1.27</b> Fluorescence Microscopy</a></li>
<li class="chapter" data-level="3.1.28" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#confoca-microscopy"><i class="fa fa-check"></i><b>3.1.28</b> Confoca Microscopy</a></li>
<li class="chapter" data-level="3.1.29" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#two-photon-microscopy"><i class="fa fa-check"></i><b>3.1.29</b> Two-Photon Microscopy</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-gram-stain"><i class="fa fa-check"></i><b>3.2</b> The Gram Stain</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-positive-bacteria"><i class="fa fa-check"></i><b>3.2.1</b> Gram-Positive Bacteria</a></li>
<li class="chapter" data-level="3.2.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-negative-bacteria"><i class="fa fa-check"></i><b>3.2.2</b> Gram-Negative Bacteria</a></li>
<li class="chapter" data-level="3.2.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-variable-and-gram-indeterminate-bacteria"><i class="fa fa-check"></i><b>3.2.3</b> Gram-Variable and Gram-Indeterminate Bacteria</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#electron-microscopy"><i class="fa fa-check"></i><b>3.3</b> Electron Microscopy</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-transmission-electron-microscope-tem"><i class="fa fa-check"></i><b>3.3.1</b> The Transmission Electron Microscope (TEM)</a></li>
<li class="chapter" data-level="3.3.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-scanning-electron-microscope-sem"><i class="fa fa-check"></i><b>3.3.2</b> The Scanning Electron Microscope (SEM)</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#microbiological-culture"><i class="fa fa-check"></i><b>3.4</b> Microbiological Culture</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#broth-cultures"><i class="fa fa-check"></i><b>3.4.1</b> Broth cultures</a></li>
<li class="chapter" data-level="3.4.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#agar-plates"><i class="fa fa-check"></i><b>3.4.2</b> Agar Plates</a></li>
<li class="chapter" data-level="3.4.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#stab-cultures"><i class="fa fa-check"></i><b>3.4.3</b> Stab Cultures</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#culture-collections"><i class="fa fa-check"></i><b>3.5</b> Culture Collections</a></li>
<li class="chapter" data-level="3.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#growth-medium"><i class="fa fa-check"></i><b>3.6</b> Growth Medium</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#culture-media"><i class="fa fa-check"></i><b>3.6.1</b> Culture Media</a></li>
<li class="chapter" data-level="3.6.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#minimal-media"><i class="fa fa-check"></i><b>3.6.2</b> Minimal Media</a></li>
<li class="chapter" data-level="3.6.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#selective-media"><i class="fa fa-check"></i><b>3.6.3</b> Selective Media</a></li>
<li class="chapter" data-level="3.6.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#differential-media"><i class="fa fa-check"></i><b>3.6.4</b> Differential Media</a></li>
<li class="chapter" data-level="3.6.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#transport-media"><i class="fa fa-check"></i><b>3.6.5</b> Transport Media</a></li>
<li class="chapter" data-level="3.6.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#enriched-media"><i class="fa fa-check"></i><b>3.6.6</b> Enriched Media</a></li>
<li class="chapter" data-level="3.6.7" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#agar-plate"><i class="fa fa-check"></i><b>3.6.7</b> Agar Plate</a></li>
<li class="chapter" data-level="3.6.8" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#blood-agar-plate"><i class="fa fa-check"></i><b>3.6.8</b> Blood Agar Plate</a></li>
<li class="chapter" data-level="3.6.9" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#chocolate-agar"><i class="fa fa-check"></i><b>3.6.9</b> Chocolate Agar</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><i class="fa fa-check"></i><b>4</b> An Introduction To Prokaryotic Cells And Microorganisms</a>
<ul>
<li class="chapter" data-level="4.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#basic-characteristics-of-life"><i class="fa fa-check"></i><b>4.1</b> Basic Characteristics Of Life</a></li>
<li class="chapter" data-level="4.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#origin-of-life"><i class="fa fa-check"></i><b>4.2</b> Origin Of Life</a></li>
<li class="chapter" data-level="4.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-foundations-of-modern-biology"><i class="fa fa-check"></i><b>4.3</b> The Foundations Of Modern Biology</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-cell-theory"><i class="fa fa-check"></i><b>4.3.1</b> The Cell Theory</a></li>
<li class="chapter" data-level="4.3.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-theory-of-evolution"><i class="fa fa-check"></i><b>4.3.2</b> The Theory Of Evolution</a></li>
<li class="chapter" data-level="4.3.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#genetics"><i class="fa fa-check"></i><b>4.3.3</b> Genetics</a></li>
<li class="chapter" data-level="4.3.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#homeostasis"><i class="fa fa-check"></i><b>4.3.4</b> Homeostasis</a></li>
<li class="chapter" data-level="4.3.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#energy-1"><i class="fa fa-check"></i><b>4.3.5</b> Energy</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#basic-characteristics-of-cells"><i class="fa fa-check"></i><b>4.4</b> Basic Characteristics Of Cells</a></li>
<li class="chapter" data-level="4.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#prokaryotic-cells"><i class="fa fa-check"></i><b>4.5</b> Prokaryotic Cells</a></li>
<li class="chapter" data-level="4.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacteria-1"><i class="fa fa-check"></i><b>4.6</b> Bacteria</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#intracellular-structure"><i class="fa fa-check"></i><b>4.6.1</b> Intracellular Structure</a></li>
<li class="chapter" data-level="4.6.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#extracellular-structures"><i class="fa fa-check"></i><b>4.6.2</b> Extracellular Structures</a></li>
<li class="chapter" data-level="4.6.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#endospores"><i class="fa fa-check"></i><b>4.6.3</b> Endospores</a></li>
<li class="chapter" data-level="4.6.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#metabolism"><i class="fa fa-check"></i><b>4.6.4</b> Metabolism</a></li>
<li class="chapter" data-level="4.6.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#growth-and-reproduction"><i class="fa fa-check"></i><b>4.6.5</b> Growth And Reproduction</a></li>
<li class="chapter" data-level="4.6.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacterial-genetics"><i class="fa fa-check"></i><b>4.6.6</b> Bacterial Genetics</a></li>
<li class="chapter" data-level="4.6.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#movement-of-bacteria"><i class="fa fa-check"></i><b>4.6.7</b> Movement Of Bacteria</a></li>
<li class="chapter" data-level="4.6.8" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacterial-communication"><i class="fa fa-check"></i><b>4.6.8</b> Bacterial Communication</a></li>
<li class="chapter" data-level="4.6.9" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#classification-and-identification"><i class="fa fa-check"></i><b>4.6.9</b> Classification And identification</a></li>
<li class="chapter" data-level="4.6.10" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#interactions-with-other-organisms"><i class="fa fa-check"></i><b>4.6.10</b> Interactions With Other Organisms</a></li>
<li class="chapter" data-level="4.6.11" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacteria-as-pathogens"><i class="fa fa-check"></i><b>4.6.11</b> Bacteria As Pathogens</a></li>
<li class="chapter" data-level="4.6.12" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#significance-in-technology-and-industry"><i class="fa fa-check"></i><b>4.6.12</b> Significance In Technology And Industry</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaea-1"><i class="fa fa-check"></i><b>4.7</b> Archaea</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#origin-and-evolution"><i class="fa fa-check"></i><b>4.7.1</b> Origin And evolution</a></li>
<li class="chapter" data-level="4.7.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#relationship-to-bacteria"><i class="fa fa-check"></i><b>4.7.2</b> Relationship To Bacteria</a></li>
<li class="chapter" data-level="4.7.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#relation-to-eukaryotes"><i class="fa fa-check"></i><b>4.7.3</b> Relation To Eukaryotes</a></li>
<li class="chapter" data-level="4.7.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#morphology"><i class="fa fa-check"></i><b>4.7.4</b> Morphology</a></li>
<li class="chapter" data-level="4.7.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#structure-composition-development-and-operation"><i class="fa fa-check"></i><b>4.7.5</b> Structure, Composition Development, And Operation</a></li>
<li class="chapter" data-level="4.7.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#cell-wall-and-flagella"><i class="fa fa-check"></i><b>4.7.6</b> Cell Wall And Flagella</a></li>
<li class="chapter" data-level="4.7.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#membranes"><i class="fa fa-check"></i><b>4.7.7</b> Membranes</a></li>
<li class="chapter" data-level="4.7.8" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-metabolism"><i class="fa fa-check"></i><b>4.7.8</b> Archaeal Metabolism</a></li>
<li class="chapter" data-level="4.7.9" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-genetics"><i class="fa fa-check"></i><b>4.7.9</b> Archaeal Genetics</a></li>
<li class="chapter" data-level="4.7.10" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-ecology"><i class="fa fa-check"></i><b>4.7.10</b> Archaeal Ecology</a></li>
<li class="chapter" data-level="4.7.11" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#significance-in-technology-and-industry-1"><i class="fa fa-check"></i><b>4.7.11</b> Significance In Technology And Industry</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><i class="fa fa-check"></i><b>5</b> An Introduction To Eukaryotic Cells And Microorganisms</a>
<ul>
<li class="chapter" data-level="5.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#characteristic-features-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.1</b> Characteristic Features Of Eukaryotic Cells</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cell-membrane"><i class="fa fa-check"></i><b>5.1.1</b> The Cell Membrane</a></li>
<li class="chapter" data-level="5.1.2" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cytoskeleton"><i class="fa fa-check"></i><b>5.1.2</b> The Cytoskeleton</a></li>
<li class="chapter" data-level="5.1.3" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-genetic-material"><i class="fa fa-check"></i><b>5.1.3</b> The Genetic Material</a></li>
<li class="chapter" data-level="5.1.4" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#organelles-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.1.4</b> Organelles of Eukaryotic Cells</a></li>
<li class="chapter" data-level="5.1.5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cell-nucleus"><i class="fa fa-check"></i><b>5.1.5</b> The Cell Nucleus</a></li>
<li class="chapter" data-level="5.1.6" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-mitochondria"><i class="fa fa-check"></i><b>5.1.6</b> The Mitochondria</a></li>
<li class="chapter" data-level="5.1.7" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-chloroplasts"><i class="fa fa-check"></i><b>5.1.7</b> The Chloroplasts</a></li>
<li class="chapter" data-level="5.1.8" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-endoplasmic-reticulum"><i class="fa fa-check"></i><b>5.1.8</b> The Endoplasmic Reticulum</a></li>
<li class="chapter" data-level="5.1.9" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-golgi-apparatus"><i class="fa fa-check"></i><b>5.1.9</b> The Golgi Apparatus</a></li>
<li class="chapter" data-level="5.1.10" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-ribosomes"><i class="fa fa-check"></i><b>5.1.10</b> The Ribosomes</a></li>
<li class="chapter" data-level="5.1.11" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#structures-outside-the-cell-membrane"><i class="fa fa-check"></i><b>5.1.11</b> Structures Outside The Cell Membrane</a></li>
<li class="chapter" data-level="5.1.12" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cellular-replication"><i class="fa fa-check"></i><b>5.1.12</b> Cellular Replication</a></li>
<li class="chapter" data-level="5.1.13" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#dna-repair"><i class="fa fa-check"></i><b>5.1.13</b> DNA Repair</a></li>
<li class="chapter" data-level="5.1.14" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cellular-growth-and-metabolism"><i class="fa fa-check"></i><b>5.1.14</b> Cellular Growth And Metabolism</a></li>
<li class="chapter" data-level="5.1.15" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protein-synthesis"><i class="fa fa-check"></i><b>5.1.15</b> Protein Synthesis</a></li>
<li class="chapter" data-level="5.1.16" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cell-motility"><i class="fa fa-check"></i><b>5.1.16</b> Cell Motility</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#origin-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.2</b> Origin Of Eukaryotic Cells</a></li>
<li class="chapter" data-level="5.3" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protists-1"><i class="fa fa-check"></i><b>5.3</b> Protists</a></li>
<li class="chapter" data-level="5.4" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protozoa"><i class="fa fa-check"></i><b>5.4</b> Protozoa</a></li>
<li class="chapter" data-level="5.5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#flagellates"><i class="fa fa-check"></i><b>5.5</b> Flagellates</a></li>
<li class="chapter" data-level="5.6" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#ciliates"><i class="fa fa-check"></i><b>5.6</b> Ciliates</a></li>
<li class="chapter" data-level="5.7" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#amoebozoa"><i class="fa fa-check"></i><b>5.7</b> Amoebozoa</a></li>
<li class="chapter" data-level="5.8" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#algae"><i class="fa fa-check"></i><b>5.8</b> Algae</a></li>
<li class="chapter" data-level="5.9" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#slime-molds"><i class="fa fa-check"></i><b>5.9</b> Slime Molds</a>
<ul>
<li class="chapter" data-level="5.9.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#reproduction-of-protists"><i class="fa fa-check"></i><b>5.9.1</b> Reproduction of Protists</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html"><i class="fa fa-check"></i><b>6</b> An Introduction To Viruses</a>
<ul>
<li class="chapter" data-level="6.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#theories-of-the-origin-of-viruses"><i class="fa fa-check"></i><b>6.1</b> Theories Of The Origin Of Viruses</a>
<ul>
<li class="chapter" data-level="6.1.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#regressive-hypothesis"><i class="fa fa-check"></i><b>6.1.1</b> Regressive Hypothesis</a></li>
<li class="chapter" data-level="6.1.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#cellular-origin-hypothesis"><i class="fa fa-check"></i><b>6.1.2</b> Cellular Origin Hypothesis</a></li>
<li class="chapter" data-level="6.1.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#co-evolution-hypothesis"><i class="fa fa-check"></i><b>6.1.3</b> Co-Evolution Hypothesis</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#structure-and-morphology-of-viruses"><i class="fa fa-check"></i><b>6.2</b> Structure And Morphology Of Viruses</a></li>
<li class="chapter" data-level="6.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#replication-of-viruses"><i class="fa fa-check"></i><b>6.3</b> Replication Of Viruses</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#role-of-viruses-in-human-disease"><i class="fa fa-check"></i><b>6.3.1</b> Role Of Viruses In Human Disease</a></li>
<li class="chapter" data-level="6.3.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#animal-viruses"><i class="fa fa-check"></i><b>6.3.2</b> Animal Viruses</a></li>
<li class="chapter" data-level="6.3.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#plant-viruses"><i class="fa fa-check"></i><b>6.3.3</b> Plant Viruses</a></li>
<li class="chapter" data-level="6.3.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#bacterial-viruses"><i class="fa fa-check"></i><b>6.3.4</b> Bacterial Viruses</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#classification-of-viruses"><i class="fa fa-check"></i><b>6.4</b> Classification Of Viruses</a></li>
<li class="chapter" data-level="6.5" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#the-baltimore-classification-of-viruses"><i class="fa fa-check"></i><b>6.5</b> The Baltimore Classification Of Viruses</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html"><i class="fa fa-check"></i><b>7</b> Microbial Nutrition, Ecology And Growth</a>
<ul>
<li class="chapter" data-level="7.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-energy"><i class="fa fa-check"></i><b>7.1</b> Primary Sources Of Energy</a></li>
<li class="chapter" data-level="7.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-reducing-equivalents"><i class="fa fa-check"></i><b>7.2</b> Primary Sources Of Reducing Equivalents</a></li>
<li class="chapter" data-level="7.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-carbon"><i class="fa fa-check"></i><b>7.3</b> Primary Sources of Carbon</a></li>
<li class="chapter" data-level="7.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#nutrients"><i class="fa fa-check"></i><b>7.4</b> Nutrients</a></li>
<li class="chapter" data-level="7.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#the-nutrient-cycle"><i class="fa fa-check"></i><b>7.5</b> The Nutrient Cycle</a></li>
<li class="chapter" data-level="7.6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#transport-and-movement-of-substances-across-the-cell-membrane"><i class="fa fa-check"></i><b>7.6</b> Transport And Movement Of Substances Across The Cell Membrane</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#diffusion"><i class="fa fa-check"></i><b>7.6.1</b> Diffusion</a></li>
<li class="chapter" data-level="7.6.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#facilitated-diffusion"><i class="fa fa-check"></i><b>7.6.2</b> Facilitated Diffusion</a></li>
<li class="chapter" data-level="7.6.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#osmosis"><i class="fa fa-check"></i><b>7.6.3</b> Osmosis</a></li>
<li class="chapter" data-level="7.6.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#active-transport"><i class="fa fa-check"></i><b>7.6.4</b> Active Transport</a></li>
<li class="chapter" data-level="7.6.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#bulk-transport"><i class="fa fa-check"></i><b>7.6.5</b> Bulk transport</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#microbial-growth"><i class="fa fa-check"></i><b>7.7</b> Microbial Growth</a></li>
<li class="chapter" data-level="7.8" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#cell-counting"><i class="fa fa-check"></i><b>7.8</b> Cell Counting</a>
<ul>
<li class="chapter" data-level="7.8.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#counting-chamber"><i class="fa fa-check"></i><b>7.8.1</b> Counting Chamber</a></li>
<li class="chapter" data-level="7.8.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#plating-and-the-colony-forming-unit-counting"><i class="fa fa-check"></i><b>7.8.2</b> Plating And The Colony-Forming Unit Counting</a></li>
<li class="chapter" data-level="7.8.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#electrical-resistance"><i class="fa fa-check"></i><b>7.8.3</b> Electrical Resistance</a></li>
<li class="chapter" data-level="7.8.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#flow-cytometry"><i class="fa fa-check"></i><b>7.8.4</b> Flow Cytometry</a></li>
<li class="chapter" data-level="7.8.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#spectrophotometry"><i class="fa fa-check"></i><b>7.8.5</b> Spectrophotometry</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#environmental-factors-influencing-growth"><i class="fa fa-check"></i><b>7.9</b> Environmental Factors Influencing Growth</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#nutrient-availability"><i class="fa fa-check"></i><b>7.9.1</b> Nutrient Availability</a></li>
<li class="chapter" data-level="7.9.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#temperature"><i class="fa fa-check"></i><b>7.9.2</b> Temperature</a></li>
<li class="chapter" data-level="7.9.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#acidity"><i class="fa fa-check"></i><b>7.9.3</b> Acidity</a></li>
<li class="chapter" data-level="7.9.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#water-activity"><i class="fa fa-check"></i><b>7.9.4</b> Water activity</a></li>
<li class="chapter" data-level="7.9.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#oxygen"><i class="fa fa-check"></i><b>7.9.5</b> Oxygen</a></li>
<li class="chapter" data-level="7.9.6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#toxic-compounds"><i class="fa fa-check"></i><b>7.9.6</b> Toxic compounds</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html"><i class="fa fa-check"></i><b>8</b> An Introduction To Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="8.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#thermodynamics-of-living-organisms"><i class="fa fa-check"></i><b>8.1</b> Thermodynamics of Living Organisms</a></li>
<li class="chapter" data-level="8.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#metabolism-1"><i class="fa fa-check"></i><b>8.2</b> Metabolism</a></li>
<li class="chapter" data-level="8.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#types-of-microbial-metabolism"><i class="fa fa-check"></i><b>8.3</b> Types Of Microbial Metabolism</a></li>
<li class="chapter" data-level="8.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#phototrophy"><i class="fa fa-check"></i><b>8.4</b> Phototrophy</a></li>
<li class="chapter" data-level="8.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#chemolithotrophy"><i class="fa fa-check"></i><b>8.5</b> Chemolithotrophy</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#hydrogen-oxidation"><i class="fa fa-check"></i><b>8.5.1</b> Hydrogen oxidation</a></li>
<li class="chapter" data-level="8.5.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#sulfur-oxidation"><i class="fa fa-check"></i><b>8.5.2</b> Sulfur oxidation</a></li>
<li class="chapter" data-level="8.5.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#ferrous-iron-fe2-oxidation"><i class="fa fa-check"></i><b>8.5.3</b> Ferrous Iron (Fe<sup>2</sup>) Oxidation</a></li>
<li class="chapter" data-level="8.5.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nitrification"><i class="fa fa-check"></i><b>8.5.4</b> Nitrification</a></li>
<li class="chapter" data-level="8.5.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nitrogen-fixation"><i class="fa fa-check"></i><b>8.5.5</b> Nitrogen Fixation</a></li>
<li class="chapter" data-level="8.5.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#catabolism"><i class="fa fa-check"></i><b>8.5.6</b> Catabolism</a></li>
<li class="chapter" data-level="8.5.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#digestion"><i class="fa fa-check"></i><b>8.5.7</b> Digestion</a></li>
<li class="chapter" data-level="8.5.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-organic-compounds"><i class="fa fa-check"></i><b>8.5.8</b> Energy From Organic Compounds</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-transformations"><i class="fa fa-check"></i><b>8.6</b> Energy Transformations</a>
<ul>
<li class="chapter" data-level="8.6.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-phosphorylation"><i class="fa fa-check"></i><b>8.6.1</b> Oxidative Phosphorylation</a></li>
<li class="chapter" data-level="8.6.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-inorganic-compounds"><i class="fa fa-check"></i><b>8.6.2</b> Energy From Inorganic Compounds</a></li>
<li class="chapter" data-level="8.6.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-light"><i class="fa fa-check"></i><b>8.6.3</b> Energy From Light</a></li>
<li class="chapter" data-level="8.6.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#anabolism"><i class="fa fa-check"></i><b>8.6.4</b> Anabolism</a></li>
</ul></li>
<li class="chapter" data-level="8.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#enzymes"><i class="fa fa-check"></i><b>8.7</b> Enzymes</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#classification-and-nomenclature-of-enzymes"><i class="fa fa-check"></i><b>8.7.1</b> Classification And Nomenclature Of Enzymes</a></li>
<li class="chapter" data-level="8.7.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#stucture-of-enzymes"><i class="fa fa-check"></i><b>8.7.2</b> Stucture Of Enzymes</a></li>
<li class="chapter" data-level="8.7.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#mechanism-of-action-of-enzymes"><i class="fa fa-check"></i><b>8.7.3</b> Mechanism Of Action Of Enzymes</a></li>
<li class="chapter" data-level="8.7.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#allosteric-modulation"><i class="fa fa-check"></i><b>8.7.4</b> Allosteric Modulation</a></li>
<li class="chapter" data-level="8.7.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cofactors-of-enzymes-and-coenzymes"><i class="fa fa-check"></i><b>8.7.5</b> Cofactors Of Enzymes And Coenzymes</a></li>
<li class="chapter" data-level="8.7.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#inhibition-of-enzymes"><i class="fa fa-check"></i><b>8.7.6</b> Inhibition Of Enzymes</a></li>
<li class="chapter" data-level="8.7.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#factors-affecting-enzyme-activity"><i class="fa fa-check"></i><b>8.7.7</b> Factors Affecting Enzyme Activity</a></li>
<li class="chapter" data-level="8.7.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#control-of-activity"><i class="fa fa-check"></i><b>8.7.8</b> Control Of Activity</a></li>
<li class="chapter" data-level="8.7.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#post-translational-modification"><i class="fa fa-check"></i><b>8.7.9</b> Post-translational Modification</a></li>
<li class="chapter" data-level="8.7.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#quantity"><i class="fa fa-check"></i><b>8.7.10</b> Quantity</a></li>
<li class="chapter" data-level="8.7.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#subcellular-distribution"><i class="fa fa-check"></i><b>8.7.11</b> Subcellular Distribution</a></li>
<li class="chapter" data-level="8.7.12" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#organ-specialization"><i class="fa fa-check"></i><b>8.7.12</b> Organ Specialization</a></li>
<li class="chapter" data-level="8.7.13" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#coenzymes"><i class="fa fa-check"></i><b>8.7.13</b> Coenzymes</a></li>
<li class="chapter" data-level="8.7.14" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#mineral-cofactors"><i class="fa fa-check"></i><b>8.7.14</b> Mineral Cofactors</a></li>
</ul></li>
<li class="chapter" data-level="8.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#heterotrophic-microbial-metabolism"><i class="fa fa-check"></i><b>8.8</b> Heterotrophic Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="8.8.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#carbon-fixation"><i class="fa fa-check"></i><b>8.8.1</b> Carbon Fixation</a></li>
<li class="chapter" data-level="8.8.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#carbohydrates-and-glycans"><i class="fa fa-check"></i><b>8.8.2</b> Carbohydrates And Glycans</a></li>
<li class="chapter" data-level="8.8.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#fatty-acids-isoprenoids-and-sterol"><i class="fa fa-check"></i><b>8.8.3</b> Fatty Acids, Isoprenoids and Sterol</a></li>
<li class="chapter" data-level="8.8.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#proteins-1"><i class="fa fa-check"></i><b>8.8.4</b> Proteins</a></li>
<li class="chapter" data-level="8.8.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nucleotide-synthesis-and-salvage"><i class="fa fa-check"></i><b>8.8.5</b> Nucleotide Synthesis And Salvage</a></li>
</ul></li>
<li class="chapter" data-level="8.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthesis-and-cellular-respiration"><i class="fa fa-check"></i><b>8.9</b> Photosynthesis And Cellular Respiration</a>
<ul>
<li class="chapter" data-level="8.9.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#the-electron-transport-chain-and-chemiosmosis"><i class="fa fa-check"></i><b>8.9.1</b> The Electron Transport Chain And Chemiosmosis</a></li>
<li class="chapter" data-level="8.9.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthesis"><i class="fa fa-check"></i><b>8.9.2</b> Photosynthesis</a></li>
<li class="chapter" data-level="8.9.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthetic-membranes-and-organelles"><i class="fa fa-check"></i><b>8.9.3</b> Photosynthetic Membranes And Organelles</a></li>
<li class="chapter" data-level="8.9.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#z-scheme"><i class="fa fa-check"></i><b>8.9.4</b> Z Scheme</a></li>
<li class="chapter" data-level="8.9.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#water-photolysis"><i class="fa fa-check"></i><b>8.9.5</b> Water Photolysis</a></li>
<li class="chapter" data-level="8.9.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#calvin-cycle"><i class="fa fa-check"></i><b>8.9.6</b> Calvin Cycle</a></li>
<li class="chapter" data-level="8.9.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#symbiosis-and-the-origin-of-chloroplasts"><i class="fa fa-check"></i><b>8.9.7</b> Symbiosis And The Origin of Chloroplasts</a></li>
<li class="chapter" data-level="8.9.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cyanobacteria-and-the-evolution-of-photosynthesis"><i class="fa fa-check"></i><b>8.9.8</b> Cyanobacteria And The Evolution of Photosynthesis</a></li>
<li class="chapter" data-level="8.9.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cellular-respiration"><i class="fa fa-check"></i><b>8.9.9</b> Cellular Respiration</a></li>
<li class="chapter" data-level="8.9.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#aerobic-respiration"><i class="fa fa-check"></i><b>8.9.10</b> Aerobic Respiration</a></li>
<li class="chapter" data-level="8.9.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#glycolysis"><i class="fa fa-check"></i><b>8.9.11</b> Glycolysis</a></li>
<li class="chapter" data-level="8.9.12" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-decarboxylation-of-pyruvate"><i class="fa fa-check"></i><b>8.9.12</b> Oxidative Decarboxylation of Pyruvate</a></li>
<li class="chapter" data-level="8.9.13" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#the-citric-acid-cycle"><i class="fa fa-check"></i><b>8.9.13</b> The Citric Acid Cycle</a></li>
<li class="chapter" data-level="8.9.14" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-phosphorylation-1"><i class="fa fa-check"></i><b>8.9.14</b> Oxidative Phosphorylation</a></li>
</ul></li>
<li class="chapter" data-level="8.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#fermentation"><i class="fa fa-check"></i><b>8.10</b> Fermentation</a></li>
<li class="chapter" data-level="8.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#anaerobic-respiration"><i class="fa fa-check"></i><b>8.11</b> Anaerobic Respiration</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="microbial-genetics.html"><a href="microbial-genetics.html"><i class="fa fa-check"></i><b>9</b> Microbial Genetics</a>
<ul>
<li class="chapter" data-level="9.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#introduction-to-genetics-and-genes"><i class="fa fa-check"></i><b>9.1</b> Introduction To Genetics And Genes</a>
<ul>
<li class="chapter" data-level="9.1.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mendelian-classical-genetics"><i class="fa fa-check"></i><b>9.1.1</b> Mendelian (Classical) Genetics</a></li>
<li class="chapter" data-level="9.1.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#molecular-genetics"><i class="fa fa-check"></i><b>9.1.2</b> Molecular Genetics</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#the-genetic-code"><i class="fa fa-check"></i><b>9.2</b> The Genetic Code</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-expression"><i class="fa fa-check"></i><b>9.2.1</b> Gene Expression</a></li>
<li class="chapter" data-level="9.2.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#nature-and-nurture"><i class="fa fa-check"></i><b>9.2.2</b> Nature And Nurture</a></li>
<li class="chapter" data-level="9.2.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-regulation"><i class="fa fa-check"></i><b>9.2.3</b> Gene Regulation</a></li>
<li class="chapter" data-level="9.2.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genetic-change"><i class="fa fa-check"></i><b>9.2.4</b> Genetic Change</a></li>
<li class="chapter" data-level="9.2.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#natural-selection-and-evolution"><i class="fa fa-check"></i><b>9.2.5</b> Natural Selection And Evolution</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-genetics-1"><i class="fa fa-check"></i><b>9.3</b> Bacterial Genetics</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-transformation"><i class="fa fa-check"></i><b>9.3.1</b> Bacterial Transformation</a></li>
<li class="chapter" data-level="9.3.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#practical-aspects-of-transformation-in-molecular-biology"><i class="fa fa-check"></i><b>9.3.2</b> Practical aspects of transformation in molecular biology</a></li>
<li class="chapter" data-level="9.3.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#selection-and-screening-in-plasmid-transformation"><i class="fa fa-check"></i><b>9.3.3</b> Selection and screening in plasmid transformation</a></li>
<li class="chapter" data-level="9.3.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-conjugation"><i class="fa fa-check"></i><b>9.3.4</b> Bacterial Conjugation</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-and-function-of-deoxyribonucleic-acid"><i class="fa fa-check"></i><b>9.4</b> Structure And Function Of Deoxyribonucleic Acid</a>
<ul>
<li class="chapter" data-level="9.4.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-damage"><i class="fa fa-check"></i><b>9.4.1</b> DNA Damage</a></li>
<li class="chapter" data-level="9.4.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genes-and-genomes"><i class="fa fa-check"></i><b>9.4.2</b> Genes And Genomes</a></li>
<li class="chapter" data-level="9.4.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-translation"><i class="fa fa-check"></i><b>9.4.3</b> Transcription And Translation</a></li>
<li class="chapter" data-level="9.4.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-replication"><i class="fa fa-check"></i><b>9.4.4</b> DNA Replication</a></li>
<li class="chapter" data-level="9.4.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-dna-replication"><i class="fa fa-check"></i><b>9.4.5</b> Eukaryotic DNA Replication</a></li>
<li class="chapter" data-level="9.4.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation"><i class="fa fa-check"></i><b>9.4.6</b> Initiation</a></li>
<li class="chapter" data-level="9.4.7" data-path="microbial-genetics.html"><a href="microbial-genetics.html#pre-replication-complex"><i class="fa fa-check"></i><b>9.4.7</b> Pre-Replication Complex</a></li>
<li class="chapter" data-level="9.4.8" data-path="microbial-genetics.html"><a href="microbial-genetics.html#preinitiation-complex"><i class="fa fa-check"></i><b>9.4.8</b> Preinitiation Complex</a></li>
<li class="chapter" data-level="9.4.9" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation"><i class="fa fa-check"></i><b>9.4.9</b> Elongation</a></li>
<li class="chapter" data-level="9.4.10" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-fork"><i class="fa fa-check"></i><b>9.4.10</b> Replication Fork</a></li>
<li class="chapter" data-level="9.4.11" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-of-the-leading-strand"><i class="fa fa-check"></i><b>9.4.11</b> Replication Of The Leading Strand</a></li>
<li class="chapter" data-level="9.4.12" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-of-the-lagging-strand"><i class="fa fa-check"></i><b>9.4.12</b> Replication Of The Lagging Strand</a></li>
<li class="chapter" data-level="9.4.13" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-replication-proteins"><i class="fa fa-check"></i><b>9.4.13</b> DNA Replication Proteins</a></li>
<li class="chapter" data-level="9.4.14" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination"><i class="fa fa-check"></i><b>9.4.14</b> Termination</a></li>
<li class="chapter" data-level="9.4.15" data-path="microbial-genetics.html"><a href="microbial-genetics.html#regulation-of-dna-replication"><i class="fa fa-check"></i><b>9.4.15</b> Regulation Of DNA Replication</a></li>
<li class="chapter" data-level="9.4.16" data-path="microbial-genetics.html"><a href="microbial-genetics.html#interactions-of-dna-with-proteins"><i class="fa fa-check"></i><b>9.4.16</b> Interactions Of DNA with Proteins</a></li>
<li class="chapter" data-level="9.4.17" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-binding-proteins"><i class="fa fa-check"></i><b>9.4.17</b> DNA-Binding Proteins</a></li>
<li class="chapter" data-level="9.4.18" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>9.4.18</b> DNA-modifying Enzymes</a></li>
<li class="chapter" data-level="9.4.19" data-path="microbial-genetics.html"><a href="microbial-genetics.html#nucleases-and-ligases"><i class="fa fa-check"></i><b>9.4.19</b> Nucleases And Ligases</a></li>
<li class="chapter" data-level="9.4.20" data-path="microbial-genetics.html"><a href="microbial-genetics.html#topoisomerases-and-helicases"><i class="fa fa-check"></i><b>9.4.20</b> Topoisomerases And Helicases</a></li>
<li class="chapter" data-level="9.4.21" data-path="microbial-genetics.html"><a href="microbial-genetics.html#polymerases"><i class="fa fa-check"></i><b>9.4.21</b> Polymerases</a></li>
<li class="chapter" data-level="9.4.22" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-recombination"><i class="fa fa-check"></i><b>9.4.22</b> DNA Recombination</a></li>
<li class="chapter" data-level="9.4.23" data-path="microbial-genetics.html"><a href="microbial-genetics.html#evolutionary-history-of-dna"><i class="fa fa-check"></i><b>9.4.23</b> Evolutionary History Of DNA</a></li>
<li class="chapter" data-level="9.4.24" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genetic-engineering"><i class="fa fa-check"></i><b>9.4.24</b> Genetic Engineering</a></li>
<li class="chapter" data-level="9.4.25" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-profiling"><i class="fa fa-check"></i><b>9.4.25</b> Dna Profiling</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-and-function-of-ribonucleic-acid"><i class="fa fa-check"></i><b>9.5</b> Structure And Function Of Ribonucleic Acid</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#comparison-with-dna"><i class="fa fa-check"></i><b>9.5.1</b> Comparison With DNA</a></li>
<li class="chapter" data-level="9.5.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-of-rna"><i class="fa fa-check"></i><b>9.5.2</b> Structure Of RNA</a></li>
<li class="chapter" data-level="9.5.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#synthesis-of-rna"><i class="fa fa-check"></i><b>9.5.3</b> Synthesis Of RNA</a></li>
<li class="chapter" data-level="9.5.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#coding-and-non-coding-rna"><i class="fa fa-check"></i><b>9.5.4</b> Coding And Non-coding RNA</a></li>
<li class="chapter" data-level="9.5.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#regulatory-rna"><i class="fa fa-check"></i><b>9.5.5</b> Regulatory RNA</a></li>
<li class="chapter" data-level="9.5.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-interference-by-mirnas"><i class="fa fa-check"></i><b>9.5.6</b> RNA Interference By miRNAs</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-expression-1"><i class="fa fa-check"></i><b>9.6</b> Gene Expression</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-rna-processing"><i class="fa fa-check"></i><b>9.6.1</b> Transcription And RNA Processing</a></li>
<li class="chapter" data-level="9.6.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-1"><i class="fa fa-check"></i><b>9.6.2</b> Initiation</a></li>
<li class="chapter" data-level="9.6.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation-1"><i class="fa fa-check"></i><b>9.6.3</b> Elongation</a></li>
<li class="chapter" data-level="9.6.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination-1"><i class="fa fa-check"></i><b>9.6.4</b> Termination</a></li>
<li class="chapter" data-level="9.6.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#inhibitors-of-transcription"><i class="fa fa-check"></i><b>9.6.5</b> Inhibitors Of Transcription</a></li>
<li class="chapter" data-level="9.6.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-factors"><i class="fa fa-check"></i><b>9.6.6</b> Transcription Factors</a></li>
<li class="chapter" data-level="9.6.7" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-processing"><i class="fa fa-check"></i><b>9.6.7</b> RNA Processing</a></li>
<li class="chapter" data-level="9.6.8" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mrna-processing"><i class="fa fa-check"></i><b>9.6.8</b> mRNA Processing</a></li>
<li class="chapter" data-level="9.6.9" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing"><i class="fa fa-check"></i><b>9.6.9</b> 5’ Processing</a></li>
<li class="chapter" data-level="9.6.10" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-1"><i class="fa fa-check"></i><b>9.6.10</b> 3’ Processing</a></li>
<li class="chapter" data-level="9.6.11" data-path="microbial-genetics.html"><a href="microbial-genetics.html#intron-splicing"><i class="fa fa-check"></i><b>9.6.11</b> Intron Splicing</a></li>
<li class="chapter" data-level="9.6.12" data-path="microbial-genetics.html"><a href="microbial-genetics.html#reverse-transcription"><i class="fa fa-check"></i><b>9.6.12</b> Reverse Transcription</a></li>
<li class="chapter" data-level="9.6.13" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-export"><i class="fa fa-check"></i><b>9.6.13</b> RNA Export</a></li>
<li class="chapter" data-level="9.6.14" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-rna-processing-1"><i class="fa fa-check"></i><b>9.6.14</b> Transcription And RNA Processing</a></li>
<li class="chapter" data-level="9.6.15" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-2"><i class="fa fa-check"></i><b>9.6.15</b> Initiation</a></li>
<li class="chapter" data-level="9.6.16" data-path="microbial-genetics.html"><a href="microbial-genetics.html#inhibitors-of-transcription-1"><i class="fa fa-check"></i><b>9.6.16</b> Inhibitors of transcription</a></li>
<li class="chapter" data-level="9.6.17" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-factors-1"><i class="fa fa-check"></i><b>9.6.17</b> Transcription Factors</a></li>
<li class="chapter" data-level="9.6.18" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-processing-1"><i class="fa fa-check"></i><b>9.6.18</b> RNA Processing</a></li>
<li class="chapter" data-level="9.6.19" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mrna-processing-1"><i class="fa fa-check"></i><b>9.6.19</b> mRNA Processing</a></li>
<li class="chapter" data-level="9.6.20" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-2"><i class="fa fa-check"></i><b>9.6.20</b> 5’ Processing</a></li>
<li class="chapter" data-level="9.6.21" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-3"><i class="fa fa-check"></i><b>9.6.21</b> 3’ Processing</a></li>
<li class="chapter" data-level="9.6.22" data-path="microbial-genetics.html"><a href="microbial-genetics.html#intron-splicing-1"><i class="fa fa-check"></i><b>9.6.22</b> Intron Splicing</a></li>
<li class="chapter" data-level="9.6.23" data-path="microbial-genetics.html"><a href="microbial-genetics.html#reverse-transcription-1"><i class="fa fa-check"></i><b>9.6.23</b> Reverse Transcription</a></li>
<li class="chapter" data-level="9.6.24" data-path="microbial-genetics.html"><a href="microbial-genetics.html#translation-and-the-genetic-code"><i class="fa fa-check"></i><b>9.6.24</b> Translation And The Genetic Code</a></li>
<li class="chapter" data-level="9.6.25" data-path="microbial-genetics.html"><a href="microbial-genetics.html#ribosomes"><i class="fa fa-check"></i><b>9.6.25</b> Ribosomes</a></li>
<li class="chapter" data-level="9.6.26" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-ribosomes"><i class="fa fa-check"></i><b>9.6.26</b> Bacterial Ribosomes</a></li>
<li class="chapter" data-level="9.6.27" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-ribosomes"><i class="fa fa-check"></i><b>9.6.27</b> Eukaryotic Ribosomes</a></li>
<li class="chapter" data-level="9.6.28" data-path="microbial-genetics.html"><a href="microbial-genetics.html#translation"><i class="fa fa-check"></i><b>9.6.28</b> Translation</a></li>
<li class="chapter" data-level="9.6.29" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-translation"><i class="fa fa-check"></i><b>9.6.29</b> Eukaryotic Translation</a></li>
<li class="chapter" data-level="9.6.30" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-3"><i class="fa fa-check"></i><b>9.6.30</b> Initiation</a></li>
<li class="chapter" data-level="9.6.31" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation-2"><i class="fa fa-check"></i><b>9.6.31</b> Elongation</a></li>
<li class="chapter" data-level="9.6.32" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination-2"><i class="fa fa-check"></i><b>9.6.32</b> Termination</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html"><i class="fa fa-check"></i><b>10</b> Genetic Engineering</a>
<ul>
<li class="chapter" data-level="10.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genetically-modified-organisms"><i class="fa fa-check"></i><b>10.1</b> Genetically Modified Organisms</a>
<ul>
<li class="chapter" data-level="10.1.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#production"><i class="fa fa-check"></i><b>10.1.1</b> Production</a></li>
<li class="chapter" data-level="10.1.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#bacteria-2"><i class="fa fa-check"></i><b>10.1.2</b> Bacteria</a></li>
<li class="chapter" data-level="10.1.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#viruses"><i class="fa fa-check"></i><b>10.1.3</b> Viruses</a></li>
<li class="chapter" data-level="10.1.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#fungi-1"><i class="fa fa-check"></i><b>10.1.4</b> Fungi</a></li>
<li class="chapter" data-level="10.1.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#plants-1"><i class="fa fa-check"></i><b>10.1.5</b> Plants</a></li>
<li class="chapter" data-level="10.1.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#crops"><i class="fa fa-check"></i><b>10.1.6</b> Crops</a></li>
<li class="chapter" data-level="10.1.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#animals"><i class="fa fa-check"></i><b>10.1.7</b> Animals</a></li>
<li class="chapter" data-level="10.1.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#laws-and-regulations"><i class="fa fa-check"></i><b>10.1.8</b> Laws and regulations</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-therapy"><i class="fa fa-check"></i><b>10.2</b> Gene therapy</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#laws-and-regulations-1"><i class="fa fa-check"></i><b>10.2.1</b> Laws and regulations</a></li>
<li class="chapter" data-level="10.2.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-isolation-and-cloning"><i class="fa fa-check"></i><b>10.2.2</b> Gene Isolation And Cloning</a></li>
<li class="chapter" data-level="10.2.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#inserting-dna-into-the-host-genome"><i class="fa fa-check"></i><b>10.2.3</b> Inserting DNA Into The Host Genome</a></li>
<li class="chapter" data-level="10.2.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#applications-of-genetic-editing"><i class="fa fa-check"></i><b>10.2.4</b> Applications Of Genetic Editing</a></li>
<li class="chapter" data-level="10.2.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#regulation-of-genetic-engineering"><i class="fa fa-check"></i><b>10.2.5</b> Regulation Of Genetic Engineering</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#recombinant-dna-technology"><i class="fa fa-check"></i><b>10.3</b> Recombinant DNA Technology</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#molecular-cloning"><i class="fa fa-check"></i><b>10.3.1</b> Molecular Cloning</a></li>
<li class="chapter" data-level="10.3.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#procedures"><i class="fa fa-check"></i><b>10.3.2</b> Procedures</a></li>
<li class="chapter" data-level="10.3.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#mutagenesis"><i class="fa fa-check"></i><b>10.3.3</b> Mutagenesis</a></li>
<li class="chapter" data-level="10.3.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#random-mutagenesis"><i class="fa fa-check"></i><b>10.3.4</b> Random Mutagenesis</a></li>
<li class="chapter" data-level="10.3.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>10.3.5</b> Site-Directed Mutagenesis</a></li>
<li class="chapter" data-level="10.3.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-synthesis"><i class="fa fa-check"></i><b>10.3.6</b> Gene Synthesis</a></li>
<li class="chapter" data-level="10.3.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knockout"><i class="fa fa-check"></i><b>10.3.7</b> Gene Knockout</a></li>
<li class="chapter" data-level="10.3.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#conditional-gene-knockout"><i class="fa fa-check"></i><b>10.3.8</b> Conditional Gene Knockout</a></li>
<li class="chapter" data-level="10.3.9" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knock-in"><i class="fa fa-check"></i><b>10.3.9</b> Gene Knock-In</a></li>
<li class="chapter" data-level="10.3.10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#rna-interference"><i class="fa fa-check"></i><b>10.3.10</b> RNA interference</a></li>
<li class="chapter" data-level="10.3.11" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#microrna"><i class="fa fa-check"></i><b>10.3.11</b> MicroRNA</a></li>
<li class="chapter" data-level="10.3.12" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#transcriptional-silencing"><i class="fa fa-check"></i><b>10.3.12</b> Transcriptional Silencing</a></li>
<li class="chapter" data-level="10.3.13" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knockdown"><i class="fa fa-check"></i><b>10.3.13</b> Gene Knockdown</a></li>
<li class="chapter" data-level="10.3.14" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#cre-lox-recombination"><i class="fa fa-check"></i><b>10.3.14</b> Cre-Lox Recombination</a></li>
<li class="chapter" data-level="10.3.15" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#crispr-gene-editing"><i class="fa fa-check"></i><b>10.3.15</b> CRISPR Gene Editing</a></li>
<li class="chapter" data-level="10.3.16" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-drive"><i class="fa fa-check"></i><b>10.3.16</b> Gene Drive</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#human-germline-modification"><i class="fa fa-check"></i><b>10.4</b> Human Germline Modification</a></li>
<li class="chapter" data-level="10.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gain-of-function-research"><i class="fa fa-check"></i><b>10.5</b> Gain-of-function Research</a></li>
<li class="chapter" data-level="10.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genomics"><i class="fa fa-check"></i><b>10.6</b> Genomics</a>
<ul>
<li class="chapter" data-level="10.6.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#complete-genomes"><i class="fa fa-check"></i><b>10.6.1</b> Complete Genomes</a></li>
<li class="chapter" data-level="10.6.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#the-omics-revolution"><i class="fa fa-check"></i><b>10.6.2</b> The “Omics” Revolution</a></li>
<li class="chapter" data-level="10.6.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-analysis"><i class="fa fa-check"></i><b>10.6.3</b> Genome Analysis</a></li>
<li class="chapter" data-level="10.6.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-annotation"><i class="fa fa-check"></i><b>10.6.4</b> Genome Annotation</a></li>
<li class="chapter" data-level="10.6.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#functional-genomics"><i class="fa fa-check"></i><b>10.6.5</b> Functional Genomics</a></li>
<li class="chapter" data-level="10.6.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#structural-genomics"><i class="fa fa-check"></i><b>10.6.6</b> Structural Genomics</a></li>
<li class="chapter" data-level="10.6.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#epigenomics"><i class="fa fa-check"></i><b>10.6.7</b> Epigenomics</a></li>
<li class="chapter" data-level="10.6.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#metagenomics"><i class="fa fa-check"></i><b>10.6.8</b> Metagenomics</a></li>
<li class="chapter" data-level="10.6.9" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#applications-of-genomics"><i class="fa fa-check"></i><b>10.6.9</b> Applications Of Genomics</a></li>
<li class="chapter" data-level="10.6.10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genomic-medicine"><i class="fa fa-check"></i><b>10.6.10</b> Genomic Medicine</a></li>
<li class="chapter" data-level="10.6.11" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#synthetic-biology-and-bioengineering"><i class="fa fa-check"></i><b>10.6.11</b> Synthetic Biology And Bioengineering</a></li>
<li class="chapter" data-level="10.6.12" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#conservation-genomics"><i class="fa fa-check"></i><b>10.6.12</b> Conservation Genomics</a></li>
<li class="chapter" data-level="10.6.13" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-wide-association-study"><i class="fa fa-check"></i><b>10.6.13</b> Genome-Wide Association Study</a></li>
</ul></li>
<li class="chapter" data-level="10.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genetic-testing"><i class="fa fa-check"></i><b>10.7</b> Genetic Testing</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html"><i class="fa fa-check"></i><b>11</b> Physical And Chemical Agents for Microbial Control</a>
<ul>
<li class="chapter" data-level="11.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#sterilization"><i class="fa fa-check"></i><b>11.1</b> Sterilization</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#methods-of-sterilization"><i class="fa fa-check"></i><b>11.1.1</b> Methods of Sterilization</a></li>
<li class="chapter" data-level="11.1.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#steam"><i class="fa fa-check"></i><b>11.1.2</b> Steam</a></li>
<li class="chapter" data-level="11.1.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#dry-heat"><i class="fa fa-check"></i><b>11.1.3</b> Dry Heat</a></li>
<li class="chapter" data-level="11.1.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#flaming"><i class="fa fa-check"></i><b>11.1.4</b> Flaming</a></li>
<li class="chapter" data-level="11.1.5" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#incineration"><i class="fa fa-check"></i><b>11.1.5</b> Incineration</a></li>
<li class="chapter" data-level="11.1.6" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#tyndallization"><i class="fa fa-check"></i><b>11.1.6</b> Tyndallization</a></li>
<li class="chapter" data-level="11.1.7" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#chemical-sterilization"><i class="fa fa-check"></i><b>11.1.7</b> Chemical Sterilization</a></li>
<li class="chapter" data-level="11.1.8" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#ethylene-oxide"><i class="fa fa-check"></i><b>11.1.8</b> Ethylene oxide</a></li>
<li class="chapter" data-level="11.1.9" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#nitrogen-dioxide"><i class="fa fa-check"></i><b>11.1.9</b> Nitrogen Dioxide</a></li>
<li class="chapter" data-level="11.1.10" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#ozone"><i class="fa fa-check"></i><b>11.1.10</b> Ozone</a></li>
<li class="chapter" data-level="11.1.11" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#glutaraldehyde-and-formaldehyde"><i class="fa fa-check"></i><b>11.1.11</b> Glutaraldehyde And Formaldehyde</a></li>
<li class="chapter" data-level="11.1.12" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#hydrogen-peroxide"><i class="fa fa-check"></i><b>11.1.12</b> Hydrogen Peroxide</a></li>
<li class="chapter" data-level="11.1.13" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#peracetic-acid"><i class="fa fa-check"></i><b>11.1.13</b> Peracetic Acid</a></li>
<li class="chapter" data-level="11.1.14" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#potential-for-chemical-sterilization-of-prions"><i class="fa fa-check"></i><b>11.1.14</b> Potential For Chemical Sterilization Of Prions</a></li>
<li class="chapter" data-level="11.1.15" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#radiation-sterilization"><i class="fa fa-check"></i><b>11.1.15</b> Radiation Sterilization</a></li>
<li class="chapter" data-level="11.1.16" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#non-ionizing-radiation-sterilization"><i class="fa fa-check"></i><b>11.1.16</b> Non-Ionizing Radiation Sterilization</a></li>
<li class="chapter" data-level="11.1.17" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#ionizing-radiation-sterilization"><i class="fa fa-check"></i><b>11.1.17</b> Ionizing Radiation Sterilization</a></li>
<li class="chapter" data-level="11.1.18" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#sterile-filtration"><i class="fa fa-check"></i><b>11.1.18</b> Sterile filtration</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#pasteurization"><i class="fa fa-check"></i><b>11.2</b> Pasteurization</a></li>
<li class="chapter" data-level="11.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#disinfectants"><i class="fa fa-check"></i><b>11.3</b> Disinfectants</a>
<ul>
<li class="chapter" data-level="11.3.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#types-of-disinfectants"><i class="fa fa-check"></i><b>11.3.1</b> Types Of Disinfectants</a></li>
<li class="chapter" data-level="11.3.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#air-disinfectants"><i class="fa fa-check"></i><b>11.3.2</b> Air Disinfectants</a></li>
<li class="chapter" data-level="11.3.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#alcohols"><i class="fa fa-check"></i><b>11.3.3</b> Alcohols</a></li>
<li class="chapter" data-level="11.3.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#aldehydes"><i class="fa fa-check"></i><b>11.3.4</b> Aldehydes</a></li>
<li class="chapter" data-level="11.3.5" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#oxidizing-agents"><i class="fa fa-check"></i><b>11.3.5</b> Oxidizing agents</a></li>
<li class="chapter" data-level="11.3.6" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#peroxy-and-peroxo-acids"><i class="fa fa-check"></i><b>11.3.6</b> Peroxy And Peroxo Acids</a></li>
<li class="chapter" data-level="11.3.7" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#phenolics"><i class="fa fa-check"></i><b>11.3.7</b> Phenolics</a></li>
<li class="chapter" data-level="11.3.8" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#quaternary-ammonium-compounds"><i class="fa fa-check"></i><b>11.3.8</b> Quaternary ammonium compounds</a></li>
<li class="chapter" data-level="11.3.9" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#inorganic-compounds"><i class="fa fa-check"></i><b>11.3.9</b> Inorganic compounds</a></li>
<li class="chapter" data-level="11.3.10" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#chlorine"><i class="fa fa-check"></i><b>11.3.10</b> Chlorine</a></li>
<li class="chapter" data-level="11.3.11" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#iodine"><i class="fa fa-check"></i><b>11.3.11</b> Iodine</a></li>
<li class="chapter" data-level="11.3.12" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#acids-and-bases-1"><i class="fa fa-check"></i><b>11.3.12</b> Acids And Bases</a></li>
<li class="chapter" data-level="11.3.13" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#metals"><i class="fa fa-check"></i><b>11.3.13</b> Metals</a></li>
<li class="chapter" data-level="11.3.14" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#terpenes"><i class="fa fa-check"></i><b>11.3.14</b> Terpenes</a></li>
<li class="chapter" data-level="11.3.15" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#other"><i class="fa fa-check"></i><b>11.3.15</b> Other</a></li>
<li class="chapter" data-level="11.3.16" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#non-chemical"><i class="fa fa-check"></i><b>11.3.16</b> Non-chemical</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antiseptics"><i class="fa fa-check"></i><b>11.4</b> Antiseptics</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html"><i class="fa fa-check"></i><b>12</b> Drugs, Microbes, And The Host</a>
<ul>
<li class="chapter" data-level="12.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antimicrobials"><i class="fa fa-check"></i><b>12.1</b> Antimicrobials</a>
<ul>
<li class="chapter" data-level="12.1.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antibacterials"><i class="fa fa-check"></i><b>12.1.1</b> Antibacterials</a></li>
<li class="chapter" data-level="12.1.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antifungals"><i class="fa fa-check"></i><b>12.1.2</b> Antifungals</a></li>
<li class="chapter" data-level="12.1.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antivirals"><i class="fa fa-check"></i><b>12.1.3</b> Antivirals</a></li>
<li class="chapter" data-level="12.1.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antiparasitics"><i class="fa fa-check"></i><b>12.1.4</b> Antiparasitics</a></li>
<li class="chapter" data-level="12.1.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#broad-spectrum-therapeutics"><i class="fa fa-check"></i><b>12.1.5</b> Broad-Spectrum Therapeutics</a></li>
</ul></li>
<li class="chapter" data-level="12.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antibiotics"><i class="fa fa-check"></i><b>12.2</b> Antibiotics</a>
<ul>
<li class="chapter" data-level="12.2.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#classes-of-antibiotics"><i class="fa fa-check"></i><b>12.2.1</b> Classes of Antibiotics</a></li>
<li class="chapter" data-level="12.2.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#penicillin-and-other-natural-antibiotics"><i class="fa fa-check"></i><b>12.2.2</b> Penicillin And Other Natural Antibiotics</a></li>
<li class="chapter" data-level="12.2.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antistaphylococcal-antibiotics"><i class="fa fa-check"></i><b>12.2.3</b> Antistaphylococcal antibiotics</a></li>
<li class="chapter" data-level="12.2.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#broad-spectrum-antibiotics"><i class="fa fa-check"></i><b>12.2.4</b> Broad-spectrum antibiotics</a></li>
<li class="chapter" data-level="12.2.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antipseudomonal-antibiotics"><i class="fa fa-check"></i><b>12.2.5</b> Antipseudomonal antibiotics</a></li>
<li class="chapter" data-level="12.2.6" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#chemical-structure-of-penicillins"><i class="fa fa-check"></i><b>12.2.6</b> Chemical Structure Of Penicillins</a></li>
<li class="chapter" data-level="12.2.7" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#mechanism-of-action-of-penicillin"><i class="fa fa-check"></i><b>12.2.7</b> Mechanism Of Action Of Penicillin</a></li>
<li class="chapter" data-level="12.2.8" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#aminoglycoside-antibiotics"><i class="fa fa-check"></i><b>12.2.8</b> Aminoglycoside Antibiotics</a></li>
<li class="chapter" data-level="12.2.9" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#mechanism-of-action-of-amimoglycoside-antibiotics"><i class="fa fa-check"></i><b>12.2.9</b> Mechanism Of Action Of Amimoglycoside Antibiotics</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#drug-resistance"><i class="fa fa-check"></i><b>12.3</b> Drug Resistance</a>
<ul>
<li class="chapter" data-level="12.3.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#bacterial-resistence-to-penicillin"><i class="fa fa-check"></i><b>12.3.1</b> Bacterial Resistence To Penicillin</a></li>
</ul></li>
<li class="chapter" data-level="12.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#testing-for-antibiotic-sensitivity"><i class="fa fa-check"></i><b>12.4</b> Testing For Antibiotic Sensitivity</a>
<ul>
<li class="chapter" data-level="12.4.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#the-disk-diffusion-test"><i class="fa fa-check"></i><b>12.4.1</b> The Disk Diffusion Test</a></li>
</ul></li>
<li class="chapter" data-level="12.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#replenishing-the-antibiotic-pipeline-and-developing-other-new-therapies"><i class="fa fa-check"></i><b>12.5</b> Replenishing The Antibiotic Pipeline And Developing Other New Therapies</a>
<ul>
<li class="chapter" data-level="12.5.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#natural-product-based-antibiotic-discovery"><i class="fa fa-check"></i><b>12.5.1</b> Natural Product-Based Antibiotic Discovery</a></li>
<li class="chapter" data-level="12.5.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#immunoglobulin-therapy"><i class="fa fa-check"></i><b>12.5.2</b> Immunoglobulin Therapy</a></li>
<li class="chapter" data-level="12.5.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#phage-therapy"><i class="fa fa-check"></i><b>12.5.3</b> Phage Therapy</a></li>
<li class="chapter" data-level="12.5.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#fecal-microbiota-transplants"><i class="fa fa-check"></i><b>12.5.4</b> Fecal Microbiota Transplants</a></li>
<li class="chapter" data-level="12.5.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antisense-rna-based-treatments"><i class="fa fa-check"></i><b>12.5.5</b> Antisense RNA-Based Treatments</a></li>
<li class="chapter" data-level="12.5.6" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#crispr-cas9-based-treatments"><i class="fa fa-check"></i><b>12.5.6</b> CRISPR-Cas9-Based Treatments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="13" data-path="infection-and-disease.html"><a href="infection-and-disease.html"><i class="fa fa-check"></i><b>13</b> Infection And Disease</a>
<ul>
<li class="chapter" data-level="13.1" data-path="infection-and-disease.html"><a href="infection-and-disease.html#pathogenic-bacteria"><i class="fa fa-check"></i><b>13.1</b> Pathogenic Bacteria</a></li>
<li class="chapter" data-level="13.2" data-path="infection-and-disease.html"><a href="infection-and-disease.html#pathogenic-susceptibility"><i class="fa fa-check"></i><b>13.2</b> Pathogenic Susceptibility</a>
<ul>
<li class="chapter" data-level="13.2.1" data-path="infection-and-disease.html"><a href="infection-and-disease.html#list-of-genera-and-microscopy-features"><i class="fa fa-check"></i><b>13.2.1</b> List of genera and microscopy features</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="infection-and-disease.html"><a href="infection-and-disease.html#infection"><i class="fa fa-check"></i><b>13.3</b> Infection</a></li>
<li class="chapter" data-level="13.4" data-path="infection-and-disease.html"><a href="infection-and-disease.html#disease"><i class="fa fa-check"></i><b>13.4</b> Disease</a></li>
<li class="chapter" data-level="13.5" data-path="infection-and-disease.html"><a href="infection-and-disease.html#epidemiology"><i class="fa fa-check"></i><b>13.5</b> Epidemiology</a></li>
<li class="chapter" data-level="13.6" data-path="infection-and-disease.html"><a href="infection-and-disease.html#pandemics"><i class="fa fa-check"></i><b>13.6</b> Pandemics</a>
<ul>
<li class="chapter" data-level="13.6.1" data-path="infection-and-disease.html"><a href="infection-and-disease.html#covid-19-pandemic"><i class="fa fa-check"></i><b>13.6.1</b> COVID-19 Pandemic</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html"><i class="fa fa-check"></i><b>14</b> An Introduction To Host Defenses And Innate Immune Systems</a>
<ul>
<li class="chapter" data-level="14.1" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#immunology"><i class="fa fa-check"></i><b>14.1</b> Immunology</a></li>
<li class="chapter" data-level="14.2" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#the-immune-system"><i class="fa fa-check"></i><b>14.2</b> The Immune System</a></li>
<li class="chapter" data-level="14.3" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#layered-defense"><i class="fa fa-check"></i><b>14.3</b> Layered Defense</a></li>
<li class="chapter" data-level="14.4" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#surface-barriers"><i class="fa fa-check"></i><b>14.4</b> Surface Barriers</a></li>
<li class="chapter" data-level="14.5" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#innate-immune-system"><i class="fa fa-check"></i><b>14.5</b> Innate Immune System</a>
<ul>
<li class="chapter" data-level="14.5.1" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#anatomical-barriers"><i class="fa fa-check"></i><b>14.5.1</b> Anatomical Barriers</a></li>
<li class="chapter" data-level="14.5.2" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#inflammation"><i class="fa fa-check"></i><b>14.5.2</b> Inflammation</a></li>
<li class="chapter" data-level="14.5.3" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#process-of-acute-inflammation"><i class="fa fa-check"></i><b>14.5.3</b> Process of Acute Inflammation</a></li>
<li class="chapter" data-level="14.5.4" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#vasodilation-and-increased-permeability"><i class="fa fa-check"></i><b>14.5.4</b> Vasodilation And Increased Permeability</a></li>
<li class="chapter" data-level="14.5.5" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#leukocyte-extravasation"><i class="fa fa-check"></i><b>14.5.5</b> Leukocyte Extravasation</a></li>
<li class="chapter" data-level="14.5.6" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#phagocytosis"><i class="fa fa-check"></i><b>14.5.6</b> Phagocytosis</a></li>
<li class="chapter" data-level="14.5.7" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#systemic-inflammation-and-obesityedit"><i class="fa fa-check"></i><b>14.5.7</b> Systemic Inflammation And Obesity[Edit]</a></li>
<li class="chapter" data-level="14.5.8" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#complement-system"><i class="fa fa-check"></i><b>14.5.8</b> Complement System</a></li>
<li class="chapter" data-level="14.5.9" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#blood"><i class="fa fa-check"></i><b>14.5.9</b> Blood</a></li>
<li class="chapter" data-level="14.5.10" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#haematopoiesis"><i class="fa fa-check"></i><b>14.5.10</b> Haematopoiesis</a></li>
<li class="chapter" data-level="14.5.11" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#white-blood-cells"><i class="fa fa-check"></i><b>14.5.11</b> White Blood Cells</a></li>
<li class="chapter" data-level="14.5.12" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#mast-cells"><i class="fa fa-check"></i><b>14.5.12</b> Mast cells</a></li>
<li class="chapter" data-level="14.5.13" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#phagocytes"><i class="fa fa-check"></i><b>14.5.13</b> Phagocytes</a></li>
<li class="chapter" data-level="14.5.14" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#macrophages"><i class="fa fa-check"></i><b>14.5.14</b> Macrophages</a></li>
<li class="chapter" data-level="14.5.15" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#neutrophils"><i class="fa fa-check"></i><b>14.5.15</b> Neutrophils</a></li>
<li class="chapter" data-level="14.5.16" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#dendritic-cells"><i class="fa fa-check"></i><b>14.5.16</b> Dendritic Cells</a></li>
<li class="chapter" data-level="14.5.17" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#basophils-and-eosinophils"><i class="fa fa-check"></i><b>14.5.17</b> Basophils And Eosinophils</a></li>
<li class="chapter" data-level="14.5.18" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#natural-killer-cells"><i class="fa fa-check"></i><b>14.5.18</b> Natural Killer Cells</a></li>
<li class="chapter" data-level="14.5.19" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#γδ-t-cells"><i class="fa fa-check"></i><b>14.5.19</b> γδ T Cells</a></li>
<li class="chapter" data-level="14.5.20" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#other-vertebrate-mechanisms"><i class="fa fa-check"></i><b>14.5.20</b> Other Vertebrate Mechanisms</a></li>
<li class="chapter" data-level="14.5.21" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#neural-regulation"><i class="fa fa-check"></i><b>14.5.21</b> Neural Regulation</a></li>
<li class="chapter" data-level="14.5.22" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#pathogen-specificity"><i class="fa fa-check"></i><b>14.5.22</b> Pathogen-specificity</a></li>
<li class="chapter" data-level="14.5.23" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#immune-evasion"><i class="fa fa-check"></i><b>14.5.23</b> Immune Evasion</a></li>
<li class="chapter" data-level="14.5.24" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#viruses-1"><i class="fa fa-check"></i><b>14.5.24</b> Viruses</a></li>
</ul></li>
<li class="chapter" data-level="14.6" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#mechanisms-of-immune-defense-in-prokaryotes"><i class="fa fa-check"></i><b>14.6</b> Mechanisms Of Immune Defense In Prokaryotes</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html"><i class="fa fa-check"></i><b>15</b> The Adaptive Immune System And Immunization</a>
<ul>
<li class="chapter" data-level="15.1" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#humoral-immunity"><i class="fa fa-check"></i><b>15.1</b> Humoral immunity</a></li>
<li class="chapter" data-level="15.2" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#antibodies"><i class="fa fa-check"></i><b>15.2</b> Antibodies</a>
<ul>
<li class="chapter" data-level="15.2.1" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#antibody-production"><i class="fa fa-check"></i><b>15.2.1</b> Antibody production</a></li>
<li class="chapter" data-level="15.2.2" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#b-cell-activation"><i class="fa fa-check"></i><b>15.2.2</b> B cell activation</a></li>
<li class="chapter" data-level="15.2.3" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#b-cell-proliferation"><i class="fa fa-check"></i><b>15.2.3</b> B cell proliferation</a></li>
<li class="chapter" data-level="15.2.4" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#antibody-antigen-reaction"><i class="fa fa-check"></i><b>15.2.4</b> Antibody-antigen reaction</a></li>
</ul></li>
<li class="chapter" data-level="15.3" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#cell-mediated-immunity"><i class="fa fa-check"></i><b>15.3</b> Cell-mediated immunity</a></li>
<li class="chapter" data-level="15.4" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#lymphocytes"><i class="fa fa-check"></i><b>15.4</b> Lymphocytes</a></li>
<li class="chapter" data-level="15.5" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#antigen-presentation"><i class="fa fa-check"></i><b>15.5</b> Antigen presentation</a></li>
<li class="chapter" data-level="15.6" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#exogenous-antigens"><i class="fa fa-check"></i><b>15.6</b> Exogenous antigens</a></li>
<li class="chapter" data-level="15.7" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#endogenous-antigens"><i class="fa fa-check"></i><b>15.7</b> Endogenous antigens</a></li>
<li class="chapter" data-level="15.8" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#t-lymphocytes"><i class="fa fa-check"></i><b>15.8</b> T lymphocytes</a>
<ul>
<li class="chapter" data-level="15.8.1" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#cd8-t-lymphocytes-and-cytotoxicity"><i class="fa fa-check"></i><b>15.8.1</b> CD8+ T lymphocytes and cytotoxicity</a></li>
<li class="chapter" data-level="15.8.2" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#helper-t-cells"><i class="fa fa-check"></i><b>15.8.2</b> Helper T-cells</a></li>
<li class="chapter" data-level="15.8.3" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#th1-and-th2-helper-t-cell-responses"><i class="fa fa-check"></i><b>15.8.3</b> Th1 And Th2: Helper T Cell Responses</a></li>
<li class="chapter" data-level="15.8.4" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#gamma-delta-t-cells"><i class="fa fa-check"></i><b>15.8.4</b> Gamma Delta T Cells</a></li>
</ul></li>
<li class="chapter" data-level="15.9" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#b-lymphocytes-and-antibody-production"><i class="fa fa-check"></i><b>15.9</b> B Lymphocytes And Antibody Production</a></li>
<li class="chapter" data-level="15.10" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#immunological-memory"><i class="fa fa-check"></i><b>15.10</b> Immunological Memory</a>
<ul>
<li class="chapter" data-level="15.10.1" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#passive-memory"><i class="fa fa-check"></i><b>15.10.1</b> Passive Memory</a></li>
<li class="chapter" data-level="15.10.2" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#active-memory"><i class="fa fa-check"></i><b>15.10.2</b> Active Memory</a></li>
<li class="chapter" data-level="15.10.3" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#immunization"><i class="fa fa-check"></i><b>15.10.3</b> Immunization</a></li>
<li class="chapter" data-level="15.10.4" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#immunological-diversity"><i class="fa fa-check"></i><b>15.10.4</b> Immunological Diversity</a></li>
</ul></li>
<li class="chapter" data-level="15.11" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#immunization-1"><i class="fa fa-check"></i><b>15.11</b> Immunization</a></li>
<li class="chapter" data-level="15.12" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#vaccination"><i class="fa fa-check"></i><b>15.12</b> Vaccination</a>
<ul>
<li class="chapter" data-level="15.12.1" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#mechanism-of-action"><i class="fa fa-check"></i><b>15.12.1</b> Mechanism of Action</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For SCI204 Microbiology at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="drugs-microbes-and-the-host" class="section level1" number="12">
<h1><span class="header-section-number">12</span> Drugs, Microbes, And The Host</h1>
<div id="antimicrobials" class="section level2" number="12.1">
<h2><span class="header-section-number">12.1</span> Antimicrobials</h2>
<p>An antimicrobial is an agent that kills microorganisms or stops their growth. Antimicrobial medicines can be grouped according to the microorganisms they act primarily against. For example, antibiotics are used against bacteria, and antifungals are used against fungi. They can also be classified according to their function. Agents that kill microbes are microbicides, while those that merely inhibit their growth are called bacteriostatic agents. The use of antimicrobial medicines to treat infection is known as antimicrobial chemotherapy, while the use of antimicrobial medicines to prevent infection is known as antimicrobial prophylaxis.</p>
<p>The main classes of antimicrobial agents are disinfectants (non-selective agents, such as bleach), which kill a wide range of microbes on non-living surfaces to prevent the spread of illness, antiseptics (which are applied to living tissue and help reduce infection during surgery), and antibiotics (which destroy microorganisms within the body). The term ‘antibiosis’, meaning “against life”, was introduced by the French bacteriologist Jean Paul Vuillemin as a descriptive name of the phenomenon exhibited by these early antibacterial drugs. Antibiosis was first described in 1877 in bacteria when Louis Pasteur and Robert Koch observed that an airborne bacillus could inhibit the growth of Bacillus anthracis. These drugs were later renamed antibiotics by Selman Waksman, an American microbiologist, in 1947. The term “antibiotic” derives from anti + βιωτικός (biōtikos), “fit for life, lively”, which comes from βίωσις (biōsis), “way of life”, and that from βίος (bios), “life”. The term “antibacterial” derives from Greek ἀντί (anti), “against” + βακτήριον (baktērion), diminutive of βακτηρία (baktēria), “staff, cane”, because the first bacteria to be discovered were rod. The term antibiotic was first used in 1942 by Selman Waksman and his collaborators in journal articles to describe any substance produced by a microorganism that is antagonistic to the growth of other microorganisms in high dilution. The term “antibiotic” originally described only those formulations derived from living microorganisms but is now also applied to synthetic agents, such as sulfonamides or fluoroquinolones. Though the term used to be restricted to antibacterials (and is often used as a synonym for them by medical professionals and in medical literature), its context has broadened to include all antimicrobials. Antibacterial agents can be further subdivided into bactericidal agents, which kill bacteria, and bacteriostatic agents, which slow down or stall bacterial growth. In response, further advancements in antimicrobial technologies have resulted in solutions that can go beyond simply inhibiting microbial growth. Instead, certain types of porous media have been developed to kill microbes on contact.</p>
<p>Antimicrobial use has been common practice for at least 2000 years. Ancient Egyptians and ancient Greeks used specific molds and plant extracts to treat infection.</p>
<p>In the 19th century, microbiologists such as Louis Pasteur and Jules Francois Joubert observed antagonism between some bacteria and discussed the merits of controlling these interactions in medicine. Louis Pasteur’s work in fermentation and spontaneous generation led to the distinction between anaerobic and aerobic bacteria. The information garnered by Pasteur led Joseph Lister to incorporate antiseptic methods, such as sterilizing surgical tools and debriding wounds into surgical procedures. The implementation of these antiseptic techniques drastically reduced the number of infections and subsequent deaths associated with surgical procedures. Louis Pasteur’s work in microbiology also led to the development of many vaccines for life-threatening diseases such as anthrax and rabies. On September 3, 1928, Alexander Fleming returned from a vacation and discovered that a Petri dish filled with Staphylococcus was separated into colonies due to the antimicrobial fungus Penicillium rubens. Fleming and his associates struggled to isolate the antimicrobial but referenced its therapeutic potential in 1929 in the British Journal of Experimental Pathology. In 1942, Howard Florey, Ernst Chain, and Edward Abraham utilized Fleming’s work to purify and extract penicillin for medicinal uses earning them the 1945 Nobel Prize in Medicine.</p>
<div id="antibacterials" class="section level3" number="12.1.1">
<h3><span class="header-section-number">12.1.1</span> Antibacterials</h3>
<p>Antibacterials are used to treat bacterial infections. Antibiotics are classified generally as beta-lactams, macrolides, quinolones, tetracyclines or aminoglycosides. Their classification within these categories depends on their antimicrobial spectra, pharmacodynamics, and chemical composition. Prolonged use of certain antibacterials can decrease the number of enteric bacteria, which may have a negative impact on health. Consumption of probiotics and reasonable eating may help to replace destroyed gut flora. Stool transplants may be considered for patients who are having difficulty recovering from prolonged antibiotic treatment, as for recurrent Clostridioides difficile infections.</p>
<p>The discovery, development and use of antibacterials during the 20th century have reduced mortality from bacterial infections. The antibiotic era began with the therapeutic application of sulfonamide drugs in 1936, followed by a “golden” period of discovery from about 1945 to 1970, when a number of structurally diverse and highly effective agents were discovered and developed. Since 1980, the introduction of new antimicrobial agents for clinical use has declined, in part because of the enormous expense of developing and testing new drugs. In parallel, there has been an alarming increase in antimicrobial resistance of bacteria, fungi, parasites and some viruses to multiple existing agents.</p>
<p>Antibacterials are among the most commonly used drugs and among the drugs commonly misused by physicians, for example, in viral respiratory tract infections. As a consequence of widespread and injudicious use of antibacterials, there has been an accelerated emergence of antibiotic-resistant pathogens, resulting in a serious threat to global public health. The resistance problem demands that a renewed effort be made to seek antibacterial agents effective against pathogenic bacteria resistant to current antibacterials. Possible strategies towards this objective include increased sampling from diverse environments and application of metagenomics to identify bioactive compounds produced by currently unknown and uncultured microorganisms as well as the development of small-molecule libraries customized for bacterial targets.</p>
</div>
<div id="antifungals" class="section level3" number="12.1.2">
<h3><span class="header-section-number">12.1.2</span> Antifungals</h3>
<p>Antifungals are used to kill or prevent further growth of fungi. In medicine, they are used as a treatment for infections such as athlete’s foot, ringworm and thrush and work by exploiting differences between mammalian and fungal cells. Unlike bacteria, both fungi and humans are eukaryotes. Thus, fungal and human cells are similar at the molecular level, making it more difficult to find a target for an antifungal drug to attack that does not also exist in the host organism. Consequently, there are often side effects to some of these drugs. Some of these side effects can be life-threatening if the drug is not used properly.</p>
<p>As well as their use in medicine, antifungals are frequently sought after to control indoor mold in damp or wet home materials. Sodium bicarbonate (baking soda) blasted on to surfaces acts as an antifungal. Another antifungal solution applied after or without blasting by soda is a mix of hydrogen peroxide and a thin surface coating that neutralizes mold and encapsulates the surface to prevent spore release. Some paints are also manufactured with an added antifungal agent for use in high humidity areas such as bathrooms or kitchens. Other antifungal surface treatments typically contain variants of metals known to suppress mold growth e.g. pigments or solutions containing copper, silver or zinc. These solutions are not usually available to the general public because of their toxicity.</p>
</div>
<div id="antivirals" class="section level3" number="12.1.3">
<h3><span class="header-section-number">12.1.3</span> Antivirals</h3>
<p>Antiviral drugs are a class of medication used specifically for treating viral infections. Like antibiotics, specific antivirals are used for specific viruses. They should be distinguished from viricides, which actively deactivate virus particles outside the body.</p>
<p>Many antiviral drugs are designed to treat infections by retroviruses, including HIV. Important antiretroviral drugs include the class of protease inhibitors. Herpes viruses, best known for causing cold sores and genital herpes, are usually treated with the nucleoside analogue acyclovir. Viral hepatitis is caused by five unrelated hepatotropic viruses (A-E) and may be treated with antiviral drugs depending on the type of infection. Some influenza A and B viruses have become resistant to neuraminidase inhibitors such as oseltamivir, and the search for new substances continues.</p>
</div>
<div id="antiparasitics" class="section level3" number="12.1.4">
<h3><span class="header-section-number">12.1.4</span> Antiparasitics</h3>
<p>Antiparasitics are a class of medications indicated for the treatment of infectious diseases such as leishmaniasis, malaria and Chagas disease, which are caused by parasites such as nematodes, cestodes, trematodes and infectious protozoa. Antiparasitic medications include metronidazole, iodoquinol and albendazole. Like all therapeutic antimicrobials, they must kill the infecting organism without serious damage to the host.</p>
</div>
<div id="broad-spectrum-therapeutics" class="section level3" number="12.1.5">
<h3><span class="header-section-number">12.1.5</span> Broad-Spectrum Therapeutics</h3>
<p>Broad-spectrum therapeutics are active against multiple classes of pathogens. Such therapeutics have been suggested as potential emergency treatments for pandemics. Azithromycin is currently the only identified broad-spectrum therapeutic.</p>
</div>
</div>
<div id="antibiotics" class="section level2" number="12.2">
<h2><span class="header-section-number">12.2</span> Antibiotics</h2>
<p>An antibiotic is a type of antimicrobial substance active against bacteria. It is the most important type of antibacterial agent for fighting bacterial infections, and antibiotic medications are widely used in the treatment and prevention of such infections. They may either kill or inhibit the growth of bacteria. A limited number of antibiotics also possess antiprotozoal activity. Antibiotics are not effective against viruses such as the common cold or influenza; drugs which inhibit viruses are termed antiviral drugs or antivirals rather than antibiotics.</p>
<p>Sometimes, the term antibiotic—literally “opposing life”, from the Greek roots ἀντι anti, “against” and βίος bios, “life”—is broadly used to refer to any substance used against microbes, but in the usual medical usage, antibiotics (such as penicillin) are those produced naturally (by one microorganism fighting another), whereas nonantibiotic antibacterials (such as sulfonamides and antiseptics) are fully synthetic. However, both classes have the same goal of killing or preventing the growth of microorganisms, and both are included in antimicrobial chemotherapy. “Antibacterials” include antiseptic drugs, antibacterial soaps, and chemical disinfectants, whereas antibiotics are an important class of antibacterials used more specifically in medicine and sometimes in livestock feed.</p>
<p>Antibiotics have been used since ancient times. Many civilizations used topical application of mouldy bread, with many references to its beneficial effects arising from ancient Egypt, Nubia, China, Serbia, Greece, and Rome. The first person to directly document the use of molds to treat infections was John Parkinson (1567–1650). Antibiotics revolutionized medicine in the 20th century. Alexander Fleming (1881–1955) discovered modern day penicillin in 1928, the widespread use of which proved significantly beneficial during wartime. However, the effectiveness and easy access to antibiotics have also led to their overuse and some bacteria have evolved resistance to them. The World Health Organization has classified antimicrobial resistance as a widespread “serious threat [that] is no longer a prediction for the future, it is happening right now in every region of the world and has the potential to affect anyone, of any age, in any country”.</p>
<p>Antibiotics are used to treat or prevent bacterial infections, and sometimes protozoan infections. (Metronidazole is effective against a number of parasitic diseases). When an infection is suspected of being responsible for an illness but the responsible pathogen has not been identified, an empiric therapy is adopted. This involves the administration of a broad-spectrum antibiotic based on the signs and symptoms presented and is initiated pending laboratory results that can take several days.</p>
<p>When the responsible pathogenic microorganism is already known or has been identified, definitive therapy can be started. This will usually involve the use of a narrow-spectrum antibiotic. The choice of antibiotic given will also be based on its cost. Identification is critically important as it can reduce the cost and toxicity of the antibiotic therapy and also reduce the possibility of the emergence of antimicrobial resistance. To avoid surgery, antibiotics may be given for non-complicated acute appendicitis.</p>

<div class="figure" style="text-align: center"><span id="fig:antibiotics"></span>
<img src="figures/drugs/Antibiotics_coverage_diagram.jpg" alt="Antibiotics coverage diagram." width="70%" />
<p class="caption">
Figure 12.1: <a href="https://commons.wikimedia.org/wiki/File:Antibiotics_coverage_diagram.jpg">Antibiotics coverage diagram.</a>
</p>
</div>
<p>Antibiotics may be given as a preventive measure and this is usually limited to at-risk populations such as those with a weakened immune system (particularly in HIV cases to prevent pneumonia), those taking immunosuppressive drugs, cancer patients, and those having surgery. Their use in surgical procedures is to help prevent infection of incisions. They have an important role in dental antibiotic prophylaxis where their use may prevent bacteremia and consequent infective endocarditis. Antibiotics are also used to prevent infection in cases of neutropenia particularly cancer-related.</p>
<p>There are many different routes of administration for antibiotic treatment. Antibiotics are usually taken by mouth. In more severe cases, particularly deep-seated systemic infections, antibiotics can be given intravenously or by injection. Where the site of infection is easily accessed, antibiotics may be given topically in the form of eye drops onto the conjunctiva for conjunctivitis or ear drops for ear infections and acute cases of swimmer’s ear. Topical use is also one of the treatment options for some skin conditions including acne and cellulitis. Advantages of topical application include achieving high and sustained concentration of antibiotic at the site of infection; reducing the potential for systemic absorption and toxicity, and total volumes of antibiotic required are reduced, thereby also reducing the risk of antibiotic misuse. Topical antibiotics applied over certain types of surgical wounds have been reported to reduce the risk of surgical site infections. However, there are certain general causes for concern with topical administration of antibiotics. Some systemic absorption of the antibiotic may occur; the quantity of antibiotic applied is difficult to accurately dose, and there is also the possibility of local hypersensitivity reactions or contact dermatitis occurring. It is recommended to administer antibiotics as soon as possible, especially in life-threatening infections. Many emergency departments stock antibiotics for this purpose.</p>
<p>Prevalence
Antibiotic consumption varies widely between countries. The WHO report on surveillance of antibiotic consumption’ published in 2018 analysed 2015 data from 65 countries. As measured in defined daily doses per 1,000 inhabitants per day. Mongolia had the highest consumption with a rate of 64.4. Burundi had the lowest at 4.4. Amoxicillin and amoxicillin/clavulanic acid were the most frequently consumed.</p>
<div id="classes-of-antibiotics" class="section level3" number="12.2.1">
<h3><span class="header-section-number">12.2.1</span> Classes of Antibiotics</h3>
<p>Antibiotics are commonly classified based on their mechanism of action, chemical structure, or spectrum of activity. Most target bacterial functions or growth processes. Those that target the bacterial cell wall (penicillins and cephalosporins) or the cell membrane (polymyxins), or interfere with essential bacterial enzymes (rifamycins, lipiarmycins, quinolones, and sulfonamides) have bactericidal activities. Protein synthesis inhibitors (macrolides, lincosamides, and tetracyclines) are usually bacteriostatic (with the exception of bactericidal aminoglycosides). Further categorization is based on their target specificity. “Narrow-spectrum” antibiotics target specific types of bacteria, such as gram-negative or gram-positive, whereas broad-spectrum antibiotics affect a wide range of bacteria. Following a 40-year break in discovering classes of antibacterial compounds, four new classes of antibiotics were introduced to clinical use in the late 2000s and early 2010s: cyclic lipopeptides (such as daptomycin), glycylcyclines (such as tigecycline), oxazolidinones (such as linezolid), and lipiarmycins (such as fidaxomicin).</p>
<p>With advances in medicinal chemistry, most modern antibacterials are semisynthetic modifications of various natural compounds. These include, for example, the beta-lactam antibiotics, which include the penicillins (produced by fungi in the genus Penicillium), the cephalosporins, and the carbapenems. Compounds that are still isolated from living organisms are the aminoglycosides, whereas other antibacterials—for example, the sulfonamides, the quinolones, and the oxazolidinones—are produced solely by chemical synthesis. Many antibacterial compounds are relatively small molecules with a molecular weight of less than 1000 daltons.</p>
<p>Since the first pioneering efforts of Howard Florey and Chain in 1939, the importance of antibiotics, including antibacterials, to medicine has led to intense research into producing antibacterials at large scales. Following screening of antibacterials against a wide range of bacteria, production of the active compounds is carried out using fermentation, usually in strongly aerobic conditions.</p>
<p>The emergence of resistance of bacteria to antibiotics is a common phenomenon. Emergence of resistance often reflects evolutionary processes that take place during antibiotic therapy. The antibiotic treatment may select for bacterial strains with physiologically or genetically enhanced capacity to survive high doses of antibiotics. Under certain conditions, it may result in preferential growth of resistant bacteria, while growth of susceptible bacteria is inhibited by the drug. For example, antibacterial selection for strains having previously acquired antibacterial-resistance genes was demonstrated in 1943 by the Luria–Delbrück experiment. Antibiotics such as penicillin and erythromycin, which used to have a high efficacy against many bacterial species and strains, have become less effective, due to the increased resistance of many bacterial strains.</p>
<p>Resistance may take the form of biodegradation of pharmaceuticals, such as sulfamethazine-degrading soil bacteria introduced to sulfamethazine through medicated pig feces. The survival of bacteria often results from an inheritable resistance, but the growth of resistance to antibacterials also occurs through horizontal gene transfer. Horizontal transfer is more likely to happen in locations of frequent antibiotic use.</p>
<p>Antibacterial resistance may impose a biological cost, thereby reducing fitness of resistant strains, which can limit the spread of antibacterial-resistant bacteria, for example, in the absence of antibacterial compounds. Additional mutations, however, may compensate for this fitness cost and can aid the survival of these bacteria.</p>
<p>Paleontological data show that both antibiotics and antibiotic resistance are ancient compounds and mechanisms. Useful antibiotic targets are those for which mutations negatively impact bacterial reproduction or viability.</p>
<p>Several molecular mechanisms of antibacterial resistance exist. Intrinsic antibacterial resistance may be part of the genetic makeup of bacterial strains. For example, an antibiotic target may be absent from the bacterial genome. Acquired resistance results from a mutation in the bacterial chromosome or the acquisition of extra-chromosomal DNA. Antibacterial-producing bacteria have evolved resistance mechanisms that have been shown to be similar to, and may have been transferred to, antibacterial-resistant strains. The spread of antibacterial resistance often occurs through vertical transmission of mutations during growth and by genetic recombination of DNA by horizontal genetic exchange. For instance, antibacterial resistance genes can be exchanged between different bacterial strains or species via plasmids that carry these resistance genes. Plasmids that carry several different resistance genes can confer resistance to multiple antibacterials. Cross-resistance to several antibacterials may also occur when a resistance mechanism encoded by a single gene conveys resistance to more than one antibacterial compound.</p>
<p>Antibacterial-resistant strains and species, sometimes referred to as “superbugs”, now contribute to the emergence of diseases that were for a while well controlled. For example, emergent bacterial strains causing tuberculosis that are resistant to previously effective antibacterial treatments pose many therapeutic challenges. Every year, nearly half a million new cases of multidrug-resistant tuberculosis (MDR-TB) are estimated to occur worldwide. For example, NDM-1 is a newly identified enzyme conveying bacterial resistance to a broad range of beta-lactam antibacterials. The United Kingdom’s Health Protection Agency has stated that “most isolates with NDM-1 enzyme are resistant to all standard intravenous antibiotics for treatment of severe infections.” On 26 May 2016, an E. coli “superbug” was identified in the United States resistant to colistin, “the last line of defence” antibiotic.</p>
<p>Before the early 20th century, treatments for infections were based primarily on medicinal folklore. Mixtures with antimicrobial properties that were used in treatments of infections were described over 2,000 years ago. Many ancient cultures, including the ancient Egyptians and ancient Greeks, used specially selected mold and plant materials to treat infections. Nubian mummies studied in the 1990s were found to contain significant levels of tetracycline. The beer brewed at that time was conjectured to have been the source.</p>
<p>The use of antibiotics in modern medicine began with the discovery of synthetic antibiotics derived from dyes.</p>
<p>Synthetic antibiotic chemotherapy as a science and development of antibacterials began in Germany with Paul Ehrlich in the late 1880s. Ehrlich noted certain dyes would color human, animal, or bacterial cells, whereas others did not. He then proposed the idea that it might be possible to create chemicals that would act as a selective drug that would bind to and kill bacteria without harming the human host. After screening hundreds of dyes against various organisms, in 1907, he discovered a medicinally useful drug, the first synthetic antibacterial organoarsenic compound salvarsan, now called arsphenamine.</p>
<p>This heralded the era of antibacterial treatment that was begun with the discovery of a series of arsenic-derived synthetic antibiotics by both Alfred Bertheim and Ehrlich in 1907. Ehrlich and Bertheim had experimented with various chemicals derived from dyes to treat trypanosomiasis in mice and spirochaeta infection in rabbits. While their early compounds were too toxic, Ehrlich and Sahachiro Hata, a Japanese bacteriologist working with Erlich in the quest for a drug to treat syphilis, achieved success with the 606th compound in their series of experiments. In 1910 Ehrlich and Hata announced their discovery, which they called drug “606”, at the Congress for Internal Medicine at Wiesbaden. The Hoechst company began to market the compound toward the end of 1910 under the name Salvarsan, now known as arsphenamine. The drug was used to treat syphilis in the first half of the 20th century. In 1908, Ehrlich received the Nobel Prize in Physiology or Medicine for his contributions to immunology. Hata was nominated for the Nobel Prize in Chemistry in 1911 and for the Nobel Prize in Physiology or Medicine in 1912 and 1913.</p>
<p>The first sulfonamide and the first systemically active antibacterial drug, Prontosil, was developed by a research team led by Gerhard Domagk in 1932 or 1933 at the Bayer Laboratories of the IG Farben conglomerate in Germany, for which Domagk received the 1939 Nobel Prize in Physiology or Medicine. Sulfanilamide, the active drug of Prontosil, was not patentable as it had already been in use in the dye industry for some years. Prontosil had a relatively broad effect against Gram-positive cocci, but not against enterobacteria. Research was stimulated apace by its success. The discovery and development of this sulfonamide drug opened the era of antibacterials.</p>
</div>
<div id="penicillin-and-other-natural-antibiotics" class="section level3" number="12.2.2">
<h3><span class="header-section-number">12.2.2</span> Penicillin And Other Natural Antibiotics</h3>
<p>Penicillins (P, PCN or PEN) are a group of antibiotics originally obtained from Penicillium moulds, principally P. chrysogenum and P. rubens. Most penicillins in clinical use are chemically synthesised from naturally-produced penicillins. A number of natural penicillins have been discovered, but only two purified compounds are in clinical use: penicillin G (intravenous use) and penicillin V (given by mouth). Penicillins were among the first medications to be effective against many bacterial infections caused by staphylococci and streptococci. They are members of the β-lactam antibiotics. They are still widely used today for different bacterial infections, though many types of bacteria have developed resistance following extensive use.</p>
<p>About 10% of people report that they are allergic to penicillin; however, up to 90% of this group may not actually be allergic. Serious allergies only occur in about 0.03%.[for whom?] Those who are allergic to penicillin are most often given cephalosporin C (another β-lactam antibiotic) because there is only 10% crossover in allergy between the penicillins and cephalosporins.</p>
<p>Starting in the late 19th century there had been reports of the antibacterial properties of Penicillium mould, but scientists were unable to discern what process was causing the effect. Scottish physician Alexander Fleming at St Mary’s Hospital in London (now part of Imperial College) was the first to show that Penicillium rubens had antibacterial properties. On 3 September 1928 he observed that fungal contamination of a bacterial culture (Staphylococcus aureus) appeared to kill the bacteria. He confirmed this observation with a new experiment on 28 September 1928. He published his experiment in 1929, and called the antibacterial substance (the fungal extract) penicillin.</p>
<p>C. J. La Touche identified the fungus as Penicillium rubrum (later reclassified by Charles Thom as P. notatum and P. chrysogenum, but later corrected as P. rubens). Fleming expressed initial optimism that penicillin would be a useful antiseptic, because of its high potency and minimal toxicity in comparison to other antiseptics of the day, and noted its laboratory value in the isolation of Bacillus influenzae (now called Haemophilus influenzae).</p>
<p>Fleming did not convince anyone that his discovery was important. This was largely because penicillin was so difficult to isolate that its development as a drug seemed impossible. It is speculated that had Fleming been more successful at making other scientists interested in his work, penicillin would possibly have been developed years earlier.</p>
<p>The importance of his work has been recognized by the placement of an International Historic Chemical Landmark at the Alexander Fleming Laboratory Museum in London on November 19, 1999.</p>
<p>In 1930, Cecil George Paine, a pathologist at the Royal Infirmary in Sheffield, successfully treated ophthalmia neonatorum, a gonococcal infection in infants, with penicillin (fungal extract) on November 25, 1930.</p>
<p>In 1940, Australian scientist Howard Florey (later Baron Florey) and a team of researchers (Ernst Chain, Edward Abraham, Arthur Duncan Gardner, Norman Heatley, Margaret Jennings, Jean Orr-Ewing and Arthur Gordon Sanders) at the Sir William Dunn School of Pathology, University of Oxford made progress in making concentrated penicillin from fungal culture broth that showed both in vitro and in vivo bactericidal action. In 1941, they treated a policeman, Albert Alexander, with a severe face infection; his condition improved, but then supplies of penicillin ran out and he died. Subsequently, several other patients were treated successfully. In December 1942, survivors of the Cocoanut Grove fire in Boston were the first burn patients to be successfully treated with penicillin.</p>
<p>The first successful use of pure penicillin was when Fleming treated Harry Lambert of fatal infection of the nervous system (streptococcal meningitis) in 1942. By that time the Oxford team could produce only small amount. Florey willingly gave the only available sample to Fleming. Lambert showed improvement from the very next day of the treatment, and was completely cured within a week. Fleming published his clinical trial in The Lancet in 1943. Following the medical breakthrough the British War Cabinet set up the Penicillin Committee on 5 April 1943 that led to projects for mass production.</p>
<p>As the medical application was established, the Oxford team found that it was impossible to produce usable amounts in their laboratory. Failing to persuade the British government, Florey and Heatley travelled to the US in June 1941 with their mould samples in order to interest the US government for large-scale production. They approached the USDA Northern Regional Research Laboratory (NRRL, now the National Center for Agricultural Utilization Research) at Peoria, Illinois, where facilities for large-scale fermentations were established. Mass culture of the mould and search for better moulds immediately followed.</p>
<p>On March 14, 1942, the first patient was treated for streptococcal sepsis with US-made penicillin produced by Merck &amp; Co. Half of the total supply produced at the time was used on that one patient, Anne Miller. By June 1942, just enough US penicillin was available to treat ten patients. In July 1943, the War Production Board drew up a plan for the mass distribution of penicillin stocks to Allied troops fighting in Europe. The results of fermentation research on corn steep liquor at the NRRL allowed the United States to produce 2.3 million doses in time for the invasion of Normandy in the spring of 1944. After a worldwide search in 1943, a mouldy cantaloupe in a Peoria, Illinois market was found to contain the best strain of mould for production using the corn steep liquor process. Pfizer scientist Jasper H. Kane suggested using a deep-tank fermentation method for producing large quantities of pharmaceutical-grade penicillin.:109 Large-scale production resulted from the development of a deep-tank fermentation plant by chemical engineer Margaret Hutchinson Rousseau. As a direct result of the war and the War Production Board, by June 1945, over 646 billion units per year were being produced.</p>
<p>After World War II, Australia was the first country to make the drug available for civilian use. In the U.S., penicillin was made available to the general public on March 15, 1945.</p>
<p>Fleming, Florey, and Chain shared the 1945 Nobel Prize in Physiology or Medicine for the development of penicillin.</p>
<p>Several semisynthetic penicillins are effective against a broader spectrum of bacteria: these include the antistaphylococcal penicillins, aminopenicillins and the antipseudomonal penicillins.</p>
<p>The term “penicillin” is defined as the natural product of Penicillium mould with antimicrobial activity. It was coined by Alexander Fleming on 7 March 1929 when he discovered the antibacterial property of Penicillium rubens. The name was “to avoid the repetition of the rather cumbersome phrase ‘Mould broth filtrate,’ the name ‘penicillin’ will be used,” as Fleming explained in his 1929 paper in the British Journal of Experimental Pathology. The name thus refers to the scientific name of the mould, as described by Fleming in his Nobel lecture in 1945:</p>
<blockquote>
<p>I have been frequently asked why I invented the name “Penicillin”. I simply followed perfectly orthodox lines and coined a word which explained that the substance penicillin was derived from a plant of the genus Penicillium just as many years ago the word “Digitalin” was invented for a substance derived from the plant Digitalis.</p>
</blockquote>
<p>In modern usage, the term penicillin is used more broadly to refer to any β-lactam antimicrobial that contains a thiazolidine ring fused to the β-lactam core, and may or may not be a natural product. Like most natural products, penicillin is present in Penicillium moulds as a mixture of active constituents (gentamicin is another example of a natural product that is an ill-defined mixture of active components).</p>
<p>The precise constitution of the penicillin extracted depends on the species of Penicillium mould used and on the nutrient media used to culture the mould. Fleming’s original strain of Penicillium rubens produces principally penicillin F, named after Fleming. But penicillin F is unstable, difficult to isolate, and produced by the mould in small quantities.</p>
<p>The principal commercial strain of Penicillium chrysogenum (the Peoria strain) produces penicillin G as the principal component when corn steep liquor is used as the culture medium. When phenoxyethanol or phenoxyacetic acid are added to the culture medium, the mould produces penicillin V as the main penicillin instead.</p>
<p>6-Aminopenicillanic acid (6-APA) is a compound derived from penicillin G. 6-APA contains the beta-lactam core of penicillin G, but with the side chains stripped off; 6-APA is a useful precursor for manufacturing other penicillins. There are many semi-synthetic penicillins derived from 6-APA and these are in three groups: antistaphylococcal penicillins, broad-spectrum penicillins, and antipseudomonal penicillins. The semi-synthetic penicillins are all referred to as penicillins because they are all derived ultimately from penicillin G.</p>
<p>Penicillin G (benzylpenicillin) was first produced from a penicillium fungus that occurs in nature. The strain of fungus used today for the manufacture of penicillin G was created by genetic engineering to improve the yield in the manufacturing process. None of the other natural penicillins (F, K, N, X, O, U1 or U6) are currently in clinical use.</p>
<p>Penicillin V (phenoxymethylpenicillin) is produced by adding the precursor phenoxyacetic acid to the medium in which a genetically modified strain of the penicillium fungus is being cultured.</p>
<p>There are three major groups of other semi-synthetic antibiotics related to the penicillins. They are synthesised by adding various side-chains to the precursor 6-APA, which is isolated from penicillin G. These are the antistaphylococcal antibiotics, broad-spectrum antibiotics, and antipseudomonal antibiotics.</p>
</div>
<div id="antistaphylococcal-antibiotics" class="section level3" number="12.2.3">
<h3><span class="header-section-number">12.2.3</span> Antistaphylococcal antibiotics</h3>
<ul>
<li>Cloxacillin (by mouth or by injection)</li>
<li>Dicloxacillin (by mouth or by injection)</li>
<li>Flucloxacillin (by mouth or by injection)</li>
<li>Methicillin (injection only)</li>
<li>Nafcillin (injection only)</li>
<li>Oxacillin (by mouth or by injection)</li>
</ul>
<p>Antistaphylococcal antibiotics are so-called because they are resistant to being broken down by staphylococcal penicillinase. They are also, therefore, referred to as being penicillinase-resistant.</p>
</div>
<div id="broad-spectrum-antibiotics" class="section level3" number="12.2.4">
<h3><span class="header-section-number">12.2.4</span> Broad-spectrum antibiotics</h3>
<p>This group of antibiotics is called “broad-spectrum” because they are active against a wide range of Gram-negative bacteria such as Escherichia coli and Salmonella typhi, for which penicillin is not suitable. However, resistance in these organisms is now common.</p>
<ul>
<li>Ampicillin</li>
<li>Amoxycillin</li>
</ul>
<p>There are many ampicillin precursors in existence. These are inactive compounds that are broken down in the gut to release ampicillin. None of these pro-drugs of ampicillin are in current use:</p>
<ul>
<li>Pivampicillin (pivaloyloxymethyl ester of ampicillin)</li>
<li>Bacampicillin</li>
<li>Metampicillin (formaldehyde ester of ampicillin)</li>
<li>Talampicillin</li>
<li>Hetacillin (ampicillin conjugated to acetone)</li>
<li>Epicillin is an aminopenicillin that has never seen widespread clinical use.</li>
</ul>
</div>
<div id="antipseudomonal-antibiotics" class="section level3" number="12.2.5">
<h3><span class="header-section-number">12.2.5</span> Antipseudomonal antibiotics</h3>
<p>The Gram-negative species, Pseudomonas aeruginosa, is naturally resistant to many antibiotic classes. There were many efforts in the 1960s and 1970s to develop antibiotics that are active against Pseudomonas species. There are two chemical classes within the group: carboxypenicillins and ureidopenicillins. All are given by injection: none can be given by mouth.</p>
<p><em>Carboxypenicillins</em>
* Carbenicillin
* Ticarcillin
* Temocillin
<em>
Ureidopenicillins</em>
* Mezlocillin
* Piperacillin
* Azlocillin</p>
<p><em>β-lactamase inhibitors</em>
* Clavulanic acid
* Sulbactam
* Tazobactam</p>
<p>The term “penicillin”, when used by itself, may refer to either of two chemical compounds, penicillin G or penicillin V.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:composition">Table 12.1: </span>The three essential polymeric macromolecules of life
</caption>
<thead>
<tr>
<th style="text-align:left;">
Common name
</th>
<th style="text-align:left;">
Chemical name
</th>
<th style="text-align:left;">
Method ofadministration
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Penicillin V
</td>
<td style="text-align:left;">
phenoxymethylpenicillin
</td>
<td style="text-align:left;">
oral
</td>
</tr>
<tr>
<td style="text-align:left;">
Penicillin G
</td>
<td style="text-align:left;">
benzylpenicillin
</td>
<td style="text-align:left;">
intravenous intramuscular
</td>
</tr>
</tbody>
</table>
<p>Penicillin G is destroyed by stomach acid, so it cannot be taken by mouth, but doses as high as 2.4 g can be given (much higher than penicillin V). It is given by intravenous or intramuscular injection. It can be formulated as an insoluble salt, and there are two such formulations in current use: procaine penicillin and benzathine benzylpenicillin, which are used only in the treatment of syphilis. When a high concentration in the blood must be maintained, penicillin G must be administered at relatively frequent intervals, because it is eliminated quite rapidly from the bloodstream by the kidney.</p>
<p>Penicillin G is licensed for use to treat septicaemia, empyema, pneumonia, pericarditis, endocarditis and meningitis caused by susceptible strains of staphylococci and streptococci. It is also licensed for the treatment of anthrax, actinomycosis, cervicofacial disease, thoracic and abdominal disease, clostridial infections, botulism, gas gangrene (with accompanying debridement and/or surgery as indicated), tetanus (as an adjunctive therapy to human tetanus immune globulin), diphtheria (as an adjunctive therapy to antitoxin and for the prevention of the carrier state), erysipelothrix endocarditis, fusospirochetosis (severe infections of the oropharynx, lower respiratory tract and genital area), Listeria infections, meningitis, endocarditis, Pasteurella infections including bacteraemia and meningitis, Haverhill fever; rat-bite fever and disseminated gonococcal infections, meningococcal meningitis and/or septicaemia caused by penicillin-susceptible organisms and syphilis.</p>
<p>Penicillin V can be taken by mouth because it is relatively resistant to stomach acid. Doses higher than 500 mg are not fully effective because of poor absorption. It is used for the same bacterial infections as those of penicillin G and is the most widely used form of penicillin. However, it is not used for diseases, such as endocarditis, where high blood levels of penicillin are required.</p>
<p>Because penicillin resistance is now so common, other antibiotics are now the preferred choice for treatments. For example, penicillin used to be the first-line treatment for infections with Neisseria gonorrhoeae and Neisseria meningitidis, but it is longer recommended for treatment of these infections.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:suszeptibility">Table 12.2: </span>The three essential polymeric macromolecules of life
</caption>
<thead>
<tr>
<th style="text-align:left;">
Bacterium
</th>
<th style="text-align:left;">
Susceptible (S)
</th>
<th style="text-align:left;">
Intermediate (I)
</th>
<th style="text-align:left;">
Resistant (R)
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Staphylococcus aureus
</td>
<td style="text-align:left;">
≤0.12 mcg/ml
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
≥0.25 mcg/ml
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcus pneumoniae meningitis
</td>
<td style="text-align:left;">
≤0.06 mcg/ml
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
≥0.12 mcg/ml
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcus pneumoniae (not meningitis)
</td>
<td style="text-align:left;">
≤2 mcg/ml
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
≥8 mcg/ml
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcus Viridans group
</td>
<td style="text-align:left;">
0.12 mcg/ml
</td>
<td style="text-align:left;">
0.25–2 mcg/ml
</td>
<td style="text-align:left;">
4 mcg/ml
</td>
</tr>
<tr>
<td style="text-align:left;">
Listeria monocytogenes
</td>
<td style="text-align:left;">
≤2 mcg/ml
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
Bacillus anthracis
</td>
<td style="text-align:left;">
≤0.12 mcg/ml
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
≥0.25 mcg/ml
</td>
</tr>
</tbody>
</table></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<p>Observations about the growth of some microorganisms inhibiting the growth of other microorganisms have been reported since the late 19th century. These observations of antibiosis between microorganisms led to the discovery of natural antibacterials. Louis Pasteur observed, “if we could intervene in the antagonism observed between some bacteria, it would offer perhaps the greatest hopes for therapeutics”.</p>
<p>Common (≥ 1% of people) adverse drug reactions associated with use of the penicillins include diarrhoea, hypersensitivity, nausea, rash, neurotoxicity, urticaria, and superinfection (including candidiasis). Infrequent adverse effects (0.1–1% of people) include fever, vomiting, erythema, dermatitis, angioedema, seizures (especially in people with epilepsy), and pseudomembranous colitis. Penicillin can also induce serum sickness or a serum sickness-like reaction in some individuals. Serum sickness is a type III hypersensitivity reaction that occurs one to three weeks after exposure to drugs including penicillin. It is not a true drug allergy, because allergies are type I hypersensitivity reactions, but repeated exposure to the offending agent can result in an anaphylactic reaction. Allergy will occur in 1-10% of people, presenting as a skin rash after exposure. IgE-mediated anaphylaxis will occur in approximately 0.01% of patients.</p>
<p>Pain and inflammation at the injection site are also common for parenterally administered benzathine benzylpenicillin, benzylpenicillin, and, to a lesser extent, procaine benzylpenicillin. The condition is known as livedoid dermatitis or Nicolau syndrome.</p>
</div>
<div id="chemical-structure-of-penicillins" class="section level3" number="12.2.6">
<h3><span class="header-section-number">12.2.6</span> Chemical Structure Of Penicillins</h3>
<p>The term “penam” is used to describe the common core skeleton of a member of the penicillins. This core has the molecular formula R-C<sub>9</sub>H<sub>11</sub>N<sub>2O</sub>4~S, where R is the variable side chain that differentiates the penicillins from one another. The penam core has a molar mass of 243 g/mol, with larger penicillins having molar mass near 450—for example, cloxacillin has a molar mass of 436 g/mol. 6-APA (C8H12N2O3S) forms the basic structure of penicillins. It is made up of an enclosed dipeptide formed by the condensation of L-cystein and D-valine. This results in the formations of β-lactam and thiazolidinic rings.</p>
<p>The key structural feature of the penicillins is the four-membered β-lactam ring; this structural moiety is essential for penicillin’s antibacterial activity. The β-lactam ring is itself fused to a five-membered thiazolidine ring. The fusion of these two rings causes the β-lactam ring to be more reactive than monocyclic β-lactams because the two fused rings distort the β-lactam amide bond and therefore remove the resonance stabilisation normally found in these chemical bonds. An acyl side side chain attached to the β-lactam ring.</p>
<p>A variety of β-lactam antibiotics have been produced following chemical modification from the 6-APA structure during synthesis, specifically by making chemical substitutions in the acyl side chain. For example, the first chemically altered penicillin, methicillin, had substitutions by methoxy groups at positions 2’ and 6’ of the 6-APA benzene ring from penicillin G. This difference makes methicillin resistant to the activity of β-lactamase, an enzyme by which many bacteria are naturally unsusceptible to penicillins.</p>
</div>
<div id="mechanism-of-action-of-penicillin" class="section level3" number="12.2.7">
<h3><span class="header-section-number">12.2.7</span> Mechanism Of Action Of Penicillin</h3>
<p>Penicillin can easily enter bacterial cell in case of Gram-positive species. This is because Gram-positive bacteria do not have an outer cell membrane and are simply enclosed in a thick cell wall. Penicillin molecules are small enough to pass through the spaces of glycoproteins in the cell wall. For this reason Gram-positive bacteria are very susceptible to penicillin (as first evidenced by the discovery of penicillin in 1928).</p>
<p>Penicillin, or any other molecule, enters Gram-negative bacteria in a different manner. The bacteria have thinner cell walls but the external surface is coated with an additional cell membrane, called the outer membrane. The outer membrane is a lipid layer (lipopolysaccharide chain) that blocks passage of water-soluble (hydrophilic) molecules like penicillin. It thus acts as the first line of defence against any toxic substance, which is the reason for relative resistance to antibiotics compared to Gram-positive species But penicillin can still enter Gram-negative species by diffusing through aqueous channels called porins (outer membrane proteins), which are dispersed among the fatty molecules and can transport nutrients and antibiotics into the bacteria. Porins are large enough to allow diffusion of most penicillins, but the rate of diffusion through them is determined by the specific size of the drug molecules. For instance, penicillin G is large and enters through porins slowly; while smaller ampicillin and amoxicillin diffuse much faster. In contrast, large vancomycin can not pass through porins and is thus ineffective for Gram-negative bacteria. The size and number of porins are different in different bacteria. As a result of the two factors—size of penicillin and porin—Gram-negative bacteria can be unsusceptible or have varying degree of susceptibility to specific penicillin.</p>
<p>The chemical structure of penicillin is triggered with a very precise, pH-dependent directed mechanism, affected by a unique spatial assembly of molecular components, which can activate by protonation. It can travel through bodily fluids, targeting and inactivating enzymes responsible for cell-wall synthesis in gram-positive bacteria, meanwhile avoiding the surrounding non-targets. Penicillin can protect itself from spontaneous hydrolysis in the body in its anionic form while storing its potential as a strong acylating agent, activated only upon approach to the target transpeptidase enzyme and protonated in the active centre. This targeted protonation neutralizes the carboxylic acid moiety, which is weakening of the β-lactam ring N–C(=O) bond, resulting in a self-activation. Specific structural requirements are equated to constructing the perfect mousetrap for catching targeted prey.</p>
<p>Penicillin kills bacteria by inhibiting the completion of the synthesis of peptidoglycans, the structural component of bacterial cell wall. It specifically inhibits the activity of enzymes that are needed for the cross-linking of peptidoglycans during the final step in cell wall biosynthesis. It does this by binding to penicillin binding proteins with the β-lactam ring, a structure found on penicillin molecules. This causes the cell wall to weaken due to fewer cross-links and means water uncontrollably flows into the cell because it cannot maintain the correct osmotic gradient. This results in cell lysis and death.</p>
<p>Bacteria constantly remodel their peptidoglycan cell walls, simultaneously building and breaking down portions of the cell wall as they grow and divide. During the last stages of peptidoglycan biosynthesis, uridine diphosphate-N-acetylmuramic acid pentapeptide (UDP-MurNAc) is formed in which the fourth and fifth amino acids are both D-alanyl-D-alanine. The transfer of D-alanine is done (catalysed) by the enzyme DD-transpeptidase (penicillin-binding proteins are such type). The structural integrity of bacterial cell wall depends on the cross linking of UDP-MurNAc and N-acetyl glucosamine. Penicillin and other β-lactam antibiotics act as an analogue of D-alanine-D-alanine (the dipeptide) in UDP-MurNAc owing to conformational similarities. The DD-transpeptidase then binds the four-membered β-lactam ring of penicillin in stead of UDP-MurNAc. As a consequence, DD-transpeptidase is inactivated, the formation of cross-links between UDP-MurNAc and N-acetyl glucosamine is blocked so that an imbalance between cell wall production and degradation develops, causing the cell to rapidly die.</p>
<p>The enzymes that hydrolyze the peptidoglycan cross-links continue to function, even while those that form such cross-links do not. This weakens the cell wall of the bacterium, and osmotic pressure becomes increasingly uncompensated—eventually causing cell death (cytolysis). In addition, the build-up of peptidoglycan precursors triggers the activation of bacterial cell wall hydrolases and autolysins, which further digest the cell wall’s peptidoglycans. The small size of the penicillins increases their potency, by allowing them to penetrate the entire depth of the cell wall. This is in contrast to the glycopeptide antibiotics vancomycin and teicoplanin, which are both much larger than the penicillins.</p>
<p>Gram-positive bacteria are called protoplasts when they lose their cell walls. Gram-negative bacteria do not lose their cell walls completely and are called spheroplasts after treatment with penicillin.</p>
<p>Penicillin shows a synergistic effect with aminoglycosides, since the inhibition of peptidoglycan synthesis allows aminoglycosides to penetrate the bacterial cell wall more easily, allowing their disruption of bacterial protein synthesis within the cell. This results in a lowered MBC for susceptible organisms.</p>
<p>Penicillins, like other β-lactam antibiotics, block not only the division of bacteria, including cyanobacteria, but also the division of cyanelles, the photosynthetic organelles of the glaucophytes, and the division of chloroplasts of bryophytes. In contrast, they have no effect on the plastids of the highly developed vascular plants. This supports the endosymbiotic theory of the evolution of plastid division in land plants.</p>
<p>Some bacteria produce enzymes that break down the β-lactam ring, called β-lactamases, which make the bacteria resistant to penicillin. Therefore, some penicillins are modified or given with other drugs for use against antibiotic-resistant bacteria or in immunocompromised patients. The use of clavulanic acid or tazobactam, β-lactamase inhibitors, alongside penicillin gives penicillin activity against β-lactamase-producing bacteria. β-Lactamase inhibitors irreversibly bind to β-lactamase preventing it from breaking down the beta-lactam rings on the antibiotic molecule. Alternatively, flucloxacillin is a modified penicillin that has activity against β-lactamase-producing bacteria due to an acyl side chain that protects the beta-lactam ring from β-lactamase.</p>
<p>Penicillin has low protein binding in plasma. The bioavailability of penicillin depends on the type: penicillin G has low bioavailability, below 30%, whereas penicillin V has higher bioavailability, between 60 and 70%.[citation needed]</p>
<p>Penicillin has a short half-life and is excreted via the kidneys. This means it must be dosed at least four times a day to maintain adequate levels of penicillin in the blood. Early manuals on the use of penicillin, therefore, recommended injections of penicillin as frequently as every three hours, and dosing penicillin has been described as being similar to trying to fill a bath with the plug out. This is no longer required since much larger doses of penicillin are cheaply and easily available; however, some authorities recommend the use of continuous penicillin infusions for this reason.</p>
<p>In 1895 Vincenzo Tiberio, Italian physician, published a paper on the antibacterial power of some extracts of mold.</p>
<p>In 1897, doctoral student Ernest Duchesne submitted a dissertation, “Contribution à l’étude de la concurrence vitale chez les micro-organismes: antagonisme entre les moisissures et les microbes” (Contribution to the study of vital competition in micro-organisms: antagonism between molds and microbes), the first known scholarly work to consider the therapeutic capabilities of molds resulting from their anti-microbial activity. In his thesis, Duchesne proposed that bacteria and molds engage in a perpetual battle for survival. Duchesne observed that E. coli was eliminated by Penicillium glaucum when they were both grown in the same culture. He also observed that when he inoculated laboratory animals with lethal doses of typhoid bacilli together with Penicillium glaucum, the animals did not contract typhoid. Unfortunately Duchesne’s army service after getting his degree prevented him from doing any further research. Duchesne died of tuberculosis, a disease now treated by antibiotics.In 1928, Sir Alexander Fleming postulated the existence of penicillin, a molecule produced by certain molds that kills or stops the growth of certain kinds of bacteria. Fleming was working on a culture of disease-causing bacteria when he noticed the spores of a green mold, Penicillium chrysogenum, in one of his culture plates. He observed that the presence of the mold killed or prevented the growth of the bacteria. Fleming postulated that the mold must secrete an antibacterial substance, which he named penicillin in 1928. Fleming believed that its antibacterial properties could be exploited for chemotherapy. He initially characterized some of its biological properties, and attempted to use a crude preparation to treat some infections, but he was unable to pursue its further development without the aid of trained chemists.</p>
<p>Ernst Chain, Howard Florey and Edward Abraham succeeded in purifying the first penicillin, penicillin G, in 1942, but it did not become widely available outside the Allied military before 1945. Later, Norman Heatley developed the back extraction technique for efficiently purifying penicillin in bulk. The chemical structure of penicillin was first proposed by Abraham in 1942 and then later confirmed by Dorothy Crowfoot Hodgkin in 1945. Purified penicillin displayed potent antibacterial activity against a wide range of bacteria and had low toxicity in humans. Furthermore, its activity was not inhibited by biological constituents such as pus, unlike the synthetic sulfonamides. (see below) The development of penicillin led to renewed interest in the search for antibiotic compounds with similar efficacy and safety. For their successful development of penicillin, which Fleming had accidentally discovered but could not develop himself, as a therapeutic drug, Chain and Florey shared the 1945 Nobel Prize in Medicine with Fleming.</p>
<p>Florey credited Rene Dubos with pioneering the approach of deliberately and systematically searching for antibacterial compounds, which had led to the discovery of gramicidin and had revived Florey’s research in penicillin. In 1939, coinciding with the start of World War II, Dubos had reported the discovery of the first naturally derived antibiotic, tyrothricin, a compound of 20% gramicidin and 80% tyrocidine, from Bacillus brevis. It was one of the first commercially manufactured antibiotics and was very effective in treating wounds and ulcers during World War II. Gramicidin, however, could not be used systemically because of toxicity. Tyrocidine also proved too toxic for systemic usage. Research results obtained during that period were not shared between the Axis and the Allied powers during World War II and limited access during the Cold War.</p>
<p>During the mid-20th century, the number of new antibiotic substances introduced for medical use increased significantly. From 1935 to 1968, 12 new classes were launched. However, after this, the number of new classes dropped markedly, with only two new classes introduced between 1969 and 2003.</p>
<p>Both the WHO and the Infectious Disease Society of America report that the weak antibiotic pipeline does not match bacteria’s increasing ability to develop resistance. The Infectious Disease Society of America report noted that the number of new antibiotics approved for marketing per year had been declining and identified seven antibiotics against the Gram-negative bacilli currently in phase 2 or phase 3 clinical trials. However, these drugs did not address the entire spectrum of resistance of Gram-negative bacilli. According to the WHO fifty one new therapeutic entities - antibiotics (including combinations), are in phase 1-3 clinical trials as of May 2017. Antibiotics targeting multidrug-resistant Gram-positive pathogens remains a high priority.</p>
<p>A few antibiotics have received marketing authorization in the last seven years. The cephalosporin ceftaroline and the lipoglycopeptides oritavancin and telavancin for the treatment of acute bacterial skin and skin structure infection and community-acquired bacterial pneumonia. The lipoglycopeptide dalbavancin and the oxazolidinone tedizolid has also been approved for use for the treatment of acute bacterial skin and skin structure infection. The first in a new class of narrow spectrum macrocyclic antibiotics, fidaxomicin, has been approved for the treatment of C. difficile colitis. New cephalosporin-lactamase inhibitor combinations also approved include ceftazidime-avibactam and ceftolozane-avibactam for complicated urinary tract infection and intra-abdominal infection.</p>
<ul>
<li>Ceftolozane/tazobactam (CXA-201; CXA-101/tazobactam): Antipseudomonal cephalosporin/β-lactamase inhibitor combination (cell wall synthesis inhibitor). FDA approved on 19 December 2014.</li>
<li>Ceftazidime/avibactam (ceftazidime/NXL104): antipseudomonal cephalosporin/β-lactamase inhibitor combination (cell wall synthesis inhibitor). FDA approved on 25 February 2015.</li>
<li>Ceftaroline/avibactam (CPT-avibactam; ceftaroline/NXL104): Anti-MRSA cephalosporin/ β-lactamase inhibitor combination (cell wall synthesis inhibitor).</li>
<li>Cefiderocol: cephalosporin siderophore. FDA approved on 14 November 2019.</li>
<li>Imipenem/relebactam: carbapenem/ β-lactamase inhibitor combination (cell wall synthesis inhibitor). FDA approved on 16 July 2019.</li>
<li>Meropenem/vaborbactam: carbapenem/ β-lactamase inhibitor combination (cell wall synthesis inhibitor). FDA approved on 29 August 2017.</li>
<li>Delafloxacin: quinolone (inhibitor of DNA synthesis). FDA approved on 19 June 2017.</li>
<li>Plazomicin (ACHN-490): semi-synthetic aminoglycoside derivative (protein synthesis inhibitor). FDA approved 25 June 2018.</li>
<li>Eravacycline (TP-434): synthetic tetracycline derivative (protein synthesis inhibitor targeting bacterial ribosomes). FDA approved on 27 August 2018.</li>
<li>Omadacycline: semi-synthetic tetracycline derivative (protein synthesis inhibitor targeting bacterial ribosomes). FDA approved on 2 October 2018.</li>
<li>Lefamulin: pleuromutilin antibiotic. FDA approved on 19 August 2019.</li>
<li>Brilacidin (PMX-30063): peptide defense protein mimetic (cell membrane disruption). In phase 2.</li>
</ul>
<p>Possible improvements include clarification of clinical trial regulations by FDA. Furthermore, appropriate economic incentives could persuade pharmaceutical companies to invest in this endeavor. In the US, the Antibiotic Development to Advance Patient Treatment (ADAPT) Act was introduced with the aim of fast tracking the drug development of antibiotics to combat the growing threat of ‘superbugs’. Under this Act, FDA can approve antibiotics and antifungals treating life-threatening infections based on smaller clinical trials. The CDC will monitor the use of antibiotics and the emerging resistance, and publish the data. The FDA antibiotics labeling process, ‘Susceptibility Test Interpretive Criteria for Microbial Organisms’ or ‘breakpoints’, will provide accurate data to healthcare professionals. According to Allan Coukell, senior director for health programs at The Pew Charitable Trusts, “By allowing drug developers to rely on smaller datasets, and clarifying FDA’s authority to tolerate a higher level of uncertainty for these drugs when making a risk/benefit calculation, ADAPT would make the clinical trials more feasible.”</p>
</div>
<div id="aminoglycoside-antibiotics" class="section level3" number="12.2.8">
<h3><span class="header-section-number">12.2.8</span> Aminoglycoside Antibiotics</h3>
<p>Aminoglycoside is a medicinal and bacteriologic category of traditional Gram-negative antibacterial medications that inhibit protein synthesis and contain as a portion of the molecule an amino-modified glycoside (sugar). The term can also refer more generally to any organic molecule that contains amino sugar substructures. Aminoglycoside antibiotics display bactericidal activity against Gram-negative aerobes and some anaerobic bacilli where resistance has not yet arisen but generally not against Gram-positive and anaerobic Gram-negative bacteria.</p>
<p>Streptomycin is the first-in-class aminoglycoside antibiotic. It is derived from Streptomyces griseus and is the earliest modern agent used against tuberculosis. Streptomycin lacks the common 2-deoxystreptamine moiety (image right, below) present in most other members of this class. Other examples of aminoglycosides include the deoxystreptamine-containing agents kanamycin, tobramycin, gentamicin, and neomycin</p>
</div>
<div id="mechanism-of-action-of-amimoglycoside-antibiotics" class="section level3" number="12.2.9">
<h3><span class="header-section-number">12.2.9</span> Mechanism Of Action Of Amimoglycoside Antibiotics</h3>
<p>Aminoglycosides display concentration-dependent bactericidal activity against “most gram-negative aerobic and facultative anaerobic bacilli” but not against gram-negative anaerobes and most gram-positive bacteria. They require only short contact time, and are most effective against susceptible bacterial populations that are rapidly multiplying. These activities are attributed to a primary mode of action as protein synthesis inhibitors, though additional mechanisms are implicated for some specific agents, and/or thorough mechanistic descriptions are as yet unavailable.</p>
<p>The inhibition of protein synthesis is mediated through aminoglycosides’ energy-dependent, sometimes irreversible binding, to the cytosolic, membrane-associated bacterial ribosome (image at right). (Aminoglycosides first cross bacterial cell walls—lipopolysaccharide in gram-negative bacteria—and cell membranes, where they are actively transported.) While specific steps in protein synthesis affected may vary somewhat between specific aminoglycoside agents, as can their affinity and degree of binding, aminoglycoside presence in the cytosol generally disturbs peptide elongation at the 30S ribosomal subunit, giving rise to inaccurate mRNA translation and therefore biosynthesis of proteins that are truncated, or bear altered amino acid compositions at particular points. Specifically, binding impairs translational proofreading leading to misreading of the RNA message, premature termination, or both, and so to inaccuracy of the translated protein product. The subset of aberrant proteins that are incorporated into the bacterial cell membrane may then lead to changes in its permeability and then to “further stimulation of aminoglycoside transport”. Recent single-molecule tracking experiments in live E. coli showed an ongoing but slower protein synthesis upon treatment with different aminoglycoside drugs.</p>
<p>Finally, a further “cell-membrane effect” also occurs with aminoglycosides; “functional integrity of the bacterial cell membrane” can be lost, later in time courses of aminoglycoside exposure and transport.</p>
<p>Aminoglycosides are useful primarily in infections involving aerobic, Gram-negative bacteria, such as Pseudomonas, Acinetobacter, and Enterobacter. Aminoglycosides are mostly ineffective against anaerobic bacteria, fungi, and viruses.</p>
<p>Aminoglycoside can cause inner ear toxicity which can result in sensorineural hearing loss. The incidence of inner ear toxicity varies from 7 to 90%, depending on the types of antibiotics used, susceptibility of the patient to such antibiotics, and the duration of antibiotic administration.</p>
</div>
</div>
<div id="drug-resistance" class="section level2" number="12.3">
<h2><span class="header-section-number">12.3</span> Drug Resistance</h2>
<p>Drug resistance is the reduction in effectiveness of a medication such as an antimicrobial or an antineoplastic in treating a disease or condition.[1] The term is used in the context of resistance that pathogens or cancers have “acquired”, that is, resistance has evolved. Antimicrobial resistance and antineoplastic resistance challenge clinical care and drive research. When an organism is resistant to more than one drug, it is said to be multidrug-resistant.</p>
<p>The development of antibiotic resistance in particular stems from the drugs targeting only specific bacterial molecules (almost always proteins). Because the drug is so specific, any mutation in these molecules will interfere with or negate its destructive effect, resulting in antibiotic resistance.[2] Furthermore, there is mounting concern over the abuse of antibiotics in the farming of livestock, which in the European Union alone accounts for three times the volume dispensed to humans – leading to development of super-resistant bacteria.[3][4]</p>
<p>Bacteria are capable of not only altering the enzyme targeted by antibiotics, but also by the use of enzymes to modify the antibiotic itself and thus neutralize it. Examples of target-altering pathogens are Staphylococcus aureus, vancomycin-resistant enterococci and macrolide-resistant Streptococcus, while examples of antibiotic-modifying microbes are Pseudomonas aeruginosa and aminoglycoside-resistant Acinetobacter baumannii.[5]</p>
<p>In short, the lack of concerted effort by governments and the pharmaceutical industry, together with the innate capacity of microbes to develop resistance at a rate that outpaces development of new drugs, suggests that existing strategies for developing viable, long-term anti-microbial therapies are ultimately doomed to failure. Without alternative strategies, the acquisition of drug resistance by pathogenic microorganisms looms as possibly one of the most significant public health threats facing humanity in the 21st century.[6] Some of the best alternative sources to reduce the chance of antibiotic resistance are probiotics, prebiotics, dietary fibers, enzymes, organic acids, phytogenics.</p>
<p>Drug, toxin, or chemical resistance is a consequence of evolution and is a response to pressures imposed on any living organism. Individual organisms vary in their sensitivity to the drug used and some with greater fitness may be capable of surviving drug treatment. Drug-resistant traits are accordingly inherited by subsequent offspring, resulting in a population that is more drug-resistant. Unless the drug used makes sexual reproduction or cell-division or horizontal gene transfer impossible in the entire target population, resistance to the drug will inevitably follow. This can be seen in cancerous tumors where some cells may develop resistance to the drugs used in chemotherapy.[9] Chemotherapy causes fibroblasts near tumors to produce large amounts of the protein WNT16B. This protein stimulates the growth of cancer cells which are drug-resistant.[10] MicroRNAs have also been shown to affect acquired drug resistance in cancer cells and this can be used for therapeutic purposes.[11] Malaria in 2012 has become a resurgent threat in South East Asia and sub-Saharan Africa, and drug-resistant strains of Plasmodium falciparum are posing massive problems for health authorities.[12][13] Leprosy has shown an increasing resistance to dapsone.</p>
<p>A rapid process of sharing resistance exists among single-celled organisms, and is termed horizontal gene transfer in which there is a direct exchange of genes, particularly in the biofilm state.[14] A similar asexual method is used by fungi and is called “parasexuality”. Examples of drug-resistant strains are to be found in microorganisms[15] such as bacteria and viruses, parasites both endo- and ecto-, plants, fungi, arthropods,[16][17] mammals,[18] birds,[19] reptiles,[20] fish, and amphibians.[20]</p>
<p>In the domestic environment, drug-resistant strains of organism may arise from seemingly safe activities such as the use of bleach,[21] tooth-brushing and mouthwashing,[22] the use of antibiotics, disinfectants and detergents, shampoos, and soaps, particularly antibacterial soaps,[23][24] hand-washing,[25] surface sprays, application of deodorants, sunblocks and any cosmetic or health-care product, insecticides, and dips.[26] The chemicals contained in these preparations, besides harming beneficial organisms, may intentionally or inadvertently target organisms that have the potential to develop resistance.[27]</p>
<p>The four main mechanisms by which microorganisms exhibit resistance to antimicrobials are:[28][29]</p>
<ul>
<li>Drug inactivation or modification: e.g., enzymatic deactivation of Penicillin G in some penicillin-resistant bacteria through the production of β-lactamases.</li>
<li>Alteration of target site: e.g., alteration of PBP — the binding target site of penicillins — in MRSA and other penicillin-resistant bacteria.</li>
<li>Alteration of metabolic pathway: e.g., some sulfonamide-resistant bacteria do not require para-aminobenzoic acid (PABA), an important precursor for the synthesis of folic acid and nucleic acids in bacteria inhibited by sulfonamides. Instead, like mammalian cells, they turn to utilizing preformed folic acid.</li>
<li>Reduced drug accumulation: by decreasing drug permeability and/or increasing active efflux (pumping out) of the drugs across the cell surface.</li>
</ul>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:resistance">Table 12.3: </span>The three essential polymeric macromolecules of life
</caption>
<thead>
<tr>
<th style="text-align:left;">
Mechanism
</th>
<th style="text-align:left;">
Antimicrobial Agent
</th>
<th style="text-align:left;">
Drug Action
</th>
<th style="text-align:left;">
Mechanism of Resistance
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Destroy drug
</td>
<td style="text-align:left;">
Aminoglycoside Beta-lactam antibiotics (penicillin and cephalosporin) Chloramphenicol
</td>
<td style="text-align:left;">
Binds to 30S Ribosome subunit, inhibiting protein synthesis Binds to penicillin-binding proteins, Inhibiting peptidoglycan synthesis Bind to 50S ribosome subunit, inhibiting formation of peptide bonds
</td>
<td style="text-align:left;">
Plasmid encode enzymes that chemically alter the drug (e.g., by acetylation or phosphorylation), thereby inactivating it. Plasmid encode beta-lactamase, which open the beta-lactam ring, inactivating it. Plasmid encode an enzyme that acetylate the drug, thereby inactivating it.
</td>
</tr>
<tr>
<td style="text-align:left;">
Alters drug target
</td>
<td style="text-align:left;">
Aminoglycosides Beta-lactam antibiotics (penicillin and cephalosporin) Erythromycin Quinolones Rifampin Trimethoprim
</td>
<td style="text-align:left;">
Binds to 30S Ribosome subunit, inhibiting protein synthesis Binds to penicillin-binding proteins, Inhibiting peptidoglycan synthesis Bind to 50S ribosome subunit, inhibiting protein synthesis Binds to DNA topoisomerase, an enzyme essential for DNA synthesis Binds to the RNA polymerase; inhibiting initiation of RNA synthesis Inhibit the enzyme dihydrofolate reduces, blocking the folic acid pathway
</td>
<td style="text-align:left;">
Bacteria make an altered 30S ribosomes that does not bind to the drug. Bacteria make an altered penicillin-binding proteins, that do not bind to the drug. Bacteria make a form of 50S ribosome that does not binds to the drug. Bacteria make an altered DNA topoisomerase that does not binds to the drug. Bacteria make an altered polymerase that does not binds to the drug. Bacteria make an altered enzyme that does not binds to the drug.
</td>
</tr>
<tr>
<td style="text-align:left;">
Inhibits drug entry or removes drug
</td>
<td style="text-align:left;">
Penicillin Erythromycin Tetracycline
</td>
<td style="text-align:left;">
Binds to penicillin-binding proteins, Inhibiting peptidoglycan synthesis Bind to 50S ribosome subunit, inhibiting protein synthesis Binds to 30S Ribosome subunit, inhibiting protein synthesis by blocking tRNA
</td>
<td style="text-align:left;">
Bacteria change shape of the outer membrane porin proteins, preventing drug from entering cell. New membrane transport system prevent drug from entering cell. New membrane transport system pumps drug out of cell.
</td>
</tr>
</tbody>
</table>
<div id="bacterial-resistence-to-penicillin" class="section level3" number="12.3.1">
<h3><span class="header-section-number">12.3.1</span> Bacterial Resistence To Penicillin</h3>
<p>When Alexander Fleming discovered the crude penicillin in 1928, one important observation he made was that many bacteria were not affected by penicillin. This phenomenon was realised by Ernst Chain and Edward Abraham while trying to identify the exact of penicillin. In 1940, they discovered that unsusceptible bacteria like Escherichia coli produced specific enzymes that can break down penicillin molecules, thus making them resistant to the antibiotic. They named the enzyme penicillinase. Penicillinase is now classified as member of enzymes called β-lactamases. These β-lactamases are naturally present in many other bacteria, and many bacteria produce them upon constant exposure to antibiotics. In most bacteria, resistance can be through three different mechanisms: reduced permeability in bacteria reduced binding affinity of the penicillin-binding proteins (PBPs) or destruction of the antibiotic through the expression of β-lactamase. Using any of these, bacteria commonly develop resistance to different antibiotics, a phenomenon called multi-drug resistance.</p>
<p>The actual process of resistance mechanism can be very complex. In case of reduced permeability in bacteria, the mechanisms are different between Gram-positive and Gram-negative bacteria. In Gram-positive bacteria, blockage of penicillin is due to changes in the cell wall. For example, resistance to vancomycin in S. aureus is due to additional peptidoglycan synthesis that makes the cell wall much thicker preventing effective penicillin entry. Resistance in Gram-negative bacteria is due to mutational variations in the structure and number of porins. In bacteria like Pseudomonas aeruginosa, there is reduced number of porins; whereas in bacteria like Enterobacter species, Escherichia coli and Klebsiella pneumoniae, there are modified porins such as non-specific porins (such as OmpC and OmpF groups) that cannot transport penicillin.</p>
<p>Resistance due to PBP alterations is highly varied. A common case is found in Streptococcus pneumoniae where there is mutation in the gene for PBP, and the mutant PBPs have decreased binding affinity for penicillins. There are six mutant PBPs in S. pneumoniae, of which PBP1a, PBP2b, PBP2x and sometimes PBP2a are responsible for reduced binding affinity. S. aureus can activate a hidden gene that produces a different PBP, PBD2, which has low binding affinity for penicillins. There is a different strain of S. aureus named methicillin-resistant S. aureus (MRSA) which is resistant not only to penicllin and other β-lactams, but also to most antibiotics. The bacterial strain developed after introduction of methicillin in 1959. In MRSA, mutations in the genes (mec system) for PBP produce a variant protein called PBP2a (also termed PBP2’), while making four normal PBPs. PBP2a has poor binding affinity for penicillin and also lacks glycosyltransferase activity required for complete peptidoglycan synthesis (which is carried out by the four normal PBPs). In Helicobacter cinaedi, there are multiple mutations in different genes that make PBP variants.</p>
<p>Enzymatic destruction by β-lactamases is the most important mechanism of penicillin resistance, and is described as “the greatest threat to the usage [of penicillins]”. It was the first discovered mechanism of penicillin resistance. During the experiments when purification and biological activity tests of penicillin were performed in 1940, it was found that E. coli was unsusceptible. The reason was discovered as production of an enzyme penicillinase (hence, the first β-lactamase known) in E. coli that easily degraded penicillin. There are over 2,000 types of β-lactamases each of which has unique amino acid sequence, and thus, enzymatic activity. All of them are able to hydrolyse β-lactam rings but their exact target sites are different. They are secreted on the bacterial surface in large quantities in Gram-positive bacteria but less so in Gram-negative species. Therefore, in a mixed bacterial infection, the Gram-positive bacteria can protect the otherwise penicillin-susceptible Gram-negative cells.</p>
<p>There are unusual mechanisms in P. aeruginosa, in which there can be biofilm-mediated resistance and formation of multidrug-tolerant persister cells.
In 1874, physician Sir William Roberts noted that cultures of the mold Penicillium glaucum that is used in the making of some types of blue cheese did not display bacterial contamination. In 1876, physicist John Tyndall also contributed to this field.</p>
</div>
</div>
<div id="testing-for-antibiotic-sensitivity" class="section level2" number="12.4">
<h2><span class="header-section-number">12.4</span> Testing For Antibiotic Sensitivity</h2>
<p>Antibiotic sensitivity testing or antibiotic susceptibility testing is the measurement of the susceptibility of bacteria to antibiotics. It is used because bacteria may have resistance to some antibiotics. Sensitivity testing results can allow a clinician to change the choice of antibiotics from empiric therapy, which is when an antibiotic is selected based on clinical suspicion about the site of an infection and common causative bacteria, to directed therapy, in which the choice of antibiotic is based on knowledge of the organism and its sensitivities.</p>
<p>Sensitivity testing usually occurs in a medical laboratory, and may be based on culture methods that expose bacteria to antibiotics, or genetic methods that test to see if bacteria have genes that confer resistance. Culture methods often involve measuring the diameter of areas without bacterial growth, called zones of inhibition, around paper discs containing antibiotics on agar culture dishes that have been evenly inoculated with bacteria. The minimum inhibitory concentration, which is the lowest concentration of the antibiotic that stops the growth of bacteria, can be estimated from the size of the zone of inhibition.</p>
<p>Antibiotic susceptibility testing has occurred since the discovery of the beta-lactam antibiotic penicillin. Initial methods were phenotypic, and involved culture or dilution. The Etest, an antibiotic impregnated strip, has been available since the 1980s, and genetic methods such as polymerase chain reaction (PCR) testing have been available since the early 2000s. Research is ongoing into improving current methods by making them faster or more accurate, as well as developing new methods for testing, such as microfluidics.</p>
<div id="the-disk-diffusion-test" class="section level3" number="12.4.1">
<h3><span class="header-section-number">12.4.1</span> The Disk Diffusion Test</h3>
<p>The disk diffusion test (also known as the agar diffusion test, Kirby–Bauer test, disc-diffusion antibiotic susceptibility test, disc-diffusion antibiotic sensitivity test and KB test) is a culture-based microbiology assay used in diagnostic and drug discovery laboratories. In diagnostic labs, the assay is used to determine the susceptibility of bacteria isolated from a patient’s infection to clinically approved antibiotics. This allows physicians to prescribe the most appropriate antibiotic treatment. In drug discovery labs, especially bioprospecting labs, the assay is used to screen biological material (e.g. plant extracts, bacterial fermentation broths) and drug candidates for antibacterial activity. When bioprospecting, the assay can be performed with paired strains of bacteria to achieve dereplication and provisionally identify antibacterial mechanism of action.</p>
<p>In diagnostic laboratories, the test is performed by inoculating the surface of an agar plate with bacteria isolated from a patient’s infection. Antibiotic-containing paper disks are then applied to the agar and the plate is incubated. If an antibiotic stops the bacteria from growing or kills the bacteria, there will be an area around the disk where the bacteria have not grown enough to be visible. This is called a zone of inhibition. The susceptibility of the bacterial isolate to each antibiotic can then be semi-quantified by comparing the size of these zones of inhibition to databases of information on known antibiotic-susceptible, moderately susceptible and resistant bacteria. In this way, it is possible to identify the most appropriate antibiotic for treating a patient’s infection. Although the disk diffusion test cannot be used to differentiate bacteriostatic and bactericidal activity, it is less cumbersome than other susceptibility test methods such as broth dilution.</p>
<p>In drug discovery labs, the disk diffusion test is performed slightly differently than in diagnostic labs. In this setting, it is not the bacterial strain that must be characterized, but a test extract (e.g. a plant or microbial extract). The agar plate is therefore inoculated with a bacterial strain of known phenotype (often an ATCC or NCTC strain), and disks containing the test extract are applied to the surface. Zone of inhibition sizes cannot be used as a semi-quantitative measure of antibacterial potency because different extracts contain molecules with different diffusion characteristics (different molecular sizes, hydrophilicities etc.). Zone of inhibition sizes can be used for the purpose of dereplication though. This is achieved by testing each extract against paired strains of bacteria (e.g. streptomycin-susceptible and -resistant strains to identify streptomycin-containing extracts). Paired strains (e.g. wild type and target overexpressing strains) can also be used to identify antibacterial mechanism of action.</p>
</div>
</div>
<div id="replenishing-the-antibiotic-pipeline-and-developing-other-new-therapies" class="section level2" number="12.5">
<h2><span class="header-section-number">12.5</span> Replenishing The Antibiotic Pipeline And Developing Other New Therapies</h2>
<p>Because antibiotic-resistant bacterial strains continue to emerge and spread, there is a constant need to develop new antibacterial treatments. Current strategies include traditional chemistry-based approaches such as natural product-based drug discovery, newer chemistry-based approaches such as drug design, traditional biology-based approaches such as immunoglobulin therapy, and experimental biology-based approaches such as phage therapy, fecal microbiota transplants, antisense RNA-based treatments, and CRISPR-Cas9-based treatments.</p>
<div id="natural-product-based-antibiotic-discovery" class="section level3" number="12.5.1">
<h3><span class="header-section-number">12.5.1</span> Natural Product-Based Antibiotic Discovery</h3>
<p>Most of the antibiotics in current use are natural products or natural product derivatives, and bacterial, fungal, plant and animal extracts are being screened in the search for new antibiotics. Organisms may be selected for testing based on ecological, ethnomedical, genomic or historical rationales. Medicinal plants, for example, are screened on the basis that they are used by traditional healers to prevent or cure infection and may therefore contain antibacterial compounds. Also, soil bacteria are screened on the basis that, historically, they have been a very rich source of antibiotics (with 70 to 80% of antibiotics in current use derived from the actinomycetes).</p>
<p>In addition to screening natural products for direct antibacterial activity, they are sometimes screened for the ability to suppress antibiotic resistance and antibiotic tolerance. For example, some secondary metabolites inhibit drug efflux pumps, thereby increasing the concentration of antibiotic able to reach its cellular target and decreasing bacterial resistance to the antibiotic. Natural products known to inhibit bacterial efflux pumps include the alkaloid lysergol, the carotenoids capsanthin and capsorubin, and the flavonoids rotenone and chrysin. Other natural products, this time primary metabolites rather than secondary metabolites, have been shown to eradicate antibiotic tolerance. For example, glucose, mannitol, and fructose reduce antibiotic tolerance in Escherichia coli and Staphylococcus aureus, rendering them more susceptible to killing by aminoglycoside antibiotics.</p>
<p>Natural products may be screened for the ability to suppress bacterial virulence factors too. Virulence factors are molecules, cellular structures and regulatory systems that enable bacteria to evade the body’s immune defenses (e.g. urease, staphyloxanthin), move towards, attach to, and/or invade human cells (e.g. type IV pili, adhesins, internalins), coordinate the activation of virulence genes (e.g. quorum sensing), and cause disease (e.g. exotoxins). Examples of natural products with antivirulence activity include the flavonoid epigallocatechin gallate (which inhibits listeriolysin O), the quinone tetrangomycin (which inhibits staphyloxanthin), and the sesquiterpene zerumbone (which inhibits Acinetobacter baumannii motility).</p>
</div>
<div id="immunoglobulin-therapy" class="section level3" number="12.5.2">
<h3><span class="header-section-number">12.5.2</span> Immunoglobulin Therapy</h3>
<p>Antibodies (anti-tetanus immunoglobulin) have been used in the treatment and prevention of tetanus since the 1910s, and this approach continues to be a useful way of controlling bacterial disease. The monoclonal antibody bezlotoxumab, for example, has been approved by the US FDA and EMA for recurrent Clostridium difficile infection, and other monoclonal antibodies are in development (e.g. AR-301 for the adjunctive treatment of S. aureus ventilator-associated pneumonia). Antibody treatments act by binding to and neutralizing bacterial exotoxins and other virulence factors.</p>
</div>
<div id="phage-therapy" class="section level3" number="12.5.3">
<h3><span class="header-section-number">12.5.3</span> Phage Therapy</h3>
<p>Phage therapy is under investigation as a method of treating antibiotic-resistant strains of bacteria. Phage therapy involves infecting bacterial pathogens with viruses. Bacteriophages and their host ranges are extremely specific for certain bacteria, thus, unlike antibiotics, they do not disturb the host organism’s intestinal microbiota. Bacteriophages, also known simply as phages, infect and kill bacteria primarily during lytic cycles. Phages insert their DNA into the bacterium, where it is transcribed and used to make new phages, after which the cell will lyse, releasing new phage that are able to infect and destroy further bacteria of the same strain. The high specificity of phage protects “good” bacteria from destruction.</p>
<p>Some disadvantages to the use of bacteriophages also exist, however. Bacteriophages may harbour virulence factors or toxic genes in their genomes and, prior to use, it may be prudent to identify genes with similarity to known virulence factors or toxins by genomic sequencing. In addition, the oral and IV administration of phages for the eradication of bacterial infections poses a much higher safety risk than topical application. Also, there is the additional concern of uncertain immune responses to these large antigenic cocktails.</p>
<p>There are considerable regulatory hurdles that must be cleared for such therapies. Despite numerous challenges, the use of bacteriophages as a replacement for antimicrobial agents against MDR pathogens that no longer respond to conventional antibiotics, remains an attractive option.</p>
</div>
<div id="fecal-microbiota-transplants" class="section level3" number="12.5.4">
<h3><span class="header-section-number">12.5.4</span> Fecal Microbiota Transplants</h3>
<p>￼
Fecal microbiota transplants are an experimental treatment for C. difficile infection.
Fecal microbiota transplants involve transferring the full intestinal microbiota from a healthy human donor (in the form of stool) to patients with C. difficile infection. Although this procedure has not been officially approved by the US FDA, its use is permitted under some conditions in patients with antibiotic-resistant C. difficile infection. Cure rates are around 90%, and work is underway to develop stool banks, standardized products, and methods of oral delivery.</p>
</div>
<div id="antisense-rna-based-treatments" class="section level3" number="12.5.5">
<h3><span class="header-section-number">12.5.5</span> Antisense RNA-Based Treatments</h3>
<p>Antisense RNA-based treatment (also known as gene silencing therapy) involves (a) identifying bacterial genes that encode essential proteins (e.g. the Pseudomonas aeruginosa genes acpP, lpxC, and rpsJ), (b) synthesizing single stranded RNA that is complementary to the mRNA encoding these essential proteins, and (c) delivering the single stranded RNA to the infection site using cell-penetrating peptides or liposomes. The antisense RNA then hybridizes with the bacterial mRNA and blocks its translation into the essential protein. Antisense RNA-based treatment has been shown to be effective in in vivo models of P. aeruginosa pneumonia.</p>
<p>In addition to silencing essential bacterial genes, antisense RNA can be used to silence bacterial genes responsible for antibiotic resistance. For example, antisense RNA has been developed that silences the S. aureus mecA gene (the gene that encodes modified penicillin-binding protein 2a and renders S. aureus strains methicillin-resistant). Antisense RNA targeting mecA mRNA has been shown to restore the susceptibility of methicillin-resistant staphylococci to oxacillin in both in vitro and in vivo studies.</p>
</div>
<div id="crispr-cas9-based-treatments" class="section level3" number="12.5.6">
<h3><span class="header-section-number">12.5.6</span> CRISPR-Cas9-Based Treatments</h3>
<p>In the early 2000s, a system was discovered that enables bacteria to defend themselves against invading viruses. The system, known as CRISPR-Cas9, consists of (a) an enzyme that destroys DNA (the nuclease Cas9) and (b) the DNA sequences of previously encountered viral invaders (CRISPR). These viral DNA sequences enable the nuclease to target foreign (viral) rather than self (bacterial) DNA.</p>
<p>Although the function of CRISPR-Cas9 in nature is to protect bacteria, the DNA sequences in the CRISPR component of the system can be modified so that the Cas9 nuclease targets bacterial resistance genes or bacterial virulence genes instead of viral genes. The modified CRISPR-Cas9 system can then be administered to bacterial pathogens using plasmids or bacteriophages. This approach has successfully been used to silence antibiotic resistance and reduce the virulence of enterohemorrhagic E. coli in an in vivo model of infection.</p>
<p>In addition to developing new antibacterial treatments, it is important to reduce the selection pressure for the emergence and spread of antibiotic resistance. Strategies to accomplish this include well-established infection control measures such as infrastructure improvement (e.g. less crowded housing), better sanitation (e.g. safe drinking water and food) and vaccine development, other approaches such as antibiotic stewardship, and experimental approaches such as the use of prebiotics and probiotics to prevent infection.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="physical-and-chemical-agents-for-microbial-control.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="infection-and-disease.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/microbiology-text/edit/master/antimicrobial_drugs.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
