<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>12 Host Defenses:The Innate Immune System | Text For SCI204 Microbiology at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Microbiology." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="12 Host Defenses:The Innate Immune System | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/microbiology_text_cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Microbiology." />
  <meta name="github-repo" content="nikolaussucher/microbiology-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="12 Host Defenses:The Innate Immune System | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Microbiology." />
  <meta name="twitter:image" content="/microbiology_text_cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="drugs-microbes-and-the-host.html"/>
<link rel="next" href="host-defenses-the-adaptive-immune-system.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />




<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">SCI204 Microbiology at RCC</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html"><i class="fa fa-check"></i><b>1</b> The Main Themes Of Microbiology</a></li>
<li class="chapter" data-level="2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html"><i class="fa fa-check"></i><b>2</b> Methods Of Studying Microorganisms</a>
<ul>
<li class="chapter" data-level="2.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microscopy"><i class="fa fa-check"></i><b>2.1</b> Microscopy</a></li>
<li class="chapter" data-level="2.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#light-microscopy"><i class="fa fa-check"></i><b>2.2</b> Light Microscopy</a></li>
<li class="chapter" data-level="2.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#the-optical-light-microscope"><i class="fa fa-check"></i><b>2.3</b> The Optical (Light) Microscope</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#simple-microscopeedit"><i class="fa fa-check"></i><b>2.3.1</b> Simple microscope[edit]</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#compound-microscope"><i class="fa fa-check"></i><b>2.3.2</b> Compound microscope</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#main-components-of-the-light-microscope"><i class="fa fa-check"></i><b>2.3.3</b> Main Components of The Light Microscope</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microscope-slide"><i class="fa fa-check"></i><b>2.4</b> Microscope Slide</a></li>
<li class="chapter" data-level="2.5" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#electron-microscopy"><i class="fa fa-check"></i><b>2.5</b> Electron microscopy</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#ransmission-electron-microscope-tem"><i class="fa fa-check"></i><b>2.5.1</b> ransmission electron microscope (TEM)</a></li>
<li class="chapter" data-level="2.5.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#scanning-electron-microscope-sem"><i class="fa fa-check"></i><b>2.5.2</b> Scanning electron microscope (SEM)</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microbiological-culture"><i class="fa fa-check"></i><b>2.6</b> Microbiological culture</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#growth-medium"><i class="fa fa-check"></i><b>2.6.1</b> Growth medium</a></li>
<li class="chapter" data-level="2.6.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#selective-media"><i class="fa fa-check"></i><b>2.6.2</b> Selective media</a></li>
<li class="chapter" data-level="2.6.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#agar-plate"><i class="fa fa-check"></i><b>2.6.3</b> Agar Plate</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html"><i class="fa fa-check"></i><b>3</b> Prokaryotic Microorganisms</a>
<ul>
<li class="chapter" data-level="3.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#bacteria"><i class="fa fa-check"></i><b>3.1</b> Bacteria</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#intracellular-structure"><i class="fa fa-check"></i><b>3.1.1</b> Intracellular Structure</a></li>
<li class="chapter" data-level="3.1.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#extracellular-structures"><i class="fa fa-check"></i><b>3.1.2</b> Extracellular Structures</a></li>
<li class="chapter" data-level="3.1.3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#endospores"><i class="fa fa-check"></i><b>3.1.3</b> Endospores</a></li>
<li class="chapter" data-level="3.1.4" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#metabolism"><i class="fa fa-check"></i><b>3.1.4</b> Metabolism</a></li>
<li class="chapter" data-level="3.1.5" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#growth-and-reproduction"><i class="fa fa-check"></i><b>3.1.5</b> Growth and reproduction</a></li>
<li class="chapter" data-level="3.1.6" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#genetics"><i class="fa fa-check"></i><b>3.1.6</b> Genetics</a></li>
<li class="chapter" data-level="3.1.7" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#movement"><i class="fa fa-check"></i><b>3.1.7</b> Movement</a></li>
<li class="chapter" data-level="3.1.8" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#communication"><i class="fa fa-check"></i><b>3.1.8</b> Communication</a></li>
<li class="chapter" data-level="3.1.9" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#classification-and-identification"><i class="fa fa-check"></i><b>3.1.9</b> Classification and identification</a></li>
<li class="chapter" data-level="3.1.10" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#interactions-with-other-organisms"><i class="fa fa-check"></i><b>3.1.10</b> Interactions With Other Organisms</a></li>
<li class="chapter" data-level="3.1.11" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#bacteria-as-pathogens"><i class="fa fa-check"></i><b>3.1.11</b> Bacteria as Pathogens</a></li>
<li class="chapter" data-level="3.1.12" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#significance-in-technology-and-industry"><i class="fa fa-check"></i><b>3.1.12</b> Significance in Technology And Industry</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#archaea"><i class="fa fa-check"></i><b>3.2</b> Archaea</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#origin-and-evolution"><i class="fa fa-check"></i><b>3.2.1</b> Origin and evolution</a></li>
<li class="chapter" data-level="3.2.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#relationship-to-bacteria"><i class="fa fa-check"></i><b>3.2.2</b> Relationship to Bacteria</a></li>
<li class="chapter" data-level="3.2.3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#relation-to-eukaryotes"><i class="fa fa-check"></i><b>3.2.3</b> Relation to Eukaryotes</a></li>
<li class="chapter" data-level="3.2.4" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#morphology"><i class="fa fa-check"></i><b>3.2.4</b> Morphology</a></li>
<li class="chapter" data-level="3.2.5" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#structure-composition-development-and-operation"><i class="fa fa-check"></i><b>3.2.5</b> Structure, Composition Development, And Operation</a></li>
<li class="chapter" data-level="3.2.6" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#cell-wall-and-flagella"><i class="fa fa-check"></i><b>3.2.6</b> Cell Wall And Flagella</a></li>
<li class="chapter" data-level="3.2.7" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#membranes"><i class="fa fa-check"></i><b>3.2.7</b> Membranes</a></li>
<li class="chapter" data-level="3.2.8" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#metabolism-1"><i class="fa fa-check"></i><b>3.2.8</b> Metabolism</a></li>
<li class="chapter" data-level="3.2.9" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#genetics-1"><i class="fa fa-check"></i><b>3.2.9</b> Genetics</a></li>
<li class="chapter" data-level="3.2.10" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#ecology"><i class="fa fa-check"></i><b>3.2.10</b> Ecology</a></li>
<li class="chapter" data-level="3.2.11" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#significance-in-technology-and-industry-1"><i class="fa fa-check"></i><b>3.2.11</b> Significance in Technology And Industry</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html"><i class="fa fa-check"></i><b>4</b> Eukaryotic Microorganisms</a>
<ul>
<li class="chapter" data-level="4.1" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#protists"><i class="fa fa-check"></i><b>4.1</b> Protists</a></li>
<li class="chapter" data-level="4.2" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#protozoa"><i class="fa fa-check"></i><b>4.2</b> Protozoa</a></li>
<li class="chapter" data-level="4.3" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#flagellates"><i class="fa fa-check"></i><b>4.3</b> Flagellates</a></li>
<li class="chapter" data-level="4.4" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#ciliates"><i class="fa fa-check"></i><b>4.4</b> Ciliates</a></li>
<li class="chapter" data-level="4.5" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#amoebozoa"><i class="fa fa-check"></i><b>4.5</b> Amoebozoa</a></li>
<li class="chapter" data-level="4.6" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#algae"><i class="fa fa-check"></i><b>4.6</b> Algae</a></li>
<li class="chapter" data-level="4.7" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#slime-molds"><i class="fa fa-check"></i><b>4.7</b> Slime Molds</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#reproduction-of-protists"><i class="fa fa-check"></i><b>4.7.1</b> Reproduction of Protists</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html"><i class="fa fa-check"></i><b>5</b> An Introduction To Viruses</a>
<ul>
<li class="chapter" data-level="5.0.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#role-in-human-disease"><i class="fa fa-check"></i><b>5.0.1</b> Role in human disease</a></li>
<li class="chapter" data-level="5.0.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#animal-viruses"><i class="fa fa-check"></i><b>5.0.2</b> Animal viruses</a></li>
<li class="chapter" data-level="5.0.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#plant-viruses"><i class="fa fa-check"></i><b>5.0.3</b> Plant viruses</a></li>
<li class="chapter" data-level="5.0.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#bacterial-viruses"><i class="fa fa-check"></i><b>5.0.4</b> Bacterial viruses</a></li>
<li class="chapter" data-level="5.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#classification"><i class="fa fa-check"></i><b>5.1</b> Classification</a></li>
<li class="chapter" data-level="5.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#baltimore-classification"><i class="fa fa-check"></i><b>5.2</b> Baltimore classification</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html"><i class="fa fa-check"></i><b>6</b> Microbial Nutrition, Ecology And Growth</a>
<ul>
<li class="chapter" data-level="6.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#thermodynamics-of-living-organisms"><i class="fa fa-check"></i><b>6.1</b> Thermodynamics of Living Organisms</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html"><i class="fa fa-check"></i><b>7</b> Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="7.0.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#catabolism"><i class="fa fa-check"></i><b>7.0.1</b> Catabolism</a></li>
<li class="chapter" data-level="7.0.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#digestion"><i class="fa fa-check"></i><b>7.0.2</b> Digestion</a></li>
<li class="chapter" data-level="7.0.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-organic-compounds"><i class="fa fa-check"></i><b>7.0.3</b> Energy From Organic Compounds</a></li>
<li class="chapter" data-level="7.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-transformations"><i class="fa fa-check"></i><b>7.1</b> Energy Transformations</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-phosphorylation"><i class="fa fa-check"></i><b>7.1.1</b> Oxidative Phosphorylation</a></li>
<li class="chapter" data-level="7.1.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-inorganic-compounds"><i class="fa fa-check"></i><b>7.1.2</b> Energy From Inorganic Compounds</a></li>
<li class="chapter" data-level="7.1.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-light"><i class="fa fa-check"></i><b>7.1.3</b> Energy From Light</a></li>
<li class="chapter" data-level="7.1.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#anabolism"><i class="fa fa-check"></i><b>7.1.4</b> Anabolism</a></li>
<li class="chapter" data-level="7.1.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbon-fixation"><i class="fa fa-check"></i><b>7.1.5</b> Carbon Fixation</a></li>
<li class="chapter" data-level="7.1.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbohydrates-and-glycans"><i class="fa fa-check"></i><b>7.1.6</b> Carbohydrates And Glycans</a></li>
<li class="chapter" data-level="7.1.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fatty-acids-isoprenoids-and-sterol"><i class="fa fa-check"></i><b>7.1.7</b> Fatty Acids, Isoprenoids and Sterol</a></li>
<li class="chapter" data-level="7.1.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#proteins"><i class="fa fa-check"></i><b>7.1.8</b> Proteins</a></li>
<li class="chapter" data-level="7.1.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#nucleotide-synthesis-and-salvage"><i class="fa fa-check"></i><b>7.1.9</b> Nucleotide Synthesis And Salvage</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#heterotrophic-microbial-metabolism"><i class="fa fa-check"></i><b>7.2</b> Heterotrophic microbial metabolism</a></li>
<li class="chapter" data-level="7.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fermentation"><i class="fa fa-check"></i><b>7.3</b> Fermentation</a></li>
<li class="chapter" data-level="7.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#chemolithotrophy"><i class="fa fa-check"></i><b>7.4</b> Chemolithotrophy</a></li>
<li class="chapter" data-level="7.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#phototrophy"><i class="fa fa-check"></i><b>7.5</b> Phototrophy</a></li>
<li class="chapter" data-level="7.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#enzymes"><i class="fa fa-check"></i><b>7.6</b> Enzymes</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#allosteric-modulation"><i class="fa fa-check"></i><b>7.6.1</b> Allosteric Modulation</a></li>
<li class="chapter" data-level="7.6.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cofactors-of-enzymes-and-coenzymes"><i class="fa fa-check"></i><b>7.6.2</b> Cofactors of Enzymes and Coenzymes</a></li>
<li class="chapter" data-level="7.6.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#inhibition-of-enzymes"><i class="fa fa-check"></i><b>7.6.3</b> Inhibition of Enzymes</a></li>
<li class="chapter" data-level="7.6.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#factors-affecting-enzyme-activity"><i class="fa fa-check"></i><b>7.6.4</b> Factors Affecting Enzyme Activity</a></li>
<li class="chapter" data-level="7.6.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#control-of-activity"><i class="fa fa-check"></i><b>7.6.5</b> Control of activity</a></li>
<li class="chapter" data-level="7.6.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#post-translational-modification"><i class="fa fa-check"></i><b>7.6.6</b> Post-translational Modification</a></li>
<li class="chapter" data-level="7.6.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#quantity"><i class="fa fa-check"></i><b>7.6.7</b> Quantity</a></li>
<li class="chapter" data-level="7.6.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#subcellular-distribution"><i class="fa fa-check"></i><b>7.6.8</b> Subcellular Distribution</a></li>
<li class="chapter" data-level="7.6.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#organ-specialization"><i class="fa fa-check"></i><b>7.6.9</b> Organ Specialization</a></li>
<li class="chapter" data-level="7.6.10" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#coenzymes"><i class="fa fa-check"></i><b>7.6.10</b> Coenzymes</a></li>
<li class="chapter" data-level="7.6.11" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#mineral-cofactors"><i class="fa fa-check"></i><b>7.6.11</b> Mineral Cofactors</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthesis-and-cellular-respiration"><i class="fa fa-check"></i><b>7.7</b> Photosynthesis And Cellular Respiration</a></li>
<li class="chapter" data-level="7.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#the-electron-transport-chain-and-chemiosmosis"><i class="fa fa-check"></i><b>7.8</b> The Electron Transport Chain And Chemiosmosis</a></li>
<li class="chapter" data-level="7.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthesis"><i class="fa fa-check"></i><b>7.9</b> Photosynthesis</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthetic-membranes-and-organelles"><i class="fa fa-check"></i><b>7.9.1</b> Photosynthetic Membranes And Organelles</a></li>
<li class="chapter" data-level="7.9.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#z-scheme"><i class="fa fa-check"></i><b>7.9.2</b> Z Scheme</a></li>
<li class="chapter" data-level="7.9.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#water-photolysis"><i class="fa fa-check"></i><b>7.9.3</b> Water Photolysis</a></li>
<li class="chapter" data-level="7.9.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#calvin-cycle"><i class="fa fa-check"></i><b>7.9.4</b> Calvin Cycle</a></li>
<li class="chapter" data-level="7.9.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbon-dioxide-levels-and-photorespiration"><i class="fa fa-check"></i><b>7.9.5</b> Carbon Dioxide Levels And Photorespiration</a></li>
<li class="chapter" data-level="7.9.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#c4-carbon-fixation"><i class="fa fa-check"></i><b>7.9.6</b> C4 Carbon Fixation</a></li>
<li class="chapter" data-level="7.9.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cam-photosynthesis"><i class="fa fa-check"></i><b>7.9.7</b> CAM Photosynthesis</a></li>
</ul></li>
<li class="chapter" data-level="7.10" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#symbiosis-and-the-origin-of-chloroplasts"><i class="fa fa-check"></i><b>7.10</b> Symbiosis And The Origin of Chloroplasts</a></li>
<li class="chapter" data-level="7.11" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cyanobacteria-and-the-evolution-of-photosynthesis"><i class="fa fa-check"></i><b>7.11</b> Cyanobacteria And The Evolution of Photosynthesis</a></li>
<li class="chapter" data-level="7.12" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cellular-respiration"><i class="fa fa-check"></i><b>7.12</b> Cellular Respiration</a>
<ul>
<li class="chapter" data-level="7.12.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#aerobic-respiration"><i class="fa fa-check"></i><b>7.12.1</b> Aerobic Respiration</a></li>
<li class="chapter" data-level="7.12.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#glycolysis"><i class="fa fa-check"></i><b>7.12.2</b> Glycolysis</a></li>
<li class="chapter" data-level="7.12.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-decarboxylation-of-pyruvate"><i class="fa fa-check"></i><b>7.12.3</b> Oxidative Decarboxylation of Pyruvate</a></li>
<li class="chapter" data-level="7.12.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#the-citric-acid-cycle"><i class="fa fa-check"></i><b>7.12.4</b> The Citric Acid Cycle</a></li>
<li class="chapter" data-level="7.12.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-phosphorylation-1"><i class="fa fa-check"></i><b>7.12.5</b> Oxidative Phosphorylation</a></li>
</ul></li>
<li class="chapter" data-level="7.13" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fermentation-1"><i class="fa fa-check"></i><b>7.13</b> Fermentation</a></li>
<li class="chapter" data-level="7.14" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#anaerobic-respiration"><i class="fa fa-check"></i><b>7.14</b> Anaerobic Respiration</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="microbial-genetics.html"><a href="microbial-genetics.html"><i class="fa fa-check"></i><b>8</b> Microbial Genetics</a></li>
<li class="chapter" data-level="9" data-path="genetic-engineering.html"><a href="genetic-engineering.html"><i class="fa fa-check"></i><b>9</b> Genetic Engineering</a>
<ul>
<li class="chapter" data-level="9.1" data-path="genetic-engineering.html"><a href="genetic-engineering.html#creatin-a-genetically-modified-organism-gmo"><i class="fa fa-check"></i><b>9.1</b> Creatin A Genetically Modified Organism (GMO)</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html"><i class="fa fa-check"></i><b>10</b> Physical And Chemical Agents for Microbial Control</a>
<ul>
<li class="chapter" data-level="10.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#disinfectant"><i class="fa fa-check"></i><b>10.1</b> Disinfectant</a></li>
<li class="chapter" data-level="10.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antiseptic"><i class="fa fa-check"></i><b>10.2</b> Antiseptic</a></li>
<li class="chapter" data-level="10.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#sterilization"><i class="fa fa-check"></i><b>10.3</b> Sterilization</a></li>
<li class="chapter" data-level="10.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#pasteurization"><i class="fa fa-check"></i><b>10.4</b> Pasteurization</a></li>
<li class="chapter" data-level="10.5" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antibiotics"><i class="fa fa-check"></i><b>10.5</b> Antibiotics</a></li>
<li class="chapter" data-level="10.6" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#classes-of-antibiotics"><i class="fa fa-check"></i><b>10.6</b> Classes of Antibiotics</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html"><i class="fa fa-check"></i><b>11</b> Drugs, Microbes, And The Host</a></li>
<li class="chapter" data-level="12" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html"><i class="fa fa-check"></i><b>12</b> Host Defenses:The Innate Immune System</a>
<ul>
<li class="chapter" data-level="12.1" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#layered-defenseedit"><i class="fa fa-check"></i><b>12.1</b> Layered defense[edit]</a></li>
<li class="chapter" data-level="12.2" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#surface-barriers"><i class="fa fa-check"></i><b>12.2</b> Surface Barriers</a></li>
<li class="chapter" data-level="12.3" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#innate-immune-system"><i class="fa fa-check"></i><b>12.3</b> Innate immune system</a></li>
<li class="chapter" data-level="12.4" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#anatomical-barriers"><i class="fa fa-check"></i><b>12.4</b> Anatomical barriers</a></li>
<li class="chapter" data-level="12.5" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#inflammation"><i class="fa fa-check"></i><b>12.5</b> Inflammation</a></li>
<li class="chapter" data-level="12.6" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#complement-systemedit"><i class="fa fa-check"></i><b>12.6</b> Complement system[edit]</a></li>
<li class="chapter" data-level="12.7" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#white-blood-cells"><i class="fa fa-check"></i><b>12.7</b> White blood cells</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html"><i class="fa fa-check"></i><b>13</b> Host Defenses: The Adaptive Immune System</a>
<ul>
<li class="chapter" data-level="13.1" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#lymphocytes"><i class="fa fa-check"></i><b>13.1</b> Lymphocytes</a></li>
<li class="chapter" data-level="13.2" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#antigen-presentation"><i class="fa fa-check"></i><b>13.2</b> Antigen presentation</a></li>
<li class="chapter" data-level="13.3" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#exogenous-antigens"><i class="fa fa-check"></i><b>13.3</b> Exogenous antigens</a></li>
<li class="chapter" data-level="13.4" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#endogenous-antigensedit"><i class="fa fa-check"></i><b>13.4</b> Endogenous antigens[edit]</a></li>
<li class="chapter" data-level="13.5" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#t-lymphocytesedit"><i class="fa fa-check"></i><b>13.5</b> T lymphocytes[edit]</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For SCI204 Microbiology at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="host-defensesthe-innate-immune-system" class="section level1" number="12">
<h1><span class="header-section-number">12</span> Host Defenses:The Innate Immune System</h1>
<p>The immune system is a network of biological processes that protects an organism from diseases. It detects and responds to a wide variety of pathogens, from viruses to parasitic worms, as well as cancer cells and objects such as wood splinters, distinguishing them from the organism’s own healthy tissue. Many species have two major subsystems of the immune system. The innate immune system provides a preconfigured response to broad groups of situations and stimuli. The adaptive immune system provides a tailored response to each stimulus by learning to recognize molecules it has previously encountered. Both use molecules and cells to perform their functions.</p>
<p>Nearly all organisms have some kind of immune system. Bacteria have a rudimentary immune system in the form of enzymes that protect against virus infections. Other basic immune mechanisms evolved in ancient plants and animals and remain in their modern descendants. These mechanisms include phagocytosis, antimicrobial peptides called defensins, and the complement system. Jawed vertebrates, including humans, have even more sophisticated defense mechanisms, including the ability to adapt to recognize pathogens more efficiently. Adaptive (or acquired) immunity creates an immunological memory leading to an enhanced response to subsequent encounters with that same pathogen. This process of acquired immunity is the basis of vaccination.</p>
<p>Dysfunction of the immune system can cause autoimmune diseases, inflammatory diseases and cancer. Immunodeficiency occurs when the immune system is less active than normal, resulting in recurring and life-threatening infections. In humans, immunodeficiency can be the result of a genetic disease such as severe combined immunodeficiency, acquired conditions such as HIV/AIDS, or the use of immunosuppressive medication. Autoimmunity results from a hyperactive immune system attacking normal tissues as if they were foreign organisms. Common autoimmune diseases include Hashimoto’s thyroiditis, rheumatoid arthritis, diabetes mellitus type 1, and systemic lupus erythematosus. Immunology covers the study of all aspects of the immune system.</p>
<div id="layered-defenseedit" class="section level2" number="12.1">
<h2><span class="header-section-number">12.1</span> Layered defense[edit]</h2>
<p>The immune system protects its host from infection with layered defenses of increasing specificity. Physical barriers prevent pathogens such as bacteria and viruses from entering the organism.[1] If a pathogen breaches these barriers, the innate immune system provides an immediate, but non-specific response. Innate immune systems are found in all animals.[2] If pathogens successfully evade the innate response, vertebrates possess a second layer of protection, the adaptive immune system, which is activated by the innate response.[3] Here, the immune system adapts its response during an infection to improve its recognition of the pathogen. This improved response is then retained after the pathogen has been eliminated, in the form of an immunological memory, and allows the adaptive immune system to mount faster and stronger attacks each time this pathogen is encountered.[4][5]</p>
<p>Both innate and adaptive immunity depend on the ability of the immune system to distinguish between self and non-self molecules. In immunology, self molecules are components of an organism’s body that can be distinguished from foreign substances by the immune system.[6] Conversely, non-self molecules are those recognized as foreign molecules. One class of non-self molecules are called antigens (originally named for being antibody generators) and are defined as substances that bind to specific immune receptors and elicit an immune response.[7]</p>
</div>
<div id="surface-barriers" class="section level2" number="12.2">
<h2><span class="header-section-number">12.2</span> Surface Barriers</h2>
<p>Several barriers protect organisms from infection, including mechanical, chemical, and biological barriers. The waxy cuticle of most leaves, the exoskeleton of insects, the shells and membranes of externally deposited eggs, and skin are examples of mechanical barriers that are the first line of defense against infection.[8] Organisms cannot be completely sealed from their environments, so systems act to protect body openings such as the lungs, intestines, and the genitourinary tract. In the lungs, coughing and sneezing mechanically eject pathogens and other irritants from the respiratory tract. The flushing action of tears and urine also mechanically expels pathogens, while mucus secreted by the respiratory and gastrointestinal tract serves to trap and entangle microorganisms.[9]</p>
<p>Chemical barriers also protect against infection. The skin and respiratory tract secrete antimicrobial peptides such as the β-defensins.[10] Enzymes such as lysozyme and phospholipase A2 in saliva, tears, and breast milk are also antibacterials.[11][12] Vaginal secretions serve as a chemical barrier following menarche, when they become slightly acidic, while semen contains defensins and zinc to kill pathogens.[13][14] In the stomach, gastric acid serves as a chemical defense against ingested pathogens.[15]</p>
<p>Within the genitourinary and gastrointestinal tracts, commensal flora serve as biological barriers by competing with pathogenic bacteria for food and space and, in some cases, changing the conditions in their environment, such as pH or available iron. As a result, the probability that pathogens will reach sufficient numbers to cause illness is reduced.[16]</p>
</div>
<div id="innate-immune-system" class="section level2" number="12.3">
<h2><span class="header-section-number">12.3</span> Innate immune system</h2>
<p>The innate immune system is one of the two main immunity strategies found in vertebrates (the other being the adaptive immune system). The innate immune system is an older evolutionary defense strategy, relatively speaking, and is the dominant immune system response found in plants, fungi, insects, and primitive multicellular organisms.[1]</p>
<p>The major functions of the vertebrate innate immune system include:</p>
<ul>
<li>Recruiting immune cells to sites of infection through the production of chemical factors, including specialized chemical mediators called cytokines</li>
<li>Activation of the complement cascade to identify bacteria, activate cells, and promote clearance of antibody complexes or dead cells</li>
<li>Identification and removal of foreign substances present in organs, tissues, blood and lymph, by specialized white blood cells</li>
<li>Activation of the adaptive immune system through a process known as antigen presentation</li>
<li>Acting as a physical and chemical barrier to infectious agents; via physical measures like skin or tree bark and chemical measures like clotting factors in blood or sap from a tree, which are released following a contusion or other injury that breaks through the first-line physical barrier (not to be confused with a second-line physical or chemical barrier, such as the blood-brain barrier, which protects the extremely vital and highly sensitive nervous system from pathogens that have already gained access to the host’s body).</li>
</ul>
</div>
<div id="anatomical-barriers" class="section level2" number="12.4">
<h2><span class="header-section-number">12.4</span> Anatomical barriers</h2>
<p>Anatomical barriers include physical, chemical and biological barriers. The epithelial surfaces form a physical barrier that is impermeable to most infectious agents, acting as the first line of defense against invading organisms.[2] Desquamation (shedding) of skin epithelium also helps remove bacteria and other infectious agents that have adhered to the epithelial surfaces. Lack of blood vessels, the inability of the epidermis to retain moisture, and the presence of sebaceous glands in the dermis, produces an environment unsuitable for the survival of microbes.[2] In the gastrointestinal and respiratory tract, movement due to peristalsis or cilia, respectively, helps remove infectious agents.[2] Also, mucus traps infectious agents.[2] The gut flora can prevent the colonization of pathogenic bacteria by secreting toxic substances or by competing with pathogenic bacteria for nutrients or attachment to cell surfaces.[2] The flushing action of tears and saliva helps prevent infection of the eyes and mouth.[2]</p>
</div>
<div id="inflammation" class="section level2" number="12.5">
<h2><span class="header-section-number">12.5</span> Inflammation</h2>
<p>Inflammation is one of the first responses of the immune system to infection or irritation. Inflammation is stimulated by chemical factors released by injured cells and serves to establish a physical barrier against the spread of infection, and to promote healing of any damaged tissue following the clearance of pathogens.[4]</p>
<p>The process of acute inflammation is initiated by cells already present in all tissues, mainly resident macrophages, dendritic cells, histiocytes, Kupffer cells, and mast cells. These cells present receptors contained on the surface or within the cell, named pattern recognition receptors (PRRs), which recognize molecules that are broadly shared by pathogens but distinguishable from host molecules, collectively referred to as pathogen-associated molecular patterns (PAMPs). At the onset of an infection, burn, or other injuries, these cells undergo activation (one of their PRRs recognizes a PAMP) and release inflammatory mediators responsible for the clinical signs of inflammation.</p>
<p>Chemical factors produced during inflammation (histamine, bradykinin, serotonin, leukotrienes, and prostaglandins) sensitize pain receptors, cause local vasodilation of the blood vessels, and attract phagocytes, especially neutrophils.[4] Neutrophils then trigger other parts of the immune system by releasing factors that summon additional leukocytes and lymphocytes. Cytokines produced by macrophages and other cells of the innate immune system mediate the inflammatory response. These cytokines include TNF, HMGB1, and IL-1.[5]</p>
<p>The inflammatory response is characterized by the following symptoms:</p>
<ul>
<li>redness of the skin, due to locally increased blood circulation;</li>
<li>heat, either increased local temperature, such as a warm feeling around a localized infection, or a systemic fever;</li>
<li>swelling of affected tissues, such as the upper throat during the common cold or joints affected by rheumatoid arthritis;</li>
<li>increased production of mucus, which can cause symptoms like a runny nose or a productive cough;</li>
<li>pain, either local pain, such as painful joints or a sore throat, or affecting the whole body, such as body aches; and</li>
<li>possible dysfunction of the organs or tissues involved.</li>
</ul>
</div>
<div id="complement-systemedit" class="section level2" number="12.6">
<h2><span class="header-section-number">12.6</span> Complement system[edit]</h2>
<p>The complement system is a biochemical cascade of the immune system that helps, or “complements”, the ability of antibodies to clear pathogens or mark them for destruction by other cells. The cascade is composed of many plasma proteins, synthesized in the liver, primarily by hepatocytes. The proteins work together to:</p>
<ul>
<li>trigger the recruitment of inflammatory cells</li>
<li>“tag” pathogens for destruction by other cells by opsonizing, or coating, the surface of the pathogen</li>
<li>form holes in the plasma membrane of the pathogen, resulting in cytolysis of the pathogen cell, causing the death of the pathogen</li>
<li>rid the body of neutralised antigen-antibody complexes.</li>
</ul>
<p>There are three different complement systems: Classical, alternative, Lectin</p>
<ul>
<li>Classical: starts when antibody binds to bacteria</li>
<li>Alternative: starts “spontaneously”</li>
<li>Lectin: starts when lectins bind to mannose on bacteria</li>
</ul>
<p>Elements of the complement cascade can be found in many non-mammalian species including plants, birds, fish, and some species of invertebrates.[6]</p>
</div>
<div id="white-blood-cells" class="section level2" number="12.7">
<h2><span class="header-section-number">12.7</span> White blood cells</h2>
<p>All white blood cells (WBCs) are known as leukocytes. Most leukocytes differ from other cells of the body in that they are not tightly associated with a particular organ or tissue; thus, their function is similar to that of independent, single-cell organisms. Most leukocytes are able to move freely and interact with and capture cellular debris, foreign particles, and invading microorganisms (although macrophages, mast cells, and dendritic cells are less mobile). Unlike many other cells in the body, most innate immune leukocytes cannot divide or reproduce on their own, but are the products of multipotent hematopoietic stem cells present in the bone marrow.[7]</p>
<p>The innate leukocytes include: natural killer cells, mast cells, eosinophils, basophils; and the phagocytic cells include macrophages, neutrophils, and dendritic cells, and function within the immune system by identifying and eliminating pathogens that might cause infection.[1]</p>
<p>Mast cells[edit]
Main article: Mast cell
Mast cells are a type of innate immune cell that resides in connective tissue and in mucous membranes. They are intimately associated with wound healing and defense against pathogens, but are also often associated with allergy and anaphylaxis (serious allergic reactions that can cause death).[4] When activated, mast cells rapidly release characteristic granules, rich in histamine and heparin, along with various hormonal mediators and chemokines, or chemotactic cytokines into the environment. Histamine dilates blood vessels, causing the characteristic signs of inflammation, and recruits neutrophils and macrophages.[4]</p>
<p>Phagocytes[edit]
Main article: Phagocyte
The word ‘phagocyte’ literally means ‘eating cell’. These are immune cells that engulf, or ‘phagocytose’, pathogens or particles. To engulf a particle or pathogen, a phagocyte extends portions of its plasma membrane, wrapping the membrane around the particle until it is enveloped (i.e., the particle is now inside the cell). Once inside the cell, the invading pathogen is contained inside a phagosome, which merges with a lysosome.[1] The lysosome contains enzymes and acids that kill and digest the particle or organism. In general, phagocytes patrol the body searching for pathogens, but are also able to react to a group of highly specialized molecular signals produced by other cells, called cytokines. The phagocytic cells of the immune system include macrophages, neutrophils, and dendritic cells.</p>
<p>Phagocytosis of the hosts’ own cells is common as part of regular tissue development and maintenance. When host cells die, either by programmed cell death (also called apoptosis) or by cell injury due to a bacterial or viral infection, phagocytic cells are responsible for their removal from the affected site.[7] By helping to remove dead cells preceding growth and development of new healthy cells, phagocytosis is an important part of the healing process following tissue injury.</p>
<p>Macrophages[edit]
Main article: Macrophages
Macrophages, from the Greek, meaning “large eaters”, are large phagocytic leukocytes, which are able to move outside of the vascular system by migrating through the walls of capillary vessels and entering the areas between cells in pursuit of invading pathogens. In tissues, organ-specific macrophages are differentiated from phagocytic cells present in the blood called monocytes. Macrophages are the most efficient phagocytes and can phagocytose substantial numbers of bacteria or other cells or microbes.[1] The binding of bacterial molecules to receptors on the surface of a macrophage triggers it to engulf and destroy the bacteria through the generation of a “respiratory burst”, causing the release of reactive oxygen species. Pathogens also stimulate the macrophage to produce chemokines, which summon other cells to the site of infection.[1]</p>
<p>Neutrophils[edit]
Main article: Neutrophils</p>
<p>Neutrophils, along with two other cell types (eosinophils and basophils; see below), are known as granulocytes due to the presence of granules in their cytoplasm, or as polymorphonuclear cells (PMNs) due to their distinctive lobed nuclei. Neutrophil granules contain a variety of toxic substances that kill or inhibit growth of bacteria and fungi. Similar to macrophages, neutrophils attack pathogens by activating a respiratory burst. The main products of the neutrophil respiratory burst are strong oxidizing agents including hydrogen peroxide, free oxygen radicals and hypochlorite. Neutrophils are the most abundant type of phagocyte, normally representing 50-60% of the total circulating leukocytes, and are usually the first cells to arrive at the site of an infection.[4] The bone marrow of a normal healthy adult produces more than 100 billion neutrophils per day, and more than 10 times that many per day during acute inflammation.[4]</p>
<p>Dendritic cells[edit]
Main article: Dendritic cell
Dendritic cells (DCs) are phagocytic cells present in tissues that are in contact with the external environment, mainly the skin (where they are often called Langerhans cells), and the inner mucosal lining of the nose, lungs, stomach, and intestines.[7] They are named for their resemblance to neuronal dendrites, but dendritic cells are not connected to the nervous system. Dendritic cells are very important in the process of antigen presentation, and serve as a link between the innate and adaptive immune systems.</p>
<p>Basophils and eosinophils[edit]
Main articles: Basophil granulocyte and Eosinophil granulocyte
Basophils and eosinophils are cells related to the neutrophil (see above). When activated by a pathogen encounter, histamine-releasing basophils are important in the defense against parasites and play a role in allergic reactions, such as asthma.[1] Upon activation, eosinophils secrete a range of highly toxic proteins and free radicals that are highly effective in killing parasites, but may also damage tissue during an allergic reaction. Activation and release of toxins by eosinophils are, therefore, tightly regulated to prevent any inappropriate tissue destruction.[4]</p>
<p>Natural killer cells[edit]
Main article: Natural killer cell
Natural killer cells (NK cells) are a component of the innate immune system that does not directly attack invading microbes. Rather, NK cells destroy compromised host cells, such as tumor cells or virus-infected cells, recognizing such cells by a condition known as “missing self.” This term describes cells with abnormally low levels of a cell-surface marker called MHC I (major histocompatibility complex) - a situation that can arise in viral infections of host cells.[8] They were named “natural killer” because of the initial notion that they do not require activation in order to kill cells that are “missing self.” For many years, it was unclear how NK cell recognize tumor cells and infected cells. It is now known that the MHC makeup on the surface of those cells is altered and the NK cells become activated through recognition of “missing self”. Normal body cells are not recognized and attacked by NK cells because they express intact self MHC antigens. Those MHC antigens are recognized by killer cell immunoglobulin receptors (KIR) that, in essence, put the brakes on NK cells. The NK-92 cell line does not express KIR and is developed for tumor therapy.[9][10][11][12]</p>
<p>γδ T cells[edit]
Main article: gamma/delta T cells
Like other ‘unconventional’ T cell subsets bearing invariant T cell receptors (TCRs), such as CD1d-restricted Natural Killer T cells, γδ T cells exhibit characteristics that place them at the border between innate and adaptive immunity. On one hand, γδ T cells may be considered a component of adaptive immunity in that they rearrange TCR genes to produce junctional diversity and develop a memory phenotype. However, the various subsets may also be considered part of the innate immune system where a restricted TCR or NK receptors may be used as a pattern recognition receptor. For example, according to this paradigm, large numbers of Vγ9/Vδ2 T cells respond within hours to common molecules produced by microbes, and highly restricted intraepithelial Vδ1 T cells will respond to stressed epithelial cells.</p>
<p>Other vertebrate mechanisms[edit]
The coagulation system overlaps with the immune system. Some products of the coagulation system can contribute to the non-specific defenses by their ability to increase vascular permeability and act as chemotactic agents for phagocytic cells. In addition, some of the products of the coagulation system are directly antimicrobial. For example, beta-lysine, a protein produced by platelets during coagulation, can cause lysis of many Gram-positive bacteria by acting as a cationic detergent.[2] Many acute-phase proteins of inflammation are involved in the coagulation system.</p>
<p>Also increased levels of lactoferrin and transferrin inhibit bacterial growth by binding iron, an essential nutrient for bacteria.[2]</p>
<p>Neural regulation[edit]
The innate immune response to infectious and sterile injury is modulated by neural circuits that control cytokine production period. The inflammatory reflex is a prototypical neural circuit that controls cytokine production in the spleen.[13] Action potentials transmitted via the vagus nerve to spleen mediate the release of acetylcholine, the neurotransmitter that inhibits cytokine release by interacting with alpha7 nicotinic acetylcholine receptors (CHRNA7) expressed on cytokine-producing cells.[14] The motor arc of the inflammatory reflex is termed the cholinergic anti-inflammatory pathway.</p>
<p>Pathogen-specificity[edit]
The parts of the innate immune system have different specificity for different pathogens.</p>
<p>Immune evasion[edit]
Cells of the innate immune system prevent free growth of microorganisms within the body, but many pathogens have evolved mechanisms to evade it.[19][20]</p>
<p>One strategy is intracellular replication, as practised by Mycobacterium tuberculosis, or wearing a protective capsule, which prevents lysis by complement and by phagocytes, as in Salmonella.[21] Bacteroides species are normally mutualistic bacteria, making up a substantial portion of the mammalian gastrointestinal flora.[22] Some species like B. fragilis for example are opportunistic pathogens, causing infections of the peritoneal cavity inhibit phagocytosis by affecting the phagocytes receptors used to engulf bacteria. They may also mimick host cells so the immune system does not recognize them as foreign. Staphylococcus aureus inhibits the ability of the phagocyte to respond to chemokine signals. M. tuberculosis, Streptococcus pyogenes, and Bacillus anthracis utilize mechanisms that directly kill the phagocyte.[citation needed]</p>
<p>Bacteria and fungi may form complex biofilms, protecting from immune cells and proteins; biofilms are present in the chronic Pseudomonas aeruginosa and Burkholderia cenocepacia infections characteristic of cystic fibrosis.[23]</p>
<p>Viruses[edit]
Type I interferons (IFN), secreted mainly by dendritic cells,[24] play a central role in antiviral host defense and a cell’s antiviral state.[25] Viral components are recognized by different receptors: Toll-like receptors are located in the endosomal membrane and recognize double-stranded RNA (dsRNA), MDA5 and RIG-I receptors are located in the cytoplasm and recognize long dsRNA and phosphate-containing dsRNA respectively.[26] When the cytoplasmic receptors MDA5 and RIG-I recognize a virus the conformation between the caspase-recruitment domain (CARD) and the CARD-containing adaptor MAVS changes. In parallel, when toll-like receptors in the endocytic compartments recognize a virus the activation of the adaptor protein TRIF is induced. Both pathways converge in the recruitment and activation of the IKKε/TBK-1 complex, inducing dimerization of transcription factors IRF3 and IRF7, which are translocated in the nucleus, where they induce IFN production with the presence of a particular transcription factor and activate transcription factor 2. IFN is secreted through secretory vesicles, where it can activate receptors on both the same cell it was released from (autocrine) or nearby cells (paracrine). This induces hundreds of interferon-stimulated genes to be expressed. This leads to antiviral protein production, such as protein kinase R, which inhibits viral protein synthesis, or the 2′,5′-oligoadenylate synthetase family, which degrades viral RNA.[25]</p>
<p>Some viruses evade this by producing molecules which interfere with IFN production. For example, the Influenza A virus produces NS1 protein, which can bind to host and viral RNA, interact with immune signaling proteins or block their activation by ubiquitination, thus inhibiting type I IFN production.[27] Influenza A also blocks protein kinase R activation and establishment of the antiviral state.[28] The dengue virus also inhibits type I IFN production by blocking IRF-3 phosophorylation using NS2B3 protease complex.[29]</p>
<p>In other species[edit]
Prokaryotes[edit]
Bacteria (and perhaps other prokaryotic organisms), utilize a unique defense mechanism, called the restriction modification system to protect themselves from pathogens, such as bacteriophages. In this system, bacteria produce enzymes, called restriction endonucleases, that attack and destroy specific regions of the viral DNA of invading bacteriophages. Methylation of the host’s own DNA marks it as “self” and prevents it from being attacked by endonucleases.[30] Restriction endonucleases and the restriction modification system exist exclusively in prokaryotes.[31]</p>
<p>Invertebrates[edit]
Invertebrates do not possess lymphocytes or an antibody-based humoral immune system, and it is likely that a multicomponent, adaptive immune system arose with the first vertebrates.[32] Nevertheless, invertebrates possess mechanisms that appear to be precursors of these aspects of vertebrate immunity. Pattern recognition receptors are proteins used by nearly all organisms to identify molecules associated with microbial pathogens. Toll-like receptors are a major class of pattern recognition receptor, that exists in all coelomates (animals with a body-cavity), including humans.[33] The complement system, as discussed above, is a biochemical cascade of the immune system that helps clear pathogens from an organism, and exists in most forms of life. Some invertebrates, including various insects, crabs, and worms utilize a modified form of the complement response known as the prophenoloxidase (proPO) system.[32]</p>
<p>Antimicrobial peptides are an evolutionarily conserved component of the innate immune response found among all classes of life and represent the main form of invertebrate systemic immunity. Several species of insect produce antimicrobial peptides known as defensins and cecropins.</p>
<p>Proteolytic cascades[edit]
In invertebrates, pattern recognition receptors (PRRs) trigger proteolytic cascades that degrade proteins and control many of the mechanisms of the innate immune system of invertebrates—including hemolymph coagulation and melanization. Proteolytic cascades are important components of the invertebrate immune system because they are turned on more rapidly than other innate immune reactions because they do not rely on gene changes. Proteolytic cascades have been found to function the same in both vertebrate and invertebrates, even though different proteins are used throughout the cascades.[34]</p>
<p>Clotting mechanisms[edit]
In the hemolymph, which makes up the fluid in the circulatory system of arthropods, a gel-like fluid surrounds pathogen invaders, similar to the way blood does in other animals. There are various different proteins and mechanisms that are involved in invertebrate clotting. In crustaceans, transglutaminase from blood cells and mobile plasma proteins make up the clotting system, where the transglutaminase polymerizes 210 kDa subunits of a plasma-clotting protein. On the other hand, in the horseshoe crab species clotting system, components of proteolytic cascades are stored as inactive forms in granules of hemocytes, which are released when foreign molecules, like lipopolysaccharides enter.[34]</p>
<p>Plants[edit]
Main article: Plant disease resistance § Immune system
Members of every class of pathogen that infect humans also infect plants. Although the exact pathogenic species vary with the infected species, bacteria, fungi, viruses, nematodes, and insects can all cause plant disease. As with animals, plants attacked by insects or other pathogens use a set of complex metabolic responses which lead to the formation of defensive chemical compounds that fight infection or make the plant less attractive to insects and other herbivores.[35] (see: plant defense against herbivory).</p>
<p>Like invertebrates, plants neither generate antibody or T-cell responses nor possess mobile cells that detect and attack pathogens. In addition, in case of infection, parts of some plants are treated as disposable and replaceable, in ways that very few animals are able to do. Walling off or discarding a part of a plant helps stop spread of an infection.[35]</p>
<p>Most plant immune responses involve systemic chemical signals sent throughout a plant. Plants use pattern-recognition receptors to recognize conserved microbial signatures. This recognition triggers an immune response. The first plant receptors of conserved microbial signatures were identified in rice (XA21, 1995)[36][37] and in Arabidopsis (FLS2, 2000).[38] Plants also carry immune receptors that recognize highly variable pathogen effectors. These include the NBS-LRR class of proteins. When a part of a plant becomes infected with a microbial or viral pathogen, in case of an incompatible interaction triggered by specific elicitors, the plant produces a localized hypersensitive response (HR), in which cells at the site of infection undergo rapid programmed cell death to prevent the spread of the disease to other parts of the plant. HR has some similarities to animal pyroptosis, such as a requirement of caspase-1-like proteolytic activity of VPEγ, a cysteine protease that regulates cell disassembly during cell death.[39]</p>
<p>“Resistance” (R) proteins, encoded by R genes, are widely present in plants and detect pathogens. These proteins contain domains similar to the NOD Like Receptors and Toll-like receptors utilized in animal innate immunity. Systemic acquired resistance (SAR) is a type of defensive response that renders the entire plant resistant to a broad spectrum of infectious agents.[40] SAR involves the production of chemical messengers, such as salicylic acid or jasmonic acid. Some of these travel through the plant and signal other cells to produce defensive compounds to protect uninfected parts, e.g., leaves.[41] Salicylic acid itself, although indispensable for expression of SAR, is not the translocated signal responsible for the systemic response. Recent evidence indicates a role for jasmonates in transmission of the signal to distal portions of the plant. RNA silencing mechanisms are also important in the plant systemic response, as they can block virus replication.[42] The jasmonic acid response, is stimulated in leaves damaged by insects, and involves the production of methyl jasmonate.[35]</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="drugs-microbes-and-the-host.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="host-defenses-the-adaptive-immune-system.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/microbiology-text/edit/master/host_defenses.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
