<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>13 Host Defenses: The Adaptive Immune System | Text For SCI204 Microbiology at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Microbiology." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="13 Host Defenses: The Adaptive Immune System | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/microbiology_text_cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Microbiology." />
  <meta name="github-repo" content="nikolaussucher/microbiology-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="13 Host Defenses: The Adaptive Immune System | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Microbiology." />
  <meta name="twitter:image" content="/microbiology_text_cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="host-defensesthe-innate-immune-system.html"/>

<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />




<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">SCI204 Microbiology at RCC</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html"><i class="fa fa-check"></i><b>1</b> The Main Themes Of Microbiology</a></li>
<li class="chapter" data-level="2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html"><i class="fa fa-check"></i><b>2</b> Methods Of Studying Microorganisms</a>
<ul>
<li class="chapter" data-level="2.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microscopy"><i class="fa fa-check"></i><b>2.1</b> Microscopy</a></li>
<li class="chapter" data-level="2.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#light-microscopy"><i class="fa fa-check"></i><b>2.2</b> Light Microscopy</a></li>
<li class="chapter" data-level="2.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#the-optical-light-microscope"><i class="fa fa-check"></i><b>2.3</b> The Optical (Light) Microscope</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#simple-microscopeedit"><i class="fa fa-check"></i><b>2.3.1</b> Simple microscope[edit]</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#compound-microscope"><i class="fa fa-check"></i><b>2.3.2</b> Compound microscope</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#main-components-of-the-light-microscope"><i class="fa fa-check"></i><b>2.3.3</b> Main Components of The Light Microscope</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microscope-slide"><i class="fa fa-check"></i><b>2.4</b> Microscope Slide</a></li>
<li class="chapter" data-level="2.5" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#electron-microscopy"><i class="fa fa-check"></i><b>2.5</b> Electron microscopy</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#ransmission-electron-microscope-tem"><i class="fa fa-check"></i><b>2.5.1</b> ransmission electron microscope (TEM)</a></li>
<li class="chapter" data-level="2.5.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#scanning-electron-microscope-sem"><i class="fa fa-check"></i><b>2.5.2</b> Scanning electron microscope (SEM)</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#microbiological-culture"><i class="fa fa-check"></i><b>2.6</b> Microbiological culture</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#growth-medium"><i class="fa fa-check"></i><b>2.6.1</b> Growth medium</a></li>
<li class="chapter" data-level="2.6.2" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#selective-media"><i class="fa fa-check"></i><b>2.6.2</b> Selective media</a></li>
<li class="chapter" data-level="2.6.3" data-path="methods-of-studying-microorganisms.html"><a href="methods-of-studying-microorganisms.html#agar-plate"><i class="fa fa-check"></i><b>2.6.3</b> Agar Plate</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html"><i class="fa fa-check"></i><b>3</b> Prokaryotic Microorganisms</a>
<ul>
<li class="chapter" data-level="3.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#bacteria"><i class="fa fa-check"></i><b>3.1</b> Bacteria</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#intracellular-structure"><i class="fa fa-check"></i><b>3.1.1</b> Intracellular Structure</a></li>
<li class="chapter" data-level="3.1.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#extracellular-structures"><i class="fa fa-check"></i><b>3.1.2</b> Extracellular Structures</a></li>
<li class="chapter" data-level="3.1.3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#endospores"><i class="fa fa-check"></i><b>3.1.3</b> Endospores</a></li>
<li class="chapter" data-level="3.1.4" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#metabolism"><i class="fa fa-check"></i><b>3.1.4</b> Metabolism</a></li>
<li class="chapter" data-level="3.1.5" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#growth-and-reproduction"><i class="fa fa-check"></i><b>3.1.5</b> Growth and reproduction</a></li>
<li class="chapter" data-level="3.1.6" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#genetics"><i class="fa fa-check"></i><b>3.1.6</b> Genetics</a></li>
<li class="chapter" data-level="3.1.7" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#movement"><i class="fa fa-check"></i><b>3.1.7</b> Movement</a></li>
<li class="chapter" data-level="3.1.8" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#communication"><i class="fa fa-check"></i><b>3.1.8</b> Communication</a></li>
<li class="chapter" data-level="3.1.9" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#classification-and-identification"><i class="fa fa-check"></i><b>3.1.9</b> Classification and identification</a></li>
<li class="chapter" data-level="3.1.10" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#interactions-with-other-organisms"><i class="fa fa-check"></i><b>3.1.10</b> Interactions With Other Organisms</a></li>
<li class="chapter" data-level="3.1.11" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#bacteria-as-pathogens"><i class="fa fa-check"></i><b>3.1.11</b> Bacteria as Pathogens</a></li>
<li class="chapter" data-level="3.1.12" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#significance-in-technology-and-industry"><i class="fa fa-check"></i><b>3.1.12</b> Significance in Technology And Industry</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#archaea"><i class="fa fa-check"></i><b>3.2</b> Archaea</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#origin-and-evolution"><i class="fa fa-check"></i><b>3.2.1</b> Origin and evolution</a></li>
<li class="chapter" data-level="3.2.2" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#relationship-to-bacteria"><i class="fa fa-check"></i><b>3.2.2</b> Relationship to Bacteria</a></li>
<li class="chapter" data-level="3.2.3" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#relation-to-eukaryotes"><i class="fa fa-check"></i><b>3.2.3</b> Relation to Eukaryotes</a></li>
<li class="chapter" data-level="3.2.4" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#morphology"><i class="fa fa-check"></i><b>3.2.4</b> Morphology</a></li>
<li class="chapter" data-level="3.2.5" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#structure-composition-development-and-operation"><i class="fa fa-check"></i><b>3.2.5</b> Structure, Composition Development, And Operation</a></li>
<li class="chapter" data-level="3.2.6" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#cell-wall-and-flagella"><i class="fa fa-check"></i><b>3.2.6</b> Cell Wall And Flagella</a></li>
<li class="chapter" data-level="3.2.7" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#membranes"><i class="fa fa-check"></i><b>3.2.7</b> Membranes</a></li>
<li class="chapter" data-level="3.2.8" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#metabolism-1"><i class="fa fa-check"></i><b>3.2.8</b> Metabolism</a></li>
<li class="chapter" data-level="3.2.9" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#genetics-1"><i class="fa fa-check"></i><b>3.2.9</b> Genetics</a></li>
<li class="chapter" data-level="3.2.10" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#ecology"><i class="fa fa-check"></i><b>3.2.10</b> Ecology</a></li>
<li class="chapter" data-level="3.2.11" data-path="prokaryotic-microorganisms.html"><a href="prokaryotic-microorganisms.html#significance-in-technology-and-industry-1"><i class="fa fa-check"></i><b>3.2.11</b> Significance in Technology And Industry</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html"><i class="fa fa-check"></i><b>4</b> Eukaryotic Microorganisms</a>
<ul>
<li class="chapter" data-level="4.1" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#protists"><i class="fa fa-check"></i><b>4.1</b> Protists</a></li>
<li class="chapter" data-level="4.2" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#protozoa"><i class="fa fa-check"></i><b>4.2</b> Protozoa</a></li>
<li class="chapter" data-level="4.3" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#flagellates"><i class="fa fa-check"></i><b>4.3</b> Flagellates</a></li>
<li class="chapter" data-level="4.4" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#ciliates"><i class="fa fa-check"></i><b>4.4</b> Ciliates</a></li>
<li class="chapter" data-level="4.5" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#amoebozoa"><i class="fa fa-check"></i><b>4.5</b> Amoebozoa</a></li>
<li class="chapter" data-level="4.6" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#algae"><i class="fa fa-check"></i><b>4.6</b> Algae</a></li>
<li class="chapter" data-level="4.7" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#slime-molds"><i class="fa fa-check"></i><b>4.7</b> Slime Molds</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="eukaryotic-microorganisms.html"><a href="eukaryotic-microorganisms.html#reproduction-of-protists"><i class="fa fa-check"></i><b>4.7.1</b> Reproduction of Protists</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html"><i class="fa fa-check"></i><b>5</b> An Introduction To Viruses</a>
<ul>
<li class="chapter" data-level="5.0.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#role-in-human-disease"><i class="fa fa-check"></i><b>5.0.1</b> Role in human disease</a></li>
<li class="chapter" data-level="5.0.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#animal-viruses"><i class="fa fa-check"></i><b>5.0.2</b> Animal viruses</a></li>
<li class="chapter" data-level="5.0.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#plant-viruses"><i class="fa fa-check"></i><b>5.0.3</b> Plant viruses</a></li>
<li class="chapter" data-level="5.0.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#bacterial-viruses"><i class="fa fa-check"></i><b>5.0.4</b> Bacterial viruses</a></li>
<li class="chapter" data-level="5.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#classification"><i class="fa fa-check"></i><b>5.1</b> Classification</a></li>
<li class="chapter" data-level="5.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#baltimore-classification"><i class="fa fa-check"></i><b>5.2</b> Baltimore classification</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html"><i class="fa fa-check"></i><b>6</b> Microbial Nutrition, Ecology And Growth</a>
<ul>
<li class="chapter" data-level="6.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#thermodynamics-of-living-organisms"><i class="fa fa-check"></i><b>6.1</b> Thermodynamics of Living Organisms</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html"><i class="fa fa-check"></i><b>7</b> Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="7.0.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#catabolism"><i class="fa fa-check"></i><b>7.0.1</b> Catabolism</a></li>
<li class="chapter" data-level="7.0.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#digestion"><i class="fa fa-check"></i><b>7.0.2</b> Digestion</a></li>
<li class="chapter" data-level="7.0.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-organic-compounds"><i class="fa fa-check"></i><b>7.0.3</b> Energy From Organic Compounds</a></li>
<li class="chapter" data-level="7.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-transformations"><i class="fa fa-check"></i><b>7.1</b> Energy Transformations</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-phosphorylation"><i class="fa fa-check"></i><b>7.1.1</b> Oxidative Phosphorylation</a></li>
<li class="chapter" data-level="7.1.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-inorganic-compounds"><i class="fa fa-check"></i><b>7.1.2</b> Energy From Inorganic Compounds</a></li>
<li class="chapter" data-level="7.1.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#energy-from-light"><i class="fa fa-check"></i><b>7.1.3</b> Energy From Light</a></li>
<li class="chapter" data-level="7.1.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#anabolism"><i class="fa fa-check"></i><b>7.1.4</b> Anabolism</a></li>
<li class="chapter" data-level="7.1.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbon-fixation"><i class="fa fa-check"></i><b>7.1.5</b> Carbon Fixation</a></li>
<li class="chapter" data-level="7.1.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbohydrates-and-glycans"><i class="fa fa-check"></i><b>7.1.6</b> Carbohydrates And Glycans</a></li>
<li class="chapter" data-level="7.1.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fatty-acids-isoprenoids-and-sterol"><i class="fa fa-check"></i><b>7.1.7</b> Fatty Acids, Isoprenoids and Sterol</a></li>
<li class="chapter" data-level="7.1.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#proteins"><i class="fa fa-check"></i><b>7.1.8</b> Proteins</a></li>
<li class="chapter" data-level="7.1.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#nucleotide-synthesis-and-salvage"><i class="fa fa-check"></i><b>7.1.9</b> Nucleotide Synthesis And Salvage</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#heterotrophic-microbial-metabolism"><i class="fa fa-check"></i><b>7.2</b> Heterotrophic microbial metabolism</a></li>
<li class="chapter" data-level="7.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fermentation"><i class="fa fa-check"></i><b>7.3</b> Fermentation</a></li>
<li class="chapter" data-level="7.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#chemolithotrophy"><i class="fa fa-check"></i><b>7.4</b> Chemolithotrophy</a></li>
<li class="chapter" data-level="7.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#phototrophy"><i class="fa fa-check"></i><b>7.5</b> Phototrophy</a></li>
<li class="chapter" data-level="7.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#enzymes"><i class="fa fa-check"></i><b>7.6</b> Enzymes</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#allosteric-modulation"><i class="fa fa-check"></i><b>7.6.1</b> Allosteric Modulation</a></li>
<li class="chapter" data-level="7.6.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cofactors-of-enzymes-and-coenzymes"><i class="fa fa-check"></i><b>7.6.2</b> Cofactors of Enzymes and Coenzymes</a></li>
<li class="chapter" data-level="7.6.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#inhibition-of-enzymes"><i class="fa fa-check"></i><b>7.6.3</b> Inhibition of Enzymes</a></li>
<li class="chapter" data-level="7.6.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#factors-affecting-enzyme-activity"><i class="fa fa-check"></i><b>7.6.4</b> Factors Affecting Enzyme Activity</a></li>
<li class="chapter" data-level="7.6.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#control-of-activity"><i class="fa fa-check"></i><b>7.6.5</b> Control of activity</a></li>
<li class="chapter" data-level="7.6.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#post-translational-modification"><i class="fa fa-check"></i><b>7.6.6</b> Post-translational Modification</a></li>
<li class="chapter" data-level="7.6.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#quantity"><i class="fa fa-check"></i><b>7.6.7</b> Quantity</a></li>
<li class="chapter" data-level="7.6.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#subcellular-distribution"><i class="fa fa-check"></i><b>7.6.8</b> Subcellular Distribution</a></li>
<li class="chapter" data-level="7.6.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#organ-specialization"><i class="fa fa-check"></i><b>7.6.9</b> Organ Specialization</a></li>
<li class="chapter" data-level="7.6.10" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#coenzymes"><i class="fa fa-check"></i><b>7.6.10</b> Coenzymes</a></li>
<li class="chapter" data-level="7.6.11" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#mineral-cofactors"><i class="fa fa-check"></i><b>7.6.11</b> Mineral Cofactors</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthesis-and-cellular-respiration"><i class="fa fa-check"></i><b>7.7</b> Photosynthesis And Cellular Respiration</a></li>
<li class="chapter" data-level="7.8" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#the-electron-transport-chain-and-chemiosmosis"><i class="fa fa-check"></i><b>7.8</b> The Electron Transport Chain And Chemiosmosis</a></li>
<li class="chapter" data-level="7.9" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthesis"><i class="fa fa-check"></i><b>7.9</b> Photosynthesis</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#photosynthetic-membranes-and-organelles"><i class="fa fa-check"></i><b>7.9.1</b> Photosynthetic Membranes And Organelles</a></li>
<li class="chapter" data-level="7.9.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#z-scheme"><i class="fa fa-check"></i><b>7.9.2</b> Z Scheme</a></li>
<li class="chapter" data-level="7.9.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#water-photolysis"><i class="fa fa-check"></i><b>7.9.3</b> Water Photolysis</a></li>
<li class="chapter" data-level="7.9.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#calvin-cycle"><i class="fa fa-check"></i><b>7.9.4</b> Calvin Cycle</a></li>
<li class="chapter" data-level="7.9.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#carbon-dioxide-levels-and-photorespiration"><i class="fa fa-check"></i><b>7.9.5</b> Carbon Dioxide Levels And Photorespiration</a></li>
<li class="chapter" data-level="7.9.6" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#c4-carbon-fixation"><i class="fa fa-check"></i><b>7.9.6</b> C4 Carbon Fixation</a></li>
<li class="chapter" data-level="7.9.7" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cam-photosynthesis"><i class="fa fa-check"></i><b>7.9.7</b> CAM Photosynthesis</a></li>
</ul></li>
<li class="chapter" data-level="7.10" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#symbiosis-and-the-origin-of-chloroplasts"><i class="fa fa-check"></i><b>7.10</b> Symbiosis And The Origin of Chloroplasts</a></li>
<li class="chapter" data-level="7.11" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cyanobacteria-and-the-evolution-of-photosynthesis"><i class="fa fa-check"></i><b>7.11</b> Cyanobacteria And The Evolution of Photosynthesis</a></li>
<li class="chapter" data-level="7.12" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#cellular-respiration"><i class="fa fa-check"></i><b>7.12</b> Cellular Respiration</a>
<ul>
<li class="chapter" data-level="7.12.1" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#aerobic-respiration"><i class="fa fa-check"></i><b>7.12.1</b> Aerobic Respiration</a></li>
<li class="chapter" data-level="7.12.2" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#glycolysis"><i class="fa fa-check"></i><b>7.12.2</b> Glycolysis</a></li>
<li class="chapter" data-level="7.12.3" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-decarboxylation-of-pyruvate"><i class="fa fa-check"></i><b>7.12.3</b> Oxidative Decarboxylation of Pyruvate</a></li>
<li class="chapter" data-level="7.12.4" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#the-citric-acid-cycle"><i class="fa fa-check"></i><b>7.12.4</b> The Citric Acid Cycle</a></li>
<li class="chapter" data-level="7.12.5" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#oxidative-phosphorylation-1"><i class="fa fa-check"></i><b>7.12.5</b> Oxidative Phosphorylation</a></li>
</ul></li>
<li class="chapter" data-level="7.13" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#fermentation-1"><i class="fa fa-check"></i><b>7.13</b> Fermentation</a></li>
<li class="chapter" data-level="7.14" data-path="microbial-metabolism.html"><a href="microbial-metabolism.html#anaerobic-respiration"><i class="fa fa-check"></i><b>7.14</b> Anaerobic Respiration</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="microbial-genetics.html"><a href="microbial-genetics.html"><i class="fa fa-check"></i><b>8</b> Microbial Genetics</a></li>
<li class="chapter" data-level="9" data-path="genetic-engineering.html"><a href="genetic-engineering.html"><i class="fa fa-check"></i><b>9</b> Genetic Engineering</a>
<ul>
<li class="chapter" data-level="9.1" data-path="genetic-engineering.html"><a href="genetic-engineering.html#creatin-a-genetically-modified-organism-gmo"><i class="fa fa-check"></i><b>9.1</b> Creatin A Genetically Modified Organism (GMO)</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html"><i class="fa fa-check"></i><b>10</b> Physical And Chemical Agents for Microbial Control</a>
<ul>
<li class="chapter" data-level="10.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#disinfectant"><i class="fa fa-check"></i><b>10.1</b> Disinfectant</a></li>
<li class="chapter" data-level="10.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antiseptic"><i class="fa fa-check"></i><b>10.2</b> Antiseptic</a></li>
<li class="chapter" data-level="10.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#sterilization"><i class="fa fa-check"></i><b>10.3</b> Sterilization</a></li>
<li class="chapter" data-level="10.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#pasteurization"><i class="fa fa-check"></i><b>10.4</b> Pasteurization</a></li>
<li class="chapter" data-level="10.5" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antibiotics"><i class="fa fa-check"></i><b>10.5</b> Antibiotics</a></li>
<li class="chapter" data-level="10.6" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#classes-of-antibiotics"><i class="fa fa-check"></i><b>10.6</b> Classes of Antibiotics</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html"><i class="fa fa-check"></i><b>11</b> Drugs, Microbes, And The Host</a></li>
<li class="chapter" data-level="12" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html"><i class="fa fa-check"></i><b>12</b> Host Defenses:The Innate Immune System</a>
<ul>
<li class="chapter" data-level="12.1" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#layered-defenseedit"><i class="fa fa-check"></i><b>12.1</b> Layered defense[edit]</a></li>
<li class="chapter" data-level="12.2" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#surface-barriers"><i class="fa fa-check"></i><b>12.2</b> Surface Barriers</a></li>
<li class="chapter" data-level="12.3" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#innate-immune-system"><i class="fa fa-check"></i><b>12.3</b> Innate immune system</a></li>
<li class="chapter" data-level="12.4" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#anatomical-barriers"><i class="fa fa-check"></i><b>12.4</b> Anatomical barriers</a></li>
<li class="chapter" data-level="12.5" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#inflammation"><i class="fa fa-check"></i><b>12.5</b> Inflammation</a></li>
<li class="chapter" data-level="12.6" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#complement-systemedit"><i class="fa fa-check"></i><b>12.6</b> Complement system[edit]</a></li>
<li class="chapter" data-level="12.7" data-path="host-defensesthe-innate-immune-system.html"><a href="host-defensesthe-innate-immune-system.html#white-blood-cells"><i class="fa fa-check"></i><b>12.7</b> White blood cells</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html"><i class="fa fa-check"></i><b>13</b> Host Defenses: The Adaptive Immune System</a>
<ul>
<li class="chapter" data-level="13.1" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#lymphocytes"><i class="fa fa-check"></i><b>13.1</b> Lymphocytes</a></li>
<li class="chapter" data-level="13.2" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#antigen-presentation"><i class="fa fa-check"></i><b>13.2</b> Antigen presentation</a></li>
<li class="chapter" data-level="13.3" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#exogenous-antigens"><i class="fa fa-check"></i><b>13.3</b> Exogenous antigens</a></li>
<li class="chapter" data-level="13.4" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#endogenous-antigensedit"><i class="fa fa-check"></i><b>13.4</b> Endogenous antigens[edit]</a></li>
<li class="chapter" data-level="13.5" data-path="host-defenses-the-adaptive-immune-system.html"><a href="host-defenses-the-adaptive-immune-system.html#t-lymphocytesedit"><i class="fa fa-check"></i><b>13.5</b> T lymphocytes[edit]</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For SCI204 Microbiology at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="host-defenses-the-adaptive-immune-system" class="section level1" number="13">
<h1><span class="header-section-number">13</span> Host Defenses: The Adaptive Immune System</h1>
<p>The adaptive immune system, also referred as the acquired immune system, is a subsystem of the immune system that is composed of specialized, systemic cells and processes that eliminate pathogens or prevent their growth. The acquired immune system is one of the two main immunity strategies found in vertebrates (the other being the innate immune system).</p>
<p>￼
Google Ngram of “acquired immunity” vs. “adaptive immunity”. The peak for “adaptive” in the 1960s reflects its introduction to immunology by Robert A. Good and use by colleagues; the explosive increase in the 1990s was correlated with the use of the phrase “innate immunity”.
Like the innate system, the adaptive immune system includes both humoral immunity components and cell-mediated immunity components and destroys invading pathogens. Unlike the innate immune system, which is pre-programmed to react to common broad categories of pathogen, the adaptive immune system is highly specific to each particular pathogen the body has encountered.</p>
<p>Adaptive immunity creates immunological memory after an initial response to a specific pathogen, and leads to an enhanced response to future encounters with that pathogen. Antibodies are a critical part of the adaptive immune system. Adaptive immunity can provide long-lasting protection, sometimes for the person’s entire lifetime. For example, someone who recovers from measles is now protected against measles for their lifetime; in other cases it does not provide lifetime protection, as with chickenpox. This process of adaptive immunity is the basis of vaccination.</p>
<p>The cells that carry out the adaptive immune response are white blood cells known as lymphocytes. B cells and T cells, two different types of lymphocytes, carry out the main activities: antibody responses, and cell-mediated immune response. In antibody responses, B cells are activated to secrete antibodies, which are proteins also known as immunoglobulins. Antibodies travel through the bloodstream and bind to the foreign antigen causing it to inactivate, which does not allow the antigen to bind to the host.[1] Antigens are any substances that elicit the adaptive immune response. Sometimes the adaptive system is unable to distinguish harmful from harmless foreign molecules; the effects of this may be hayfever, asthma, or any other allergy.</p>
<p>In adaptive immunity, pathogen-specific receptors are “acquired” during the lifetime of the organism (whereas in innate immunity pathogen-specific receptors are already encoded in the genome). This acquired response is called “adaptive” because it prepares the body’s immune system for future challenges (though it can actually also be maladaptive when it results in allergies or autoimmunity).</p>
<p>The system is highly adaptable because of two factors. First, somatic hypermutation is a process of accelerated random genetic mutations in the antibody-coding genes, which allows antibodies with novel specificity to be created. Second, V(D)J recombination randomly selects one variable (V), one diversity (D), and one joining (J) region for genetic recombination and discards the rest, which produces a highly unique combination of antigen-receptor gene segments in each lymphocyte. This mechanism allows a small number of genetic segments to generate a vast number of different antigen receptors, which are then uniquely expressed on each individual lymphocyte. Since the gene rearrangement leads to an irreversible change in the DNA of each cell, all progeny (offspring) of that cell inherit genes that encode the same receptor specificity, including the memory B cells and memory T cells that are the keys to long-lived specific immunity.</p>
<p>The term “adaptive” was first used by Robert Good in reference to antibody responses in frogs as a synonym for “acquired immune response” in 1964. Good acknowledged he used the terms as synonyms but explained only that he preferred to use the term “adaptive”. He might have been thinking of the then not implausible theory of antibody formation in which antibodies were plastic and could adapt themselves to the molecular shape of antigens, and/or to the concept of “adaptive enzymes” as described by Monod in bacteria, that is, enzymes whose expression could be induced by their substrates. The phrase was used almost exclusively by Good and his students and a few other immunologists working with marginal organisms until the 1990s when it became widely used in tandem with the term “innate immunity” which became a popular subject after the discovery of the Toll receptor system in Drosophila, a previously marginal organism for the study of immunology. The term “adaptive” as used in immunology is problematic as acquired immune responses can be both adaptive and maladaptive in the physiological sense. Indeed, both acquired and innate immune responses can be both adaptive and maladaptive in the evolutionary sense. Most textbooks today, following the early use by Janeway, use “adaptive” almost exclusively and noting in glossaries that the term is synonymous with “acquired”.</p>
<p>The classic sense of “acquired immunity” came to mean, since Tonegawa’s discovery, “antigen-specific immunity mediated by somatic gene rearrangements that create clone-defining antigen receptors”. In the last decade, the term “adaptive” has been increasingly applied to another class of immune response not so-far associated with somatic gene rearrangements. These include expansion of natural killer (NK) cells with so-far unexplained specificity for antigens, expansion of NK cells expressing germ-line encoded receptors, and activation of other innate immune cells to an activated state that confers a short-term “immune memory”. In this sense, “adaptive immunity” more closely resembles the concept of “activated state” or “heterostasis”, thus returning in sense to the physiological sense of “adaptation” to environmental changes.</p>
<p>Acquired immunity is triggered in vertebrates when a pathogen evades the innate immune system and (1) generates a threshold level of antigen and (2) generates “stranger” or “danger” signals activating dendritic cells.[2]</p>
<p>The major functions of the acquired immune system include:</p>
<ul>
<li>Recognition of specific “non-self” antigens in the presence of “self”, during the process of antigen presentation.</li>
<li>Generation of responses that are tailored to maximally eliminate specific pathogens or pathogen-infected cells.</li>
<li>Development of immunological memory, in which pathogens are “remembered” through memory B cells and memory T cells.</li>
</ul>
<p>In humans, it takes 4-7 days for the adaptive immune system to mount a significant response.[3]</p>
<div id="lymphocytes" class="section level2" number="13.1">
<h2><span class="header-section-number">13.1</span> Lymphocytes</h2>
<p>T and B lymphocytes are the cells of the adaptive immune system. The human body has about 2 trillion lymphocytes, which are 20–40% of white blood cells; their total mass is about the same as the brain or liver. The peripheral bloodstream contains only 2% of all circulating lymphocytes; the other 98% move within tissues and the lymphatic system, which includes the lymph nodes and spleen.[1] In humans, approximately 1–2% of the lymphocyte pool recirculates each hour to increase the opportunity for the cells to encounter the specific pathogen and antigen that they react to.[4]</p>
<p>B cells and T cells are derived from the same multipotent hematopoietic stem cells, and look identical to one another until after they are activated. B cells play a large role in the humoral immune response, whereas T cells are intimately involved in cell-mediated immune responses. In all vertebrates except Agnatha, B cells and T cells are produced by stem cells in the bone marrow.[5] T cell progenitors then migrate from the bone marrow to the thymus, where they develop further.</p>
<p>In an adult animal, the peripheral lymphoid organs contain a mixture of B and T cells in at least three stages of differentiation:</p>
<ul>
<li>Naive B and naive T cells, which have left the bone marrow or thymus and entered the lymphatic system, but have yet to encounter their matching antigen</li>
<li>Effector cells that have been activated by their matching antigen, and are actively involved in eliminating a pathogen</li>
<li>Memory cells, the survivors of past infections</li>
</ul>
</div>
<div id="antigen-presentation" class="section level2" number="13.2">
<h2><span class="header-section-number">13.2</span> Antigen presentation</h2>
<p>Acquired immunity relies on the capacity of immune cells to distinguish between the body’s own cells and unwanted invaders. The host’s cells express “self” antigens. These antigens are different from those on the surface of bacteria or on the surface of virus-infected host cells (“non-self” or “foreign” antigens). The acquired immune response is triggered by recognizing foreign antigen in the cellular context of an activated dendritic cell.</p>
<p>With the exception of non-nucleated cells (including erythrocytes), all cells are capable of presenting antigen through the function of major histocompatibility complex (MHC) molecules.[5] Some cells are specially equipped to present antigen, and to prime naive T cells. Dendritic cells, B-cells, and macrophages are equipped with special “co-stimulatory” ligands recognized by co-stimulatory receptors on T cells, and are termed professional antigen-presenting cells (APCs).</p>
<p>Several T cells subgroups can be activated by professional APCs, and each type of T cell is specially equipped to deal with each unique toxin or microbial pathogen. The type of T cell activated, and the type of response generated, depends, in part, on the context in which the APC first encountered the antigen.[2]</p>
</div>
<div id="exogenous-antigens" class="section level2" number="13.3">
<h2><span class="header-section-number">13.3</span> Exogenous antigens</h2>
<p>Dendritic cells engulf exogenous pathogens, such as bacteria, parasites or toxins in the tissues and then migrate, via chemotactic signals, to the T cell-enriched lymph nodes. During migration, dendritic cells undergo a process of maturation in which they lose most of their ability to engulf other pathogens, and develop an ability to communicate with T-cells. The dendritic cell uses enzymes to chop the pathogen into smaller pieces, called antigens. In the lymph node, the dendritic cell displays these non-self antigens on its surface by coupling them to a receptor called the major histocompatibility complex, or MHC (also known in humans as human leukocyte antigen (HLA)). This MHC-antigen complex is recognized by T-cells passing through the lymph node. Exogenous antigens are usually displayed on MHC class II molecules, which activate CD4+T helper cells.[2]</p>
</div>
<div id="endogenous-antigensedit" class="section level2" number="13.4">
<h2><span class="header-section-number">13.4</span> Endogenous antigens[edit]</h2>
<p>Endogenous antigens are produced by intracellular bacteria and viruses replicating within a host cell. The host cell uses enzymes to digest virally associated proteins and displays these pieces on its surface to T-cells by coupling them to MHC. Endogenous antigens are typically displayed on MHC class I molecules, and activate CD8+ cytotoxic T-cells. With the exception of non-nucleated cells (including erythrocytes), MHC class I is expressed by all host cells.[2]</p>
</div>
<div id="t-lymphocytesedit" class="section level2" number="13.5">
<h2><span class="header-section-number">13.5</span> T lymphocytes[edit]</h2>
<p>CD8+ T lymphocytes and cytotoxicity[edit]
Main article: Cytotoxic T cell
Cytotoxic T cells (also known as TC, killer T cell, or cytotoxic T-lymphocyte (CTL)) are a sub-group of T cells that induce the death of cells that are infected with viruses (and other pathogens), or are otherwise damaged or dysfunctional.[2]</p>
<p>Naive cytotoxic T cells are activated when their T-cell receptor (TCR) strongly interacts with a peptide-bound MHC class I molecule. This affinity depends on the type and orientation of the antigen/MHC complex, and is what keeps the CTL and infected cell bound together.[2] Once activated, the CTL undergoes a process called clonal selection, in which it gains functions and divides rapidly to produce an army of “armed” effector cells. Activated CTL then travels throughout the body searching for cells that bear that unique MHC Class I + peptide.[citation needed]</p>
<p>When exposed to these infected or dysfunctional somatic cells, effector CTL release perforin and granulysin: cytotoxins that form pores in the target cell’s plasma membrane, allowing ions and water to flow into the infected cell, and causing it to burst or lyse. CTL release granzyme, a serine protease encapsulated in a granule that enters cells via pores to induce apoptosis (cell death). To limit extensive tissue damage during an infection, CTL activation is tightly controlled and in general requires a very strong MHC/antigen activation signal, or additional activation signals provided by “helper” T-cells (see below).[2]</p>
<p>On resolution of the infection, most effector cells die and phagocytes clear them away—but a few of these cells remain as memory cells.[5] On a later encounter with the same antigen, these memory cells quickly differentiate into effector cells, dramatically shortening the time required to mount an effective response.[citation needed]</p>
<p>Helper T-cells[edit]
Main article: T helper cell</p>
<p>CD4+ lymphocytes, also called “helper” T cells, are immune response mediators, and play an important role in establishing and maximizing the capabilities of the acquired immune response.[2] These cells have no cytotoxic or phagocytic activity; and cannot kill infected cells or clear pathogens, but, in essence “manage” the immune response, by directing other cells to perform these tasks.</p>
<p>Helper T cells express T cell receptors (TCR) that recognize antigen bound to Class II MHC molecules. The activation of a naive helper T-cell causes it to release cytokines, which influences the activity of many cell types, including the APC (Antigen-Presenting Cell) that activated it. Helper T-cells require a much milder activation stimulus than cytotoxic T cells. Helper T cells can provide extra signals that “help” activate cytotoxic cells.[5]</p>
<p>Th1 and Th2: helper T cell responses[edit]
Classically, two types of effector CD4+ T helper cell responses can be induced by a professional APC, designated Th1 and Th2, each designed to eliminate different types of pathogens. The factors that dictate whether an infection triggers a Th1 or Th2 type response are not fully understood, but the response generated does play an important role in the clearance of different pathogens.[2]</p>
<p>The Th1 response is characterized by the production of Interferon-gamma, which activates the bactericidal activities of macrophages, and induces B cells to make opsonizing (marking for phagocytosis) and complement-fixing antibodies, and leads to cell-mediated immunity.[2] In general, Th1 responses are more effective against intracellular pathogens (viruses and bacteria that are inside host cells).</p>
<p>The Th2 response is characterized by the release of Interleukin 5, which induces eosinophils in the clearance of parasites.[7] Th2 also produce Interleukin 4, which facilitates B cell isotype switching.[2] In general, Th2 responses are more effective against extracellular bacteria, parasites including helminths and toxins.[2] Like cytotoxic T cells, most of the CD4+ helper cells die on resolution of infection, with a few remaining as CD4+ memory cells.</p>
<p>Increasingly, there is strong evidence from mouse and human-based scientific studies of a broader diversity in CD4+ effector T helper cell subsets.[8][9] Regulatory T (Treg) cells, have been identified as important negative regulators of adaptive immunity as they limit and suppress the immune system to control aberrant immune responses to self-antigens; an important mechanism in controlling the development of autoimmune diseases.[5] Follicular helper T (Tfh) cells are another distinct population of effector CD4+ T cells that develop from naive T cells post-antigen activation. Tfh cells are specialized in helping B cell humoral immunity as they are uniquely capable of migrating to follicular B cells in secondary lymphoid organs and provide them positive paracrine signals to enable the generation and recall production of high-quality affinity-matured antibodies. Similar to Tregs, Tfh cells also play a role in immunological tolerance as an abnormal expansion of Tfh cell numbers can lead to unrestricted autoreactive antibody production causing severe systemic autoimmune disorders.[10]</p>
<p>The relevance of CD4+ T helper cells is highlighted during an HIV infection. HIV is able to subvert the immune system by specifically attacking the CD4+ T cells, precisely the cells that could drive the clearance of the virus, but also the cells that drive immunity against all other pathogens encountered during an organism’s lifetime.[5]</p>
<p>Gamma delta T cells[edit]
Main article: Gamma delta T cell
Gamma delta T cells (γδ T cells) possess an alternative T cell receptor (TCR) as opposed to CD4+ and CD8+ αβ T cells and share characteristics of helper T cells, cytotoxic T cells and natural killer cells. Like other ‘unconventional’ T cell subsets bearing invariant TCRs, such as CD1d-restricted natural killer T cells, γδ T cells exhibit characteristics that place them at the border between innate and acquired immunity. On one hand, γδ T cells may be considered a component of adaptive immunity in that they rearrange TCR genes via V(D)J recombination, which also produces junctional diversity, and develop a memory phenotype. On the other hand, however, the various subsets may also be considered part of the innate immune system where a restricted TCR or NK receptors may be used as a pattern recognition receptor. For example, according to this paradigm, large numbers of Vγ9/Vδ2 T cells respond within hours to common molecules produced by microbes, and highly restricted intraepithelial Vδ1 T cells respond to stressed epithelial cells.</p>
<p>B lymphocytes and antibody production</p>
<p>B Cells are the major cells involved in the creation of antibodies that circulate in blood plasma and lymph, known as humoral immunity. Antibodies (also known as immunoglobulin, Ig), are large Y-shaped proteins used by the immune system to identify and neutralize foreign objects. In mammals, there are five types of antibody: IgA, IgD, IgE, IgG, and IgM, differing in biological properties; each has evolved to handle different kinds of antigens. Upon activation, B cells produce antibodies, each of which recognize a unique antigen, and neutralizing specific pathogens.[2]</p>
<p>Antigen and antibody binding would cause five different protective mechanisms:</p>
<p>Agglutination: Reduces number of infectious units to be dealt with
Activation of complement: Cause inflammation and cell lysis
Opsonization: Coating antigen with antibody enhances phagocytosis
Antibody-dependent cell-mediated cytotoxicity: Antibodies attached to target cell cause destruction by macrophages, eosinophils, and NK cells
Neutralization: Blocks adhesion of bacteria and viruses to mucosa
Like the T cell, B cells express a unique B cell receptor (BCR), in this case, a membrane-bound antibody molecule. All the BCR of any one clone of B cells recognizes and binds to only one particular antigen. A critical difference between B cells and T cells is how each cell “sees” an antigen. T cells recognize their cognate antigen in a processed form – as a peptide in the context of an MHC molecule,[2] whereas B cells recognize antigens in their native form.[2] Once a B cell encounters its cognate (or specific) antigen (and receives additional signals from a helper T cell (predominately Th2 type)), it further differentiates into an effector cell, known as a plasma cell.[2]</p>
<p>Plasma cells are short-lived cells (2–3 days) that secrete antibodies. These antibodies bind to antigens, making them easier targets for phagocytes, and trigger the complement cascade.[2] About 10% of plasma cells survive to become long-lived antigen-specific memory B cells.[2] Already primed to produce specific antibodies, these cells can be called upon to respond quickly if the same pathogen re-infects the host, while the host experiences few, if any, symptoms.</p>
<p>Alternative systems[edit]
In jawless vertebrates[edit]
Main article: Adaptive immunity in jawless vertebrates
Primitive jawless vertebrates, such as the lamprey and hagfish, have an adaptive immune system that shows 3 different cell lineages, each sharing a common origin with B cells, αβ T cells, and innate-like γΔ T cells.[11][12] Instead of the classical antibodies and T cell receptors, these animals possess a large array of molecules called variable lymphocyte receptors (VLRs for short) that, like the antigen receptors of jawed vertebrates, are produced from only a small number (one or two) of genes. These molecules are believed to bind pathogenic antigens in a similar way to antibodies, and with the same degree of specificity.[13]</p>
<p>In insects[edit]
For a long time it was thought that insects and other invertebrates possess only innate immune system. However, in recent years some of the basic hallmarks of adaptive immunity have been discovered in insects. Those traits are immune memory and specificity. Although the hallmarks are present the mechanisms are different from those in vertebrates.</p>
<p>Immune memory in insects was discovered through the phenomenon of priming. When insects are exposed to non-lethal dose or heat killed bacteria they are able to develop a memory of that infection that allows them to withstand otherwise lethal dose of the same bacteria they were exposed to before.[14][15] Unlike in vertebrates, insects do not possess cells specific for adaptive immunity. Instead those mechanisms are mediated by hemocytes. Hemocytes function similarly to phagocytes and after priming they are able to more effectively recognize and engulf the pathogen.[16] It was also shown that it is possible to transfer the memory into offspring. For example, in honeybees if the queen is infected with bacteria then the newly born workers have enhanced abilities in fighting with the same bacteria.[17] Other experimental model based on red flour beetle also showed pathogen specific primed memory transfer into offspring from both mothers and fathers.[18]</p>
<p>Most commonly accepted theory of the specificity is based on Dscam gene. Dscam gene also known as Down syndrome cell adhesive molecule is a gene that contains 3 variable Ig domains. Those domains can be alternatively spliced reaching high numbers of variations.[19] It was shown that after exposure to different pathogens there are different splice forms of dscam produced. After the animals with different splice forms are exposed to the same pathogen only the individuals with the splice form specific for that pathogen survive.[19]</p>
<p>Other mechanisms supporting the specificity of insect immunity is RNA interference (RNAi). RNAi is a form of antiviral immunity with high specificity.[20] It has several different pathways that all end with the virus being unable to replicate. One of the pathways is siRNA in which long double stranded RNA is cut into pieces that serve as templates for protein complex Ago2-RISC that finds and degrades complementary RNA of the virus. MiRNA pathway in cytoplasm binds to Ago1-RISC complex and functions as a template for viral RNA degradation. Last one is piRNA where small RNA binds to the Piwi protein family and controls transposones and other mobile elements.[21] Despite the research the exact mechanisms responsible for immune priming and specificity in insects are not well described.</p>
<p>Immunological memory[edit]
Further information: Immunity (medical)
When B cells and T cells are activated some become memory B cells and some memory T cells. Throughout the lifetime of an animal these memory cells form a database of effective B and T lymphocytes. Upon interaction with a previously encountered antigen, the appropriate memory cells are selected and activated. In this manner, the second and subsequent exposures to an antigen produce a stronger and faster immune response. This is “adaptive” in the sense that the body’s immune system prepares itself for future challenges, but is “maladaptive” of course if the receptors are autoimmune. Immunological memory can be in the form of either passive short-term memory or active long-term memory.</p>
<p>Passive memory[edit]
Passive memory is usually short-term, lasting between a few days and several months. Newborn infants have had no prior exposure to microbes and are particularly vulnerable to infection. Several layers of passive protection are provided by the mother. In utero, maternal IgG is transported directly across the placenta, so that, at birth, human babies have high levels of antibodies, with the same range of antigen specificities as their mother.[2] Breast milk contains antibodies (mainly IgA) that are transferred to the gut of the infant, protecting against bacterial infections, until the newborn can synthesize its own antibodies.[2]</p>
<p>This is passive immunity because the fetus does not actually make any memory cells or antibodies: It only borrows them. Short-term passive immunity can also be transferred artificially from one individual to another via antibody-rich serum.</p>
<p>Active memory[edit]
In general, active immunity is long-term and can be acquired by infection followed by B cell and T cell activation, or artificially acquired by vaccines, in a process called immunization.</p>
<p>Immunization[edit]
Historically, infectious disease has been the leading cause of death in the human population. Over the last century, two important factors have been developed to combat their spread: sanitation and immunization.[5] Immunization (commonly referred to as vaccination) is the deliberate induction of an immune response, and represents the single most effective manipulation of the immune system that scientists have developed.[5] Immunizations are successful because they utilize the immune system’s natural specificity as well as its inducibility.</p>
<p>The principle behind immunization is to introduce an antigen, derived from a disease-causing organism, that stimulates the immune system to develop protective immunity against that organism, but that does not itself cause the pathogenic effects of that organism. An antigen (short for antibody generator), is defined as any substance that binds to a specific antibody and elicits an adaptive immune response.[1]</p>
<p>Most viral vaccines are based on live attenuated viruses, whereas many bacterial vaccines are based on acellular components of microorganisms, including harmless toxin components.[1] Many antigens derived from acellular vaccines do not strongly induce an adaptive response, and most bacterial vaccines require the addition of adjuvants that activate the antigen-presenting cells of the innate immune system to enhance immunogenicity.[5]</p>
<p>Immunological diversity</p>
<p>Most large molecules, including virtually all proteins and many polysaccharides, can serve as antigens.[2] The parts of an antigen that interact with an antibody molecule or a lymphocyte receptor, are called epitopes, or antigenic determinants. Most antigens contain a variety of epitopes and can stimulate the production of antibodies, specific T cell responses, or both.[2] A very small proportion (less than 0.01%) of the total lymphocytes are able to bind to a particular antigen, which suggests that only a few cells respond to each antigen.[5]</p>
<p>For the acquired response to “remember” and eliminate a large number of pathogens the immune system must be able to distinguish between many different antigens,[1] and the receptors that recognize antigens must be produced in a huge variety of configurations, in essence one receptor (at least) for each different pathogen that might ever be encountered. Even in the absence of antigen stimulation, a human can produce more than 1 trillion different antibody molecules.[5] Millions of genes would be required to store the genetic information that produces these receptors, but, the entire human genome contains fewer than 25,000 genes.[22]</p>
<p>Myriad receptors are produced through a process known as clonal selection.[1][2] According to the clonal selection theory, at birth, an animal randomly generates a vast diversity of lymphocytes (each bearing a unique antigen receptor) from information encoded in a small family of genes. To generate each unique antigen receptor, these genes have undergone a process called V(D)J recombination, or combinatorial diversification, in which one gene segment recombines with other gene segments to form a single unique gene. This assembly process generates the enormous diversity of receptors and antibodies, before the body ever encounters antigens, and enables the immune system to respond to an almost unlimited diversity of antigens.[2] Throughout an animal’s lifetime, lymphocytes that can react against the antigens an animal actually encounters are selected for action—directed against anything that expresses that antigen.</p>
<p>Note that the innate and acquired portions of the immune system work together, not in spite of each other. The acquired arm, B, and T cells couldn’t function without the innate system input. T cells are useless without antigen-presenting cells to activate them, and B cells are crippled without T cell help. On the other hand, the innate system would likely be overrun with pathogens without the specialized action of the adaptive immune response.</p>
<p>Acquired immunity during pregnancy[edit]
The cornerstone of the immune system is the recognition of “self” versus “non-self”. Therefore, the mechanisms that protect the human fetus (which is considered “non-self”) from attack by the immune system, are particularly interesting. Although no comprehensive explanation has emerged to explain this mysterious, and often repeated, lack of rejection, two classical reasons may explain how the fetus is tolerated. The first is that the fetus occupies a portion of the body protected by a non-immunological barrier, the uterus, which the immune system does not routinely patrol.[2] The second is that the fetus itself may promote local immunosuppression in the mother, perhaps by a process of active nutrient depletion.[2] A more modern explanation for this induction of tolerance is that specific glycoproteins expressed in the uterus during pregnancy suppress the uterine immune response (see eu-FEDS).</p>
<p>During pregnancy in viviparous mammals (all mammals except Monotremes), endogenous retroviruses (ERVs) are activated and produced in high quantities during the implantation of the embryo. They are currently known to possess immunosuppressive properties, suggesting a role in protecting the embryo from its mother’s immune system. Also, viral fusion proteins cause the formation of the placental syncytium[23] to limit exchange of migratory cells between the developing embryo and the body of the mother (something an epithelium can’t do sufficiently, as certain blood cells specialize to insert themselves between adjacent epithelial cells). The immunodepressive action was the initial normal behavior of the virus, similar to HIV. The fusion proteins were a way to spread the infection to other cells by simply merging them with the infected one (HIV does this too). It is believed that the ancestors of modern viviparous mammals evolved after an infection by this virus, enabling the fetus to survive the immune system of the mother.[24]</p>
<p>The human genome project found several thousand ERVs classified into 24 families.[25]</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="host-defensesthe-innate-immune-system.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/microbiology-text/edit/master/adaptive_immunity.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
