<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>9 Microbial Genetics | Text For SCI204 Microbiology at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Microbiology." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="9 Microbial Genetics | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/microbiology_text_cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Microbiology." />
  <meta name="github-repo" content="nikolaussucher/microbiology-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="9 Microbial Genetics | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Microbiology." />
  <meta name="twitter:image" content="/microbiology_text_cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="an-introduction-to-microbial-metabolism.html"/>
<link rel="next" href="genetic-engineering-1.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />




<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">SCI204 Microbiology at RCC</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html"><i class="fa fa-check"></i><b>1</b> The Main Themes Of Microbiology</a>
<ul>
<li class="chapter" data-level="1.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#general-characteristics-and-evolution-of-microorganisms"><i class="fa fa-check"></i><b>1.1</b> General Characteristics And Evolution Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#archaea"><i class="fa fa-check"></i><b>1.1.1</b> Archaea</a></li>
<li class="chapter" data-level="1.1.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#bacteria"><i class="fa fa-check"></i><b>1.1.2</b> Bacteria</a></li>
<li class="chapter" data-level="1.1.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#eukaryotes"><i class="fa fa-check"></i><b>1.1.3</b> Eukaryotes</a></li>
<li class="chapter" data-level="1.1.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#protists"><i class="fa fa-check"></i><b>1.1.4</b> Protists</a></li>
<li class="chapter" data-level="1.1.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#fungi"><i class="fa fa-check"></i><b>1.1.5</b> Fungi</a></li>
<li class="chapter" data-level="1.1.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#plants"><i class="fa fa-check"></i><b>1.1.6</b> Plants</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#ecology-of-microorganisms"><i class="fa fa-check"></i><b>1.2</b> Ecology Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbiome"><i class="fa fa-check"></i><b>1.2.1</b> Microbiome</a></li>
<li class="chapter" data-level="1.2.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbiota-members-of-the-microbiome"><i class="fa fa-check"></i><b>1.2.2</b> Microbiota – members of the microbiome</a></li>
<li class="chapter" data-level="1.2.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbial-networks-and-interaction"><i class="fa fa-check"></i><b>1.2.3</b> Microbial Networks And Interaction</a></li>
<li class="chapter" data-level="1.2.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-human-microbiome"><i class="fa fa-check"></i><b>1.2.4</b> The Human Microbiome</a></li>
<li class="chapter" data-level="1.2.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#extremophiles"><i class="fa fa-check"></i><b>1.2.5</b> Extremophiles</a></li>
<li class="chapter" data-level="1.2.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#soil-microorganisms"><i class="fa fa-check"></i><b>1.2.6</b> Soil Microorganisms</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#human-interactions-with-microbes"><i class="fa fa-check"></i><b>1.3</b> Human Interactions With Microbes</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#food-production"><i class="fa fa-check"></i><b>1.3.1</b> Food production</a></li>
<li class="chapter" data-level="1.3.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#water-treatment"><i class="fa fa-check"></i><b>1.3.2</b> Water Treatment</a></li>
<li class="chapter" data-level="1.3.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#fuel-production"><i class="fa fa-check"></i><b>1.3.3</b> Fuel Production</a></li>
<li class="chapter" data-level="1.3.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#production-of-chemicals-and-enzymes"><i class="fa fa-check"></i><b>1.3.4</b> Production Of Chemicals And Enzymes</a></li>
<li class="chapter" data-level="1.3.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#use-of-microorganisms-in-science"><i class="fa fa-check"></i><b>1.3.5</b> Use Of Microorganisms In Science</a></li>
<li class="chapter" data-level="1.3.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#use-of-microorganisms-for-warfare"><i class="fa fa-check"></i><b>1.3.6</b> Use of Microorganisms For Warfare</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#role-of-microorganisms-in-disease"><i class="fa fa-check"></i><b>1.4</b> Role Of Microorganisms In Disease</a></li>
<li class="chapter" data-level="1.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#role-of-microorganisms-in-agriculture-and-horticulture"><i class="fa fa-check"></i><b>1.5</b> Role Of Microorganisms In Agriculture And Horticulture</a></li>
<li class="chapter" data-level="1.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#history-of-microbiology"><i class="fa fa-check"></i><b>1.6</b> History Of Microbiology</a></li>
<li class="chapter" data-level="1.7" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#taxonomy-classification-and-naming-of-microorganisms"><i class="fa fa-check"></i><b>1.7</b> Taxonomy: Classification And Naming Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-concept-of-species-in-biology"><i class="fa fa-check"></i><b>1.7.1</b> The Concept of Species in Biology</a></li>
<li class="chapter" data-level="1.7.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#binomial-nomenclature"><i class="fa fa-check"></i><b>1.7.2</b> Binomial Nomenclature</a></li>
<li class="chapter" data-level="1.7.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#taxonomy-and-systematics"><i class="fa fa-check"></i><b>1.7.3</b> Taxonomy And Systematics</a></li>
<li class="chapter" data-level="1.7.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#kingdoms-and-domains"><i class="fa fa-check"></i><b>1.7.4</b> Kingdoms And Domains</a></li>
<li class="chapter" data-level="1.7.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-three-domain-system"><i class="fa fa-check"></i><b>1.7.5</b> The Three-Domain System</a></li>
<li class="chapter" data-level="1.7.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#bergeys-manual-of-systematic-bacteriology"><i class="fa fa-check"></i><b>1.7.6</b> Bergey’s Manual of Systematic Bacteriology</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html"><i class="fa fa-check"></i><b>2</b> The Chemistry Of Biology</a>
<ul>
<li class="chapter" data-level="2.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#matter"><i class="fa fa-check"></i><b>2.1</b> Matter</a></li>
<li class="chapter" data-level="2.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-atom"><i class="fa fa-check"></i><b>2.2</b> The Atom</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structure-of-the-atom"><i class="fa fa-check"></i><b>2.2.1</b> Structure of The Atom</a></li>
<li class="chapter" data-level="2.2.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-atomic-nucleus"><i class="fa fa-check"></i><b>2.2.2</b> The Atomic Nucleus</a></li>
<li class="chapter" data-level="2.2.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-electron-cloud"><i class="fa fa-check"></i><b>2.2.3</b> The Electron Cloud</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-periodic-table-of-the-elements"><i class="fa fa-check"></i><b>2.3</b> The Periodic Table of The Elements</a></li>
<li class="chapter" data-level="2.4" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#chemical-bonds-molecules-and-compounds"><i class="fa fa-check"></i><b>2.4</b> Chemical Bonds, Molecules, And Compounds</a></li>
<li class="chapter" data-level="2.5" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#energy"><i class="fa fa-check"></i><b>2.5</b> Energy</a></li>
<li class="chapter" data-level="2.6" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#chemical-reactions"><i class="fa fa-check"></i><b>2.6</b> Chemical Reactions</a></li>
<li class="chapter" data-level="2.7" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#radioactive-decay-of-atoms"><i class="fa fa-check"></i><b>2.7</b> Radioactive Decay of Atoms</a></li>
<li class="chapter" data-level="2.8" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#ions-and-salts"><i class="fa fa-check"></i><b>2.8</b> Ions And Salts</a></li>
<li class="chapter" data-level="2.9" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#solutions"><i class="fa fa-check"></i><b>2.9</b> Solutions</a></li>
<li class="chapter" data-level="2.10" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#acids-and-bases"><i class="fa fa-check"></i><b>2.10</b> Acids And Bases</a></li>
<li class="chapter" data-level="2.11" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#reduction-oxidation-redox-reactions"><i class="fa fa-check"></i><b>2.11</b> Reduction-oxidation (Redox) Reactions</a></li>
<li class="chapter" data-level="2.12" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-chemistry-of-water"><i class="fa fa-check"></i><b>2.12</b> The Chemistry Of Water</a></li>
<li class="chapter" data-level="2.13" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#basic-organic-chemistry-for-biology"><i class="fa fa-check"></i><b>2.13</b> Basic Organic Chemistry For Biology</a>
<ul>
<li class="chapter" data-level="2.13.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#functional-groups"><i class="fa fa-check"></i><b>2.13.1</b> Functional Groups</a></li>
<li class="chapter" data-level="2.13.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#biomolecules"><i class="fa fa-check"></i><b>2.13.2</b> Biomolecules</a></li>
<li class="chapter" data-level="2.13.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#proteins"><i class="fa fa-check"></i><b>2.13.3</b> Proteins</a></li>
<li class="chapter" data-level="2.13.4" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structure"><i class="fa fa-check"></i><b>2.13.4</b> Structure</a></li>
<li class="chapter" data-level="2.13.5" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#cellular-functions-of-proteins"><i class="fa fa-check"></i><b>2.13.5</b> Cellular Functions of Proteins</a></li>
<li class="chapter" data-level="2.13.6" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#cell-signaling-and-ligand-binding"><i class="fa fa-check"></i><b>2.13.6</b> Cell Signaling And Ligand Binding</a></li>
<li class="chapter" data-level="2.13.7" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structural-proteins"><i class="fa fa-check"></i><b>2.13.7</b> Structural Proteins</a></li>
<li class="chapter" data-level="2.13.8" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#carbohydrates"><i class="fa fa-check"></i><b>2.13.8</b> Carbohydrates</a></li>
<li class="chapter" data-level="2.13.9" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#lipids"><i class="fa fa-check"></i><b>2.13.9</b> Lipids</a></li>
<li class="chapter" data-level="2.13.10" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#nucleic-acids"><i class="fa fa-check"></i><b>2.13.10</b> Nucleic Acids</a></li>
<li class="chapter" data-level="2.13.11" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#deoxyribonucleic-acid-dna"><i class="fa fa-check"></i><b>2.13.11</b> Deoxyribonucleic Acid (DNA)</a></li>
<li class="chapter" data-level="2.13.12" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#ribonucleic-acid-rna"><i class="fa fa-check"></i><b>2.13.12</b> Ribonucleic Acid (RNA)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html"><i class="fa fa-check"></i><b>3</b> An Introduction To Methods Of Studying Microorganisms</a>
<ul>
<li class="chapter" data-level="3.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#microscopy"><i class="fa fa-check"></i><b>3.1</b> Microscopy</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#light-microscopy"><i class="fa fa-check"></i><b>3.1.1</b> Light Microscopy</a></li>
<li class="chapter" data-level="3.1.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-optical-light-microscope"><i class="fa fa-check"></i><b>3.1.2</b> The Optical (Light) Microscope</a></li>
<li class="chapter" data-level="3.1.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#simple-microscope"><i class="fa fa-check"></i><b>3.1.3</b> Simple Microscope</a></li>
<li class="chapter" data-level="3.1.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#compound-microscope"><i class="fa fa-check"></i><b>3.1.4</b> Compound Microscope</a></li>
<li class="chapter" data-level="3.1.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-main-components-of-the-light-microscope"><i class="fa fa-check"></i><b>3.1.5</b> The Main Components Of The Light Microscope</a></li>
<li class="chapter" data-level="3.1.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#eyepiece-ocular-lens"><i class="fa fa-check"></i><b>3.1.6</b> Eyepiece (Ocular Lens)</a></li>
<li class="chapter" data-level="3.1.7" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#objective-lens"><i class="fa fa-check"></i><b>3.1.7</b> Objective Lens</a></li>
<li class="chapter" data-level="3.1.8" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#oil-immersion-objective"><i class="fa fa-check"></i><b>3.1.8</b> Oil Immersion Objective</a></li>
<li class="chapter" data-level="3.1.9" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#focus-knobs"><i class="fa fa-check"></i><b>3.1.9</b> Focus Knobs</a></li>
<li class="chapter" data-level="3.1.10" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-frame"><i class="fa fa-check"></i><b>3.1.10</b> The Frame</a></li>
<li class="chapter" data-level="3.1.11" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-stage"><i class="fa fa-check"></i><b>3.1.11</b> The Stage</a></li>
<li class="chapter" data-level="3.1.12" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-light-source"><i class="fa fa-check"></i><b>3.1.12</b> The Light Source</a></li>
<li class="chapter" data-level="3.1.13" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-condenser"><i class="fa fa-check"></i><b>3.1.13</b> The Condenser</a></li>
<li class="chapter" data-level="3.1.14" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#magnification"><i class="fa fa-check"></i><b>3.1.14</b> Magnification</a></li>
<li class="chapter" data-level="3.1.15" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-microscope-slide"><i class="fa fa-check"></i><b>3.1.15</b> The Microscope Slide</a></li>
<li class="chapter" data-level="3.1.16" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dry-mount"><i class="fa fa-check"></i><b>3.1.16</b> Dry Mount</a></li>
<li class="chapter" data-level="3.1.17" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#wet-mount-or-temporary-mount"><i class="fa fa-check"></i><b>3.1.17</b> Wet Mount Or Temporary Mount</a></li>
<li class="chapter" data-level="3.1.18" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#prepared-mount-or-permanent-mount"><i class="fa fa-check"></i><b>3.1.18</b> Prepared Mount Or Permanent Mount</a></li>
<li class="chapter" data-level="3.1.19" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#techniques-of-optical-microscopy"><i class="fa fa-check"></i><b>3.1.19</b> Techniques Of Optical Microscopy</a></li>
<li class="chapter" data-level="3.1.20" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#bright-field"><i class="fa fa-check"></i><b>3.1.20</b> Bright Field</a></li>
<li class="chapter" data-level="3.1.21" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#oblique-illumination"><i class="fa fa-check"></i><b>3.1.21</b> Oblique Illumination</a></li>
<li class="chapter" data-level="3.1.22" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dark-field"><i class="fa fa-check"></i><b>3.1.22</b> Dark Field</a></li>
<li class="chapter" data-level="3.1.23" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dispersion-staining"><i class="fa fa-check"></i><b>3.1.23</b> Dispersion Staining</a></li>
<li class="chapter" data-level="3.1.24" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#phase-contrast"><i class="fa fa-check"></i><b>3.1.24</b> Phase Contrast</a></li>
<li class="chapter" data-level="3.1.25" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#differential-interference-contrast"><i class="fa fa-check"></i><b>3.1.25</b> Differential Interference Contrast</a></li>
<li class="chapter" data-level="3.1.26" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#interference-reflection"><i class="fa fa-check"></i><b>3.1.26</b> Interference Reflection</a></li>
<li class="chapter" data-level="3.1.27" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#fluorescence-microscopy"><i class="fa fa-check"></i><b>3.1.27</b> Fluorescence Microscopy</a></li>
<li class="chapter" data-level="3.1.28" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#confoca-microscopy"><i class="fa fa-check"></i><b>3.1.28</b> Confoca Microscopy</a></li>
<li class="chapter" data-level="3.1.29" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#two-photon-microscopy"><i class="fa fa-check"></i><b>3.1.29</b> Two-Photon Microscopy</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-gram-stain"><i class="fa fa-check"></i><b>3.2</b> The Gram Stain</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-positive-bacteria"><i class="fa fa-check"></i><b>3.2.1</b> Gram-Positive Bacteria</a></li>
<li class="chapter" data-level="3.2.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-negative-bacteria"><i class="fa fa-check"></i><b>3.2.2</b> Gram-Negative Bacteria</a></li>
<li class="chapter" data-level="3.2.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-variable-and-gram-indeterminate-bacteria"><i class="fa fa-check"></i><b>3.2.3</b> Gram-Variable and Gram-Indeterminate Bacteria</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#electron-microscopy"><i class="fa fa-check"></i><b>3.3</b> Electron Microscopy</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-transmission-electron-microscope-tem"><i class="fa fa-check"></i><b>3.3.1</b> The Transmission Electron Microscope (TEM)</a></li>
<li class="chapter" data-level="3.3.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-scanning-electron-microscope-sem"><i class="fa fa-check"></i><b>3.3.2</b> The Scanning Electron Microscope (SEM)</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#microbiological-culture"><i class="fa fa-check"></i><b>3.4</b> Microbiological Culture</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#broth-cultures"><i class="fa fa-check"></i><b>3.4.1</b> Broth cultures</a></li>
<li class="chapter" data-level="3.4.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#agar-plates"><i class="fa fa-check"></i><b>3.4.2</b> Agar Plates</a></li>
<li class="chapter" data-level="3.4.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#stab-cultures"><i class="fa fa-check"></i><b>3.4.3</b> Stab Cultures</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#culture-collections"><i class="fa fa-check"></i><b>3.5</b> Culture Collections</a></li>
<li class="chapter" data-level="3.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#growth-medium"><i class="fa fa-check"></i><b>3.6</b> Growth Medium</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#culture-media"><i class="fa fa-check"></i><b>3.6.1</b> Culture Media</a></li>
<li class="chapter" data-level="3.6.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#minimal-media"><i class="fa fa-check"></i><b>3.6.2</b> Minimal Media</a></li>
<li class="chapter" data-level="3.6.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#selective-media"><i class="fa fa-check"></i><b>3.6.3</b> Selective Media</a></li>
<li class="chapter" data-level="3.6.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#differential-media"><i class="fa fa-check"></i><b>3.6.4</b> Differential Media</a></li>
<li class="chapter" data-level="3.6.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#transport-media"><i class="fa fa-check"></i><b>3.6.5</b> Transport Media</a></li>
<li class="chapter" data-level="3.6.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#enriched-media"><i class="fa fa-check"></i><b>3.6.6</b> Enriched Media</a></li>
<li class="chapter" data-level="3.6.7" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#agar-plate"><i class="fa fa-check"></i><b>3.6.7</b> Agar Plate</a></li>
<li class="chapter" data-level="3.6.8" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#blood-agar-plate"><i class="fa fa-check"></i><b>3.6.8</b> Blood Agar Plate</a></li>
<li class="chapter" data-level="3.6.9" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#chocolate-agar"><i class="fa fa-check"></i><b>3.6.9</b> Chocolate Agar</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><i class="fa fa-check"></i><b>4</b> An Introduction To Prokaryotic Cells And Microorganisms</a>
<ul>
<li class="chapter" data-level="4.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#basic-characteristics-of-life"><i class="fa fa-check"></i><b>4.1</b> Basic Characteristics Of Life</a></li>
<li class="chapter" data-level="4.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#origin-of-life"><i class="fa fa-check"></i><b>4.2</b> Origin Of Life</a></li>
<li class="chapter" data-level="4.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-foundations-of-modern-biology"><i class="fa fa-check"></i><b>4.3</b> The Foundations Of Modern Biology</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-cell-theory"><i class="fa fa-check"></i><b>4.3.1</b> The Cell Theory</a></li>
<li class="chapter" data-level="4.3.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-theory-of-evolution"><i class="fa fa-check"></i><b>4.3.2</b> The Theory Of Evolution</a></li>
<li class="chapter" data-level="4.3.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#genetics"><i class="fa fa-check"></i><b>4.3.3</b> Genetics</a></li>
<li class="chapter" data-level="4.3.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#homeostasis"><i class="fa fa-check"></i><b>4.3.4</b> Homeostasis</a></li>
<li class="chapter" data-level="4.3.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#energy-1"><i class="fa fa-check"></i><b>4.3.5</b> Energy</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#basic-characteristics-of-cells"><i class="fa fa-check"></i><b>4.4</b> Basic Characteristics Of Cells</a></li>
<li class="chapter" data-level="4.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#prokaryotic-cells"><i class="fa fa-check"></i><b>4.5</b> Prokaryotic Cells</a></li>
<li class="chapter" data-level="4.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacteria-1"><i class="fa fa-check"></i><b>4.6</b> Bacteria</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#intracellular-structure"><i class="fa fa-check"></i><b>4.6.1</b> Intracellular Structure</a></li>
<li class="chapter" data-level="4.6.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#extracellular-structures"><i class="fa fa-check"></i><b>4.6.2</b> Extracellular Structures</a></li>
<li class="chapter" data-level="4.6.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#endospores"><i class="fa fa-check"></i><b>4.6.3</b> Endospores</a></li>
<li class="chapter" data-level="4.6.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#metabolism"><i class="fa fa-check"></i><b>4.6.4</b> Metabolism</a></li>
<li class="chapter" data-level="4.6.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#growth-and-reproduction"><i class="fa fa-check"></i><b>4.6.5</b> Growth And Reproduction</a></li>
<li class="chapter" data-level="4.6.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacterial-genetics"><i class="fa fa-check"></i><b>4.6.6</b> Bacterial Genetics</a></li>
<li class="chapter" data-level="4.6.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#movement-of-bacteria"><i class="fa fa-check"></i><b>4.6.7</b> Movement Of Bacteria</a></li>
<li class="chapter" data-level="4.6.8" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacterial-communication"><i class="fa fa-check"></i><b>4.6.8</b> Bacterial Communication</a></li>
<li class="chapter" data-level="4.6.9" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#classification-and-identification"><i class="fa fa-check"></i><b>4.6.9</b> Classification And identification</a></li>
<li class="chapter" data-level="4.6.10" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#interactions-with-other-organisms"><i class="fa fa-check"></i><b>4.6.10</b> Interactions With Other Organisms</a></li>
<li class="chapter" data-level="4.6.11" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacteria-as-pathogens"><i class="fa fa-check"></i><b>4.6.11</b> Bacteria As Pathogens</a></li>
<li class="chapter" data-level="4.6.12" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#significance-in-technology-and-industry"><i class="fa fa-check"></i><b>4.6.12</b> Significance In Technology And Industry</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaea-1"><i class="fa fa-check"></i><b>4.7</b> Archaea</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#origin-and-evolution"><i class="fa fa-check"></i><b>4.7.1</b> Origin And evolution</a></li>
<li class="chapter" data-level="4.7.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#relationship-to-bacteria"><i class="fa fa-check"></i><b>4.7.2</b> Relationship To Bacteria</a></li>
<li class="chapter" data-level="4.7.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#relation-to-eukaryotes"><i class="fa fa-check"></i><b>4.7.3</b> Relation To Eukaryotes</a></li>
<li class="chapter" data-level="4.7.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#morphology"><i class="fa fa-check"></i><b>4.7.4</b> Morphology</a></li>
<li class="chapter" data-level="4.7.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#structure-composition-development-and-operation"><i class="fa fa-check"></i><b>4.7.5</b> Structure, Composition Development, And Operation</a></li>
<li class="chapter" data-level="4.7.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#cell-wall-and-flagella"><i class="fa fa-check"></i><b>4.7.6</b> Cell Wall And Flagella</a></li>
<li class="chapter" data-level="4.7.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#membranes"><i class="fa fa-check"></i><b>4.7.7</b> Membranes</a></li>
<li class="chapter" data-level="4.7.8" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-metabolism"><i class="fa fa-check"></i><b>4.7.8</b> Archaeal Metabolism</a></li>
<li class="chapter" data-level="4.7.9" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-genetics"><i class="fa fa-check"></i><b>4.7.9</b> Archaeal Genetics</a></li>
<li class="chapter" data-level="4.7.10" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-ecology"><i class="fa fa-check"></i><b>4.7.10</b> Archaeal Ecology</a></li>
<li class="chapter" data-level="4.7.11" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#significance-in-technology-and-industry-1"><i class="fa fa-check"></i><b>4.7.11</b> Significance In Technology And Industry</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><i class="fa fa-check"></i><b>5</b> An Introduction To Eukaryotic Cells And Microorganisms</a>
<ul>
<li class="chapter" data-level="5.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#characteristic-features-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.1</b> Characteristic Features Of Eukaryotic Cells</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cell-membrane"><i class="fa fa-check"></i><b>5.1.1</b> The Cell Membrane</a></li>
<li class="chapter" data-level="5.1.2" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cytoskeleton"><i class="fa fa-check"></i><b>5.1.2</b> The Cytoskeleton</a></li>
<li class="chapter" data-level="5.1.3" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-genetic-material"><i class="fa fa-check"></i><b>5.1.3</b> The Genetic Material</a></li>
<li class="chapter" data-level="5.1.4" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#organelles-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.1.4</b> Organelles of Eukaryotic Cells</a></li>
<li class="chapter" data-level="5.1.5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cell-nucleus"><i class="fa fa-check"></i><b>5.1.5</b> The Cell Nucleus</a></li>
<li class="chapter" data-level="5.1.6" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-mitochondria"><i class="fa fa-check"></i><b>5.1.6</b> The Mitochondria</a></li>
<li class="chapter" data-level="5.1.7" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-chloroplasts"><i class="fa fa-check"></i><b>5.1.7</b> The Chloroplasts</a></li>
<li class="chapter" data-level="5.1.8" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-endoplasmic-reticulum"><i class="fa fa-check"></i><b>5.1.8</b> The Endoplasmic Reticulum</a></li>
<li class="chapter" data-level="5.1.9" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-golgi-apparatus"><i class="fa fa-check"></i><b>5.1.9</b> The Golgi Apparatus</a></li>
<li class="chapter" data-level="5.1.10" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-ribosomes"><i class="fa fa-check"></i><b>5.1.10</b> The Ribosomes</a></li>
<li class="chapter" data-level="5.1.11" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#structures-outside-the-cell-membrane"><i class="fa fa-check"></i><b>5.1.11</b> Structures Outside The Cell Membrane</a></li>
<li class="chapter" data-level="5.1.12" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cellular-replication"><i class="fa fa-check"></i><b>5.1.12</b> Cellular Replication</a></li>
<li class="chapter" data-level="5.1.13" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#dna-repair"><i class="fa fa-check"></i><b>5.1.13</b> DNA Repair</a></li>
<li class="chapter" data-level="5.1.14" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cellular-growth-and-metabolism"><i class="fa fa-check"></i><b>5.1.14</b> Cellular Growth And Metabolism</a></li>
<li class="chapter" data-level="5.1.15" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protein-synthesis"><i class="fa fa-check"></i><b>5.1.15</b> Protein Synthesis</a></li>
<li class="chapter" data-level="5.1.16" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cell-motility"><i class="fa fa-check"></i><b>5.1.16</b> Cell Motility</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#origin-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.2</b> Origin Of Eukaryotic Cells</a></li>
<li class="chapter" data-level="5.3" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protists-1"><i class="fa fa-check"></i><b>5.3</b> Protists</a></li>
<li class="chapter" data-level="5.4" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protozoa"><i class="fa fa-check"></i><b>5.4</b> Protozoa</a></li>
<li class="chapter" data-level="5.5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#flagellates"><i class="fa fa-check"></i><b>5.5</b> Flagellates</a></li>
<li class="chapter" data-level="5.6" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#ciliates"><i class="fa fa-check"></i><b>5.6</b> Ciliates</a></li>
<li class="chapter" data-level="5.7" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#amoebozoa"><i class="fa fa-check"></i><b>5.7</b> Amoebozoa</a></li>
<li class="chapter" data-level="5.8" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#algae"><i class="fa fa-check"></i><b>5.8</b> Algae</a></li>
<li class="chapter" data-level="5.9" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#slime-molds"><i class="fa fa-check"></i><b>5.9</b> Slime Molds</a>
<ul>
<li class="chapter" data-level="5.9.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#reproduction-of-protists"><i class="fa fa-check"></i><b>5.9.1</b> Reproduction of Protists</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html"><i class="fa fa-check"></i><b>6</b> An Introduction To Viruses</a>
<ul>
<li class="chapter" data-level="6.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#theories-of-the-origin-of-viruses"><i class="fa fa-check"></i><b>6.1</b> Theories Of The Origin Of Viruses</a></li>
<li class="chapter" data-level="6.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#structure-and-morphology-of-viruses"><i class="fa fa-check"></i><b>6.2</b> Structure And Morphology Of Viruses</a></li>
<li class="chapter" data-level="6.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#replication-of-viruses"><i class="fa fa-check"></i><b>6.3</b> Replication Of Viruses</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#role-of-viruses-in-human-disease"><i class="fa fa-check"></i><b>6.3.1</b> Role Of Viruses In Human Disease</a></li>
<li class="chapter" data-level="6.3.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#animal-viruses"><i class="fa fa-check"></i><b>6.3.2</b> Animal Viruses</a></li>
<li class="chapter" data-level="6.3.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#plant-viruses"><i class="fa fa-check"></i><b>6.3.3</b> Plant Viruses</a></li>
<li class="chapter" data-level="6.3.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#bacterial-viruses"><i class="fa fa-check"></i><b>6.3.4</b> Bacterial Viruses</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#classification-of-viruses"><i class="fa fa-check"></i><b>6.4</b> Classification Of Viruses</a></li>
<li class="chapter" data-level="6.5" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#the-baltimore-classification-of-viruses"><i class="fa fa-check"></i><b>6.5</b> The Baltimore Classification Of Viruses</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html"><i class="fa fa-check"></i><b>7</b> Microbial Nutrition, Ecology And Growth</a>
<ul>
<li class="chapter" data-level="7.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-energy"><i class="fa fa-check"></i><b>7.1</b> Primary Sources Of Energy</a></li>
<li class="chapter" data-level="7.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-reducing-equivalents"><i class="fa fa-check"></i><b>7.2</b> Primary Sources Of Reducing Equivalents</a></li>
<li class="chapter" data-level="7.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-carbon"><i class="fa fa-check"></i><b>7.3</b> Primary Sources of Carbon</a></li>
<li class="chapter" data-level="7.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#nutrients"><i class="fa fa-check"></i><b>7.4</b> Nutrients</a></li>
<li class="chapter" data-level="7.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#the-nutrient-cycle"><i class="fa fa-check"></i><b>7.5</b> The Nutrient Cycle</a></li>
<li class="chapter" data-level="7.6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#transport-and-movement-of-substances-across-the-cell-membrane"><i class="fa fa-check"></i><b>7.6</b> Transport And Movement Of Substances Across The Cell Membrane</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#diffusion"><i class="fa fa-check"></i><b>7.6.1</b> Diffusion</a></li>
<li class="chapter" data-level="7.6.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#facilitated-diffusion"><i class="fa fa-check"></i><b>7.6.2</b> Facilitated Diffusion</a></li>
<li class="chapter" data-level="7.6.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#osmosis"><i class="fa fa-check"></i><b>7.6.3</b> Osmosis</a></li>
<li class="chapter" data-level="7.6.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#active-transport"><i class="fa fa-check"></i><b>7.6.4</b> Active Transport</a></li>
<li class="chapter" data-level="7.6.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#bulk-transport"><i class="fa fa-check"></i><b>7.6.5</b> Bulk transport</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#microbial-growth"><i class="fa fa-check"></i><b>7.7</b> Microbial Growth</a></li>
<li class="chapter" data-level="7.8" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#cell-counting"><i class="fa fa-check"></i><b>7.8</b> Cell Counting</a>
<ul>
<li class="chapter" data-level="7.8.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#counting-chamber"><i class="fa fa-check"></i><b>7.8.1</b> Counting Chamber</a></li>
<li class="chapter" data-level="7.8.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#plating-and-the-colony-forming-unit-counting"><i class="fa fa-check"></i><b>7.8.2</b> Plating And The Colony-Forming Unit Counting</a></li>
<li class="chapter" data-level="7.8.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#electrical-resistance"><i class="fa fa-check"></i><b>7.8.3</b> Electrical Resistance</a></li>
<li class="chapter" data-level="7.8.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#flow-cytometry"><i class="fa fa-check"></i><b>7.8.4</b> Flow Cytometry</a></li>
<li class="chapter" data-level="7.8.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#spectrophotometry"><i class="fa fa-check"></i><b>7.8.5</b> Spectrophotometry</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#environmental-factors-influencing-growth"><i class="fa fa-check"></i><b>7.9</b> Environmental Factors Influencing Growth</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#nutrient-availability"><i class="fa fa-check"></i><b>7.9.1</b> Nutrient Availability</a></li>
<li class="chapter" data-level="7.9.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#temperature"><i class="fa fa-check"></i><b>7.9.2</b> Temperature</a></li>
<li class="chapter" data-level="7.9.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#acidity"><i class="fa fa-check"></i><b>7.9.3</b> Acidity</a></li>
<li class="chapter" data-level="7.9.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#water-activity"><i class="fa fa-check"></i><b>7.9.4</b> Water activity</a></li>
<li class="chapter" data-level="7.9.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#oxygen"><i class="fa fa-check"></i><b>7.9.5</b> Oxygen</a></li>
<li class="chapter" data-level="7.9.6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#toxic-compounds"><i class="fa fa-check"></i><b>7.9.6</b> Toxic compounds</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html"><i class="fa fa-check"></i><b>8</b> An Introduction To Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="8.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#thermodynamics-of-living-organisms"><i class="fa fa-check"></i><b>8.1</b> Thermodynamics of Living Organisms</a></li>
<li class="chapter" data-level="8.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#metabolism-1"><i class="fa fa-check"></i><b>8.2</b> Metabolism</a></li>
<li class="chapter" data-level="8.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#types-of-microbial-metabolism"><i class="fa fa-check"></i><b>8.3</b> Types Of Microbial Metabolism</a></li>
<li class="chapter" data-level="8.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#phototrophy"><i class="fa fa-check"></i><b>8.4</b> Phototrophy</a></li>
<li class="chapter" data-level="8.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#chemolithotrophy"><i class="fa fa-check"></i><b>8.5</b> Chemolithotrophy</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#hydrogen-oxidation"><i class="fa fa-check"></i><b>8.5.1</b> Hydrogen oxidation</a></li>
<li class="chapter" data-level="8.5.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#sulfur-oxidation"><i class="fa fa-check"></i><b>8.5.2</b> Sulfur oxidation</a></li>
<li class="chapter" data-level="8.5.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#ferrous-iron-fe2-oxidation"><i class="fa fa-check"></i><b>8.5.3</b> Ferrous Iron (Fe<sup>2</sup>) Oxidation</a></li>
<li class="chapter" data-level="8.5.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nitrification"><i class="fa fa-check"></i><b>8.5.4</b> Nitrification</a></li>
<li class="chapter" data-level="8.5.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nitrogen-fixation"><i class="fa fa-check"></i><b>8.5.5</b> Nitrogen Fixation</a></li>
<li class="chapter" data-level="8.5.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#catabolism"><i class="fa fa-check"></i><b>8.5.6</b> Catabolism</a></li>
<li class="chapter" data-level="8.5.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#digestion"><i class="fa fa-check"></i><b>8.5.7</b> Digestion</a></li>
<li class="chapter" data-level="8.5.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-organic-compounds"><i class="fa fa-check"></i><b>8.5.8</b> Energy From Organic Compounds</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-transformations"><i class="fa fa-check"></i><b>8.6</b> Energy Transformations</a>
<ul>
<li class="chapter" data-level="8.6.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-phosphorylation"><i class="fa fa-check"></i><b>8.6.1</b> Oxidative Phosphorylation</a></li>
<li class="chapter" data-level="8.6.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-inorganic-compounds"><i class="fa fa-check"></i><b>8.6.2</b> Energy From Inorganic Compounds</a></li>
<li class="chapter" data-level="8.6.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-light"><i class="fa fa-check"></i><b>8.6.3</b> Energy From Light</a></li>
<li class="chapter" data-level="8.6.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#anabolism"><i class="fa fa-check"></i><b>8.6.4</b> Anabolism</a></li>
</ul></li>
<li class="chapter" data-level="8.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#enzymes"><i class="fa fa-check"></i><b>8.7</b> Enzymes</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#classification-and-nomenclature-of-enzymes"><i class="fa fa-check"></i><b>8.7.1</b> Classification And Nomenclature Of Enzymes</a></li>
<li class="chapter" data-level="8.7.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#stucture-of-enzymes"><i class="fa fa-check"></i><b>8.7.2</b> Stucture Of Enzymes</a></li>
<li class="chapter" data-level="8.7.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#mechanism-of-action-of-enzymes"><i class="fa fa-check"></i><b>8.7.3</b> Mechanism Of Action Of Enzymes</a></li>
<li class="chapter" data-level="8.7.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#allosteric-modulation"><i class="fa fa-check"></i><b>8.7.4</b> Allosteric Modulation</a></li>
<li class="chapter" data-level="8.7.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cofactors-of-enzymes-and-coenzymes"><i class="fa fa-check"></i><b>8.7.5</b> Cofactors Of Enzymes And Coenzymes</a></li>
<li class="chapter" data-level="8.7.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#inhibition-of-enzymes"><i class="fa fa-check"></i><b>8.7.6</b> Inhibition Of Enzymes</a></li>
<li class="chapter" data-level="8.7.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#factors-affecting-enzyme-activity"><i class="fa fa-check"></i><b>8.7.7</b> Factors Affecting Enzyme Activity</a></li>
<li class="chapter" data-level="8.7.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#control-of-activity"><i class="fa fa-check"></i><b>8.7.8</b> Control Of Activity</a></li>
<li class="chapter" data-level="8.7.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#post-translational-modification"><i class="fa fa-check"></i><b>8.7.9</b> Post-translational Modification</a></li>
<li class="chapter" data-level="8.7.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#quantity"><i class="fa fa-check"></i><b>8.7.10</b> Quantity</a></li>
<li class="chapter" data-level="8.7.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#subcellular-distribution"><i class="fa fa-check"></i><b>8.7.11</b> Subcellular Distribution</a></li>
<li class="chapter" data-level="8.7.12" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#organ-specialization"><i class="fa fa-check"></i><b>8.7.12</b> Organ Specialization</a></li>
<li class="chapter" data-level="8.7.13" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#coenzymes"><i class="fa fa-check"></i><b>8.7.13</b> Coenzymes</a></li>
<li class="chapter" data-level="8.7.14" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#mineral-cofactors"><i class="fa fa-check"></i><b>8.7.14</b> Mineral Cofactors</a></li>
</ul></li>
<li class="chapter" data-level="8.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#heterotrophic-microbial-metabolism"><i class="fa fa-check"></i><b>8.8</b> Heterotrophic Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="8.8.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#carbon-fixation"><i class="fa fa-check"></i><b>8.8.1</b> Carbon Fixation</a></li>
<li class="chapter" data-level="8.8.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#carbohydrates-and-glycans"><i class="fa fa-check"></i><b>8.8.2</b> Carbohydrates And Glycans</a></li>
<li class="chapter" data-level="8.8.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#fatty-acids-isoprenoids-and-sterol"><i class="fa fa-check"></i><b>8.8.3</b> Fatty Acids, Isoprenoids and Sterol</a></li>
<li class="chapter" data-level="8.8.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#proteins-1"><i class="fa fa-check"></i><b>8.8.4</b> Proteins</a></li>
<li class="chapter" data-level="8.8.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nucleotide-synthesis-and-salvage"><i class="fa fa-check"></i><b>8.8.5</b> Nucleotide Synthesis And Salvage</a></li>
</ul></li>
<li class="chapter" data-level="8.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthesis-and-cellular-respiration"><i class="fa fa-check"></i><b>8.9</b> Photosynthesis And Cellular Respiration</a>
<ul>
<li class="chapter" data-level="8.9.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#the-electron-transport-chain-and-chemiosmosis"><i class="fa fa-check"></i><b>8.9.1</b> The Electron Transport Chain And Chemiosmosis</a></li>
<li class="chapter" data-level="8.9.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthesis"><i class="fa fa-check"></i><b>8.9.2</b> Photosynthesis</a></li>
<li class="chapter" data-level="8.9.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthetic-membranes-and-organelles"><i class="fa fa-check"></i><b>8.9.3</b> Photosynthetic Membranes And Organelles</a></li>
<li class="chapter" data-level="8.9.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#z-scheme"><i class="fa fa-check"></i><b>8.9.4</b> Z Scheme</a></li>
<li class="chapter" data-level="8.9.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#water-photolysis"><i class="fa fa-check"></i><b>8.9.5</b> Water Photolysis</a></li>
<li class="chapter" data-level="8.9.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#calvin-cycle"><i class="fa fa-check"></i><b>8.9.6</b> Calvin Cycle</a></li>
<li class="chapter" data-level="8.9.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#symbiosis-and-the-origin-of-chloroplasts"><i class="fa fa-check"></i><b>8.9.7</b> Symbiosis And The Origin of Chloroplasts</a></li>
<li class="chapter" data-level="8.9.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cyanobacteria-and-the-evolution-of-photosynthesis"><i class="fa fa-check"></i><b>8.9.8</b> Cyanobacteria And The Evolution of Photosynthesis</a></li>
<li class="chapter" data-level="8.9.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cellular-respiration"><i class="fa fa-check"></i><b>8.9.9</b> Cellular Respiration</a></li>
<li class="chapter" data-level="8.9.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#aerobic-respiration"><i class="fa fa-check"></i><b>8.9.10</b> Aerobic Respiration</a></li>
<li class="chapter" data-level="8.9.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#glycolysis"><i class="fa fa-check"></i><b>8.9.11</b> Glycolysis</a></li>
<li class="chapter" data-level="8.9.12" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-decarboxylation-of-pyruvate"><i class="fa fa-check"></i><b>8.9.12</b> Oxidative Decarboxylation of Pyruvate</a></li>
<li class="chapter" data-level="8.9.13" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#the-citric-acid-cycle"><i class="fa fa-check"></i><b>8.9.13</b> The Citric Acid Cycle</a></li>
<li class="chapter" data-level="8.9.14" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-phosphorylation-1"><i class="fa fa-check"></i><b>8.9.14</b> Oxidative Phosphorylation</a></li>
</ul></li>
<li class="chapter" data-level="8.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#fermentation"><i class="fa fa-check"></i><b>8.10</b> Fermentation</a></li>
<li class="chapter" data-level="8.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#anaerobic-respiration"><i class="fa fa-check"></i><b>8.11</b> Anaerobic Respiration</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="microbial-genetics.html"><a href="microbial-genetics.html"><i class="fa fa-check"></i><b>9</b> Microbial Genetics</a>
<ul>
<li class="chapter" data-level="9.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#introduction-to-genetics-and-genes"><i class="fa fa-check"></i><b>9.1</b> Introduction To Genetics And Genes</a>
<ul>
<li class="chapter" data-level="9.1.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mendelian-classical-genetics"><i class="fa fa-check"></i><b>9.1.1</b> Mendelian (Classical) Genetics</a></li>
<li class="chapter" data-level="9.1.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#molecular-genetics"><i class="fa fa-check"></i><b>9.1.2</b> Molecular Genetics</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#the-genetic-code"><i class="fa fa-check"></i><b>9.2</b> The Genetic Code</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-expression"><i class="fa fa-check"></i><b>9.2.1</b> Gene Expression</a></li>
<li class="chapter" data-level="9.2.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#nature-and-nurture"><i class="fa fa-check"></i><b>9.2.2</b> Nature And Nurture</a></li>
<li class="chapter" data-level="9.2.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-regulation"><i class="fa fa-check"></i><b>9.2.3</b> Gene Regulation</a></li>
<li class="chapter" data-level="9.2.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genetic-change"><i class="fa fa-check"></i><b>9.2.4</b> Genetic Change</a></li>
<li class="chapter" data-level="9.2.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#natural-selection-and-evolution"><i class="fa fa-check"></i><b>9.2.5</b> Natural Selection And Evolution</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-genetics-1"><i class="fa fa-check"></i><b>9.3</b> Bacterial Genetics</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-transformation"><i class="fa fa-check"></i><b>9.3.1</b> Bacterial Transformation</a></li>
<li class="chapter" data-level="9.3.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#practical-aspects-of-transformation-in-molecular-biology"><i class="fa fa-check"></i><b>9.3.2</b> Practical aspects of transformation in molecular biology</a></li>
<li class="chapter" data-level="9.3.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#selection-and-screening-in-plasmid-transformation"><i class="fa fa-check"></i><b>9.3.3</b> Selection and screening in plasmid transformation</a></li>
<li class="chapter" data-level="9.3.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-conjugation"><i class="fa fa-check"></i><b>9.3.4</b> Bacterial Conjugation</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-and-function-of-deoxyribonucleic-acid"><i class="fa fa-check"></i><b>9.4</b> Structure And Function Of Deoxyribonucleic Acid</a>
<ul>
<li class="chapter" data-level="9.4.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-damage"><i class="fa fa-check"></i><b>9.4.1</b> DNA Damage</a></li>
<li class="chapter" data-level="9.4.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genes-and-genomes"><i class="fa fa-check"></i><b>9.4.2</b> Genes And Genomes</a></li>
<li class="chapter" data-level="9.4.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-translation"><i class="fa fa-check"></i><b>9.4.3</b> Transcription And Translation</a></li>
<li class="chapter" data-level="9.4.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-replication"><i class="fa fa-check"></i><b>9.4.4</b> DNA Replication</a></li>
<li class="chapter" data-level="9.4.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-dna-replication"><i class="fa fa-check"></i><b>9.4.5</b> Eukaryotic DNA Replication</a></li>
<li class="chapter" data-level="9.4.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation"><i class="fa fa-check"></i><b>9.4.6</b> Initiation</a></li>
<li class="chapter" data-level="9.4.7" data-path="microbial-genetics.html"><a href="microbial-genetics.html#pre-replication-complex"><i class="fa fa-check"></i><b>9.4.7</b> Pre-Replication Complex</a></li>
<li class="chapter" data-level="9.4.8" data-path="microbial-genetics.html"><a href="microbial-genetics.html#preinitiation-complex"><i class="fa fa-check"></i><b>9.4.8</b> Preinitiation Complex</a></li>
<li class="chapter" data-level="9.4.9" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation"><i class="fa fa-check"></i><b>9.4.9</b> Elongation</a></li>
<li class="chapter" data-level="9.4.10" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-fork"><i class="fa fa-check"></i><b>9.4.10</b> Replication Fork</a></li>
<li class="chapter" data-level="9.4.11" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-of-the-leading-strand"><i class="fa fa-check"></i><b>9.4.11</b> Replication Of The Leading Strand</a></li>
<li class="chapter" data-level="9.4.12" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-of-the-lagging-strand"><i class="fa fa-check"></i><b>9.4.12</b> Replication Of The Lagging Strand</a></li>
<li class="chapter" data-level="9.4.13" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-replication-proteins"><i class="fa fa-check"></i><b>9.4.13</b> DNA Replication Proteins</a></li>
<li class="chapter" data-level="9.4.14" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination"><i class="fa fa-check"></i><b>9.4.14</b> Termination</a></li>
<li class="chapter" data-level="9.4.15" data-path="microbial-genetics.html"><a href="microbial-genetics.html#regulation-of-dna-replication"><i class="fa fa-check"></i><b>9.4.15</b> Regulation Of DNA Replication</a></li>
<li class="chapter" data-level="9.4.16" data-path="microbial-genetics.html"><a href="microbial-genetics.html#interactions-of-dna-with-proteins"><i class="fa fa-check"></i><b>9.4.16</b> Interactions Of DNA with Proteins</a></li>
<li class="chapter" data-level="9.4.17" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-binding-proteins"><i class="fa fa-check"></i><b>9.4.17</b> DNA-Binding Proteins</a></li>
<li class="chapter" data-level="9.4.18" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>9.4.18</b> DNA-modifying Enzymes</a></li>
<li class="chapter" data-level="9.4.19" data-path="microbial-genetics.html"><a href="microbial-genetics.html#nucleases-and-ligases"><i class="fa fa-check"></i><b>9.4.19</b> Nucleases And Ligases</a></li>
<li class="chapter" data-level="9.4.20" data-path="microbial-genetics.html"><a href="microbial-genetics.html#topoisomerases-and-helicases"><i class="fa fa-check"></i><b>9.4.20</b> Topoisomerases And Helicases</a></li>
<li class="chapter" data-level="9.4.21" data-path="microbial-genetics.html"><a href="microbial-genetics.html#polymerases"><i class="fa fa-check"></i><b>9.4.21</b> Polymerases</a></li>
<li class="chapter" data-level="9.4.22" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-recombination"><i class="fa fa-check"></i><b>9.4.22</b> DNA Recombination</a></li>
<li class="chapter" data-level="9.4.23" data-path="microbial-genetics.html"><a href="microbial-genetics.html#evolutionary-history-of-dna"><i class="fa fa-check"></i><b>9.4.23</b> Evolutionary History Of DNA</a></li>
<li class="chapter" data-level="9.4.24" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genetic-engineering"><i class="fa fa-check"></i><b>9.4.24</b> Genetic Engineering</a></li>
<li class="chapter" data-level="9.4.25" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-profiling"><i class="fa fa-check"></i><b>9.4.25</b> Dna Profiling</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-and-function-of-ribonucleic-acid"><i class="fa fa-check"></i><b>9.5</b> Structure And Function Of Ribonucleic Acid</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#comparison-with-dna"><i class="fa fa-check"></i><b>9.5.1</b> Comparison With DNA</a></li>
<li class="chapter" data-level="9.5.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-of-rna"><i class="fa fa-check"></i><b>9.5.2</b> Structure Of RNA</a></li>
<li class="chapter" data-level="9.5.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#synthesis-of-rna"><i class="fa fa-check"></i><b>9.5.3</b> Synthesis Of RNA</a></li>
<li class="chapter" data-level="9.5.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#coding-and-non-coding-rna"><i class="fa fa-check"></i><b>9.5.4</b> Coding And Non-coding RNA</a></li>
<li class="chapter" data-level="9.5.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#regulatory-rna"><i class="fa fa-check"></i><b>9.5.5</b> Regulatory RNA</a></li>
<li class="chapter" data-level="9.5.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-interference-by-mirnas"><i class="fa fa-check"></i><b>9.5.6</b> RNA Interference By miRNAs</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-expression-1"><i class="fa fa-check"></i><b>9.6</b> Gene Expression</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-rna-processing"><i class="fa fa-check"></i><b>9.6.1</b> Transcription And RNA Processing</a></li>
<li class="chapter" data-level="9.6.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-1"><i class="fa fa-check"></i><b>9.6.2</b> Initiation</a></li>
<li class="chapter" data-level="9.6.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation-1"><i class="fa fa-check"></i><b>9.6.3</b> Elongation</a></li>
<li class="chapter" data-level="9.6.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination-1"><i class="fa fa-check"></i><b>9.6.4</b> Termination</a></li>
<li class="chapter" data-level="9.6.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#inhibitors-of-transcription"><i class="fa fa-check"></i><b>9.6.5</b> Inhibitors Of Transcription</a></li>
<li class="chapter" data-level="9.6.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-factors"><i class="fa fa-check"></i><b>9.6.6</b> Transcription Factors</a></li>
<li class="chapter" data-level="9.6.7" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-processing"><i class="fa fa-check"></i><b>9.6.7</b> RNA Processing</a></li>
<li class="chapter" data-level="9.6.8" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mrna-processing"><i class="fa fa-check"></i><b>9.6.8</b> mRNA Processing</a></li>
<li class="chapter" data-level="9.6.9" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing"><i class="fa fa-check"></i><b>9.6.9</b> 5’ Processing</a></li>
<li class="chapter" data-level="9.6.10" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-1"><i class="fa fa-check"></i><b>9.6.10</b> 3’ Processing</a></li>
<li class="chapter" data-level="9.6.11" data-path="microbial-genetics.html"><a href="microbial-genetics.html#intron-splicing"><i class="fa fa-check"></i><b>9.6.11</b> Intron Splicing</a></li>
<li class="chapter" data-level="9.6.12" data-path="microbial-genetics.html"><a href="microbial-genetics.html#reverse-transcription"><i class="fa fa-check"></i><b>9.6.12</b> Reverse Transcription</a></li>
<li class="chapter" data-level="9.6.13" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-export"><i class="fa fa-check"></i><b>9.6.13</b> RNA Export</a></li>
<li class="chapter" data-level="9.6.14" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-rna-processing-1"><i class="fa fa-check"></i><b>9.6.14</b> Transcription And RNA Processing</a></li>
<li class="chapter" data-level="9.6.15" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-2"><i class="fa fa-check"></i><b>9.6.15</b> Initiation</a></li>
<li class="chapter" data-level="9.6.16" data-path="microbial-genetics.html"><a href="microbial-genetics.html#inhibitors-of-transcription-1"><i class="fa fa-check"></i><b>9.6.16</b> Inhibitors of transcription</a></li>
<li class="chapter" data-level="9.6.17" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-factors-1"><i class="fa fa-check"></i><b>9.6.17</b> Transcription Factors</a></li>
<li class="chapter" data-level="9.6.18" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-processing-1"><i class="fa fa-check"></i><b>9.6.18</b> RNA Processing</a></li>
<li class="chapter" data-level="9.6.19" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mrna-processing-1"><i class="fa fa-check"></i><b>9.6.19</b> mRNA Processing</a></li>
<li class="chapter" data-level="9.6.20" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-2"><i class="fa fa-check"></i><b>9.6.20</b> 5’ Processing</a></li>
<li class="chapter" data-level="9.6.21" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-3"><i class="fa fa-check"></i><b>9.6.21</b> 3’ Processing</a></li>
<li class="chapter" data-level="9.6.22" data-path="microbial-genetics.html"><a href="microbial-genetics.html#intron-splicing-1"><i class="fa fa-check"></i><b>9.6.22</b> Intron Splicing</a></li>
<li class="chapter" data-level="9.6.23" data-path="microbial-genetics.html"><a href="microbial-genetics.html#reverse-transcription-1"><i class="fa fa-check"></i><b>9.6.23</b> Reverse Transcription</a></li>
<li class="chapter" data-level="9.6.24" data-path="microbial-genetics.html"><a href="microbial-genetics.html#translation-and-the-genetic-code"><i class="fa fa-check"></i><b>9.6.24</b> Translation And The Genetic Code</a></li>
<li class="chapter" data-level="9.6.25" data-path="microbial-genetics.html"><a href="microbial-genetics.html#ribosomes"><i class="fa fa-check"></i><b>9.6.25</b> Ribosomes</a></li>
<li class="chapter" data-level="9.6.26" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-ribosomes"><i class="fa fa-check"></i><b>9.6.26</b> Bacterial Ribosomes</a></li>
<li class="chapter" data-level="9.6.27" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-ribosomes"><i class="fa fa-check"></i><b>9.6.27</b> Eukaryotic Ribosomes</a></li>
<li class="chapter" data-level="9.6.28" data-path="microbial-genetics.html"><a href="microbial-genetics.html#translation"><i class="fa fa-check"></i><b>9.6.28</b> Translation</a></li>
<li class="chapter" data-level="9.6.29" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-translation"><i class="fa fa-check"></i><b>9.6.29</b> Eukaryotic Translation</a></li>
<li class="chapter" data-level="9.6.30" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-3"><i class="fa fa-check"></i><b>9.6.30</b> Initiation</a></li>
<li class="chapter" data-level="9.6.31" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation-2"><i class="fa fa-check"></i><b>9.6.31</b> Elongation</a></li>
<li class="chapter" data-level="9.6.32" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination-2"><i class="fa fa-check"></i><b>9.6.32</b> Termination</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html"><i class="fa fa-check"></i><b>10</b> Genetic Engineering</a>
<ul>
<li class="chapter" data-level="10.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genetically-modified-organisms"><i class="fa fa-check"></i><b>10.1</b> Genetically Modified Organisms</a>
<ul>
<li class="chapter" data-level="10.1.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#production"><i class="fa fa-check"></i><b>10.1.1</b> Production</a></li>
<li class="chapter" data-level="10.1.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#bacteria-2"><i class="fa fa-check"></i><b>10.1.2</b> Bacteria</a></li>
<li class="chapter" data-level="10.1.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#viruses"><i class="fa fa-check"></i><b>10.1.3</b> Viruses</a></li>
<li class="chapter" data-level="10.1.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#fungi-1"><i class="fa fa-check"></i><b>10.1.4</b> Fungi</a></li>
<li class="chapter" data-level="10.1.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#plants-1"><i class="fa fa-check"></i><b>10.1.5</b> Plants</a></li>
<li class="chapter" data-level="10.1.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#crops"><i class="fa fa-check"></i><b>10.1.6</b> Crops</a></li>
<li class="chapter" data-level="10.1.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#animals"><i class="fa fa-check"></i><b>10.1.7</b> Animals</a></li>
<li class="chapter" data-level="10.1.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#laws-and-regulations"><i class="fa fa-check"></i><b>10.1.8</b> Laws and regulations</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-therapy"><i class="fa fa-check"></i><b>10.2</b> Gene therapy</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#laws-and-regulations-1"><i class="fa fa-check"></i><b>10.2.1</b> Laws and regulations</a></li>
<li class="chapter" data-level="10.2.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-isolation-and-cloning"><i class="fa fa-check"></i><b>10.2.2</b> Gene Isolation And Cloning</a></li>
<li class="chapter" data-level="10.2.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#inserting-dna-into-the-host-genome"><i class="fa fa-check"></i><b>10.2.3</b> Inserting DNA Into The Host Genome</a></li>
<li class="chapter" data-level="10.2.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#applications-of-genetic-editing"><i class="fa fa-check"></i><b>10.2.4</b> Applications Of Genetic Editing</a></li>
<li class="chapter" data-level="10.2.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#regulation-of-genetic-engineering"><i class="fa fa-check"></i><b>10.2.5</b> Regulation Of Genetic Engineering</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#recombinant-dna-technology"><i class="fa fa-check"></i><b>10.3</b> Recombinant DNA Technology</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#molecular-cloning"><i class="fa fa-check"></i><b>10.3.1</b> Molecular Cloning</a></li>
<li class="chapter" data-level="10.3.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#procedures"><i class="fa fa-check"></i><b>10.3.2</b> Procedures</a></li>
<li class="chapter" data-level="10.3.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#mutagenesis"><i class="fa fa-check"></i><b>10.3.3</b> Mutagenesis</a></li>
<li class="chapter" data-level="10.3.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#random-mutagenesis"><i class="fa fa-check"></i><b>10.3.4</b> Random Mutagenesis</a></li>
<li class="chapter" data-level="10.3.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>10.3.5</b> Site-Directed Mutagenesis</a></li>
<li class="chapter" data-level="10.3.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-synthesis"><i class="fa fa-check"></i><b>10.3.6</b> Gene Synthesis</a></li>
<li class="chapter" data-level="10.3.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knockout"><i class="fa fa-check"></i><b>10.3.7</b> Gene Knockout</a></li>
<li class="chapter" data-level="10.3.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#conditional-gene-knockout"><i class="fa fa-check"></i><b>10.3.8</b> Conditional Gene Knockout</a></li>
<li class="chapter" data-level="10.3.9" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knock-in"><i class="fa fa-check"></i><b>10.3.9</b> Gene Knock-In</a></li>
<li class="chapter" data-level="10.3.10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#rna-interference"><i class="fa fa-check"></i><b>10.3.10</b> RNA interference</a></li>
<li class="chapter" data-level="10.3.11" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#microrna"><i class="fa fa-check"></i><b>10.3.11</b> MicroRNA</a></li>
<li class="chapter" data-level="10.3.12" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#transcriptional-silencing"><i class="fa fa-check"></i><b>10.3.12</b> Transcriptional Silencing</a></li>
<li class="chapter" data-level="10.3.13" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knockdown"><i class="fa fa-check"></i><b>10.3.13</b> Gene Knockdown</a></li>
<li class="chapter" data-level="10.3.14" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#cre-lox-recombination"><i class="fa fa-check"></i><b>10.3.14</b> Cre-Lox Recombination</a></li>
<li class="chapter" data-level="10.3.15" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#crispr-gene-editing"><i class="fa fa-check"></i><b>10.3.15</b> CRISPR Gene Editing</a></li>
<li class="chapter" data-level="10.3.16" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-drive"><i class="fa fa-check"></i><b>10.3.16</b> Gene Drive</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#human-germline-modification"><i class="fa fa-check"></i><b>10.4</b> Human Germline Modification</a></li>
<li class="chapter" data-level="10.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gain-of-function-research"><i class="fa fa-check"></i><b>10.5</b> Gain-of-function Research</a></li>
<li class="chapter" data-level="10.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genomics"><i class="fa fa-check"></i><b>10.6</b> Genomics</a>
<ul>
<li class="chapter" data-level="10.6.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#complete-genomes"><i class="fa fa-check"></i><b>10.6.1</b> Complete Genomes</a></li>
<li class="chapter" data-level="10.6.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#the-omics-revolution"><i class="fa fa-check"></i><b>10.6.2</b> The “Omics” Revolution</a></li>
<li class="chapter" data-level="10.6.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-analysis"><i class="fa fa-check"></i><b>10.6.3</b> Genome Analysis</a></li>
<li class="chapter" data-level="10.6.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-annotation"><i class="fa fa-check"></i><b>10.6.4</b> Genome Annotation</a></li>
<li class="chapter" data-level="10.6.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#functional-genomics"><i class="fa fa-check"></i><b>10.6.5</b> Functional Genomics</a></li>
<li class="chapter" data-level="10.6.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#structural-genomics"><i class="fa fa-check"></i><b>10.6.6</b> Structural Genomics</a></li>
<li class="chapter" data-level="10.6.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#epigenomics"><i class="fa fa-check"></i><b>10.6.7</b> Epigenomics</a></li>
<li class="chapter" data-level="10.6.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#metagenomics"><i class="fa fa-check"></i><b>10.6.8</b> Metagenomics</a></li>
<li class="chapter" data-level="10.6.9" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#applications-of-genomics"><i class="fa fa-check"></i><b>10.6.9</b> Applications Of Genomics</a></li>
<li class="chapter" data-level="10.6.10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genomic-medicine"><i class="fa fa-check"></i><b>10.6.10</b> Genomic Medicine</a></li>
<li class="chapter" data-level="10.6.11" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#synthetic-biology-and-bioengineering"><i class="fa fa-check"></i><b>10.6.11</b> Synthetic Biology And Bioengineering</a></li>
<li class="chapter" data-level="10.6.12" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#conservation-genomics"><i class="fa fa-check"></i><b>10.6.12</b> Conservation Genomics</a></li>
<li class="chapter" data-level="10.6.13" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-wide-association-study"><i class="fa fa-check"></i><b>10.6.13</b> Genome-Wide Association Study</a></li>
</ul></li>
<li class="chapter" data-level="10.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genetic-testing"><i class="fa fa-check"></i><b>10.7</b> Genetic Testing</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html"><i class="fa fa-check"></i><b>11</b> Physical And Chemical Agents for Microbial Control</a>
<ul>
<li class="chapter" data-level="11.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#disinfectant"><i class="fa fa-check"></i><b>11.1</b> Disinfectant</a></li>
<li class="chapter" data-level="11.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antiseptic"><i class="fa fa-check"></i><b>11.2</b> Antiseptic</a></li>
<li class="chapter" data-level="11.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#sterilization"><i class="fa fa-check"></i><b>11.3</b> Sterilization</a></li>
<li class="chapter" data-level="11.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#pasteurization"><i class="fa fa-check"></i><b>11.4</b> Pasteurization</a></li>
<li class="chapter" data-level="11.5" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antibiotics"><i class="fa fa-check"></i><b>11.5</b> Antibiotics</a></li>
<li class="chapter" data-level="11.6" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#classes-of-antibiotics"><i class="fa fa-check"></i><b>11.6</b> Classes of Antibiotics</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html"><i class="fa fa-check"></i><b>12</b> Drugs, Microbes, And The Host</a>
<ul>
<li class="chapter" data-level="12.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antibacterials"><i class="fa fa-check"></i><b>12.1</b> Antibacterials</a></li>
<li class="chapter" data-level="12.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#pathogenic-bacteria"><i class="fa fa-check"></i><b>12.2</b> Pathogenic bacteria</a></li>
<li class="chapter" data-level="12.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#pathogenic-susceptibility"><i class="fa fa-check"></i><b>12.3</b> Pathogenic susceptibility</a></li>
<li class="chapter" data-level="12.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#infection"><i class="fa fa-check"></i><b>12.4</b> Infection</a></li>
<li class="chapter" data-level="12.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#historic-pandemics"><i class="fa fa-check"></i><b>12.5</b> Historic pandemics</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html"><i class="fa fa-check"></i><b>13</b> An Introduction To Host Defenses And Innate Immune Systems</a>
<ul>
<li class="chapter" data-level="13.1" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#layered-defense"><i class="fa fa-check"></i><b>13.1</b> Layered defense</a></li>
<li class="chapter" data-level="13.2" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#surface-barriers"><i class="fa fa-check"></i><b>13.2</b> Surface Barriers</a></li>
<li class="chapter" data-level="13.3" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#innate-immune-system"><i class="fa fa-check"></i><b>13.3</b> Innate immune system</a></li>
<li class="chapter" data-level="13.4" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#anatomical-barriers"><i class="fa fa-check"></i><b>13.4</b> Anatomical barriers</a></li>
<li class="chapter" data-level="13.5" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#inflammation"><i class="fa fa-check"></i><b>13.5</b> Inflammation</a></li>
<li class="chapter" data-level="13.6" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#complement-system"><i class="fa fa-check"></i><b>13.6</b> Complement system</a></li>
<li class="chapter" data-level="13.7" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#white-blood-cells"><i class="fa fa-check"></i><b>13.7</b> White blood cells</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html"><i class="fa fa-check"></i><b>14</b> The Adaptive Immune System And Immunization</a>
<ul>
<li class="chapter" data-level="14.1" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#lymphocytes"><i class="fa fa-check"></i><b>14.1</b> Lymphocytes</a></li>
<li class="chapter" data-level="14.2" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#antigen-presentation"><i class="fa fa-check"></i><b>14.2</b> Antigen presentation</a></li>
<li class="chapter" data-level="14.3" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#exogenous-antigens"><i class="fa fa-check"></i><b>14.3</b> Exogenous antigens</a></li>
<li class="chapter" data-level="14.4" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#endogenous-antigens"><i class="fa fa-check"></i><b>14.4</b> Endogenous antigens</a></li>
<li class="chapter" data-level="14.5" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#t-lymphocytes"><i class="fa fa-check"></i><b>14.5</b> T lymphocytes</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For SCI204 Microbiology at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="microbial-genetics" class="section level1" number="9">
<h1><span class="header-section-number">9</span> Microbial Genetics</h1>
<div id="introduction-to-genetics-and-genes" class="section level2" number="9.1">
<h2><span class="header-section-number">9.1</span> Introduction To Genetics And Genes</h2>
<p><a href="https://en.wikipedia.org/wiki/Genetics">Genetics</a> is a branch of biology concerned with the study of genes, genetic variation, and heredity in organisms.</p>
<p>The word genetics stems from the ancient Greek γενετικός genetikos meaning “genitive”/“generative”, which in turn derives from γένεσις genesis meaning “origin”.</p>
<p>Though heredity had been observed for millennia, <a href="https://en.wikipedia.org/wiki/Gregor_Mendel">Gregor Mendel</a>, a scientist and Augustinian friar working in the 19th century, was the first to study genetics scientifically. Mendel studied “trait inheritance”, patterns in the way traits are handed down from parents to offspring. He observed that organisms (pea plants) inherit traits by way of discrete “units of inheritance”. This term, still used today, is a somewhat ambiguous definition of what is referred to as a gene. Mendel’s conclusions were largely ignored by the vast majority of scientists at the time. In 1900, however, his work was “re-discovered” by three European scientists, <a href="https://en.wikipedia.org/wiki/Hugo_de_Vries">Hugo de Vries</a>, <a href="https://en.wikipedia.org/wiki/Carl_Correns">Carl Correns</a>, and <a href="https://en.wikipedia.org/wiki/Erich_von_Tschermak">Erich von Tschermak</a>. In 1905, <a href="https://en.wikipedia.org/wiki/Wilhelm_Johannsen">Wilhelm Johannsen</a> introduced the term <em>gene</em> and <a href="https://en.wikipedia.org/wiki/William_Bateson">William Bateson</a> the term <em>genetics</em> (the adjective genetic predates the noun and was first used in a biological sense in 1860). Our understanding of what a <a href="https://en.wikipedia.org/wiki/Gene#Discovery_of_discrete_inherited_units"><em>gene</em></a> is has undergone quite a bit of change. Currently, genes are considered to be pieces of DNA that contain information for synthesis of ribonuclec acids (RNAs) that can be directly functional or serve as the intermediate template for a protein that performs a function.</p>
<p>Trait inheritance and molecular inheritance mechanisms of genes are still primary principles of genetics in the 21st century, but modern genetics has expanded beyond inheritance to studying the function and behavior of genes. Gene structure and function, variation, and distribution are studied within the context of the cell, the organism (e.g. dominance), and within the context of a population. Genetics has given rise to a number of subfields, including molecular genetics, epigenetics and population genetics. Organisms studied within the broad field span the three domains of life (archaea, bacteria, and eukarya).</p>
<p>Genetic processes work in combination with an organism’s environment and experiences to influence development and behavior, often referred to as nature versus nurture. The intracellular or extracellular environment of a living cell or organism may switch gene transcription on or off. A classic example is two seeds of genetically identical corn, one placed in a temperate climate and one in an arid climate (lacking sufficient waterfall or rain). While the average height of the two corn stalks may be genetically determined to be equal, the one in the arid climate only grows to half the height of the one in the temperate climate due to lack of water and nutrients in its environment.</p>
<p>The observation that living things inherit traits from their parents has been used since prehistoric times to improve crop plants and animals through selective breeding. The modern science of genetics, seeking to understand this process, is generally considered to have began with the work of the Augustinian friar Gregor Mendel in the mid-19th century.</p>
<p>Other theories of inheritance preceded Mendel’s work. A popular theory during the 19th century, and implied by Charles Darwin’s 1859 On the Origin of Species, was blending inheritance: the idea that individuals inherit a smooth blend of traits from their parents. Mendel’s work provided examples where traits were definitely not blended after hybridization, showing that traits are produced by combinations of distinct genes rather than a continuous blend. Blending of traits in the progeny is now explained by the action of multiple genes with quantitative effects. Another theory that had some support at that time was the inheritance of acquired characteristics: the belief that individuals inherit traits strengthened by their parents. This theory (commonly associated with Jean-Baptiste Lamarck) is now known to be wrong—the experiences of individuals do not affect the genes they pass to their children, although evidence in the field of epigenetics has revived some aspects of Lamarck’s theory. Other theories included the pangenesis of Charles Darwin (which had both acquired and inherited aspects) and Francis Galton’s reformulation of pangenesis as both particulate and inherited.</p>
<div id="mendelian-classical-genetics" class="section level3" number="9.1.1">
<h3><span class="header-section-number">9.1.1</span> Mendelian (Classical) Genetics</h3>
<p>Modern genetics started with Mendel’s studies of the nature of inheritance in plants. In his paper “Versuche über Pflanzenhybriden” (“Experiments on Plant Hybridization”), presented in 1865 to the Naturforschender Verein (Society for Research in Nature) in Brünn, Mendel traced the inheritance patterns of certain traits in pea plants and described them mathematically. Although this pattern of inheritance could only be observed for a few traits, Mendel’s work suggested that heredity was particulate, not acquired, and that the inheritance patterns of many traits could be explained through simple rules and ratios.</p>

<div class="figure" style="text-align: center"><span id="fig:sexlinked"></span>
<img src="figures/genetics/Sexlinked_inheritance_white.jpg" alt="Morgan’s observation of sex-linked inheritance of a mutation causing white eyes in Drosophila led him to the hypothesis that genes are located upon chromosomes." width="70%" />
<p class="caption">
Figure 9.1: <a href="https://commons.wikimedia.org/wiki/File:Sexlinked_inheritance_white.jpg">Morgan’s observation of sex-linked inheritance of a mutation causing white eyes in Drosophila led him to the hypothesis that genes are located upon chromosomes.</a>
</p>
</div>
<p>The importance of Mendel’s work did not gain wide understanding until 1900, after his death, when Hugo de Vries and other scientists rediscovered his research. William Bateson, a proponent of Mendel’s work, coined the word genetics in 1905 . Bateson both acted as a mentor and was aided significantly by the work of other scientists from Newnham College at Cambridge, specifically the work of Becky Saunders, Nora Darwin Barlow, and Muriel Wheldale Onslow. Bateson popularized the usage of the word genetics to describe the study of inheritance in his inaugural address to the Third International Conference on Plant Hybridization in London in 1906.</p>
<p>After the rediscovery of Mendel’s work, scientists tried to determine which molecules in the cell were responsible for inheritance. In 1911, Thomas Hunt Morgan argued that genes are on chromosomes, based on observations of a sex-linked white eye mutation in fruit flies. In 1913, his student <a href="https://en.wikipedia.org/wiki/Alfred_Sturtevant">Alfred Sturtevant</a> used the phenomenon of genetic linkage to show that genes are arranged linearly on the chromosome.</p>
</div>
<div id="molecular-genetics" class="section level3" number="9.1.2">
<h3><span class="header-section-number">9.1.2</span> Molecular Genetics</h3>
<p>In an influential published in 1941 paper, <a href="https://en.wikipedia.org/wiki/George_Beadle">George Beadle</a> and <a href="https://en.wikipedia.org/wiki/Edward_Tatum">Edward Tatum</a> proposed the idea that genes act through the production of enzymes, with each gene responsible for producing a single enzyme that in turn affects a single step in a metabolic pathway. The concept arose from work on genetic mutations in the mold <em>Neurospora crassa</em>, and subsequently was dubbed the “one gene–one enzyme hypothesis” by their collaborator Norman Horowitz. In 2004 Norman Horowitz reminisced that “these experiments founded the science of what Beadle and Tatum called ‘biochemical genetics.’ These experiments are by some considered to constitute the begining of what became molecular genetics and the development of the one gene–one enzyme hypothesis is often considered the first significant result in what came to be called molecular biology. Although it has been extremely influential, the hypothesis was recognized soon after its proposal to be an oversimplification. Even the subsequent reformulation of the”one gene–one polypeptide" hypothesis is now considered too simple to describe the relationship between genes and proteins. In attributing an instructional role to genes, Beadle and Tatum implicitly accorded genes an informational capability. This insight provided the foundation for the concept of a genetic code. However, it was not until the experiments were performed showing that DNA was the genetic material, that proteins consist of a defined linear sequence of amino acids, and that DNA structure contained a linear sequence of base pairs, was there a clear basis for solving the genetic code.</p>
<p>In attributing an instructional role to genes, Beadle and Tatum implicitly accorded genes an informational capability. This insight provided the foundation for the concept of a genetic code. However, it was not until the experiments were performed showing that DNA was the genetic material, that proteins consist of a defined linear sequence of amino acids, and that DNA structure contained a linear sequence of base pairs, was there a clear basis for solving the genetic code.</p>
<p>Although genes were known to exist on chromosomes, chromosomes are composed of both protein and DNA, and scientists did not know which of the two is responsible for inheritance. In 1928, <a href="https://en.wikipedia.org/wiki/Frederick_Griffith">Frederick Griffith</a> discovered the phenomenon of transformation: dead bacteria could transfer genetic material to “transform” other still-living bacteria. Sixteen years later, in 1944, the Avery–MacLeod–McCarty experiment identified DNA as the molecule responsible for transformation. The role of the nucleus as the repository of genetic information in eukaryotes had been established by Hämmerling in 1943 in his work on the single celled alga Acetabularia. The Hershey–Chase experiment in 1952 confirmed that DNA (rather than protein) is the genetic material of the viruses that infect bacteria, providing further evidence that DNA is the molecule responsible for inheritance.</p>
<p><a href="https://en.wikipedia.org/wiki/James_Watson">James Watson</a> and <a href="https://en.wikipedia.org/wiki/Francis_Crick">Francis Crick</a> determined the structure of DNA in 1953, using the X-ray crystallography work of <a href="https://en.wikipedia.org/wiki/Rosalind_Franklin">Rosalind Franklin</a> and <a href="https://en.wikipedia.org/wiki/Maurice_Wilkins">Maurice Wilkins</a> that indicated DNA has a helical structure (i.e., shaped like a corkscrew). Their double-helix model had two strands of DNA with the nucleotides pointing inward, each matching a complementary nucleotide on the other strand to form what look like rungs on a twisted ladder. This structure showed that genetic information exists in the sequence of nucleotides on each strand of DNA. The structure also suggested a simple method for replication: if the strands are separated, new partner strands can be reconstructed for each based on the sequence of the old strand. This property is what gives DNA its semi-conservative nature where one strand of new DNA is from an original parent strand.</p>

<div class="figure" style="text-align: center"><span id="fig:dnacartoon"></span>
<img src="figures/genetics/DNA_cartoon.png" alt="A cartoon representation of DNA based on atomic coordinates of PDB 1BNA, rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 9.2: A cartoon representation of DNA based on atomic coordinates of <a href="https://www.rcsb.org/structure/1bna">PDB 1BNA</a>, rendered with open source molecular visualization tool PyMol.
</p>
</div>
<p>Although the structure of DNA showed how inheritance works, it was still not known how DNA influences the behavior of cells. In the following years, scientists tried to understand how DNA controls the process of protein production. It was discovered that the cell uses DNA as a template to create matching messenger RNA, molecules with nucleotides very similar to DNA. The nucleotide sequence of a messenger RNA is used as a template by ribosomes to create an amino acid sequence in protein; this correspondence between nucleotide sequences and amino acid sequences is known as the genetic code.</p>
<p>With the newfound molecular understanding of inheritance came an explosion of research. One important development was chain-termination DNA sequencing in 1977 by <a href="https://en.wikipedia.org/wiki/Frederick_Sanger">Frederick Sanger</a>. This technology allows scientists to read the nucleotide sequence of a DNA molecule. In 1983, <a href="https://en.wikipedia.org/wiki/Kary_Mullis">Kary Banks Mullis</a> developed the polymerase chain reaction, providing a quick way to isolate and amplify a specific section of DNA from a mixture. The efforts of the Human Genome Project, Department of Energy, NIH, and parallel private efforts by Celera Genomics led to the sequencing of the human genome in 2003.</p>
</div>
</div>
<div id="the-genetic-code" class="section level2" number="9.2">
<h2><span class="header-section-number">9.2</span> The Genetic Code</h2>
<p>Whereas other aspects such as the 3D structure, called tertiary structure, of protein can only be predicted using sophisticated algorithms, the amino acid sequence, called primary structure, can be determined solely from the nucleic acid sequence with the aid of a translation table.</p>
<table class="table" style="font-size: 7px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:geneticcode">Table 9.1: </span>The genetic code: RNA codons.
</caption>
<thead>
<tr>
<th style="text-align:center;">
</th>
<th style="text-align:center;">
U
</th>
<th style="text-align:center;">
C
</th>
<th style="text-align:center;">
A
</th>
<th style="text-align:center;">
G
</th>
<th style="text-align:center;">
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
U
</td>
<td style="text-align:center;">
UUU Phenylalanine (Phe)
</td>
<td style="text-align:center;">
UCU Serine (Ser)
</td>
<td style="text-align:center;">
UAU Tyrosine (Tyr)
</td>
<td style="text-align:center;">
UGU Cysteine (Cys)
</td>
<td style="text-align:center;font-weight: bold;">
U
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
U
</td>
<td style="text-align:center;">
UUC Phe
</td>
<td style="text-align:center;">
UCC Ser
</td>
<td style="text-align:center;">
UAC Tyr
</td>
<td style="text-align:center;">
UGC Cys
</td>
<td style="text-align:center;font-weight: bold;">
C
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
U
</td>
<td style="text-align:center;">
UUA Leucine (Leu)
</td>
<td style="text-align:center;">
UCA Ser
</td>
<td style="text-align:center;">
UAA STOP
</td>
<td style="text-align:center;">
UGA STOP
</td>
<td style="text-align:center;font-weight: bold;">
A
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
U
</td>
<td style="text-align:center;">
UUG Leu
</td>
<td style="text-align:center;">
UCG Ser
</td>
<td style="text-align:center;">
UAG STOP
</td>
<td style="text-align:center;">
UGG Tryptophan (Trp)
</td>
<td style="text-align:center;font-weight: bold;">
G
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
C
</td>
<td style="text-align:center;">
CUU Leucine (Leu)
</td>
<td style="text-align:center;">
CCU Proline (Pro)
</td>
<td style="text-align:center;">
CAU Histidine (His)
</td>
<td style="text-align:center;">
CGU Arginine (Arg)
</td>
<td style="text-align:center;font-weight: bold;">
U
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
C
</td>
<td style="text-align:center;">
CUC Leu
</td>
<td style="text-align:center;">
CCC Pro
</td>
<td style="text-align:center;">
CAC His
</td>
<td style="text-align:center;">
CGC Arg
</td>
<td style="text-align:center;font-weight: bold;">
C
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
C
</td>
<td style="text-align:center;">
CUA Leu
</td>
<td style="text-align:center;">
CCA Pro
</td>
<td style="text-align:center;">
CAA Glutamine (Gln)
</td>
<td style="text-align:center;">
CGA Arg
</td>
<td style="text-align:center;font-weight: bold;">
A
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
C
</td>
<td style="text-align:center;">
CUG Leu
</td>
<td style="text-align:center;">
CCG Pro
</td>
<td style="text-align:center;">
CAG Gln
</td>
<td style="text-align:center;">
CGG Arg
</td>
<td style="text-align:center;font-weight: bold;">
G
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
A
</td>
<td style="text-align:center;">
AUU Isoleucine (Ile)
</td>
<td style="text-align:center;">
ACU Threonine (Thr)
</td>
<td style="text-align:center;">
AAU Asparagine (Asn)
</td>
<td style="text-align:center;">
AGU Serine (Ser)
</td>
<td style="text-align:center;font-weight: bold;">
U
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
A
</td>
<td style="text-align:center;">
AUC Ile
</td>
<td style="text-align:center;">
ACC Thr
</td>
<td style="text-align:center;">
AAC Asn
</td>
<td style="text-align:center;">
AGC Ser
</td>
<td style="text-align:center;font-weight: bold;">
C
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
A
</td>
<td style="text-align:center;">
AUA Ile
</td>
<td style="text-align:center;">
ACA Thr
</td>
<td style="text-align:center;">
AAA Lysine (Lys)
</td>
<td style="text-align:center;">
AGA Arginine (Arg)
</td>
<td style="text-align:center;font-weight: bold;">
A
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
A
</td>
<td style="text-align:center;">
AUG Methionine (Met) or START
</td>
<td style="text-align:center;">
ACG Thr
</td>
<td style="text-align:center;">
AAG Lys
</td>
<td style="text-align:center;">
AGG Arg
</td>
<td style="text-align:center;font-weight: bold;">
G
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
G
</td>
<td style="text-align:center;">
GUU Valine Val
</td>
<td style="text-align:center;">
GCU Alanine (Ala)
</td>
<td style="text-align:center;">
GAU Aspartic acid (Asp)
</td>
<td style="text-align:center;">
GGU Glycine (Gly)
</td>
<td style="text-align:center;font-weight: bold;">
U
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
G
</td>
<td style="text-align:center;">
GUC (Val)
</td>
<td style="text-align:center;">
GCC Ala
</td>
<td style="text-align:center;">
GAC Asp
</td>
<td style="text-align:center;">
GGC Gly
</td>
<td style="text-align:center;font-weight: bold;">
C
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
G
</td>
<td style="text-align:center;">
GUA Val
</td>
<td style="text-align:center;">
GCA Ala
</td>
<td style="text-align:center;">
GAA Glutamic acid (Glu)
</td>
<td style="text-align:center;">
GGA Gly
</td>
<td style="text-align:center;font-weight: bold;">
A
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
G
</td>
<td style="text-align:center;">
GUG Val
</td>
<td style="text-align:center;">
GCG Ala
</td>
<td style="text-align:center;">
GAG Glu
</td>
<td style="text-align:center;">
GGG Gly
</td>
<td style="text-align:center;font-weight: bold;">
G
</td>
</tr>
</tbody>
</table>
<p>There are many computer programs capable of translating a DNA/RNA sequence into a protein sequence. Normally this is performed using the Standard Genetic Code (Table <a href="microbial-genetics.html#tab:geneticcode">9.1</a>), however, few programs can handle all the “special” cases, such as the use of the alternative initiation codons. For instance, the rare alternative start codon CTG codes for Methionine when used as a start codon, and for Leucine in all other positions.</p>
<div id="gene-expression" class="section level3" number="9.2.1">
<h3><span class="header-section-number">9.2.1</span> Gene Expression</h3>
<p>Genes generally express their functional effect through the production of proteins, which are complex molecules responsible for most functions in the cell. Proteins are made up of one or more polypeptide chains, each of which is composed of a sequence of amino acids, and the DNA sequence of a gene encodes the amino acid sequence of the corresponding protein. This process begins with the production of a messenger RNA (mRNA) molecule with a sequence matching the gene’s DNA sequence, a process called transcription.</p>
<p>This messenger RNA molecule is then used to produce a corresponding amino acid sequence through a process called translation. Each group of three nucleotides in the sequence, called a codon, corresponds either to one of the twenty possible amino acids in a protein or an instruction to end the amino acid sequence; this correspondence is called the genetic code. The flow of information is unidirectional: information is transferred from nucleotide sequences into the amino acid sequence of proteins, but it never transfers from protein back into the sequence of DNA—a phenomenon Francis Crick called the central dogma of molecular biology.</p>
<p>The specific sequence of amino acids results in a unique three-dimensional structure for that protein, and the three-dimensional structures of proteins are related to their functions. Some are simple structural molecules, like the fibers formed by the protein collagen. Proteins can bind to other proteins and simple molecules, sometimes acting as enzymes by facilitating chemical reactions within the bound molecules (without changing the structure of the protein itself). Protein structure is dynamic; the protein hemoglobin bends into slightly different forms as it facilitates the capture, transport, and release of oxygen molecules within mammalian blood.</p>
<p>A single nucleotide difference within DNA can cause a change in the amino acid sequence of a protein. Because protein structures are the result of their amino acid sequences, some changes can dramatically change the properties of a protein by destabilizing the structure or changing the surface of the protein in a way that changes its interaction with other proteins and molecules. For example, sickle-cell anemia is a human genetic disease that results from a single base difference within the coding region for the β-globin section of hemoglobin, causing a single amino acid change that changes hemoglobin’s physical properties. Sickle-cell versions of hemoglobin stick to themselves, stacking to form fibers that distort the shape of red blood cells carrying the protein. These sickle-shaped cells no longer flow smoothly through blood vessels, having a tendency to clog or degrade, causing the medical problems associated with this disease.</p>
<p>Some DNA sequences are transcribed into RNA but are not translated into protein products—such RNA molecules are called non-coding RNA. In some cases, these products fold into structures which are involved in critical cell functions (e.g. ribosomal RNA and transfer RNA). RNA can also have regulatory effects through hybridization interactions with other RNA molecules (e.g. microRNA).</p>
</div>
<div id="nature-and-nurture" class="section level3" number="9.2.2">
<h3><span class="header-section-number">9.2.2</span> Nature And Nurture</h3>
<p>Although genes contain all the information an organism uses to function, the environment plays an important role in determining the ultimate phenotypes an organism displays. The phrase “nature and nurture” refers to this complementary relationship. The phenotype of an organism depends on the interaction of genes and the environment. An interesting example is the coat coloration of the Siamese cat. In this case, the body temperature of the cat plays the role of the environment. The cat’s genes code for dark hair, thus the hair-producing cells in the cat make cellular proteins resulting in dark hair. But these dark hair-producing proteins are sensitive to temperature (i.e. have a mutation causing temperature-sensitivity) and denature in higher-temperature environments, failing to produce dark-hair pigment in areas where the cat has a higher body temperature. In a low-temperature environment, however, the protein’s structure is stable and produces dark-hair pigment normally. The protein remains functional in areas of skin that are colder—such as its legs, ears, tail and face—so the cat has dark hair at its extremities.</p>
<p>Environment plays a major role in effects of the human genetic disease phenylketonuria. The mutation that causes phenylketonuria disrupts the ability of the body to break down the amino acid phenylalanine, causing a toxic build-up of an intermediate molecule that, in turn, causes severe symptoms of progressive intellectual disability and seizures. However, if someone with the phenylketonuria mutation follows a strict diet that avoids this amino acid, they remain normal and healthy.</p>
<p>A common method for determining how genes and environment (“nature and nurture”) contribute to a phenotype involves studying identical and fraternal twins, or other siblings of multiple births. Identical siblings are genetically the same since they come from the same zygote. Meanwhile, fraternal twins are as genetically different from one another as normal siblings. By comparing how often a certain disorder occurs in a pair of identical twins to how often it occurs in a pair of fraternal twins, scientists can determine whether that disorder is caused by genetic or postnatal environmental factors. However, such tests cannot separate genetic factors from environmental factors affecting fetal development.</p>
</div>
<div id="gene-regulation" class="section level3" number="9.2.3">
<h3><span class="header-section-number">9.2.3</span> Gene Regulation</h3>
<p>The genome of a given organism contains thousands of genes, but not all these genes need to be active at any given moment. A gene is expressed when it is being transcribed into mRNA and there exist many cellular methods of controlling the expression of genes such that proteins are produced only when needed by the cell. Transcription factors are regulatory proteins that bind to DNA, either promoting or inhibiting the transcription of a gene. Within the genome of <em>Escherichia coli</em> bacteria, for example, there exists a series of genes necessary for the synthesis of the amino acid tryptophan. However, when tryptophan is already available to the cell, these genes for tryptophan synthesis are no longer needed. The presence of tryptophan directly affects the activity of the genes—tryptophan molecules bind to the tryptophan repressor (a transcription factor), changing the repressor’s structure such that the repressor binds to the genes. The tryptophan repressor blocks the transcription and expression of the genes, thereby creating negative feedback regulation of the tryptophan synthesis process.</p>

<div class="figure" style="text-align: center"><span id="fig:transcriptionfactor"></span>
<img src="figures/genetics/Transcription_factor_DNA_1A1L.png" alt="Transcription factors bind to DNA, influencing the transcription of associated genes. Based on atomic coordinates of PDB 1A1L, rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 9.3: Transcription factors bind to DNA, influencing the transcription of associated genes. Based on atomic coordinates of <a href="https://www.rcsb.org/structure/1a1l">PDB 1A1L</a>, rendered with open source molecular visualization tool PyMol.
</p>
</div>
<p>Differences in gene expression are especially clear within multicellular organisms, where cells all contain the same genome but have very different structures and behaviors due to the expression of different sets of genes. All the cells in a multicellular organism derive from a single cell, differentiating into variant cell types in response to external and intercellular signals and gradually establishing different patterns of gene expression to create different behaviors. As no single gene is responsible for the development of structures within multicellular organisms, these patterns arise from the complex interactions between many cells.</p>
<p>Within eukaryotes, there exist structural features of chromatin that influence the transcription of genes, often in the form of modifications to DNA and chromatin that are stably inherited by daughter cells. These features are called “epigenetic” because they exist “on top” of the DNA sequence and retain inheritance from one cell generation to the next. Because of epigenetic features, different cell types grown within the same medium can retain very different properties. Although epigenetic features are generally dynamic over the course of development, some, like the phenomenon of paramutation, have multigenerational inheritance and exist as rare exceptions to the general rule of DNA as the basis for inheritance.</p>
</div>
<div id="genetic-change" class="section level3" number="9.2.4">
<h3><span class="header-section-number">9.2.4</span> Genetic Change</h3>
<p>During the process of DNA replication, errors occasionally occur in the polymerization of the second strand. These errors, called mutations, can affect the phenotype of an organism, especially if they occur within the protein coding sequence of a gene. Error rates are usually very low—1 error in every 10–100 million bases—due to the “proofreading” ability of DNA polymerases. Processes that increase the rate of changes in DNA are called mutagenic: mutagenic chemicals promote errors in DNA replication, often by interfering with the structure of base-pairing, while UV radiation induces mutations by causing damage to the DNA structure. Chemical damage to DNA occurs naturally as well and cells use DNA repair mechanisms to repair mismatches and breaks. The repair does not, however, always restore the original sequence.</p>
<p>In organisms that use chromosomal crossover to exchange DNA and recombine genes, errors in alignment during meiosis can also cause mutations. Errors in crossover are especially likely when similar sequences cause partner chromosomes to adopt a mistaken alignment; this makes some regions in genomes more prone to mutating in this way. These errors create large structural changes in DNA sequence – duplications, inversions, deletions of entire regions – or the accidental exchange of whole parts of sequences between different chromosomes (chromosomal translocation).</p>
</div>
<div id="natural-selection-and-evolution" class="section level3" number="9.2.5">
<h3><span class="header-section-number">9.2.5</span> Natural Selection And Evolution</h3>
<p>Mutations alter an organism’s genotype and occasionally this causes different phenotypes to appear. Most mutations have little effect on an organism’s phenotype, health, or reproductive fitness. Mutations that do have an effect are usually detrimental, but occasionally some can be beneficial. Studies in the fly <em>Drosophila melanogaster</em> suggest that if a mutation changes a protein produced by a gene, about 70 percent of these mutations will be harmful with the remainder being either neutral or weakly beneficial.</p>
<p>Population genetics studies the distribution of genetic differences within populations and how these distributions change over time. Changes in the frequency of an allele in a population are mainly influenced by natural selection, where a given allele provides a selective or reproductive advantage to the organism, as well as other factors such as mutation, genetic drift, genetic hitchhiking, artificial selection and migration.</p>
<p>Over many generations, the genomes of organisms can change significantly, resulting in evolution. In the process called adaptation, selection for beneficial mutations can cause a species to evolve into forms better able to survive in their environment. New species are formed through the process of speciation, often caused by geographical separations that prevent populations from exchanging genes with each other.</p>
<p>By comparing the homology between different species’ genomes, it is possible to calculate the evolutionary distance between them and when they may have diverged. Genetic comparisons are generally considered a more accurate method of characterizing the relatedness between species than the comparison of phenotypic characteristics. The evolutionary distances between species can be used to form evolutionary trees; these trees represent the common descent and divergence of species over time, although they do not show the transfer of genetic material between unrelated species (known as horizontal gene transfer and most common in bacteria).</p>
</div>
</div>
<div id="bacterial-genetics-1" class="section level2" number="9.3">
<h2><span class="header-section-number">9.3</span> Bacterial Genetics</h2>
<p>Bacterial genetics is the subfield of genetics devoted to the study of bacteria. Bacterial genetics are subtly different from eukaryotic genetics, however bacteria still serve as a good model for animal genetic studies. One of the major distinctions between bacterial and eukaryotic genetics stems from the bacteria’s lack of membrane-bound organelles (this is true of all prokaryotes. While it is a fact that there are prokaryotic organelles, they are never bound by a lipid membrane, but by a shell of proteins), necessitating protein synthesis occur in the cytoplasm.</p>
<p>Since the discovery of microorganisms by Robert Hooke and Antoni van Leeuwenhoek during the period 1665-1885 they have been used to study many processes and have had applications in various areas of study in genetics. For example: Microorganisms’ rapid growth rates and short generation times are used by scientists to study evolution. Robert Hooke and Antoni van Leeuwenhoek discoveries involved depictions, observations, and descriptions of microorganisms. Mucor is the microfungus that Hooke presented and gave a depiction of. His contribution being, Mucor as the first microorganism to be illustrated. Antoni van Leeuwenhoek’s contribution to the microscopic protozoa and microscopic bacteria yielded to scientific observations and descriptions. These contributions were accomplished by a simple microscope, which led to the understanding of microbes today and continues to progress scientists understanding. Microbial genetics also has applications in being able to study processes and pathways that are similar to those found in humans such as drug metabolism.
Genetically modified bacteria were the first organisms to be modified in the laboratory, due to their simple genetics. These organisms are now used for several purposes, and are particularly important in producing large amounts of pure human proteins for use in medicine.</p>
<p>Most bacteria have a single circular chromosome that can range in size from only 160,000 base pairs in the endosymbiotic bacteria Carsonella ruddii, to 12,200,000 base pairs (12.2 Mbp) in the soil-dwelling bacteria Sorangium cellulosum. There are many exceptions to this, for example some Streptomyces and Borrelia species contain a single linear chromosome, while some Vibrio species contain more than one chromosome. Bacteria can also contain plasmids, small extra-chromosomal molecules of DNA that may contain genes for various useful functions such as antibiotic resistance, metabolic capabilities, or various virulence factors.</p>
<p>Bacteria genomes usually encode a few hundred to a few thousand genes. The genes in bacterial genomes are usually a single continuous stretch of DNA and although several different types of introns do exist in bacteria, these are much rarer than in eukaryotes.</p>
<p>Bacteria, as asexual organisms, inherit an identical copy of the parent’s genomes and are clonal. However, all bacteria can evolve by selection on changes to their genetic material DNA caused by genetic recombination or mutations. Mutations come from errors made during the replication of DNA or from exposure to mutagens. Mutation rates vary widely among different species of bacteria and even among different clones of a single species of bacteria. Genetic changes in bacterial genomes come from either random mutation during replication or “stress-directed mutation”, where genes involved in a particular growth-limiting process have an increased mutation rate.</p>
<p>Some bacteria also transfer genetic material between cells. This can occur in three main ways. First, bacteria can take up exogenous DNA from their environment, in a process called transformation. Many bacteria can naturally take up DNA from the environment, while others must be chemically altered in order to induce them to take up DNA. The development of competence in nature is usually associated with stressful environmental conditions, and seems to be an adaptation for facilitating repair of DNA damage in recipient cells. The second way bacteria transfer genetic material is by transduction, when the integration of a bacteriophage introduces foreign DNA into the chromosome. Many types of bacteriophage exist, some simply infect and lyse their host bacteria, while others insert into the bacterial chromosome. Bacteria resist phage infection through restriction modification systems that degrade foreign DNA, and a system that uses CRISPR sequences to retain fragments of the genomes of phage that the bacteria have come into contact with in the past, which allows them to block virus replication through a form of RNA interference. The third method of gene transfer is conjugation, whereby DNA is transferred through direct cell contact. In ordinary circumstances, transduction, conjugation, and transformation involve transfer of DNA between individual bacteria of the same species, but occasionally transfer may occur between individuals of different bacterial species and this may have significant consequences, such as the transfer of antibiotic resistance. In such cases, gene acquisition from other bacteria or the environment is called horizontal gene transfer and may be common under natural conditions.</p>
<p>Like other organisms, bacteria also breed true and maintain their characteristics from generation to generation, yet at the same time, exhibit variations in particular properties in a small proportion of their progeny. Though heritability and variations in bacteria had been noticed from the early days of bacteriology, it was not realised then that bacteria too obey the laws of genetics. Even the existence of a bacterial nucleus was a subject of controversy. The differences in morphology and other properties were attributed by Nageli in 1877, to bacterial pleomorphism, which postulated the existence of a single, a few species of bacteria, which possessed a protein capacity for a variation. With the development and application of precise methods of pure culture, it became apparent that different types of bacteria retained constant form and function through successive generations. This led to the concept of monomorphism.</p>
<div id="bacterial-transformation" class="section level3" number="9.3.1">
<h3><span class="header-section-number">9.3.1</span> Bacterial Transformation</h3>
<p>In molecular biology and genetics, transformation is the genetic alteration of a cell resulting from the direct uptake and incorporation of exogenous genetic material from its surroundings through the cell membrane(s). For transformation to take place, the recipient bacterium must be in a state of competence, which might occur in nature as a time-limited response to environmental conditions such as starvation and cell density, and may also be induced in a laboratory.</p>

<div class="figure" style="text-align: center"><span id="fig:transormation"></span>
<img src="figures/genetics/Bacterial_Transformation.svg" alt="Schematic drawing of bacterial conjugation. Conjugation diagram 1) Donor cell produces pilus. 2) Pilus attaches to recipient cell, brings the two cells together. 3) The mobile plasmid is nicked and a single strand of DNA is then transferred to the recipient cell. 4) Both cells recircularize their plasmids, synthesize second strands, and reproduce pili; both cells are now viable donors." width="70%" />
<p class="caption">
Figure 9.4: <a href="https://commons.wikimedia.org/wiki/File:Conjugation.svg">Schematic drawing of bacterial conjugation</a>. Conjugation diagram 1) Donor cell produces pilus. 2) Pilus attaches to recipient cell, brings the two cells together. 3) The mobile plasmid is nicked and a single strand of DNA is then transferred to the recipient cell. 4) Both cells recircularize their plasmids, synthesize second strands, and reproduce pili; both cells are now viable donors.
</p>
</div>
<p>Transformation is one of three processes for horizontal gene transfer, in which exogenous genetic material passes from one bacterium to another, the other two being conjugation (transfer of genetic material between two bacterial cells in direct contact) and transduction (injection of foreign DNA by a bacteriophage virus into the host bacterium). In transformation, the genetic material passes through the intervening medium, and uptake is completely dependent on the recipient bacterium.</p>
<p>As of 2014 about 80 species of bacteria were known to be capable of transformation, about evenly divided between Gram-positive and Gram-negative bacteria; the number might be an overestimate since several of the reports are supported by single papers.</p>
<p>“Transformation” may also be used to describe the insertion of new genetic material into nonbacterial cells, including animal and plant cells; however, because “transformation” has a special meaning in relation to animal cells, indicating progression to a cancerous state, the process is usually called “transfection”.</p>
<p>Transformation in bacteria was first demonstrated in 1928 by the British bacteriologist Frederick Griffith. Griffith was interested in determining whether injections of heat-killed bacteria could be used to vaccinate mice against pneumonia. However, he discovered that a non-virulent strain of Streptococcus pneumoniae could be made virulent after being exposed to heat-killed virulent strains. Griffith hypothesized that some “transforming principle” from the heat-killed strain was responsible for making the harmless strain virulent. In 1944 this “transforming principle” was identified as being genetic by Oswald Avery, Colin MacLeod, and Maclyn McCarty. They isolated DNA from a virulent strain of S. pneumoniae and using just this DNA were able to make a harmless strain virulent. They called this uptake and incorporation of DNA by bacteria “transformation” (See Avery-MacLeod-McCarty experiment) The results of Avery et al.’s experiments were at first skeptically received by the scientific community and it was not until the development of genetic markers and the discovery of other methods of genetic transfer (conjugation in 1947 and transduction in 1953) by Joshua Lederberg that Avery’s experiments were accepted.</p>
<p>It was originally thought that Escherichia coli, a commonly used laboratory organism, was refractory to transformation. However, in 1970, Morton Mandel and Akiko Higa showed that E. coli may be induced to take up DNA from bacteriophage λ without the use of helper phage after treatment with calcium chloride solution. Two years later in 1972, Stanley Norman Cohen, Annie Chang and Leslie Hsu showed that CaCl
2 treatment is also effective for transformation of plasmid DNA. The method of transformation by Mandel and Higa was later improved upon by Douglas Hanahan. The discovery of artificially induced competence in E. coli created an efficient and convenient procedure for transforming bacteria which allows for simpler molecular cloning methods in biotechnology and research, and it is now a routinely used laboratory procedure.</p>
<p>Transformation using electroporation was developed in the late 1980s, increasing the efficiency of in-vitro transformation and increasing the number of bacterial strains that could be transformed. Transformation of animal and plant cells was also investigated with the first transgenic mouse being created by injecting a gene for a rat growth hormone into a mouse embryo in 1982. In 1897 a bacterium that caused plant tumors, Agrobacterium tumefaciens, was discovered and in the early 1970s the tumor-inducing agent was found to be a DNA plasmid called the Ti plasmid. By removing the genes in the plasmid that caused the tumor and adding in novel genes, researchers were able to infect plants with A. tumefaciens and let the bacteria insert their chosen DNA into the genomes of the plants. Not all plant cells are susceptible to infection by A. tumefaciens, so other methods were developed, including electroporation and micro-injection. Particle bombardment was made possible with the invention of the Biolistic Particle Delivery System (gene gun) by John Sanford in the 1980s.</p>
<p>As of 2014 about 80 species of bacteria were known to be capable of transformation, about evenly divided between Gram-positive and Gram-negative bacteria; the number might be an overestimate since several of the reports are supported by single papers.</p>
<p>Naturally competent bacteria carry sets of genes that provide the protein machinery to bring DNA across the cell membrane(s). The transport of the exogenous DNA into the cells may require proteins that are involved in the assembly of type IV pili and type II secretion system, as well as DNA translocase complex at the cytoplasmic membrane.</p>
<p>Due to the differences in structure of the cell envelope between Gram-positive and Gram-negative bacteria, there are some differences in the mechanisms of DNA uptake in these cells, however most of them share common features that involve related proteins. The DNA first binds to the surface of the competent cells on a DNA receptor, and passes through the cytoplasmic membrane via DNA translocase. Only single-stranded DNA may pass through, the other strand being degraded by nucleases in the process. The translocated single-stranded DNA may then be integrated into the bacterial chromosomes by a RecA-dependent process. In Gram-negative cells, due to the presence of an extra membrane, the DNA requires the presence of a channel formed by secretins on the outer membrane. Pilin may be required for competence, but its role is uncertain. The uptake of DNA is generally non-sequence specific, although in some species the presence of specific DNA uptake sequences may facilitate efficient DNA uptake.</p>
<p>Natural transformation is a bacterial adaptation for DNA transfer that depends on the expression of numerous bacterial genes whose products appear to be responsible for this process. In general, transformation is a complex, energy-requiring developmental process. In order for a bacterium to bind, take up and recombine exogenous DNA into its chromosome, it must become competent, that is, enter a special physiological state. Competence development in Bacillus subtilis requires expression of about 40 genes. The DNA integrated into the host chromosome is usually (but with rare exceptions) derived from another bacterium of the same species, and is thus homologous to the resident chromosome.</p>
<p>In B. subtilis the length of the transferred DNA is greater than 1271 kb (more than 1 million bases). The length transferred is likely double stranded DNA and is often more than a third of the total chromosome length of 4215 kb. It appears that about 7-9% of the recipient cells take up an entire chromosome.</p>
<p>The capacity for natural transformation appears to occur in a number of prokaryotes, and thus far 67 prokaryotic species (in seven different phyla) are known to undergo this process.</p>
<p>Competence for transformation is typically induced by high cell density and/or nutritional limitation, conditions associated with the stationary phase of bacterial growth. Transformation in Haemophilus influenzae occurs most efficiently at the end of exponential growth as bacterial growth approaches stationary phase. Transformation in Streptococcus mutans, as well as in many other streptococci, occurs at high cell density and is associated with biofilm formation. Competence in B. subtilis is induced toward the end of logarithmic growth, especially under conditions of amino acid limitation. Similarly, in Micrococcus luteus (a representative of the less well studied Actinobacteria phylum), competence develops during the mid-late exponential growth phase and is also triggered by amino acids starvation.</p>
<p>By releasing intact host and plasmid DNA, certain bacteriophages are thought to contribute to transformation.</p>
<p>Competence is specifically induced by DNA damaging conditions. For instance, transformation is induced in Streptococcus pneumoniae by the DNA damaging agents mitomycin C (a DNA cross-linking agent) and fluoroquinolone (a topoisomerase inhibitor that causes double-strand breaks). In B. subtilis, transformation is increased by UV light, a DNA damaging agent. In Helicobacter pylori, ciprofloxacin, which interacts with DNA gyrase and introduces double-strand breaks, induces expression of competence genes, thus enhancing the frequency of transformation Using Legionella pneumophila, Charpentier et al. tested 64 toxic molecules to determine which of these induce competence. Of these, only six, all DNA damaging agents, caused strong induction. These DNA damaging agents were mitomycin C (which causes DNA inter-strand crosslinks), norfloxacin, ofloxacin and nalidixic acid (inhibitors of DNA gyrase that cause double-strand breaks), bicyclomycin (causes single- and double-strand breaks), and hydroxyurea (induces DNA base oxidation). UV light also induced competence in L. pneumophila. Charpentier et al. suggested that competence for transformation probably evolved as a DNA damage response.</p>
<p>Logarithmically growing bacteria differ from stationary phase bacteria with respect to the number of genome copies present in the cell, and this has implications for the capability to carry out an important DNA repair process. During logarithmic growth, two or more copies of any particular region of the chromosome may be present in a bacterial cell, as cell division is not precisely matched with chromosome replication. The process of homologous recombinational repair (HRR) is a key DNA repair process that is especially effective for repairing double-strand damages, such as double-strand breaks. This process depends on a second homologous chromosome in addition to the damaged chromosome. During logarithmic growth, a DNA damage in one chromosome may be repaired by HRR using sequence information from the other homologous chromosome. Once cells approach stationary phase, however, they typically have just one copy of the chromosome, and HRR requires input of homologous template from outside the cell by transformation.</p>
<p>To test whether the adaptive function of transformation is repair of DNA damages, a series of experiments were carried out using B. subtilis irradiated by UV light as the damaging agent (reviewed by Michod et al. and Bernstein et al.) The results of these experiments indicated that transforming DNA acts to repair potentially lethal DNA damages introduced by UV light in the recipient DNA. The particular process responsible for repair was likely HRR. Transformation in bacteria can be viewed as a primitive sexual process, since it involves interaction of homologous DNA from two individuals to form recombinant DNA that is passed on to succeeding generations. Bacterial transformation in prokaryotes may have been the ancestral process that gave rise to meiotic sexual reproduction in eukaryotes (see Evolution of sexual reproduction; Meiosis.)</p>
<p>Artificial competence can be induced in laboratory procedures that involve making the cell passively permeable to DNA by exposing it to conditions that do not normally occur in nature. Typically the cells are incubated in a solution containing divalent cations (often calcium chloride) under cold conditions, before being exposed to a heat pulse (heat shock). Calcium chloride partially disrupts the cell membrane, which allows the recombinant DNA to enter the host cell. Cells that are able to take up the DNA are called competent cells.</p>
<p>It has been found that growth of Gram-negative bacteria in 20 mM Mg reduces the number of protein-to-lipopolysaccharide bonds by increasing the ratio of ionic to covalent bonds, which increases membrane fluidity, facilitating transformation. The role of lipopolysaccharides here are verified from the observation that shorter O-side chains are more effectively transformed – perhaps because of improved DNA accessibility.</p>
<p>The surface of bacteria such as E. coli is negatively charged due to phospholipids and lipopolysaccharides on its cell surface, and the DNA is also negatively charged. One function of the divalent cation therefore would be to shield the charges by coordinating the phosphate groups and other negative charges, thereby allowing a DNA molecule to adhere to the cell surface.</p>
<p>DNA entry into E. coli cells is through channels known as zones of adhesion or Bayer’s junction, with a typical cell carrying as many as 400 such zones. Their role was established when cobalamine (which also uses these channels) was found to competitively inhibit DNA uptake. Another type of channel implicated in DNA uptake consists of poly (HB):poly P:Ca. In this poly (HB) is envisioned to wrap around DNA (itself a polyphosphate), and is carried in a shield formed by Ca ions.</p>
<p>It is suggested that exposing the cells to divalent cations in cold condition may also change or weaken the cell surface structure, making it more permeable to DNA. The heat-pulse is thought to create a thermal imbalance across the cell membrane, which forces the DNA to enter the cells through either cell pores or the damaged cell wall.</p>
<p>Electroporation is another method of promoting competence. In this method the cells are briefly shocked with an electric field of 10-20 kV/cm, which is thought to create holes in the cell membrane through which the plasmid DNA may enter. After the electric shock, the holes are rapidly closed by the cell’s membrane-repair mechanisms.</p>
</div>
<div id="practical-aspects-of-transformation-in-molecular-biology" class="section level3" number="9.3.2">
<h3><span class="header-section-number">9.3.2</span> Practical aspects of transformation in molecular biology</h3>
<p>The discovery of artificially induced competence in bacteria allow bacteria such as Escherichia coli to be used as a convenient host for the manipulation of DNA as well as expressing proteins. Typically plasmids are used for transformation in E. coli. In order to be stably maintained in the cell, a plasmid DNA molecule must contain an origin of replication, which allows it to be replicated in the cell independently of the replication of the cell’s own chromosome.</p>
<p>The efficiency with which a competent culture can take up exogenous DNA and express its genes is known as transformation efficiency and is measured in colony forming unit (cfu) per μg DNA used. A transformation efficiency of 1×108 cfu/μg for a small plasmid like pUC19 is roughly equivalent to 1 in 2000 molecules of the plasmid used being transformed.</p>
<p>In calcium chloride transformation, the cells are prepared by chilling cells in the presence of Ca<sup>2+</sup>
(in CaCl<sub>2</sub> solution), making the cell become permeable to plasmid DNA. The cells are incubated on ice with the DNA, and then briefly heat-shocked (e.g., at 42 °C for 30–120 seconds). This method works very well for circular plasmid DNA. Non-commercial preparations should normally give 106 to 107 transformants per microgram of plasmid; a poor preparation will be about 104/μg or less, but a good preparation of competent cells can give up to ~108 colonies per microgram of plasmid. Protocols, however, exist for making supercompetent cells that may yield a transformation efficiency of over 109. The chemical method, however, usually does not work well for linear DNA, such as fragments of chromosomal DNA, probably because the cell’s native exonuclease enzymes rapidly degrade linear DNA. In contrast, cells that are naturally competent are usually transformed more efficiently with linear DNA than with plasmid DNA.</p>
<p>The transformation efficiency using the CaCl<sub>2</sub> method decreases with plasmid size, and electroporation therefore may be a more effective method for the uptake of large plasmid DNA. Cells used in electroporation should be prepared first by washing in cold double-distilled water to remove charged particles that may create sparks during the electroporation process.</p>
</div>
<div id="selection-and-screening-in-plasmid-transformation" class="section level3" number="9.3.3">
<h3><span class="header-section-number">9.3.3</span> Selection and screening in plasmid transformation</h3>
<p>Because transformation usually produces a mixture of relatively few transformed cells and an abundance of non-transformed cells, a method is necessary to select for the cells that have acquired the plasmid. The plasmid therefore requires a selectable marker such that those cells without the plasmid may be killed or have their growth arrested. Antibiotic resistance is the most commonly used marker for prokaryotes. The transforming plasmid contains a gene that confers resistance to an antibiotic that the bacteria are otherwise sensitive to. The mixture of treated cells is cultured on media that contain the antibiotic so that only transformed cells are able to grow. Another method of selection is the use of certain auxotrophic markers that can compensate for an inability to metabolise certain amino acids, nucleotides, or sugars. This method requires the use of suitably mutated strains that are deficient in the synthesis or utility of a particular biomolecule, and the transformed cells are cultured in a medium that allows only cells containing the plasmid to grow.</p>
<p>In a cloning experiment, a gene may be inserted into a plasmid used for transformation. However, in such experiment, not all the plasmids may contain a successfully inserted gene. Additional techniques may therefore be employed further to screen for transformed cells that contain plasmid with the insert. Reporter genes can be used as markers, such as the lacZ gene which codes for β-galactosidase used in blue-white screening. This method of screening relies on the principle of α-complementation, where a fragment of the lacZ gene (lacZα) in the plasmid can complement another mutant lacZ gene (lacZΔM15) in the cell. Both genes by themselves produce non-functional peptides, however, when expressed together, as when a plasmid containing lacZ-α is transformed into a lacZΔM15 cells, they form a functional β-galactosidase. The presence of an active β-galactosidase may be detected when cells are grown in plates containing X-gal, forming characteristic blue colonies. However, the multiple cloning site, where a gene of interest may be ligated into the plasmid vector, is located within the lacZα gene. Successful ligation therefore disrupts the lacZα gene, and no functional β-galactosidase can form, resulting in white colonies. Cells containing successfully ligated insert can then be easily identified by its white coloration from the unsuccessful blue ones.</p>
<p>Other commonly used reporter genes are green fluorescent protein (GFP), which produces cells that glow green under blue light, and the enzyme luciferase, which catalyzes a reaction with luciferin to emit light. The recombinant DNA may also be detected using other methods such as nucleic acid hybridization with radioactive RNA probe, while cells that expressed the desired protein from the plasmid may also be detected using immunological methods.</p>
</div>
<div id="bacterial-conjugation" class="section level3" number="9.3.4">
<h3><span class="header-section-number">9.3.4</span> Bacterial Conjugation</h3>
<p>Bacterial conjugation is the transfer of genetic material between bacterial cells by direct cell-to-cell contact or by a bridge-like connection between two cells. This takes place through a pilus. It is a parasexual mode of reproduction in bacteria. The process was discovered by Joshua Lederberg and Edward Tatum in 1946.</p>
<p>It is a mechanism of horizontal gene transfer as are transformation and transduction although these two other mechanisms do not involve cell-to-cell contact.</p>
<p>Classical E. coli bacterial conjugation is often regarded as the bacterial equivalent of sexual reproduction or mating since it involves the exchange of genetic material. However, it is not sexual reproduction, since no exchange of gamete occurs, and indeed no generation of a new organism: instead an existing organism is transformed. During classical E. coli conjugation the donor cell provides a conjugative or mobilizable genetic element that is most often a plasmid or transposon. Most conjugative plasmids have systems ensuring that the recipient cell does not already contain a similar element.</p>
<p>The genetic information transferred is often beneficial to the recipient. Benefits may include antibiotic resistance, xenobiotic tolerance or the ability to use new metabolites. Other elements can be detrimental and may be viewed as bacterial parasites.</p>
<p>Conjugation in Escherichia coli by spontaneous zygogenesis and in Mycobacterium smegmatis by distributive conjugal transfer differ from the better studied classical E. coli conjugation in that these cases involve substantial blending of the parental genomes.</p>

<div class="figure" style="text-align: center"><span id="fig:conjugation"></span>
<img src="./figures/genetics/Conjugation.svg" alt="Schematic drawing of bacterial conjugation. Conjugation diagram 1) Donor cell produces pilus. 2) Pilus attaches to recipient cell, brings the two cells together. 3) The mobile plasmid is nicked and a single strand of DNA is then transferred to the recipient cell. 4) Both cells recircularize their plasmids, synthesize second strands, and reproduce pili; both cells are now viable donors." width="70%" />
<p class="caption">
Figure 9.5: <a href="https://commons.wikimedia.org/wiki/File:Conjugation.svg">Schematic drawing of bacterial conjugation</a>. Conjugation diagram 1) Donor cell produces pilus. 2) Pilus attaches to recipient cell, brings the two cells together. 3) The mobile plasmid is nicked and a single strand of DNA is then transferred to the recipient cell. 4) Both cells recircularize their plasmids, synthesize second strands, and reproduce pili; both cells are now viable donors.
</p>
</div>
<p>The F-plasmid is an episome (a plasmid that can integrate itself into the bacterial chromosome by homologous recombination) with a length of about 100 kb. It carries its own origin of replication, the oriV, and an origin of transfer, or oriT. There can only be one copy of the F-plasmid in a given bacterium, either free or integrated, and bacteria that possess a copy are called F-positive or F-plus (denoted F+). Cells that lack F plasmids are called F-negative or F-minus (F−) and as such can function as recipient cells.</p>
<p>Among other genetic information, the F-plasmid carries a tra and trb locus, which together are about 33 kb long and consist of about 40 genes. The tra locus includes the pilin gene and regulatory genes, which together form pili on the cell surface. The locus also includes the genes for the proteins that attach themselves to the surface of F− bacteria and initiate conjugation. Though there is some debate on the exact mechanism of conjugation it seems that the pili are not the structures through which DNA exchange occurs. This has been shown in experiments where the pilus are allowed to make contact, but then are denatured with SDS and yet DNA transformation still proceeds. Several proteins coded for in the tra or trb locus seem to open a channel between the bacteria and it is thought that the traD enzyme, located at the base of the pilus, initiates membrane fusion.</p>
<p>When conjugation is initiated by a signal the relaxase enzyme creates a nick in one of the strands of the conjugative plasmid at the oriT. Relaxase may work alone or in a complex of over a dozen proteins known collectively as a relaxosome. In the F-plasmid system the relaxase enzyme is called TraI and the relaxosome consists of TraI, TraY, TraM and the integrated host factor IHF. The nicked strand, or T-strand, is then unwound from the unbroken strand and transferred to the recipient cell in a 5’-terminus to 3’-terminus direction. The remaining strand is replicated either independent of conjugative action (vegetative replication beginning at the oriV) or in concert with conjugation (conjugative replication similar to the rolling circle replication of lambda phage). Conjugative replication may require a second nick before successful transfer can occur. A recent report claims to have inhibited conjugation with chemicals that mimic an intermediate step of this second nicking event.</p>
<p>If the F-plasmid that is transferred has previously been integrated into the donor’s genome (producing an Hfr strain [“High Frequency of Recombination”]) some of the donor’s chromosomal DNA may also be transferred with the plasmid DNA. The amount of chromosomal DNA that is transferred depends on how long the two conjugating bacteria remain in contact. In common laboratory strains of E. coli the transfer of the entire bacterial chromosome takes about 100 minutes. The transferred DNA can then be integrated into the recipient genome via homologous recombination.</p>
<p>A cell culture that contains in its population cells with non-integrated F-plasmids usually also contains a few cells that have accidentally integrated their plasmids. It is these cells that are responsible for the low-frequency chromosomal gene transfers that occur in such cultures. Some strains of bacteria with an integrated F-plasmid can be isolated and grown in pure culture. Because such strains transfer chromosomal genes very efficiently they are called Hfr (high frequency of recombination). The E. coli genome was originally mapped by interrupted mating experiments in which various Hfr cells in the process of conjugation were sheared from recipients after less than 100 minutes (initially using a Waring blender). The genes that were transferred were then investigated.</p>
<p>Since integration of the F-plasmid into the E. coli chromosome is a rare spontaneous occurrence, and since the numerous genes promoting DNA transfer are in the plasmid genome rather than in the bacterial genome, it has been argued that conjugative bacterial gene transfer, as it occurs in the E. coli Hfr system, is not an evolutionary adaptation of the bacterial host, nor is it likely ancestral to eukaryotic sex.</p>
</div>
</div>
<div id="structure-and-function-of-deoxyribonucleic-acid" class="section level2" number="9.4">
<h2><span class="header-section-number">9.4</span> Structure And Function Of Deoxyribonucleic Acid</h2>
<p><a href="https://en.wikipedia.org/wiki/DNA">Deoxyribonucleic acid</a> (DNA) is a molecule composed of two chains that coil around each other to form a double helix carrying genetic instructions for the development, functioning, growth and reproduction of all known organisms and many viruses. DNA and ribonucleic acid (RNA) are nucleic acids; alongside proteins, lipids and complex carbohydrates (polysaccharides), nucleic acids are one of the four major types of macromolecules that are essential for all known forms of life.</p>

<div class="figure" style="text-align: center"><span id="fig:dnastructure"></span>
<img src="figures/genetics/dna_licorice.png" alt="The structure of the DNA double helix. A section of DNA. The bases lie horizontally between the two spiraling strands. The atoms in the structure are colour-coded by element (based on atomic coordinates of PDB 1bna rendered with open source molecular visualization tool PyMol.)" width="70%" />
<p class="caption">
Figure 9.6: The structure of the DNA double helix. A section of DNA. The bases lie horizontally between the two spiraling strands. The atoms in the structure are colour-coded by element (based on atomic coordinates of <a href="https://www.rcsb.org/structure/1bna">PDB 1bna</a> rendered with open source molecular visualization tool PyMol.)
</p>
</div>
<p>The two DNA strands are also known as polynucleotides as they are composed of simpler monomeric units called nucleotides. Each nucleotide is composed of one of four nitrogen-containing nucleobases (cytosine [C], guanine [G], adenine [A] or thymine [T]), a sugar called deoxyribose, and a phosphate group. The nucleotides are joined to one another in a chain by covalent bonds between the sugar of one nucleotide and the phosphate of the next, resulting in an alternating sugar-phosphate backbone. The nitrogenous bases of the two separate polynucleotide strands are bound together, according to base pairing rules (A with T and C with G), with hydrogen bonds to make double-stranded DNA. The complementary nitrogenous bases are divided into two groups, pyrimidines and purines. In DNA, the pyrimidines are thymine and cytosine; the purines are adenine and guanine.</p>

<div class="figure" style="text-align: center"><span id="fig:basestruc"></span>
<img src="figures/genetics/bases_hydrogen.png" alt="The structure of the four nucleotides and their base pairing in the DNA double helix. The atoms in the structure are colour-coded by element (based on atomic coordinates of PDB 1bna rendered with open source molecular visualization tool PyMol.)" width="70%" />
<p class="caption">
Figure 9.7: The structure of the four nucleotides and their base pairing in the DNA double helix. The atoms in the structure are colour-coded by element (based on atomic coordinates of <a href="https://www.rcsb.org/structure/1bna">PDB 1bna</a> rendered with open source molecular visualization tool PyMol.)
</p>
</div>
<p>Both strands of DNA store biological information. This information is replicated as and when the two strands separate. A large part of DNA (more than 98% for humans) is non-coding, meaning that these sections do not serve as patterns for protein sequences. The two strands of DNA run in opposite directions to each other and are thus antiparallel. Attached to each sugar is one of four types of nucleobases (informally, bases). It is the sequence of these four nucleobases along the backbone that encodes genetic information. RNA strands are created using DNA strands as a template in a process called transcription, where DNA bases are exchanged for their corresponding bases except in the case of thymine (T), which RNA substitutes for uracil (U). Under the genetic code, these RNA strands specify the sequence of amino acids within proteins in a process called translation.</p>
<p>Within eukaryotic cells, DNA is organized into long structures called chromosomes. Before typical cell division, these chromosomes are duplicated in the process of DNA replication, providing a complete set of chromosomes for each daughter cell. Eukaryotic organisms (animals, plants, fungi and protists) store most of their DNA inside the cell nucleus as nuclear DNA, and some in the mitochondria as mitochondrial DNA or in chloroplasts as chloroplast DNA. In contrast, prokaryotes (bacteria and archaea) store their DNA only in the cytoplasm, in circular chromosomes. Within eukaryotic chromosomes, chromatin proteins, such as histones, compact and organize DNA. These compacting structures guide the interactions between DNA and other proteins, helping control which parts of the DNA are transcribed.</p>
<p>DNA was first isolated by <a href="https://en.wikipedia.org/wiki/Friedrich_Miescher">Friedrich Miescher</a> in 1869. Its molecular structure was first identified by <a href="https://en.wikipedia.org/wiki/Francis_Crick">Francis Crick</a> and <a href="https://en.wikipedia.org/wiki/James_Watson">James Watson</a> at the Cavendish Laboratory within the University of Cambridge in 1953, whose model-building efforts were guided by X-ray diffraction data acquired by <a href="https://en.wikipedia.org/wiki/Raymond_Gosling">Raymond Gosling</a>, who was a post-graduate student of <a href="https://en.wikipedia.org/wiki/Rosalind_Franklin">Rosalind Franklin</a>.</p>
<p>DNA is a long polymer made from repeating units called nucleotides, each of which is usually symbolized by a single letter: either A, T, C, or G. The structure of DNA is dynamic along its length, being capable of coiling into tight loops and other shapes. In all species it is composed of two helical chains, bound to each other by hydrogen bonds. Both chains are coiled around the same axis, and have the same pitch of 34 angstroms (Å) (3.4 nanometres). The pair of chains has a radius of 10 angstroms (1.0 nanometre). Although each individual nucleotide is very small, a DNA polymer can be very large and contain hundreds of millions, such as in chromosome 1. Chromosome 1 is the largest human chromosome with approximately 220 million base pairs, and would be 85 mm long if straightened.</p>
<p>DNA does not usually exist as a single strand, but instead as a pair of strands that are held tightly together. These two long strands coil around each other, in the shape of a double helix. The nucleotide contains both a segment of the backbone of the molecule (which holds the chain together) and a nucleobase (which interacts with the other DNA strand in the helix). A nucleobase linked to a sugar is called a nucleoside, and a base linked to a sugar and to one or more phosphate groups is called a nucleotide. A biopolymer comprising multiple linked nucleotides (as in DNA) is called a polynucleotide.</p>
<p>The backbone of the DNA strand is made from alternating phosphate and sugar residues. The sugar in DNA is 2-deoxyribose, which is a pentose (five-carbon) sugar. The sugars are joined together by phosphate groups that form phosphodiester bonds between the third and fifth carbon atoms of adjacent sugar rings. These are known as the 3′-end (three prime end), and 5′-end (five prime end) carbons, the prime symbol being used to distinguish these carbon atoms from those of the base to which the deoxyribose forms a glycosidic bond. When imagining DNA, each phosphoryl is normally considered to “belong” to the nucleotide whose 5′ carbon forms a bond therewith. Any DNA strand therefore normally has one end at which there is a phosphoryl attached to the 5′ carbon of a ribose (the 5′ phosphoryl) and another end at which there is a free hydroxyl attached to the 3′ carbon of a ribose (the 3′ hydroxyl). The orientation of the 3′ and 5′ carbons along the sugar-phosphate backbone confers directionality (sometimes called polarity) to each DNA strand. In a nucleic acid double helix, the direction of the nucleotides in one strand is opposite to their direction in the other strand: the strands are antiparallel. The asymmetric ends of DNA strands are said to have a directionality of five prime end (5′ ), and three prime end (3′), with the 5′ end having a terminal phosphate group and the 3′ end a terminal hydroxyl group. One major difference between DNA and RNA is the sugar, with the 2-deoxyribose in DNA being replaced by the alternative pentose sugar ribose in RNA.</p>
<p>Twin helical strands form the DNA backbone. Another double helix may be found tracing the spaces, or grooves, between the strands (Figure <a href="microbial-genetics.html#fig:groove">9.8</a>). These voids are adjacent to the base pairs and may provide a binding site. As the strands are not symmetrically located with respect to each other, the grooves are unequally sized. One groove, the major groove, is 22 angstroms (Å) wide and the other, the minor groove, is 12 Å wide. The width of the major groove means that the edges of the bases are more accessible in the major groove than in the minor groove. As a result, proteins such as transcription factors that can bind to specific sequences in double-stranded DNA usually make contact with the sides of the bases exposed in the major groove.</p>

<div class="figure" style="text-align: center"><span id="fig:groove"></span>
<img src="figures/genetics/dna_solid.png" alt="DNA major and minor grooves. PDB 1bna rendered with open source molecular visualization tool PyMol.)" width="70%" />
<p class="caption">
Figure 9.8: DNA major and minor grooves. <a href="https://www.rcsb.org/structure/1bna">PDB 1bna</a> rendered with open source molecular visualization tool PyMol.)
</p>
</div>
<p>In a DNA double helix, each type of nucleobase on one strand bonds with just one type of nucleobase on the other strand. This is called complementary base pairing. Here, purines form hydrogen bonds to pyrimidines, with adenine bonding only to thymine in two hydrogen bonds, and cytosine bonding only to guanine in three hydrogen bonds (Figure <a href="microbial-genetics.html#fig:pairing">9.9</a>). This arrangement of two nucleotides binding together across the double helix is called a Watson-Crick base pair. As hydrogen bonds are not covalent, they can be broken and rejoined relatively easily. The two strands of DNA in a double helix can thus be pulled apart like a zipper, either by a mechanical force or high temperature. As a result of this base pair complementarity, all the information in the double-stranded sequence of a DNA helix is duplicated on each strand, which is vital in DNA replication. This reversible and specific interaction between complementary base pairs is critical for all the functions of DNA in organisms.</p>

<div class="figure" style="text-align: center"><span id="fig:pairing"></span>
<img src="figures/genetics/Base_pair_GC.svg" alt="Top, a GC base pair with three hydrogen bonds. Bottom, an AT base pair with two hydrogen bonds. Non-covalent hydrogen bonds between the pairs are shown as dashed lines. The two types of base pairs form different numbers of hydrogen bonds, AT forming two hydrogen bonds, and GC forming three hydrogen bonds (see figures, right). DNA with high GC-content is more stable than DNA with low GC-content." width="70%" /><img src="figures/genetics/Base_pair_AT.svg" alt="Top, a GC base pair with three hydrogen bonds. Bottom, an AT base pair with two hydrogen bonds. Non-covalent hydrogen bonds between the pairs are shown as dashed lines. The two types of base pairs form different numbers of hydrogen bonds, AT forming two hydrogen bonds, and GC forming three hydrogen bonds (see figures, right). DNA with high GC-content is more stable than DNA with low GC-content." width="70%" />
<p class="caption">
Figure 9.9: Top, a <a href="https://commons.wikimedia.org/wiki/File:Base_pair_GC.svg">GC base pair</a> with three hydrogen bonds. Bottom, an <a href="https://commons.wikimedia.org/wiki/File:Base_pair_AT.svg">AT base pair</a> with two hydrogen bonds. Non-covalent hydrogen bonds between the pairs are shown as dashed lines. The two types of base pairs form different numbers of hydrogen bonds, AT forming two hydrogen bonds, and GC forming three hydrogen bonds (see figures, right). DNA with high GC-content is more stable than DNA with low GC-content.
</p>
</div>
<p>As noted above, most DNA molecules are actually two polymer strands, bound together in a helical fashion by noncovalent bonds; this double-stranded (dsDNA) structure is maintained largely by the intrastrand base stacking interactions, which are strongest for G,C stacks. The two strands can come apart—a process known as melting—to form two single-stranded DNA (ssDNA) molecules. Melting occurs at high temperature, low salt and high pH (low pH also melts DNA, but since DNA is unstable due to acid depurination, low pH is rarely used).</p>
<p>The stability of the dsDNA form depends not only on the GC-content (% G,C basepairs) but also on sequence (since stacking is sequence specific) and also length (longer molecules are more stable). The stability can be measured in various ways; a common way is the “melting temperature”, which is the temperature at which 50% of the ds molecules are converted to ss molecules; melting temperature is dependent on ionic strength and the concentration of DNA. As a result, it is both the percentage of GC base pairs and the overall length of a DNA double helix that determines the strength of the association between the two strands of DNA. Long DNA helices with a high GC-content have stronger-interacting strands, while short helices with high AT content have weaker-interacting strands. In biology, parts of the DNA double helix that need to separate easily, such as the TATAAT <a href="https://en.wikipedia.org/wiki/Pribnow_box">Pribnow box</a> in some promoters, tend to have a high AT content, making the strands easier to pull apart.</p>
<p>In the laboratory, the strength of this interaction can be measured by finding the temperature necessary to break the hydrogen bonds, their melting temperature (also called T<sub>m</sub> value). When all the base pairs in a DNA double helix melt, the strands separate and exist in solution as two entirely independent molecules. These single-stranded DNA molecules have no single common shape, but some conformations are more stable than others.</p>
<p>A DNA sequence is called a “sense” sequence if it is the same as that of a messenger RNA copy that is translated into protein. The sequence on the opposite strand is called the “antisense” sequence. Both sense and antisense sequences can exist on different parts of the same strand of DNA (i.e. both strands can contain both sense and antisense sequences). In both prokaryotes and eukaryotes, antisense RNA sequences are produced, but the functions of these RNAs are not entirely clear. One proposal is that antisense RNAs are involved in regulating gene expression through RNA-RNA base pairing.</p>
<p>A few DNA sequences in prokaryotes and eukaryotes, and more in plasmids and viruses, blur the distinction between sense and antisense strands by having overlapping genes. In these cases, some DNA sequences do double duty, encoding one protein when read along one strand, and a second protein when read in the opposite direction along the other strand. In bacteria, this overlap may be involved in the regulation of gene transcription, while in viruses, overlapping genes increase the amount of information that can be encoded within the small viral genome.</p>
<p>DNA can be twisted like a rope in a process called DNA supercoiling. With DNA in its “relaxed” state, a strand usually circles the axis of the double helix once every 10.4 base pairs, but if the DNA is twisted the strands become more tightly or more loosely wound. If the DNA is twisted in the direction of the helix, this is positive supercoiling, and the bases are held more tightly together. If they are twisted in the opposite direction, this is negative supercoiling, and the bases come apart more easily. In nature, most DNA has slight negative supercoiling that is introduced by enzymes called topoisomerases. These enzymes are also needed to relieve the twisting stresses introduced into DNA strands during processes such as transcription and DNA replication.</p>
<p>The expression of genes is influenced by how the DNA is packaged in chromosomes, in a structure called chromatin. Base modifications can be involved in packaging, with regions that have low or no gene expression usually containing high levels of methylation of cytosine bases. DNA packaging and its influence on gene expression can also occur by covalent modifications of the histone protein core around which DNA is wrapped in the chromatin structure or else by remodeling carried out by chromatin remodeling complexes. There is, further, crosstalk between DNA methylation and histone modification, so they can coordinately affect chromatin and gene expression.</p>
<p>For one example, cytosine methylation produces 5-methylcytosine, which is important for X-inactivation of chromosomes. The average level of methylation varies between organisms—the worm <em>Caenorhabditis elegans</em> lacks cytosine methylation, while vertebrates have higher levels, with up to 1% of their DNA containing 5-methylcytosine. Despite the importance of 5-methylcytosine, it can deaminate to leave a thymine base, so methylated cytosines are particularly prone to mutations. Other base modifications include adenine methylation in bacteria, the presence of 5-hydroxymethylcytosine in the brain, and the glycosylation of uracil to produce the “J-base” in kinetoplastids.</p>
<div id="dna-damage" class="section level3" number="9.4.1">
<h3><span class="header-section-number">9.4.1</span> DNA Damage</h3>
<p>DNA can be damaged by many sorts of <a href="https://en.wikipedia.org/wiki/Mutagen">mutagens</a>, which change the DNA sequence. Mutagens include oxidizing agents, alkylating agents and also high-energy electromagnetic radiation such as ultraviolet light and X-rays. The type of DNA damage produced depends on the type of mutagen. For example, UV light can damage DNA by producing thymine dimers, which are cross-links between pyrimidine bases. On the other hand, oxidants such as free radicals or hydrogen peroxide produce multiple forms of damage, including base modifications, particularly of guanosine, and double-strand breaks. A typical human cell contains about 150,000 bases that have suffered oxidative damage. Of these oxidative lesions, the most dangerous are double-strand breaks, as these are difficult to repair and can produce point mutations, insertions, deletions from the DNA sequence, and chromosomal translocations. These mutations can cause cancer. DNA damage that is naturally occurring, due to normal cellular processes that produce reactive oxygen species, the hydrolytic activities of cellular water, etc., also occurs frequently. Although most of this damage is repaired, in any cell some DNA damage may remain despite the action of repair processes. This DNA damage accumulates with age in mammalian postmitotic tissues. This accumulation appears to be an important underlying cause of aging.</p>
<p>Many mutagens fit into the space between two adjacent base pairs, this is called intercalation. Most intercalators are aromatic and planar molecules; examples include ethidium bromide, acridines, daunomycin, and doxorubicin. For an intercalator to fit between base pairs, the bases must separate, distorting the DNA strands by unwinding of the double helix. This inhibits both transcription and DNA replication, causing toxicity and mutations. As a result, DNA intercalators may be carcinogens, and in the case of thalidomide, a teratogen. Others such as benzo[a]pyrene diol epoxide and aflatoxin form DNA adducts that induce errors in replication. Nevertheless, due to their ability to inhibit DNA transcription and replication, other similar toxins are also used in chemotherapy to inhibit rapidly growing cancer cells.</p>
<p>DNA usually occurs as linear chromosomes in eukaryotes, and circular chromosomes in prokaryotes. The set of chromosomes in a cell makes up its genome; the human genome has approximately 3 billion base pairs of DNA arranged into 46 chromosomes. Transmission of genetic information in genes is achieved via complementary base pairing. For example, in transcription, the DNA sequence is copied into a complementary RNA sequence. Usually, this RNA copy is then used to make a matching protein sequence in a process called translation. In alternative fashion, a cell may simply copy its genetic information in a process called DNA replication.</p>
</div>
<div id="genes-and-genomes" class="section level3" number="9.4.2">
<h3><span class="header-section-number">9.4.2</span> Genes And Genomes</h3>
<p>Genomic DNA is tightly and orderly packed in the process called DNA condensation, to fit the small available volumes of the cell. In eukaryotes, DNA is located in the cell nucleus, with small amounts in mitochondria and chloroplasts. In prokaryotes, the DNA is held within an irregularly shaped body in the cytoplasm called the nucleoid.</p>
<p>In many species, only a small fraction of the total sequence of the genome encodes protein. For example, only about 1.5% of the human genome consists of protein-coding exons, with over 50% of human DNA consisting of non-coding repetitive sequences. The reasons for the presence of so much noncoding DNA in eukaryotic genomes and the extraordinary differences in genome size, or C-value, among species, represent a long-standing puzzle known as the “C-value enigma”. However, some DNA sequences that do not code protein may still encode functional non-coding RNA molecules, which are involved in the regulation of gene expression.</p>
<p>Some noncoding DNA sequences play structural roles in chromosomes. Telomeres and centromeres typically contain few genes but are important for the function and stability of chromosomes. An abundant form of noncoding DNA in humans are pseudogenes, which are copies of genes that have been disabled by mutation. These sequences are usually just molecular fossils, although they can occasionally serve as raw genetic material for the creation of new genes through the process of gene duplication and divergence.</p>
</div>
<div id="transcription-and-translation" class="section level3" number="9.4.3">
<h3><span class="header-section-number">9.4.3</span> Transcription And Translation</h3>
<p>A gene is a sequence of DNA that contains genetic information and can influence the phenotype of an organism. Within a gene, the sequence of bases along a DNA strand defines a messenger RNA sequence, which then defines one or more protein sequences. The relationship between the nucleotide sequences of genes and the amino-acid sequences of proteins is determined by the rules of translation, known collectively as the genetic code. The genetic code consists of three-letter ‘words’ called codons formed from a sequence of three nucleotides (e.g. ACT, CAG, TTT).</p>
<p>In transcription, the codons of a gene are copied into messenger RNA by RNA polymerase. This RNA copy is then decoded by a ribosome that reads the RNA sequence by base-pairing the messenger RNA to transfer RNA, which carries amino acids. Since there are 4 bases in 3-letter combinations, there are 64 possible codons. These encode the twenty standard amino acids, giving most amino acids more than one possible codon. There are also three ‘stop’ or ‘nonsense’ codons signifying the end of the coding region; these are the TAA, TGA, and TAG codons.</p>
</div>
<div id="dna-replication" class="section level3" number="9.4.4">
<h3><span class="header-section-number">9.4.4</span> DNA Replication</h3>
<p>Cell division is essential for an organism to grow, but, when a cell divides, it must replicate the DNA in its genome so that the two daughter cells have the same genetic information as their parent. The double-stranded structure of DNA provides a simple mechanism for DNA replication. Here, the two strands are separated and then each strand’s complementary DNA sequence is recreated by an enzyme called DNA polymerase. This enzyme makes the complementary strand by finding the correct base through complementary base pairing and bonding it onto the original strand. As DNA polymerases can only extend a DNA strand in a 5′ to 3′ direction, different mechanisms are used to copy the antiparallel strands of the double helix. In this way, the base on the old strand dictates which base appears on the new strand, and the cell ends up with a perfect copy of its DNA.</p>
<p>In molecular biology, DNA replication is the biological process of producing two identical replicas of DNA from one original DNA molecule. DNA replication occurs in all living organisms acting as the basis for biological inheritance.</p>
<p>DNA is made up of a double helix of two complementary strands. During replication, these strands are separated. Each strand of the original DNA molecule then serves as a template for the production of its counterpart, a process referred to as semi-conservative replication. As a result of semi-conservative replication, the new helix will be composed of an original DNA strand as well as a newly synthesized strand. Cellular proofreading and error-checking mechanisms ensure near perfect fidelity for DNA replication.</p>

<div class="figure" style="text-align: center"><span id="fig:repfork"></span>
<img src="figures/genetics/DNA_replication_split.svg" alt="DNA polymerases adds nucleotides to the 3′ end of a strand of DNA. If a mismatch is accidentally incorporated, the polymerase is inhibited from further extension. Proofreading removes the mismatched nucleotide and extension continues." width="70%" />
<p class="caption">
Figure 9.10: <a href="https://commons.wikimedia.org/wiki/File:DNA_replication_split.svg">DNA polymerases adds nucleotides to the 3′ end of a strand of DNA. If a mismatch is accidentally incorporated, the polymerase is inhibited from further extension. Proofreading removes the mismatched nucleotide and extension continues.</a>
</p>
</div>
<p>In a cell, DNA replication begins at specific locations, or origins of replication, in the genome. Unwinding of DNA at the origin and synthesis of new strands, accommodated by an enzyme known as helicase, results in replication forks growing bi-directionally from the origin. A number of proteins are associated with the replication fork to help in the initiation and continuation of DNA synthesis. Most prominently, DNA polymerase synthesizes the new strands by adding nucleotides that complement each (template) strand. DNA replication occurs during the S-stage of interphase.</p>
<p>DNA replication (DNA amplification) can also be performed in vitro (artificially, outside a cell). DNA polymerases isolated from cells and artificial DNA primers can be used to start DNA synthesis at known sequences in a template DNA molecule. Polymerase chain reaction (PCR), ligase chain reaction (LCR), and transcription-mediated amplification (TMA) are examples.</p>
<p>The <a href="https://en.wikipedia.org/wiki/Replisome">replisome</a> is a complex molecular machine that carries out replication of DNA. The replisome first unwinds double stranded DNA into two single strands. For each of the resulting single strands, a new complementary sequence of DNA is synthesized. The net result is formation of two new double stranded DNA sequences that are exact copies of the original double stranded DNA sequence.</p>
<p>In terms of structure, the replisome is composed of two replicative polymerase complexes, one of which synthesizes the leading strand, while the other synthesizes the lagging strand. The replisome is composed of a number of proteins including helicase, RFC, PCNA, gyrase/topoisomerase, SSB/RPA, primase, DNA polymerase III, RNAse H, and ligase.</p>
<p>For prokaryotes, each dividing nucleoid (region containing genetic material which is not a nucleus) requires two replisomes for bidirectional replication. The two replisomes continue replication at both forks in the middle of the cell. Finally, as the termination site replicates, the two replisomes separate from the DNA. The replisome remains at a fixed, midcell location in the cell, attached to the membrane, and the template DNA threads through it. DNA is fed through the stationary pair of replisomes located at the cell membrane.</p>
<p>For eukaryotes, numerous replication bubbles form at origins of replication throughout the chromosome. As with prokaryotes, two replisomes are required, one at each replication fork located at the terminus of the replication bubble. Because of significant differences in chromosome size, and the associated complexities of highly condensed chromosomes, various aspects of the DNA replication process in eukaryotes, including the terminal phases, are less well-characterised than for prokaryotes.</p>
<p>The replisome is responsible for copying the entirety of genomic DNA in each proliferative cell. This process allows for the high-fidelity passage of hereditary/genetic information from parental cell to daughter cell and is thus essential to all organisms. Much of the cell cycle is built around ensuring that DNA replication occurs without errors.</p>

<div class="figure" style="text-align: center"><span id="fig:repforkform"></span>
<img src="figures/genetics/Eukaryotic_DNA_replication.svg" alt="Many enzymes are involved in forming the DNA replication fork and DNA polymerization." width="70%" />
<p class="caption">
Figure 9.11: <a href="https://commons.wikimedia.org/wiki/File:Eukaryotic_DNA_replication.svg">Many enzymes are involved in forming the DNA replication fork and DNA polymerization.</a>
</p>
</div>
<p>In G1 phase of the cell cycle, many of the DNA replication regulatory processes are initiated. In eukaryotes, the vast majority of DNA synthesis occurs during S phase of the cell cycle, and the entire genome must be unwound and duplicated to form two daughter copies. During G2, any damaged DNA or replication errors are corrected. Finally, one copy of the genomes is segregated to each daughter cell at mitosis or M phase. These daughter copies each contain one strand from the parental duplex DNA and one nascent antiparallel strand.</p>
</div>
<div id="eukaryotic-dna-replication" class="section level3" number="9.4.5">
<h3><span class="header-section-number">9.4.5</span> Eukaryotic DNA Replication</h3>
<p>Eukaryotic DNA replication is a conserved mechanism that restricts DNA replication to once per cell cycle. Eukaryotic DNA replication of chromosomal DNA is central for the duplication of a cell and is necessary for the maintenance of the eukaryotic genome.</p>
<p>DNA replication is the action of DNA polymerases synthesizing a DNA strand complementary to the original template strand. To synthesize DNA, the double-stranded DNA is unwound by DNA helicases ahead of polymerases, forming a replication fork containing two single-stranded templates. Replication processes permit the copying of a single DNA double helix into two DNA helices, which are divided into the daughter cells at mitosis. The major enzymatic functions carried out at the replication fork are well conserved from prokaryotes to eukaryotes, but the replication machinery in eukaryotic DNA replication is a much larger complex, coordinating many proteins at the site of replication, forming the replisome.</p>
<p>After the replicative helicase has unwound the parental DNA duplex, exposing two single-stranded DNA templates, replicative polymerases are needed to generate two copies of the parental genome. DNA polymerase function is highly specialized and accomplish replication on specific templates and in narrow localizations. At the eukaryotic replication fork, there are three distinct replicative polymerase complexes that contribute to DNA replication: Polymerase α, Polymerase δ, and Polymerase ε. These three polymerases are essential for viability of the cell.</p>

<div class="figure" style="text-align: center"><span id="fig:polymerase"></span>
<img src="figures/genetics/DNA_polymerase.svg" alt="DNA polymerases adds nucleotides to the 3′ end of a strand of DNA. If a mismatch is accidentally incorporated, the polymerase is inhibited from further extension. Proofreading removes the mismatched nucleotide and extension continues." width="70%" />
<p class="caption">
Figure 9.12: <a href="https://commons.wikimedia.org/wiki/File:DNA_polymerase.svg">DNA polymerases adds nucleotides to the 3′ end of a strand of DNA. If a mismatch is accidentally incorporated, the polymerase is inhibited from further extension. Proofreading removes the mismatched nucleotide and extension continues.</a>
</p>
</div>
<p>Because DNA polymerases require a primer on which to begin DNA synthess, polymerase α (Pol α) acts as a replicative primase. Pol α is associated with an RNA primase and this complex accomplishes the priming task by synthesizing a primer that contains a short 10 nucleotide stretch of RNA followed by 10 to 20 DNA bases. Importantly, this priming action occurs at replication initiation at origins to begin leading-strand synthesis and also at the 5’ end of each Okazaki fragment on the lagging strand.</p>
<p>However, Pol α is not able to continue DNA replication and must be replaced with another polymerase to continue DNA synthesis. Polymerase switching requires clamp loaders and it has been proven that normal DNA replication requires the coordinated actions of all three DNA polymerases: Pol α for priming synthesis, Pol ε for leading-strand replication, and the Pol δ, which is constantly loaded, for generating Okazaki fragments during lagging-strand synthesis.</p>
<ul>
<li>Polymerase α (Pol α): Forms a complex with a small catalytic subunit (PriS) and a large noncatalytic (PriL) subunit. First, synthesis of an RNA primer allows DNA synthesis by DNA polymerase alpha. Occurs once at the origin on the leading strand and at the start of each Okazaki fragment on the lagging strand. Pri subunits act as a primase, synthesizing an RNA primer. DNA Pol α elongates the newly formed primer with DNA nucleotides. After around 20 nucleotides, elongation is taken over by Pol ε on the leading strand and Pol δ on the lagging strand.</li>
<li>Polymerase δ (Pol δ): Highly processive and has proofreading, 3’-&gt;5’ exonuclease activity. In vivo, it is the main polymerase involved in both lagging strand and leading strand synthesis.</li>
<li>Polymerase ε (Pol ε): Highly processive and has proofreading, 3’-&gt;5’ exonuclease activity. Highly related to pol δ, in vivo it functions mainly in error checking of pol δ.</li>
</ul>
<p>DNA replication, like all biological polymerization processes, proceeds in three enzymatically catalyzed and coordinated steps: initiation, elongation and termination.</p>
</div>
<div id="initiation" class="section level3" number="9.4.6">
<h3><span class="header-section-number">9.4.6</span> Initiation</h3>
<p>For a cell to divide, it must first replicate its DNA. DNA replication is an all-or-none process; once replication begins, it proceeds to completion. Once replication is complete, it does not occur again in the same cell cycle. This is made possible by the division of initiation into two temporally distinct steps: formation of the pre-replication complex and the preinitiation complex.</p>
</div>
<div id="pre-replication-complex" class="section level3" number="9.4.7">
<h3><span class="header-section-number">9.4.7</span> Pre-Replication Complex</h3>
<p>In late mitosis and early G1 phase, a large complex of initiator proteins assembles into the pre-replication complex at particular points in the DNA, known as “origins”. In <em>E. coli</em> the primary initiator protein is DnaA; in yeast, this is the origin recognition complex. Sequences used by initiator proteins tend to be “AT-rich” (rich in adenine and thymine bases), because A-T base pairs have two hydrogen bonds (rather than the three formed in a C-G pair) and thus are easier to strand-separate. In eukaryotes, the origin recognition complex catalyzes the assembly of initiator proteins into the pre-replication complex. Cdc6 and Cdt1 then associate with the bound origin recognition complex at the origin in order to form a larger complex necessary to load the Mcm complex onto the DNA. The Mcm complex is the helicase that will unravel the DNA helix at the replication origins and replication forks in eukaryotes. The Mcm complex is recruited at late G1 phase and loaded by the ORC-Cdc6-Cdt1 complex onto the DNA via ATP-dependent protein remodeling. The loading of the Mcm complex onto the origin DNA marks the completion of pre-replication complex formation.</p>
<p>If environmental conditions are right in late G1 phase, the G1 and G1/S cyclin-Cdk complexes are activated, which stimulate expression of genes that encode components of the DNA synthetic machinery. G1/S-Cdk activation also promotes the expression and activation of S-Cdk complexes, which may play a role in activating replication origins depending on species and cell type. Control of these Cdks vary depending cell type and stage of development.</p>
<p>In a similar manner, Cdc7 is also required through S phase to activate replication origins. Cdc7 is not active throughout the cell cycle, and its activation is strictly timed to avoid premature initiation of DNA replication. In late G1, Cdc7 activity rises abruptly as a result of association with the regulatory subunit Dbf4, which binds Cdc7 directly and promotes its protein kinase activity. Cdc7 has been found to be a rate-limiting regulator of origin activity. Together, the G1/S-Cdks and/or S-Cdks and Cdc7 collaborate to directly activate the replication origins, leading to initiation of DNA synthesis.</p>
</div>
<div id="preinitiation-complex" class="section level3" number="9.4.8">
<h3><span class="header-section-number">9.4.8</span> Preinitiation Complex</h3>
<p>In early S phase, S-Cdk and Cdc7 activation lead to the assembly of the preinitiation complex, a massive protein complex formed at the origin. Formation of the preinitiation complex displaces Cdc6 and Cdt1 from the origin replication complex, inactivating and disassembling the pre-replication complex. Loading the preinitiation complex onto the origin activates the Mcm helicase, causing unwinding of the DNA helix. The preinitiation complex also loads α-primase and other DNA polymerases onto the DNA.</p>
<p>After α-primase synthesizes the first primers, the primer-template junctions interact with the clamp loader, which loads the sliding clamp onto the DNA to begin DNA synthesis. The components of the preinitiation complex remain associated with replication forks as they move out from the origin.</p>
</div>
<div id="elongation" class="section level3" number="9.4.9">
<h3><span class="header-section-number">9.4.9</span> Elongation</h3>
<p>DNA polymerase has 5′–3′ activity. All known DNA replication systems require a free 3′ hydroxyl group before synthesis can be initiated (note: the DNA template is read in 3′ to 5′ direction whereas a new strand is synthesized in the 5′ to 3′ direction—this is often confused). Four distinct mechanisms for DNA synthesis are recognized:</p>
<p>All cellular life forms and many DNA viruses, phages and plasmids use a primase to synthesize a short RNA primer with a free 3′ OH group which is subsequently elongated by a DNA polymerase. The 5′ end of the nicked strand is transferred to a tyrosine residue on the nuclease and the free 3′ OH group is then used by the DNA polymerase to synthesize the new strand.
The first is the best known of these mechanisms and is used by the cellular organisms. In this mechanism, once the two strands are separated, primase adds RNA primers to the template strands. The leading strand receives one RNA primer while the lagging strand receives several. The leading strand is continuously extended from the primer by a DNA polymerase with high processivity, while the lagging strand is extended discontinuously from each primer forming Okazaki fragments. RNase removes the primer RNA fragments, and a low processivity DNA polymerase distinct from the replicative polymerase enters to fill the gaps. When this is complete, a single nick on the leading strand and several nicks on the lagging strand can be found. Ligase works to fill these nicks in, thus completing the newly replicated DNA molecule.</p>
<p>Multiple DNA polymerases take on different roles in the DNA replication process. In <em>E. coli</em>, DNA Pol III is the polymerase enzyme primarily responsible for DNA replication. It assembles into a replication complex at the replication fork that exhibits extremely high processivity, remaining intact for the entire replication cycle. In contrast, DNA Pol I is the enzyme responsible for replacing RNA primers with DNA. DNA Pol I has a 5′ to 3′ exonuclease activity in addition to its polymerase activity, and uses its exonuclease activity to degrade the RNA primers ahead of it as it extends the DNA strand behind it, in a process called nick translation. Pol I is much less processive than Pol III because its primary function in DNA replication is to create many short DNA regions rather than a few very long regions.</p>
<p>In eukaryotes, the low-processivity enzyme, Pol α, helps to initiate replication because it forms a complex with primase. In eukaryotes, leading strand synthesis is thought to be conducted by Pol ε; however, this view has recently been challenged, suggesting a role for Pol δ. Primer removal is completed by Pol δ while repair of DNA during replication is completed by Pol ε.</p>
<p>As DNA synthesis continues, the original DNA strands continue to unwind on each side of the bubble, forming a replication fork with two prongs. In bacteria, which have a single origin of replication on their circular chromosome, this process creates a “theta structure” (resembling the Greek letter theta: θ). In contrast, eukaryotes have longer linear chromosomes and initiate replication at multiple origins within these.</p>
</div>
<div id="replication-fork" class="section level3" number="9.4.10">
<h3><span class="header-section-number">9.4.10</span> Replication Fork</h3>
<p>The replication fork is a structure that forms within the long helical DNA during DNA replication. It is created by helicases, which break the hydrogen bonds holding the two DNA strands together in the helix. The resulting structure has two branching “prongs”, each one made up of a single strand of DNA. These two strands serve as the template for the leading and lagging strands, which will be created as DNA polymerase matches complementary nucleotides to the templates; the templates may be properly referred to as the leading strand template and the lagging strand template.</p>
<p>DNA is always synthesized by adding nucleotides to the 3′ end of a strand. Since the leading and lagging strand templates are oriented in opposite directions at the replication fork, a major issue is how to achieve synthesis of nascent (new) lagging strand DNA, whose direction of synthesis is opposite to the direction of the growing replication fork.</p>
</div>
<div id="replication-of-the-leading-strand" class="section level3" number="9.4.11">
<h3><span class="header-section-number">9.4.11</span> Replication Of The Leading Strand</h3>
<p>The leading strand is the strand of nascent DNA which is synthesized in the same direction as the growing replication fork. This sort of DNA replication is continuous.</p>
</div>
<div id="replication-of-the-lagging-strand" class="section level3" number="9.4.12">
<h3><span class="header-section-number">9.4.12</span> Replication Of The Lagging Strand</h3>
<p>The lagging strand is the strand of nascent DNA whose direction of synthesis is opposite to the direction of the growing replication fork. Because of its orientation, replication of the lagging strand is more complicated as compared to that of the leading strand. As a consequence, the DNA polymerase on this strand is seen to “lag behind” the other strand.</p>
<p>The lagging strand is synthesized in short, separated segments. On the lagging strand template, a primase “reads” the template DNA and initiates synthesis of a short complementary RNA primer. A DNA polymerase extends the primed segments, forming Okazaki fragments. The RNA primers are then removed and replaced with DNA, and the fragments of DNA are joined together by DNA ligase.</p>
<p>In all cases the helicase is composed of six polypeptides that wrap around only one strand of the DNA being replicated. The two polymerases are bound to the helicase heximer. In eukaryotes the helicase wraps around the leading strand, and in prokaryotes it wraps around the lagging strand.</p>
<p>As helicase unwinds DNA at the replication fork, the DNA ahead is forced to rotate. This process results in a build-up of twists in the DNA ahead. This build-up forms a torsional resistance that would eventually halt the progress of the replication fork. Topoisomerases are enzymes that temporarily break the strands of DNA, relieving the tension caused by unwinding the two strands of the DNA helix; topoisomerases (including DNA gyrase) achieve this by adding negative supercoils to the DNA helix.</p>
<p>Bare single-stranded DNA tends to fold back on itself forming secondary structures; these structures can interfere with the movement of DNA polymerase. To prevent this, single-strand binding proteins bind to the DNA until a second strand is synthesized, preventing secondary structure formation.</p>
<p>Clamp proteins form a sliding clamp around DNA, helping the DNA polymerase maintain contact with its template, thereby assisting with processivity. The inner face of the clamp enables DNA to be threaded through it. Once the polymerase reaches the end of the template or detects double-stranded DNA, the sliding clamp undergoes a conformational change that releases the DNA polymerase. Clamp-loading proteins are used to initially load the clamp, recognizing the junction between template and RNA primers.</p>
</div>
<div id="dna-replication-proteins" class="section level3" number="9.4.13">
<h3><span class="header-section-number">9.4.13</span> DNA Replication Proteins</h3>
<p>At the replication fork, many replication enzymes assemble on the DNA into a complex molecular machine called the replisome. The following is a list of major DNA replication enzymes that participate in the replisome:</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:heteroploidy">Table 9.2: </span>A list of major DNA replication enzymes that participate in the replisome
</caption>
<thead>
<tr>
<th style="text-align:left;">
Enzymes
</th>
<th style="text-align:left;">
Function in DNA replication
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 10em; ">
DNA helicase
</td>
<td style="text-align:left;width: 30em; ">
Also known as helix destabilizing enzyme. Helicase separates the two strands of DNA at the Replication Fork behind the topoisomerase.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
DNA polymerase
</td>
<td style="text-align:left;width: 30em; ">
The enzyme responsible for catalyzing the addition of nucleotide substrates to DNA in the 5′ to 3′ direction during DNA replication. Also performs proof-reading and error correction. There exist many different types of DNA Polymerase, each of which perform different functions in different types of cells.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
DNA clamp
</td>
<td style="text-align:left;width: 30em; ">
A protein which prevents elongating DNA polymerases from dissociating from the DNA parent strand.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Single-strand DNA-binding protein
</td>
<td style="text-align:left;width: 30em; ">
Bind to ssDNA and prevent the DNA double helix from re-annealing after DNA helicase unwinds it, thus maintaining the strand separation, and facilitating the synthesis of the nascent strand.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Topoisomerase
</td>
<td style="text-align:left;width: 30em; ">
Relaxes the DNA from its super-coiled nature.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
DNA gyrase
</td>
<td style="text-align:left;width: 30em; ">
Relieves strain of unwinding by DNA helicase; this is a specific type of topoisomerase
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
DNA ligase
</td>
<td style="text-align:left;width: 30em; ">
Re-anneals the semi-conservative strands and joins Okazaki Fragments of the lagging strand.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Primase
</td>
<td style="text-align:left;width: 30em; ">
Provides a starting point of RNA (or DNA) for DNA polymerase to begin synthesis of the new DNA strand.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Telomerase
</td>
<td style="text-align:left;width: 30em; ">
Lengthens telomeric DNA by adding repetitive nucleotide sequences to the ends of eukaryotic chromosomes. This allows germ cells and stem cells to avoid the Hayflick limit on cell divisi
</td>
</tr>
</tbody>
</table>
</div>
<div id="termination" class="section level3" number="9.4.14">
<h3><span class="header-section-number">9.4.14</span> Termination</h3>
<p>Eukaryotes initiate DNA replication at multiple points in the chromosome, so replication forks meet and terminate at many points in the chromosome. Because eukaryotes have linear chromosomes, DNA replication is unable to reach the very end of the chromosomes. Due to this problem, DNA is lost in each replication cycle from the end of the chromosome. Telomeres are regions of repetitive DNA close to the ends and help prevent loss of genes due to this shortening. Shortening of the telomeres is a normal process in somatic cells. This shortens the telomeres of the daughter DNA chromosome. As a result, cells can only divide a certain number of times before the DNA loss prevents further division. (This is known as the <a href="https://en.wikipedia.org/wiki/Leonard_Hayflick">Hayflick</a> limit.) Within the germ cell line, which passes DNA to the next generation, telomerase extends the repetitive sequences of the telomere region to prevent degradation. Telomerase can become mistakenly active in somatic cells, sometimes leading to cancer formation. Increased telomerase activity is one of the hallmarks of cancer.</p>
<p>Termination requires that the progress of the DNA replication fork must stop or be blocked. Termination at a specific locus, when it occurs, involves the interaction between two components: (1) a termination site sequence in the DNA, and (2) a protein which binds to this sequence to physically stop DNA replication. In various bacterial species, this is named the DNA replication terminus site-binding protein, or Ter protein.</p>
<p>Because bacteria have circular chromosomes, termination of replication occurs when the two replication forks meet each other on the opposite end of the parental chromosome. <em>E. coli</em> regulates this process through the use of termination sequences that, when bound by the Tus protein, enable only one direction of replication fork to pass through. As a result, the replication forks are constrained to always meet within the termination region of the chromosome.</p>
</div>
<div id="regulation-of-dna-replication" class="section level3" number="9.4.15">
<h3><span class="header-section-number">9.4.15</span> Regulation Of DNA Replication</h3>
<p>Within eukaryotes, DNA replication is controlled within the context of the cell cycle. As the cell grows and divides, it progresses through stages in the cell cycle; DNA replication takes place during the S phase (synthesis phase). The progress of the eukaryotic cell through the cycle is controlled by cell cycle checkpoints. Progression through checkpoints is controlled through complex interactions between various proteins, including cyclins and cyclin-dependent kinases.</p>
<p>The G1/S checkpoint (or restriction checkpoint) regulates whether eukaryotic cells enter the process of DNA replication and subsequent division. Cells that do not proceed through this checkpoint remain in the G0 stage and do not replicate their DNA.</p>
<p>After passing through the G1/S checkpoint, DNA must be replicated only once in each cell cycle. When the Mcm complex moves away from the origin, the pre-replication complex is dismantled. Because a new Mcm complex cannot be loaded at an origin until the pre-replication subunits are reactivated, one origin of replication can not be used twice in the same cell cycle.</p>
<p>Activation of S-Cdks in early S phase promotes the destruction or inhibition of individual pre-replication complex components, preventing immediate reassembly. S and M-Cdks continue to block pre-replication complex assembly even after S phase is complete, ensuring that assembly cannot occur again until all Cdk activity is reduced in late mitosis.</p>
<p>Replication of chloroplast and mitochondrial genomes occurs independently of the cell cycle, through the process of D-loop replication.</p>
</div>
<div id="interactions-of-dna-with-proteins" class="section level3" number="9.4.16">
<h3><span class="header-section-number">9.4.16</span> Interactions Of DNA with Proteins</h3>
<p>All the functions of DNA depend on interactions with proteins. These protein interactions can be non-specific, or the protein can bind specifically to a single DNA sequence. Enzymes can also bind to DNA and of these, the polymerases that copy the DNA base sequence in transcription and DNA replication are particularly important.</p>
</div>
<div id="dna-binding-proteins" class="section level3" number="9.4.17">
<h3><span class="header-section-number">9.4.17</span> DNA-Binding Proteins</h3>
<p>Structural proteins that bind DNA are well-understood examples of non-specific DNA-protein interactions. Within chromosomes, DNA is held in complexes with structural proteins. These proteins organize the DNA into a compact structure called chromatin. In eukaryotes, this structure involves DNA binding to a complex of small basic proteins called histones, while in prokaryotes multiple types of proteins are involved. The histones form a disk-shaped complex called a nucleosome, which contains two complete turns of double-stranded DNA wrapped around its surface. These non-specific interactions are formed through basic residues in the histones, making ionic bonds to the acidic sugar-phosphate backbone of the DNA, and are thus largely independent of the base sequence. Chemical modifications of these basic amino acid residues include methylation, phosphorylation, and acetylation. These chemical changes alter the strength of the interaction between the DNA and the histones, making the DNA more or less accessible to transcription factors and changing the rate of transcription. Other non-specific DNA-binding proteins in chromatin include the high-mobility group proteins, which bind to bent or distorted DNA. These proteins are important in bending arrays of nucleosomes and arranging them into the larger structures that make up chromosomes.</p>
<p>A distinct group of DNA-binding proteins is the DNA-binding proteins that specifically bind single-stranded DNA. In humans, replication protein A is the best-understood member of this family and is used in processes where the double helix is separated, including DNA replication, recombination, and DNA repair. These binding proteins seem to stabilize single-stranded DNA and protect it from forming stem-loops or being degraded by nucleases.</p>
<p>In contrast, other proteins have evolved to bind to particular DNA sequences. The most intensively studied of these are the various transcription factors, which are proteins that regulate transcription. Each transcription factor binds to one particular set of DNA sequences and activates or inhibits the transcription of genes that have these sequences close to their promoters. The transcription factors do this in two ways. Firstly, they can bind the RNA polymerase responsible for transcription, either directly or through other mediator proteins; this locates the polymerase at the promoter and allows it to begin transcription. Alternatively, transcription factors can bind enzymes that modify the histones at the promoter. This changes the accessibility of the DNA template to the polymerase.</p>
<p>As these DNA targets can occur throughout an organism’s genome, changes in the activity of one type of transcription factor can affect thousands of genes. Consequently, these proteins are often the targets of the signal transduction processes that control responses to environmental changes or cellular differentiation and development. The specificity of these transcription factors’ interactions with DNA come from the proteins making multiple contacts to the edges of the DNA bases, allowing them to “read” the DNA sequence. Most of these base-interactions are made in the major groove, where the bases are most accessible.</p>
</div>
<div id="dna-modifying-enzymes" class="section level3" number="9.4.18">
<h3><span class="header-section-number">9.4.18</span> DNA-modifying Enzymes</h3>
</div>
<div id="nucleases-and-ligases" class="section level3" number="9.4.19">
<h3><span class="header-section-number">9.4.19</span> Nucleases And Ligases</h3>
<p>Nucleases are enzymes that cut DNA strands by catalyzing the hydrolysis of the phosphodiester bonds. Nucleases that hydrolyse nucleotides from the ends of DNA strands are called exonucleases, while endonucleases cut within strands. The most frequently used nucleases in molecular biology are the restriction endonucleases, which cut DNA at specific sequences. For instance, the EcoRI enzyme recognizes the 6-base sequence 5′-GAATTC-3′ and cuts each strand after the G creating 4 nucleotide sticky ends with a 5’ end overhang of AATT. In nature, these enzymes protect bacteria against phage infection by digesting the phage DNA when it enters the bacterial cell, acting as part of the restriction modification system. These sequence-specific nucleases are used in molecular cloning and DNA fingerprinting.</p>
<p>Enzymes called DNA ligases can rejoin cut or broken DNA strands. Ligases are particularly important in lagging strand DNA replication, as they join together the short segments of DNA produced at the replication fork into a complete copy of the DNA template. They are also used in DNA repair and genetic recombination.</p>
</div>
<div id="topoisomerases-and-helicases" class="section level3" number="9.4.20">
<h3><span class="header-section-number">9.4.20</span> Topoisomerases And Helicases</h3>
<p>Topoisomerases are enzymes with both nuclease and ligase activity. These proteins change the amount of supercoiling in DNA. Some of these enzymes work by cutting the DNA helix and allowing one section to rotate, thereby reducing its level of supercoiling; the enzyme then seals the DNA break. Other types of these enzymes are capable of cutting one DNA helix and then passing a second strand of DNA through this break, before rejoining the helix. Topoisomerases are required for many processes involving DNA, such as DNA replication and transcription.</p>
<p>Helicases are proteins that are a type of molecular motor. They use the chemical energy in nucleoside triphosphates, predominantly adenosine triphosphate (ATP), to break hydrogen bonds between bases and unwind the DNA double helix into single strands. These enzymes are essential for most processes where enzymes need to access the DNA bases.</p>
</div>
<div id="polymerases" class="section level3" number="9.4.21">
<h3><span class="header-section-number">9.4.21</span> Polymerases</h3>
<p>Polymerases are enzymes that synthesize polynucleotide chains from nucleoside triphosphates. The sequence of their products is created based on existing polynucleotide chains—which are called templates. These enzymes function by repeatedly adding a nucleotide to the 3′ hydroxyl group at the end of the growing polynucleotide chain. As a consequence, all polymerases work in a 5′ to 3′ direction. In the active site of these enzymes, the incoming nucleoside triphosphate base-pairs to the template: this allows polymerases to accurately synthesize the complementary strand of their template. Polymerases are classified according to the type of template that they use.</p>
<p>In DNA replication, DNA-dependent DNA polymerases make copies of DNA polynucleotide chains. To preserve biological information, it is essential that the sequence of bases in each copy are precisely complementary to the sequence of bases in the template strand. Many DNA polymerases have a proofreading activity. Here, the polymerase recognizes the occasional mistakes in the synthesis reaction by the lack of base pairing between the mismatched nucleotides. If a mismatch is detected, a 3′ to 5′ exonuclease activity is activated and the incorrect base removed. In most organisms, DNA polymerases function in a large complex called the <a href="https://en.wikipedia.org/wiki/Replisome">replisome</a> that contains multiple accessory subunits, such as the DNA clamp or helicases.</p>
<p>RNA-dependent DNA polymerases are a specialized class of polymerases that copy the sequence of an RNA strand into DNA. They include reverse transcriptase, which is a viral enzyme involved in the infection of cells by retroviruses, and telomerase, which is required for the replication of telomeres. For example, HIV reverse transcriptase is an enzyme for AIDS virus replication. Telomerase is an unusual polymerase because it contains its own RNA template as part of its structure. It synthesizes telomeres at the ends of chromosomes. Telomeres prevent fusion of the ends of neighboring chromosomes and protect chromosome ends from damage.</p>
<p>Transcription is carried out by a DNA-dependent RNA polymerase that copies the sequence of a DNA strand into RNA. To begin transcribing a gene, the RNA polymerase binds to a sequence of DNA called a promoter and separates the DNA strands. It then copies the gene sequence into a messenger RNA transcript until it reaches a region of DNA called the terminator, where it halts and detaches from the DNA. As with human DNA-dependent DNA polymerases, RNA polymerase II, the enzyme that transcribes most of the genes in the human genome, operates as part of a large protein complex with multiple regulatory and accessory subunits.</p>
</div>
<div id="dna-recombination" class="section level3" number="9.4.22">
<h3><span class="header-section-number">9.4.22</span> DNA Recombination</h3>
<p>A DNA helix usually does not interact with other segments of DNA, and in human cells, the different chromosomes even occupy separate areas in the nucleus called “chromosome territories”. This physical separation of different chromosomes is important for the ability of DNA to function as a stable repository for information, as one of the few times chromosomes interact is in chromosomal crossover which occurs during sexual reproduction, when genetic recombination occurs. Chromosomal crossover is when two DNA helices break, swap a section and then rejoin.</p>
<p>Recombination allows chromosomes to exchange genetic information and produces new combinations of genes, which increases the efficiency of natural selection and can be important in the rapid evolution of new proteins. Genetic recombination can also be involved in DNA repair, particularly in the cell’s response to double-strand breaks.</p>
<p>The most common form of chromosomal crossover is homologous recombination, where the two chromosomes involved share very similar sequences. Non-homologous recombination can be damaging to cells, as it can produce chromosomal translocations and genetic abnormalities. The recombination reaction is catalyzed by enzymes known as recombinases, such as RAD51. The first step in recombination is a double-stranded break caused by either an endonuclease or damage to the DNA. A series of steps catalyzed in part by the recombinase then leads to joining of the two helices by at least one Holliday junction, in which a segment of a single strand in each helix is annealed to the complementary strand in the other helix. The Holliday junction is a tetrahedral junction structure that can be moved along the pair of chromosomes, swapping one strand for another. The recombination reaction is then halted by cleavage of the junction and re-ligation of the released DNA. Only strands of like polarity exchange DNA during recombination. There are two types of cleavage: east-west cleavage and north-south cleavage. The north-south cleavage nicks both strands of DNA, while the east-west cleavage has one strand of DNA intact.</p>
</div>
<div id="evolutionary-history-of-dna" class="section level3" number="9.4.23">
<h3><span class="header-section-number">9.4.23</span> Evolutionary History Of DNA</h3>
<p>DNA contains the genetic information that allows all forms of life to function, grow and reproduce. However, it is unclear how long in the 4-billion-year history of life DNA has performed this function, as it has been proposed that the earliest forms of life may have used RNA as their genetic material. RNA may have acted as the central part of early cell metabolism as it can both transmit genetic information and carry out catalysis as part of ribozymes. This ancient RNA world where nucleic acid would have been used for both catalysis and genetics may have influenced the evolution of the current genetic code based on four nucleotide bases. This would occur, since the number of different bases in such an organism is a trade-off between a small number of bases increasing replication accuracy and a large number of bases increasing the catalytic efficiency of ribozymes. However, there is no direct evidence of ancient genetic systems, as recovery of DNA from most fossils is impossible because DNA survives in the environment for less than one million years, and slowly degrades into short fragments in solution.</p>
<p>Building blocks of DNA (adenine, guanine, and related organic molecules) may have been formed extraterrestrially in outer space. Complex DNA and RNA organic compounds of life, including uracil, cytosine, and thymine, have also been formed in the laboratory under conditions mimicking those found in outer space, using starting chemicals, such as pyrimidine, found in meteorites. Pyrimidine, like polycyclic aromatic hydrocarbons (PAHs), the most carbon-rich chemical found in the universe, may have been formed in red giants or in interstellar cosmic dust and gas clouds.</p>
</div>
<div id="genetic-engineering" class="section level3" number="9.4.24">
<h3><span class="header-section-number">9.4.24</span> Genetic Engineering</h3>
<p>Methods have been developed to purify DNA from organisms, such as phenol-chloroform extraction, and to manipulate it in the laboratory, such as restriction digests and the polymerase chain reaction. Modern biology and biochemistry make intensive use of these techniques in recombinant DNA technology. Recombinant DNA is a man-made DNA sequence that has been assembled from other DNA sequences. They can be transformed into organisms in the form of plasmids or in the appropriate format, by using a viral vector. The genetically modified organisms produced can be used to produce products such as recombinant proteins, used in medical research, or be grown in agriculture.</p>
</div>
<div id="dna-profiling" class="section level3" number="9.4.25">
<h3><span class="header-section-number">9.4.25</span> Dna Profiling</h3>
<p>Forensic scientists can use DNA in blood, semen, skin, saliva or hair found at a crime scene to identify a matching DNA of an individual, such as a perpetrator. This process is formally termed DNA profiling, also called DNA fingerprinting. In DNA profiling, the lengths of variable sections of repetitive DNA, such as short tandem repeats and minisatellites, are compared between people. This method is usually an extremely reliable technique for identifying a matching DNA. However, identification can be complicated if the scene is contaminated with DNA from several people. DNA profiling was developed in 1984 by British geneticist Sir Alec Jeffreys, and first used in forensic science to convict Colin Pitchfork in the 1988 Enderby murders case.</p>
<p>DNA profiling is also used in DNA paternity testing to determine if someone is the biological parent or grandparent of a child with the probability of parentage is typically 99.99% when the alleged parent is biologically related to the child. Normally, paternity testing is performed after birth, but recently developed methods allow isolation and sequencing of fetal DNA from the blood of the mother.</p>
</div>
</div>
<div id="structure-and-function-of-ribonucleic-acid" class="section level2" number="9.5">
<h2><span class="header-section-number">9.5</span> Structure And Function Of Ribonucleic Acid</h2>
<p><a href="https://en.wikipedia.org/wiki/RNA">Ribonucleic acid (RNA)</a> is a polymeric molecule essential in various biological roles in coding, decoding, regulation and expression of genes. Like DNA, RNA is assembled as a chain of nucleotides, but unlike DNA it is more often found in nature as a single-strand folded onto itself, rather than a paired double-strand. Cellular organisms use messenger RNA (mRNA) to convey genetic information (using the nitrogenous bases of guanine, uracil, adenine, and cytosine, denoted by the letters G, U, A, and C) and direct synthesis of proteins by ribosomes. Many viruses encode their genetic information using an RNA genome.</p>
<p>Some RNA molecules play an active role within cells by catalyzing biological reactions, controlling gene expression, or sensing and communicating responses to cellular signals. One of these active processes is protein synthesis, a universal function in which RNA molecules direct the synthesis of proteins on ribosomes. This process uses transfer RNA (tRNA) molecules to deliver amino acids to the ribosome, where ribosomal RNA (rRNA) then links amino acids together to form coded proteins.</p>
<div id="comparison-with-dna" class="section level3" number="9.5.1">
<h3><span class="header-section-number">9.5.1</span> Comparison With DNA</h3>
<p>The chemical structure of RNA is very similar to that of DNA, but differs in three primary ways:</p>
<ul>
<li>Unlike double-stranded DNA, RNA is a single-stranded molecule in many of its biological roles and consists of much shorter chains of nucleotides. However, a single RNA molecule can, by complementary base pairing, form intrastrand double helixes, as in tRNA.</li>
<li>While the sugar-phosphate “backbone” of DNA contains deoxyribose, RNA contains ribose instead. Ribose has a hydroxyl group attached to the pentose ring in the 2’ position, whereas deoxyribose does not. The hydroxyl groups in the ribose backbone make RNA more chemically labile than DNA by lowering the activation energy of hydrolysis.</li>
<li>The complementary base to adenine in DNA is thymine, whereas in RNA, it is uracil, which is an unmethylated form of thymine.</li>
</ul>
<p>Like DNA, most biologically active RNAs, including mRNA, tRNA, rRNA, snRNAs, and other non-coding RNAs, contain self-complementary sequences that allow parts of the RNA to fold and pair with itself to form double helices. Analysis of these RNAs has revealed that they are highly structured. Unlike DNA, their structures do not consist of long double helices, but rather collections of short helices packed together into structures akin to proteins. In this fashion, RNAs can achieve chemical catalysis (like enzymes). For instance, determination of the structure of the ribosome—an RNA-protein complex that catalyzes peptide bond formation—revealed that its active site is composed entirely of RNA.</p>
</div>
<div id="structure-of-rna" class="section level3" number="9.5.2">
<h3><span class="header-section-number">9.5.2</span> Structure Of RNA</h3>
<p>Each nucleotide in RNA contains a ribose sugar, with carbons numbered 1’ through 5’. A base is attached to the 1’ position, in general, adenine (A), cytosine (C), guanine (G), or uracil (U). Adenine and guanine are purines, cytosine and uracil are pyrimidines. A phosphate group is attached to the 3’ position of one ribose and the 5’ position of the next. The phosphate groups have a negative charge each, making RNA a charged molecule (polyanion). The bases form hydrogen bonds between cytosine and guanine, between adenine and uracil and between guanine and uracil. However, other interactions are possible, such as a group of adenine bases binding to each other in a bulge, or the GNRA tetraloop that has a guanine–adenine base-pair.</p>
<p>An important structural component of RNA that distinguishes it from DNA is the presence of a hydroxyl group at the 2’ position of the ribose sugar.</p>
<p>RNA is transcribed with only four bases (adenine, cytosine, guanine and uracil), but these bases and attached sugars can be modified in numerous ways as the RNAs mature. Pseudouridine (Ψ), in which the linkage between uracil and ribose is changed from a C–N bond to a C–C bond, and ribothymidine (T) are found in various places (the most notable ones being in the TΨC loop of tRNA). Another notable modified base is hypoxanthine, a deaminated adenine base whose nucleoside is called inosine (I). Inosine plays a key role in the wobble hypothesis of the genetic code.</p>
<p>There are more than 100 other naturally occurring modified nucleosides. The greatest structural diversity of modifications can be found in tRNA, while pseudouridine and nucleosides with 2’-O-methylribose often present in rRNA are the most common. The specific roles of many of these modifications in RNA are not fully understood. However, it is notable that, in ribosomal RNA, many of the post-transcriptional modifications occur in highly functional regions, such as the peptidyl transferase center and the subunit interface, implying that they are important for normal function.</p>
<p>The functional form of single-stranded RNA molecules, just like proteins, frequently requires a specific tertiary structure. The scaffold for this structure is provided by secondary structural elements that are hydrogen bonds within the molecule. This leads to several recognizable “domains” of secondary structure like hairpin loops, bulges, and internal loops. Since RNA is charged, metal ions such as Mg2<sup>+</sup> are needed to stabilise many secondary and tertiary structures.</p>
<p>The naturally occurring enantiomer of RNA is D-RNA composed of D-ribonucleotides. All chirality centers are located in the D-ribose. By the use of L-ribose or rather L-ribonucleotides, L-RNA can be synthesized. L-RNA is much more stable against degradation by RNase.</p>
</div>
<div id="synthesis-of-rna" class="section level3" number="9.5.3">
<h3><span class="header-section-number">9.5.3</span> Synthesis Of RNA</h3>
<p>Synthesis of RNA is usually catalyzed by an enzyme—RNA polymerase—using DNA as a template, a process known as transcription. Initiation of transcription begins with the binding of the enzyme to a promoter sequence in the DNA (usually found “upstream” of a gene). The DNA double helix is unwound by the helicase activity of the enzyme. The enzyme then progresses along the template strand in the 3’ to 5’ direction, synthesizing a complementary RNA molecule with elongation occurring in the 5’ to 3’ direction. The DNA sequence also dictates where termination of RNA synthesis will occur.</p>
<p>Primary transcript RNAs are often modified by enzymes after transcription. For example, a poly(A) tail and a 5’ cap are added to eukaryotic pre-mRNA and introns are removed by the spliceosome.</p>
<p>There are also a number of RNA-dependent RNA polymerases that use RNA as their template for synthesis of a new strand of RNA. For instance, a number of RNA viruses (such as poliovirus) use this type of enzyme to replicate their genetic material. Also, RNA-dependent RNA polymerase is part of the RNA interference pathway in many organisms.</p>
</div>
<div id="coding-and-non-coding-rna" class="section level3" number="9.5.4">
<h3><span class="header-section-number">9.5.4</span> Coding And Non-coding RNA</h3>
<p>Messenger RNA (mRNA) is the RNA that carries information from DNA to the ribosome, the sites of protein synthesis (translation) in the cell. The coding sequence of the mRNA determines the amino acid sequence in the protein that is produced. However, many RNAs do not code for protein (about 97% of the transcriptional output is non-protein-coding in eukaryotes).</p>
<p>These so-called non-coding RNAs (“ncRNA”) can be encoded by their own genes (RNA genes), but can also derive from mRNA introns. The most prominent examples of non-coding RNAs are transfer RNA (tRNA) and ribosomal RNA (rRNA), both of which are involved in the process of translation. There are also non-coding RNAs involved in gene regulation, RNA processing and other roles. Certain RNAs are able to catalyse chemical reactions such as cutting and ligating other RNA molecules, and the catalysis of peptide bond formation in the ribosome; these are known as ribozymes.</p>
<p>According to the length of RNA chain, RNA includes small RNA and long RNA. Usually, small RNAs are shorter than 200 nt in length, and long RNAs are greater than 200 nt long. Long RNAs, also called large RNAs, mainly include long non-coding RNA (lncRNA) and mRNA. Small RNAs mainly include 5.8S ribosomal RNA (rRNA), 5S rRNA, transfer RNA (tRNA), microRNA (miRNA), small interfering RNA (siRNA), small nucleolar RNA (snoRNAs), Piwi-interacting RNA (piRNA), tRNA-derived small RNA (tsRNA) and small rDNA-derived RNA (srRNA).</p>
<p>Messenger RNA (mRNA) carries genetic information from the nucleus to the cytoplasm and serves as template for the synthesis of protein by ribosomes, a process called translation. In eukaryotic cells, once precursor mRNA (pre-mRNA) has been transcribed from DNA, it is processed to mature mRNA. This removes its introns (non-coding stretches of DNA sequence) from the pre-mRNA. The mRNA is then exported from the nucleus to the cytoplasm, where ribosomes bind to it and translate its the corresponding protein form with the help of tRNA. In prokaryotic cells, which do not have nucleus and cytoplasm compartments, mRNA can bind to ribosomes while it is being transcribed from DNA. After a certain amount of time, the message degrades into its component nucleotides with the assistance of ribonucleases.</p>
<p>Transfer RNA (tRNA) is a small RNA chain of about 80 nucleotides that transfers a specific amino acid to a growing polypeptide chain at the ribosomal site of protein synthesis during translation. It has sites for amino acid attachment and an anticodon region for codon recognition that binds to a specific sequence on the messenger RNA chain through hydrogen bonding.</p>

<div class="figure" style="text-align: center"><span id="fig:trnastruc1"></span>
<img src="figures/genetics/trna.png" alt="Tertiary structure of tRNA (based on atomic coordinates of PDB 1ehz rendered with open source molecular visualization tool PyMol.)" width="70%" />
<p class="caption">
Figure 9.13: Tertiary structure of tRNA (based on atomic coordinates of <a href="https://www.rcsb.org/structure/1ehz">PDB 1ehz</a> rendered with open source molecular visualization tool PyMol.)
</p>
</div>
<p>Ribosomal RNA (rRNA) is the catalytic component of the ribosomes. Eukaryotic ribosomes contain four different rRNA molecules: 18S, 5.8S, 28S and 5S rRNA. Three of the rRNA molecules are synthesized in the nucleolus, and one is synthesized elsewhere. In the cytoplasm, ribosomal RNA and protein combine to form a nucleoprotein called a ribosome. The ribosome binds mRNA and carries out protein synthesis. Several ribosomes may be attached to a single mRNA at any time. Nearly all the RNA found in a typical eukaryotic cell is rRNA.</p>
<p>Transfer-messenger RNA (tmRNA) is found in many bacteria and plastids. It tags proteins encoded by mRNAs that lack stop codons for degradation and prevents the ribosome from stalling.</p>
</div>
<div id="regulatory-rna" class="section level3" number="9.5.5">
<h3><span class="header-section-number">9.5.5</span> Regulatory RNA</h3>
<p>The earliest known regulators of gene expression were proteins known as repressors and activators, regulators with specific short binding sites within enhancer regions near the genes to be regulated. More recently, RNAs have been found to regulate genes as well. There are several kinds of RNA-dependent processes in eukaryotes regulating the expression of genes at various points, such as RNAi repressing genes post-transcriptionally, long non-coding RNAs shutting down blocks of chromatin epigenetically, and enhancer RNAs inducing increased gene expression. In addition to these mechanisms in eukaryotes, both bacteria and archaea have been found to use regulatory RNAs extensively. Bacterial small RNA and the CRISPR system are examples of such prokaryotic regulatory RNA systems.</p>
</div>
<div id="rna-interference-by-mirnas" class="section level3" number="9.5.6">
<h3><span class="header-section-number">9.5.6</span> RNA Interference By miRNAs</h3>
<p>Post-transcriptional expression levels of many genes can be controlled by RNA interference, in which miRNAs, specific short RNA molecules, pair with mRNA regions and target them for degradation. This antisense-based process involves steps that first process the RNA so that it can base-pair with a region of its target mRNAs. Once the base pairing occurs, other proteins direct the mRNA to be destroyed by nucleases.</p>
<p>Next to be linked to regulation were Xist and other long noncoding RNAs associated with X chromosome inactivation. Their roles, at first mysterious, were shown to be the silencing of blocks of chromatin via recruitment of Polycomb complex so that messenger RNA could not be transcribed from them. Additional lncRNAs, currently defined as RNAs of more than 200 base pairs that do not appear to have coding potential, have been found associated with regulation of stem cell pluripotency and cell division.</p>
<p>The third major group of regulatory RNAs is called enhancer RNAs. It is not clear at present whether they are a unique category of RNAs of various lengths or constitute a distinct subset of lncRNAs. In any case, they are transcribed from enhancers, which are known regulatory sites in the DNA near genes they regulate. They up-regulate the transcription of the gene(s) under control of the enhancer from which they are transcribed.</p>
<p>At first, regulatory RNA was thought to be a eukaryotic phenomenon, a part of the explanation for why so much more transcription in higher organisms was seen than had been predicted. But as soon as researchers began to look for possible RNA regulators in bacteria, they turned up there as well, termed as small RNA (sRNA). Bacterial small RNAs generally act via antisense pairing with mRNA to down-regulate its translation, either by affecting stability or affecting cis-binding ability. Riboswitches have also been discovered. They are cis-acting regulatory RNA sequences acting allosterically. They change shape when they bind metabolites so that they gain or lose the ability to bind chromatin to regulate expression of genes.</p>
<p>Archaea also have systems of regulatory RNA. The CRISPR system, recently being used to edit DNA in situ, acts via regulatory RNAs in archaea and bacteria to provide protection against virus invaders.</p>
<p>Many RNAs are involved in modifying other RNAs. Introns are spliced out of pre-mRNA by spliceosomes, which contain several small nuclear RNAs (snRNA), or the introns can be ribozymes that are spliced by themselves. RNA can also be altered by having its nucleotides modified to nucleotides other than A, C, G and U. In eukaryotes, modifications of RNA nucleotides are in general directed by small nucleolar RNAs (snoRNA; 60–300 nt), found in the nucleolus and cajal bodies. snoRNAs associate with enzymes and guide them to a spot on an RNA by basepairing to that RNA. These enzymes then perform the nucleotide modification. rRNAs and tRNAs are extensively modified, but snRNAs and mRNAs can also be the target of base modification. RNA can also be methylated.</p>
<p>RNA viruses have genomes composed of RNA that encodes a number of proteins. The viral genome is replicated by some of those proteins, while other proteins protect the genome as the virus particle moves to a new host cell. Viroids are another group of pathogens, but they consist only of RNA, do not encode any protein and are replicated by a host plant cell’s polymerase.</p>
<p>Reverse transcribing viruses replicate their genomes by reverse transcribing DNA copies from their RNA; these DNA copies are then transcribed to new RNA. Retrotransposons also spread by copying DNA and RNA from one another, and telomerase contains an RNA that is used as template for building the ends of eukaryotic chromosomes.</p>
<p>Research on RNA has led to many important biological discoveries and numerous Nobel Prizes. Nucleic acids were discovered in 1868 by Friedrich Miescher, who called the material ‘nuclein’ since it was found in the nucleus. It was later discovered that prokaryotic cells, which do not have a nucleus, also contain nucleic acids. The role of RNA in protein synthesis was suspected already in 1939. <a href="">Severo Ochoa</a> won the 1959 Nobel Prize in Medicine (shared with <a href="">Arthur Kornberg</a>) after he discovered an enzyme that can synthesize RNA in the laboratory. However, the enzyme discovered by Ochoa (polynucleotide phosphorylase) was later shown to be responsible for RNA degradation, not RNA synthesis. In 1956 <a href="https://en.wikipedia.org/wiki/Alexander_Rich">Alex Rich</a> and David Davies hybridized two separate strands of RNA to form the first crystal of RNA whose structure could be determined by X-ray crystallography.</p>
<p>The sequence of the 77 nucleotides of a yeast tRNA was found by <a href="https://en.wikipedia.org/wiki/Robert_W._Holley">Robert W. Holley</a> in 1965, winning Holley the 1968 Nobel Prize in Medicine (shared with <a href="https://en.wikipedia.org/wiki/Har_Gobind_Khorana">Har Gobind Khorana</a> and <a href="https://en.wikipedia.org/wiki/Marshall_Warren_Nirenberg">Marshall Nirenberg</a>).</p>
<p>During the early 1970s, retroviruses and reverse transcriptase were discovered, showing for the first time that enzymes could copy RNA into DNA (the opposite of the usual route for transmission of genetic information). For this work, <a href="https://en.wikipedia.org/wiki/David_Baltimore">David Baltimore</a>, <a href="https://en.wikipedia.org/wiki/Renato_Dulbecco">Renato Dulbecco</a> and <a href="https://en.wikipedia.org/wiki/Howard_Martin_Temin">Howard Temin</a> were awarded a Nobel Prize in 1975. In 1976, <a href="">Walter Fiers</a> and his team determined the first complete nucleotide sequence of an RNA virus genome, that of bacteriophage MS2.</p>
<p>In 1977, introns and RNA splicing were discovered in both mammalian viruses and in cellular genes, resulting in a 1993 Nobel to <a href="https://en.wikipedia.org/wiki/Phillip_Allen_Sharp">Philip Sharp</a> and <a href="https://en.wikipedia.org/wiki/Richard_J._Roberts">Richard Roberts</a>. Catalytic RNA molecules (ribozymes) were discovered in the early 1980s, leading to a 1989 Nobel award to <a href="https://en.wikipedia.org/wiki/Thomas_Cech">Thomas Cech</a> and <a href="https://en.wikipedia.org/wiki/Sidney_Altman">Sidney Altman</a>. In 1990, it was found in Petunia that introduced genes can silence similar genes of the plant’s own, now known to be a result of RNA interference.</p>
<p>At about the same time, 22 nt long RNAs, now called microRNAs, were found to have a role in the development of <em>C. elegans</em>. Studies on RNA interference gleaned a Nobel Prize for <a href="https://en.wikipedia.org/wiki/Andrew_Fire">Andrew Fire</a> and <a href="https://en.wikipedia.org/wiki/Craig_Mello">Craig Mello</a> in 2006, and another Nobel was awarded for studies on the transcription of RNA to <a href="">Roger Kornberg</a> in the same year. The discovery of gene regulatory RNAs has led to attempts to develop drugs made of RNA, such as siRNA, to silence genes. Adding to the Nobel prizes awarded for research on RNA in 2009 it was awarded for the elucidation of the atomic structure of the ribosome to <a href="https://en.wikipedia.org/wiki/Venki_Ramakrishnan">Venki Ramakrishnan</a>, <a href="https://en.wikipedia.org/wiki/Thomas_A._Steitz">Tom Steitz</a>, and <a href="https://en.wikipedia.org/wiki/Ada_Yonath">Ada Yonath</a>.</p>
<p>In 1967, <a href="https://en.wikipedia.org/wiki/Carl_Woese">Carl Woese</a> hypothesized that RNA might be catalytic and suggested that the earliest forms of life (self-replicating molecules) could have relied on RNA both to carry genetic information and to catalyze biochemical reactions—an RNA world.</p>
</div>
</div>
<div id="gene-expression-1" class="section level2" number="9.6">
<h2><span class="header-section-number">9.6</span> Gene Expression</h2>
<p>Gene expression is the process by which information from a gene is used in the synthesis of a functional gene product. These products are often proteins, but in non-protein-coding genes such as transfer RNA (tRNA) or small nuclear RNA (snRNA) genes, the product is a functional RNA. Gene expression is summarized in the Central Dogma first formulated by Francis Crick in 1958, further developed in his 1970 article, and expanded by the subsequent discoveries of reverse transcription and RNA replication.</p>
<p>The process of gene expression is used by all known life—eukaryotes (including multicellular organisms), prokaryotes (bacteria and archaea), and utilized by viruses—to generate the macromolecular machinery for life.</p>
<p>In genetics, gene expression is the most fundamental level at which the genotype gives rise to the phenotype, i.e. observable trait. The genetic information stored in DNA represents the genotype, whereas the phenotype results from the “interpretation” of that information. Such phenotypes are often expressed by the synthesis of proteins that control the organism’s structure and development, or that act as enzymes catalyzing specific metabolic pathways.</p>
<p>All steps in the gene expression process may be modulated (regulated), including the transcription, RNA splicing, translation, and post-translational modification of a protein. Regulation of gene expression gives control over the timing, location, and amount of a given gene product (protein or ncRNA) present in a cell and can have a profound effect on the cellular structure and function. Regulation of gene expression is the basis for cellular differentiation, development, morphogenesis and the versatility and adaptability of any organism. Gene regulation may therefore serve as a substrate for evolutionary change.</p>
<div id="transcription-and-rna-processing" class="section level3" number="9.6.1">
<h3><span class="header-section-number">9.6.1</span> Transcription And RNA Processing</h3>
<p><a href="https://en.wikipedia.org/wiki/Transcription_(biology)">Transcription</a> is the first of several steps of DNA based gene expression, in which a particular segment of DNA is copied into RNA (especially mRNA) by the enzyme RNA polymerase. During transcription, a DNA sequence is read by an RNA polymerase, which produces a complementary, antiparallel RNA strand called a primary transcript.</p>
<p>Transcription proceeds in the following general steps:</p>
<ol style="list-style-type: decimal">
<li>RNA polymerase, together with one or more general transcription factors, binds to promoter DNA.</li>
<li>RNA polymerase creates a transcription bubble, which separates the two strands of the DNA helix. This is done by breaking the hydrogen bonds between complementary DNA nucleotides.</li>
<li>RNA polymerase adds RNA nucleotides (which are complementary to the nucleotides of one DNA strand).</li>
<li>RNA sugar-phosphate backbone forms with assistance from RNA polymerase to form an RNA strand.</li>
<li>Hydrogen bonds of the RNA–DNA helix break, freeing the newly synthesized RNA strand.</li>
<li>If the cell has a nucleus, the RNA may be further processed. This may include polyadenylation, capping, and splicing.</li>
<li>The RNA may remain in the nucleus or exit to the cytoplasm through the nuclear pore complex.</li>
</ol>
<p>The stretch of DNA transcribed into an RNA molecule is called a transcription unit. If the DNA encodes a protein, the transcription produces messenger RNA (mRNA); the mRNA, in turn, serves as a template for the protein’s synthesis through translation. Alternatively, the transcribed DNA may encode for non-coding RNA such as microRNA, ribosomal RNA (rRNA), transfer RNA (tRNA), or enzymatic RNA molecules called ribozymes.</p>
<p>A DNA transcription unit encoding for a protein may contain both a coding sequence, which will be translated into the protein, and regulatory sequences, which direct and regulate the synthesis of that protein. The regulatory sequence before (“upstream” from) the coding sequence is called the five prime untranslated region (5’UTR); the sequence after (“downstream” from) the coding sequence is called the three prime untranslated region (3’UTR).</p>
<p>As opposed to DNA replication, transcription results in an RNA complement that includes the nucleotide uracil (U) in all instances where thymine (T) would have occurred in a DNA complement.</p>
<p>Only one of the two DNA strands serve as a template for transcription. The antisense strand of DNA is read by RNA polymerase from the 3’ end to the 5’ end during transcription (3’ → 5’). The complementary RNA is created in the opposite direction, in the 5’ → 3’ direction, matching the sequence of the sense strand with the exception of switching uracil for thymine. This directionality is because RNA polymerase can only add nucleotides to the 3’ end of the growing mRNA chain. This use of only the 3’ → 5’ DNA strand eliminates the need for the Okazaki fragments that are seen in DNA replication. This also removes the need for an RNA primer to initiate RNA synthesis, as is the case in DNA replication.</p>
<p>The non-template (sense) strand of DNA is called the coding strand, because its sequence is the same as the newly created RNA transcript (except for the substitution of uracil for thymine). This is the strand that is used by convention when presenting a DNA sequence.</p>
<p>Transcription has some proofreading mechanisms, but they are fewer and less effective than the controls for copying DNA. As a result, transcription has a lower copying fidelity than DNA replication.</p>
<p>Transcription is divided into initiation, promoter escape, elongation, and termination.</p>
</div>
<div id="initiation-1" class="section level3" number="9.6.2">
<h3><span class="header-section-number">9.6.2</span> Initiation</h3>
<p>Transcription begins with the binding of RNA polymerase, together with one or more general transcription factors, to a specific DNA sequence referred to as a “promoter” to form an RNA polymerase-promoter “closed complex”. In the “closed complex” the promoter DNA is still fully double-stranded.</p>
<p>RNA polymerase, assisted by one or more general transcription factors, then unwinds approximately 14 base pairs of DNA to form an RNA polymerase-promoter “open complex”. In the “open complex” the promoter DNA is partly unwound and single-stranded. The exposed, single-stranded DNA is referred to as the “transcription bubble.”</p>
<p>RNA polymerase, assisted by one or more general transcription factors, then selects a transcription start site in the transcription bubble, binds to an initiating NTP and an extending NTP (or a short RNA primer and an extending NTP) complementary to the transcription start site sequence, and catalyzes bond formation to yield an initial RNA product.</p>
<p>In bacteria, RNA polymerase holoenzyme consists of five subunits: 2 α subunits, 1 β subunit, 1 β’ subunit, and 1 ω subunit. In bacteria, there is one general RNA transcription factor known as a sigma factor. RNA polymerase core enzyme binds to the bacterial general transcription (sigma) factor to form RNA polymerase holoenzyme and then binds to a promoter. (RNA polymerase is called a holoenzyme when sigma subunit is attached to the core enzyme which is consist of 2 α subunits, 1 β subunit, 1 β’ subunit only).</p>
<p>In archaea and eukaryotes, RNA polymerase contains subunits homologous to each of the five RNA polymerase subunits in bacteria and also contains additional subunits. In archaea and eukaryotes, the functions of the bacterial general transcription factor sigma are performed by multiple general transcription factors that work together. In archaea, there are three general transcription factors: TBP, TFB, and TFE. In eukaryotes, in RNA polymerase II-dependent transcription, there are six general transcription factors: TFIIA, TFIIB (an ortholog of archaeal TFB), TFIID (a multisubunit factor in which the key subunit, TBP, is an ortholog of archaeal TBP), TFIIE (an ortholog of archaeal TFE), TFIIF, and TFIIH. The TFIID is the first component to bind to DNA due to binding of TBP, while TFIIH is the last component to be recruited. In archaea and eukaryotes, the RNA polymerase-promoter closed complex is usually referred to as the “preinitiation complex.”</p>
<p>Transcription initiation is regulated by additional proteins, known as activators and repressors, and, in some cases, associated coactivators or corepressors, which modulate formation and function of the transcription initiation complex.</p>
<p>After the first bond is synthesized, the RNA polymerase must escape the promoter. During this time there is a tendency to release the RNA transcript and produce truncated transcripts. This is called abortive initiation, and is common for both eukaryotes and prokaryotes. Abortive initiation continues to occur until an RNA product of a threshold length of approximately 10 nucleotides is synthesized, at which point promoter escape occurs and a transcription elongation complex is formed.</p>
<p>Mechanistically, promoter escape occurs through DNA scrunching, providing the energy needed to break interactions between RNA polymerase holoenzyme and the promoter.</p>
<p>In eukaryotes, at an RNA polymerase II-dependent promoter, upon promoter clearance, TFIIH phosphorylates serine 5 on the carboxy terminal domain of RNA polymerase II, leading to the recruitment of capping enzyme (CE). The exact mechanism of how CE induces promoter clearance in eukaryotes is not yet known.</p>
</div>
<div id="elongation-1" class="section level3" number="9.6.3">
<h3><span class="header-section-number">9.6.3</span> Elongation</h3>
<p>One strand of the DNA, the template strand (or noncoding strand), is used as a template for RNA synthesis. As transcription proceeds, RNA polymerase traverses the template strand and uses base pairing complementarity with the DNA template to create an RNA copy (which elongates during the traversal). Although RNA polymerase traverses the template strand from 3’ → 5’, the coding (non-template) strand and newly formed RNA can also be used as reference points, so transcription can be described as occurring 5’ → 3’. This produces an RNA molecule from 5’ → 3’, an exact copy of the coding strand (except that thymines are replaced with uracils, and the nucleotides are composed of a ribose (5-carbon) sugar where DNA has deoxyribose (one fewer oxygen atom) in its sugar-phosphate backbone).</p>

<div class="figure" style="text-align: center"><span id="fig:elongation"></span>
<img src="figures/genetics/Simple_transcription_elongation1.svg" alt="RNA polymerase (RNAP) at work. Note the coding and template strands. The resulting RNA is a synthesized from the template strand and identical in sequence to the coding strand." width="70%" />
<p class="caption">
Figure 9.14: <a href="https://commons.wikimedia.org/wiki/File:Simple_transcription_elongation1.svg">RNA polymerase (RNAP) at work. Note the coding and template strands. The resulting RNA is a synthesized from the template strand and identical in sequence to the coding strand.</a>
</p>
</div>
<p>mRNA transcription can involve multiple RNA polymerases on a single DNA template and multiple rounds of transcription (amplification of particular mRNA), so many mRNA molecules can be rapidly produced from a single copy of a gene. The characteristic elongation rates in prokaryotes and eukaryotes are about 10-100 nts/sec. In eukaryotes, however, nucleosomes act as major barriers to transcribing polymerases during transcription elongation. In these organisms, the pausing induced by nucleosomes can be regulated by transcription elongation factors such as TFIIS.</p>
<p>Elongation also involves a proofreading mechanism that can replace incorrectly incorporated bases. In eukaryotes, this may correspond with short pauses during transcription that allow appropriate RNA editing factors to bind. These pauses may be intrinsic to the RNA polymerase or due to chromatin structure.</p>
</div>
<div id="termination-1" class="section level3" number="9.6.4">
<h3><span class="header-section-number">9.6.4</span> Termination</h3>
<p>Bacteria use two different strategies for transcription termination – Rho-independent termination and Rho-dependent termination. In Rho-independent transcription termination, RNA transcription stops when the newly synthesized RNA molecule forms a G-C-rich hairpin loop followed by a run of Us. When the hairpin forms, the mechanical stress breaks the weak rU-dA bonds, now filling the DNA–RNA hybrid. This pulls the poly-U transcript out of the active site of the RNA polymerase, terminating transcription. In the “Rho-dependent” type of termination, a protein factor called “Rho” destabilizes the interaction between the template and the mRNA, thus releasing the newl synthesized mRNA from the elongation complex.</p>
<p>Transcription termination in eukaryotes is less well understood than in bacteria, but involves cleavage of the new transcript followed by template-independent addition of adenines at its new 3’ end, in a process called polyadenylation.</p>
</div>
<div id="inhibitors-of-transcription" class="section level3" number="9.6.5">
<h3><span class="header-section-number">9.6.5</span> Inhibitors Of Transcription</h3>
<p>Transcription inhibitors can be used as antibiotics against, for example, pathogenic bacteria (antibacterials) and fungi (antifungals). An example of such an antibacterial is rifampicin, which inhibits bacterial transcription of DNA into mRNA by inhibiting DNA-dependent RNA polymerase by binding its beta-subunit, while 8-hydroxyquinoline is an antifungal transcription inhibitor. The effects of histone methylation may also work to inhibit the action of transcription.</p>
<p>In vertebrates, the majority of gene promoters contain a CpG island with numerous CpG sites. When many of a gene’s promoter CpG sites are methylated the gene becomes inhibited (silenced).</p>
</div>
<div id="transcription-factors" class="section level3" number="9.6.6">
<h3><span class="header-section-number">9.6.6</span> Transcription Factors</h3>
<p>Active transcription units are clustered in the nucleus, in discrete sites called transcription factories or euchromatin. Such sites can be visualized by allowing engaged polymerases to extend their transcripts in tagged precursors (Br-UTP or Br-U) and immuno-labeling the tagged nascent RNA. Transcription factories can also be localized using fluorescence in situ hybridization or marked by antibodies directed against polymerases. There are ~10,000 factories in the nucleoplasm of a HeLa cell, among which are ~8,000 polymerase II factories and ~2,000 polymerase III factories. Each polymerase II factory contains ~8 polymerases. As most active transcription units are associated with only one polymerase, each factory usually contains ~8 different transcription units. These units might be associated through promoters and/or enhancers, with loops forming a “cloud” around the factor.</p>
<p>A molecule that allows the genetic material to be realized as a protein was first hypothesized by François Jacob and Jacques Monod. Severo Ochoa won a Nobel Prize in Physiology or Medicine in 1959 for developing a process for synthesizing RNA in vitro with polynucleotide phosphorylase, which was useful for cracking the genetic code. RNA synthesis by RNA polymerase was established in vitro by several laboratories by 1965.</p>
<p>Roger D. Kornberg won the 2006 Nobel Prize in Chemistry “for his studies of the molecular basis of eukaryotic transcription”.</p>
</div>
<div id="rna-processing" class="section level3" number="9.6.7">
<h3><span class="header-section-number">9.6.7</span> RNA Processing</h3>
<p><a href="https://en.wikipedia.org/wiki/Post-transcriptional_modification">Post-transcriptional modification</a> or co-transcriptional modification is a set of biological processes common to most eukaryotic cells by which an RNA primary transcript is chemically altered following transcription from a gene to produce a mature, functional RNA molecule that can then leave the nucleus and perform any of a variety of different functions in the cell. There are many types of post-transcriptional modifications achieved through a diverse class of molecular mechanisms.</p>
<p>Perhaps the most notable example is the conversion of precursor messenger RNA transcripts into mature messenger RNA that is subsequently capable of being translated into protein. This process includes three major steps that significantly modify the chemical structure of the RNA molecule: the addition of a 5’ cap, the addition of a 3’ polyadenylated tail, and RNA splicing. Such processing is vital for the correct translation of eukaryotic genomes because the initial precursor mRNA produced by transcription often contains both exons (coding sequences) and introns (non-coding sequences); splicing removes the introns and links the exons directly, while the cap and tail facilitate the transport of the mRNA to a ribosome and protect it from molecular degradation.</p>
<p>Post-transcriptional modifications may also occur during the processing of other transcripts which ultimately become transfer RNA, ribosomal RNA, or any of the other types of RNA used by the cell.</p>
</div>
<div id="mrna-processing" class="section level3" number="9.6.8">
<h3><span class="header-section-number">9.6.8</span> mRNA Processing</h3>
<p>The pre-mRNA molecule undergoes three main modifications. These modifications are 5’ capping, 3’ polyadenylation, and RNA splicing, which occur in the cell nucleus before the RNA is translated.</p>
</div>
<div id="processing" class="section level3" number="9.6.9">
<h3><span class="header-section-number">9.6.9</span> 5’ Processing</h3>
<p>Capping of the pre-mRNA involves the addition of 7-methylguanosine (m7G) to the 5’ end. To achieve this, the terminal 5’ phosphate requires removal, which is done with the aid of a phosphatase enzyme. The enzyme guanosyl transferase then catalyses the reaction, which produces the diphosphate 5’ end. The diphosphate 5’ end then attacks the alpha phosphorus atom of a GTP molecule in order to add the guanine residue in a 5’5’ triphosphate link. The enzyme (guanine-N7-)-methyltransferase (“cap MTase”) transfers a methyl group from S-adenosyl methionine to the guanine ring. This type of cap, with just the (m7G) in position is called a cap 0 structure. The ribose of the adjacent nucleotide may also be methylated to give a cap 1. Methylation of nucleotides downstream of the RNA molecule produce cap 2, cap 3 structures and so on. In these cases the methyl groups are added to the 2’ OH groups of the ribose sugar. The cap protects the 5’ end of the primary RNA transcript from attack by ribonucleases that have specificity to the 3’5’ phosphodiester bonds.</p>
</div>
<div id="processing-1" class="section level3" number="9.6.10">
<h3><span class="header-section-number">9.6.10</span> 3’ Processing</h3>
<p>The pre-mRNA processing at the 3’ end of the RNA molecule involves cleavage of its 3’ end and then the addition of about 250 adenine residues to form a poly(A) tail. The cleavage and adenylation reactions occur primarily if a polyadenylation signal sequence (5’- AAUAAA-3’) is located near the 3’ end of the pre-mRNA molecule, which is followed by another sequence, which is usually (5’-CA-3’) and is the site of cleavage. A GU-rich sequence is also usually present further downstream on the pre-mRNA molecule. More recently, it has been demonstrated that alternate signal sequences such as UGUA upstream off the cleavage site can also direct cleavage and polyadenylation in the absence of the AAUAAA signal. It is important to understand that these two signals are not mutually independent and often coexist. After the synthesis of the sequence elements, several multi-subunit proteins are transferred to the RNA molecule. The transfer of these sequence specific binding proteins cleavage and polyadenylation specificity factor (CPSF), Cleavage Factor I (CF I) and cleavage stimulation factor (CStF) occurs from RNA Polymerase II. The three factors bind to the sequence elements. The AAUAAA signal is directly bound by CPSF. For UGUA dependent processing sites, binding of the multi protein complex is done by Cleavage Factor I (CF I). The resultant protein complex formed contains additional cleavage factors and the enzyme Polyadenylate Polymerase (PAP). This complex cleaves the RNA between the polyadenylation sequence and the GU-rich sequence at the cleavage site marked by the (5’-CA-3’) sequences. Poly(A) polymerase then adds about 200 adenine units to the new 3’ end of the RNA molecule using ATP as a precursor. As the poly(A) tail is synthesised, it binds multiple copies of poly(A) binding protein, which protects the 3’end from ribonuclease digestion.</p>
</div>
<div id="intron-splicing" class="section level3" number="9.6.11">
<h3><span class="header-section-number">9.6.11</span> Intron Splicing</h3>
<p>RNA splicing is the process by which introns, regions of RNA that do not code for proteins, are removed from the pre-mRNA and the remaining exons connected to re-form a single continuous molecule. Exons are sections of mRNA which become “expressed” or translated into a protein. They are the coding portions of a mRNA molecule. Although most RNA splicing occurs after the complete synthesis and end-capping of the pre-mRNA, transcripts with many exons can be spliced co-transcriptionally. The splicing reaction is catalyzed by a large protein complex called the spliceosome assembled from proteins and small nuclear RNA molecules that recognize splice sites in the pre-mRNA sequence. Many pre-mRNAs, including those encoding antibodies, can be spliced in multiple ways to produce different mature mRNAs that encode different protein sequences. This process is known as alternative splicing, and allows production of a large variety of proteins from a limited amount of DNA.</p>
</div>
<div id="reverse-transcription" class="section level3" number="9.6.12">
<h3><span class="header-section-number">9.6.12</span> Reverse Transcription</h3>
<p>Some viruses (such as HIV, the cause of AIDS), have the ability to transcribe RNA into DNA. HIV has an RNA genome that is reverse transcribed into DNA. The resulting DNA can be merged with the DNA genome of the host cell. The main enzyme responsible for synthesis of DNA from an RNA template is called reverse transcriptase.</p>
<p>In the case of HIV, reverse transcriptase is responsible for synthesizing a complementary DNA strand (cDNA) to the viral RNA genome. The enzyme ribonuclease H then digests the RNA strand, and reverse transcriptase synthesises a complementary strand of DNA to form a double helix DNA structure (“cDNA”). The cDNA is integrated into the host cell’s genome by the enzyme integrase, which causes the host cell to generate viral proteins that reassemble into new viral particles. In HIV, subsequent to this, the host cell undergoes programmed cell death, or apoptosis of T cells. However, in other retroviruses, the host cell remains intact as the virus buds out of the cell.</p>
<p>Some eukaryotic cells contain an enzyme with reverse transcription activity called telomerase. Telomerase is a reverse transcriptase that lengthens the ends of linear chromosomes. Telomerase carries an RNA template from which it synthesizes a repeating sequence of DNA, or “junk” DNA. This repeated sequence of DNA is called a telomere and can be thought of as a “cap” for a chromosome. It is important because every time a linear chromosome is duplicated, it is shortened. With this “junk” DNA or “cap” at the ends of chromosomes, the shortening eliminates some of the non-essential, repeated sequence rather than the protein-encoding DNA sequence, that is farther away from the chromosome end.</p>
<p>Telomerase is often activated in cancer cells to enable cancer cells to duplicate their genomes indefinitely without losing important protein-coding DNA sequence. Activation of telomerase could be part of the process that allows cancer cells to become immortal. The immortalizing factor of cancer via telomere lengthening due to telomerase has been proven to occur in 90% of all carcinogenic tumors in vivo with the remaining 10% using an alternative telomere maintenance route called ALT or Alternative Lengthening of Telomeres.</p>
</div>
<div id="rna-export" class="section level3" number="9.6.13">
<h3><span class="header-section-number">9.6.13</span> RNA Export</h3>
<p>In eukaryotes most mature RNA must be exported to the cytoplasm from the nucleus. While some RNAs function in the nucleus, many RNAs are transported through the nuclear pores and into the cytosol. Export of RNAs requires association with specific proteins known as exportins. Specific exportin molecules are responsible for the export of a given RNA type. mRNA transport also requires the correct association with Exon Junction Complex (EJC), which ensures that correct processing of the mRNA is completed before export. In some cases RNAs are additionally transported to a specific part of the cytoplasm, such as a synapse; they are then towed by motor proteins that bind through linker proteins to specific sequences (called “zipcodes”) on the RNA.</p>
</div>
<div id="transcription-and-rna-processing-1" class="section level3" number="9.6.14">
<h3><span class="header-section-number">9.6.14</span> Transcription And RNA Processing</h3>
<p><a href="https://en.wikipedia.org/wiki/Transcription_(biology)">Transcription</a> is the first of several steps of DNA based gene expression, in which a particular segment of DNA is copied into RNA (especially mRNA) by the enzyme RNA polymerase. During transcription, a DNA sequence is read by an RNA polymerase, which produces a complementary, antiparallel RNA strand called a primary transcript.</p>
<p>Transcription proceeds in the following general steps:</p>
<ol style="list-style-type: decimal">
<li>RNA polymerase, together with one or more general transcription factors, binds to promoter DNA.</li>
<li>RNA polymerase creates a transcription bubble, which separates the two strands of the DNA helix. This is done by breaking the hydrogen bonds between complementary DNA nucleotides.</li>
<li>RNA polymerase adds RNA nucleotides (which are complementary to the nucleotides of one DNA strand).</li>
<li>RNA sugar-phosphate backbone forms with assistance from RNA polymerase to form an RNA strand.</li>
<li>Hydrogen bonds of the RNA–DNA helix break, freeing the newly synthesized RNA strand.</li>
<li>If the cell has a nucleus, the RNA may be further processed. This may include polyadenylation, capping, and splicing.</li>
<li>The RNA may remain in the nucleus or exit to the cytoplasm through the nuclear pore complex.</li>
</ol>
<p>The stretch of DNA transcribed into an RNA molecule is called a transcription unit. If the DNA encodes a protein, the transcription produces messenger RNA (mRNA); the mRNA, in turn, serves as a template for the protein’s synthesis through translation. Alternatively, the transcribed DNA may encode for non-coding RNA such as microRNA, ribosomal RNA (rRNA), transfer RNA (tRNA), or enzymatic RNA molecules called ribozymes.</p>
<p>A DNA transcription unit encoding for a protein may contain both a coding sequence, which will be translated into the protein, and regulatory sequences, which direct and regulate the synthesis of that protein. The regulatory sequence before (“upstream” from) the coding sequence is called the five prime untranslated region (5’UTR); the sequence after (“downstream” from) the coding sequence is called the three prime untranslated region (3’UTR).</p>
<p>As opposed to DNA replication, transcription results in an RNA complement that includes the nucleotide uracil (U) in all instances where thymine (T) would have occurred in a DNA complement.</p>
<p>Only one of the two DNA strands serve as a template for transcription. The antisense strand of DNA is read by RNA polymerase from the 3’ end to the 5’ end during transcription (3’ → 5’). The complementary RNA is created in the opposite direction, in the 5’ → 3’ direction, matching the sequence of the sense strand with the exception of switching uracil for thymine. This directionality is because RNA polymerase can only add nucleotides to the 3’ end of the growing mRNA chain. This use of only the 3’ → 5’ DNA strand eliminates the need for the Okazaki fragments that are seen in DNA replication. This also removes the need for an RNA primer to initiate RNA synthesis, as is the case in DNA replication.</p>
<p>The non-template (sense) strand of DNA is called the coding strand, because its sequence is the same as the newly created RNA transcript (except for the substitution of uracil for thymine). This is the strand that is used by convention when presenting a DNA sequence.</p>
<p>Transcription has some proofreading mechanisms, but they are fewer and less effective than the controls for copying DNA. As a result, transcription has a lower copying fidelity than DNA replication.</p>
<p>Transcription is divided into initiation, promoter escape, elongation, and termination.</p>
</div>
<div id="initiation-2" class="section level3" number="9.6.15">
<h3><span class="header-section-number">9.6.15</span> Initiation</h3>
<p>Transcription begins with the binding of RNA polymerase, together with one or more general transcription factors, to a specific DNA sequence referred to as a “promoter” to form an RNA polymerase-promoter “closed complex”. In the “closed complex” the promoter DNA is still fully double-stranded.</p>
<p>RNA polymerase, assisted by one or more general transcription factors, then unwinds approximately 14 base pairs of DNA to form an RNA polymerase-promoter “open complex”. In the “open complex” the promoter DNA is partly unwound and single-stranded. The exposed, single-stranded DNA is referred to as the “transcription bubble.”</p>
<p>RNA polymerase, assisted by one or more general transcription factors, then selects a transcription start site in the transcription bubble, binds to an initiating NTP and an extending NTP (or a short RNA primer and an extending NTP) complementary to the transcription start site sequence, and catalyzes bond formation to yield an initial RNA product.</p>
<p>In bacteria, RNA polymerase holoenzyme consists of five subunits: 2 α subunits, 1 β subunit, 1 β’ subunit, and 1 ω subunit. In bacteria, there is one general RNA transcription factor known as a sigma factor. RNA polymerase core enzyme binds to the bacterial general transcription (sigma) factor to form RNA polymerase holoenzyme and then binds to a promoter. (RNA polymerase is called a holoenzyme when sigma subunit is attached to the core enzyme which is consist of 2 α subunits, 1 β subunit, 1 β’ subunit only).</p>
<p>In archaea and eukaryotes, RNA polymerase contains subunits homologous to each of the five RNA polymerase subunits in bacteria and also contains additional subunits. In archaea and eukaryotes, the functions of the bacterial general transcription factor sigma are performed by multiple general transcription factors that work together. In archaea, there are three general transcription factors: TBP, TFB, and TFE. In eukaryotes, in RNA polymerase II-dependent transcription, there are six general transcription factors: TFIIA, TFIIB (an ortholog of archaeal TFB), TFIID (a multisubunit factor in which the key subunit, TBP, is an ortholog of archaeal TBP), TFIIE (an ortholog of archaeal TFE), TFIIF, and TFIIH. The TFIID is the first component to bind to DNA due to binding of TBP, while TFIIH is the last component to be recruited. In archaea and eukaryotes, the RNA polymerase-promoter closed complex is usually referred to as the “preinitiation complex.”</p>
<p>Transcription initiation is regulated by additional proteins, known as activators and repressors, and, in some cases, associated coactivators or corepressors, which modulate formation and function of the transcription initiation complex.</p>
<p>After the first bond is synthesized, the RNA polymerase must escape the promoter. During this time there is a tendency to release the RNA transcript and produce truncated transcripts. This is called abortive initiation, and is common for both eukaryotes and prokaryotes. Abortive initiation continues to occur until an RNA product of a threshold length of approximately 10 nucleotides is synthesized, at which point promoter escape occurs and a transcription elongation complex is formed.</p>
<p>Mechanistically, promoter escape occurs through DNA scrunching, providing the energy needed to break interactions between RNA polymerase holoenzyme and the promoter.</p>
<p>In eukaryotes, at an RNA polymerase II-dependent promoter, upon promoter clearance, TFIIH phosphorylates serine 5 on the carboxy terminal domain of RNA polymerase II, leading to the recruitment of capping enzyme (CE). The exact mechanism of how CE induces promoter clearance in eukaryotes is not yet known.</p>
</div>
<div id="inhibitors-of-transcription-1" class="section level3" number="9.6.16">
<h3><span class="header-section-number">9.6.16</span> Inhibitors of transcription</h3>
<p>Transcription inhibitors can be used as antibiotics against, for example, pathogenic bacteria (antibacterials) and fungi (antifungals). An example of such an antibacterial is rifampicin, which inhibits bacterial transcription of DNA into mRNA by inhibiting DNA-dependent RNA polymerase by binding its beta-subunit, while 8-hydroxyquinoline is an antifungal transcription inhibitor. The effects of histone methylation may also work to inhibit the action of transcription.</p>
<p>In vertebrates, the majority of gene promoters contain a CpG island with numerous CpG sites. When many of a gene’s promoter CpG sites are methylated the gene becomes inhibited (silenced).</p>
</div>
<div id="transcription-factors-1" class="section level3" number="9.6.17">
<h3><span class="header-section-number">9.6.17</span> Transcription Factors</h3>
<p>Active transcription units are clustered in the nucleus, in discrete sites called transcription factories or euchromatin. Such sites can be visualized by allowing engaged polymerases to extend their transcripts in tagged precursors (Br-UTP or Br-U) and immuno-labeling the tagged nascent RNA. Transcription factories can also be localized using fluorescence in situ hybridization or marked by antibodies directed against polymerases. There are ~10,000 factories in the nucleoplasm of a HeLa cell, among which are ~8,000 polymerase II factories and ~2,000 polymerase III factories. Each polymerase II factory contains ~8 polymerases. As most active transcription units are associated with only one polymerase, each factory usually contains ~8 different transcription units. These units might be associated through promoters and/or enhancers, with loops forming a “cloud” around the factor.</p>
<p>A molecule that allows the genetic material to be realized as a protein was first hypothesized by François Jacob and Jacques Monod. Severo Ochoa won a Nobel Prize in Physiology or Medicine in 1959 for developing a process for synthesizing RNA in vitro with polynucleotide phosphorylase, which was useful for cracking the genetic code. RNA synthesis by RNA polymerase was established in vitro by several laboratories by 1965.</p>
<p>Roger D. Kornberg won the 2006 Nobel Prize in Chemistry “for his studies of the molecular basis of eukaryotic transcription”.</p>
</div>
<div id="rna-processing-1" class="section level3" number="9.6.18">
<h3><span class="header-section-number">9.6.18</span> RNA Processing</h3>
<p><a href="https://en.wikipedia.org/wiki/Post-transcriptional_modification">Post-transcriptional modification</a> or co-transcriptional modification is a set of biological processes common to most eukaryotic cells by which an RNA primary transcript is chemically altered following transcription from a gene to produce a mature, functional RNA molecule that can then leave the nucleus and perform any of a variety of different functions in the cell. There are many types of post-transcriptional modifications achieved through a diverse class of molecular mechanisms.</p>
<p>Perhaps the most notable example is the conversion of precursor messenger RNA transcripts into mature messenger RNA that is subsequently capable of being translated into protein. This process includes three major steps that significantly modify the chemical structure of the RNA molecule: the addition of a 5’ cap, the addition of a 3’ polyadenylated tail, and RNA splicing. Such processing is vital for the correct translation of eukaryotic genomes because the initial precursor mRNA produced by transcription often contains both exons (coding sequences) and introns (non-coding sequences); splicing removes the introns and links the exons directly, while the cap and tail facilitate the transport of the mRNA to a ribosome and protect it from molecular degradation.</p>
<p>Post-transcriptional modifications may also occur during the processing of other transcripts which ultimately become transfer RNA, ribosomal RNA, or any of the other types of RNA used by the cell.</p>
</div>
<div id="mrna-processing-1" class="section level3" number="9.6.19">
<h3><span class="header-section-number">9.6.19</span> mRNA Processing</h3>
<p>The pre-mRNA molecule undergoes three main modifications. These modifications are 5’ capping, 3’ polyadenylation, and RNA splicing, which occur in the cell nucleus before the RNA is translated.</p>
</div>
<div id="processing-2" class="section level3" number="9.6.20">
<h3><span class="header-section-number">9.6.20</span> 5’ Processing</h3>
<p>Capping of the pre-mRNA involves the addition of 7-methylguanosine (m7G) to the 5’ end. To achieve this, the terminal 5’ phosphate requires removal, which is done with the aid of a phosphatase enzyme. The enzyme guanosyl transferase then catalyses the reaction, which produces the diphosphate 5’ end. The diphosphate 5’ end then attacks the alpha phosphorus atom of a GTP molecule in order to add the guanine residue in a 5’5’ triphosphate link. The enzyme (guanine-N7-)-methyltransferase (“cap MTase”) transfers a methyl group from S-adenosyl methionine to the guanine ring. This type of cap, with just the (m7G) in position is called a cap 0 structure. The ribose of the adjacent nucleotide may also be methylated to give a cap 1. Methylation of nucleotides downstream of the RNA molecule produce cap 2, cap 3 structures and so on. In these cases the methyl groups are added to the 2’ OH groups of the ribose sugar. The cap protects the 5’ end of the primary RNA transcript from attack by ribonucleases that have specificity to the 3’5’ phosphodiester bonds.</p>
</div>
<div id="processing-3" class="section level3" number="9.6.21">
<h3><span class="header-section-number">9.6.21</span> 3’ Processing</h3>
<p>The pre-mRNA processing at the 3’ end of the RNA molecule involves cleavage of its 3’ end and then the addition of about 250 adenine residues to form a poly(A) tail. The cleavage and adenylation reactions occur primarily if a polyadenylation signal sequence (5’- AAUAAA-3’) is located near the 3’ end of the pre-mRNA molecule, which is followed by another sequence, which is usually (5’-CA-3’) and is the site of cleavage. A GU-rich sequence is also usually present further downstream on the pre-mRNA molecule. More recently, it has been demonstrated that alternate signal sequences such as UGUA upstream off the cleavage site can also direct cleavage and polyadenylation in the absence of the AAUAAA signal. It is important to understand that these two signals are not mutually independent and often coexist. After the synthesis of the sequence elements, several multi-subunit proteins are transferred to the RNA molecule. The transfer of these sequence specific binding proteins cleavage and polyadenylation specificity factor (CPSF), Cleavage Factor I (CF I) and cleavage stimulation factor (CStF) occurs from RNA Polymerase II. The three factors bind to the sequence elements. The AAUAAA signal is directly bound by CPSF. For UGUA dependent processing sites, binding of the multi protein complex is done by Cleavage Factor I (CF I). The resultant protein complex formed contains additional cleavage factors and the enzyme Polyadenylate Polymerase (PAP). This complex cleaves the RNA between the polyadenylation sequence and the GU-rich sequence at the cleavage site marked by the (5’-CA-3’) sequences. Poly(A) polymerase then adds about 200 adenine units to the new 3’ end of the RNA molecule using ATP as a precursor. As the poly(A) tail is synthesised, it binds multiple copies of poly(A) binding protein, which protects the 3’end from ribonuclease digestion.</p>
</div>
<div id="intron-splicing-1" class="section level3" number="9.6.22">
<h3><span class="header-section-number">9.6.22</span> Intron Splicing</h3>
<p>RNA splicing is the process by which introns, regions of RNA that do not code for proteins, are removed from the pre-mRNA and the remaining exons connected to re-form a single continuous molecule. Exons are sections of mRNA which become “expressed” or translated into a protein. They are the coding portions of a mRNA molecule. Although most RNA splicing occurs after the complete synthesis and end-capping of the pre-mRNA, transcripts with many exons can be spliced co-transcriptionally. The splicing reaction is catalyzed by a large protein complex called the spliceosome assembled from proteins and small nuclear RNA molecules that recognize splice sites in the pre-mRNA sequence. Many pre-mRNAs, including those encoding antibodies, can be spliced in multiple ways to produce different mature mRNAs that encode different protein sequences. This process is known as alternative splicing, and allows production of a large variety of proteins from a limited amount of DNA.</p>
</div>
<div id="reverse-transcription-1" class="section level3" number="9.6.23">
<h3><span class="header-section-number">9.6.23</span> Reverse Transcription</h3>
<p>Some viruses (such as HIV, the cause of AIDS), have the ability to transcribe RNA into DNA. HIV has an RNA genome that is reverse transcribed into DNA. The resulting DNA can be merged with the DNA genome of the host cell. The main enzyme responsible for synthesis of DNA from an RNA template is called reverse transcriptase.</p>
<p>In the case of HIV, reverse transcriptase is responsible for synthesizing a complementary DNA strand (cDNA) to the viral RNA genome. The enzyme ribonuclease H then digests the RNA strand, and reverse transcriptase synthesises a complementary strand of DNA to form a double helix DNA structure (“cDNA”). The cDNA is integrated into the host cell’s genome by the enzyme integrase, which causes the host cell to generate viral proteins that reassemble into new viral particles. In HIV, subsequent to this, the host cell undergoes programmed cell death, or apoptosis of T cells. However, in other retroviruses, the host cell remains intact as the virus buds out of the cell.</p>
<p>Some eukaryotic cells contain an enzyme with reverse transcription activity called telomerase. Telomerase is a reverse transcriptase that lengthens the ends of linear chromosomes. Telomerase carries an RNA template from which it synthesizes a repeating sequence of DNA, or “junk” DNA. This repeated sequence of DNA is called a telomere and can be thought of as a “cap” for a chromosome. It is important because every time a linear chromosome is duplicated, it is shortened. With this “junk” DNA or “cap” at the ends of chromosomes, the shortening eliminates some of the non-essential, repeated sequence rather than the protein-encoding DNA sequence, that is farther away from the chromosome end.</p>
<p>Telomerase is often activated in cancer cells to enable cancer cells to duplicate their genomes indefinitely without losing important protein-coding DNA sequence. Activation of telomerase could be part of the process that allows cancer cells to become immortal. The immortalizing factor of cancer via telomere lengthening due to telomerase has been proven to occur in 90% of all carcinogenic tumors in vivo with the remaining 10% using an alternative telomere maintenance route called ALT or Alternative Lengthening of Telomeres.</p>
</div>
<div id="translation-and-the-genetic-code" class="section level3" number="9.6.24">
<h3><span class="header-section-number">9.6.24</span> Translation And The Genetic Code</h3>
<p>In molecular biology and genetics, <a href="https://en.wikipedia.org/wiki/Translation_(biology)">translation</a> is the process in which ribosomes in the cytoplasm or endoplasmic reticulum (ER) synthesize proteins after the process of transcription of DNA to RNA in the cell’s nucleus. The entire process is called gene expression.</p>
</div>
<div id="ribosomes" class="section level3" number="9.6.25">
<h3><span class="header-section-number">9.6.25</span> Ribosomes</h3>
<p><a href="https://en.wikipedia.org/wiki/Ribosome">Ribosomes</a> are complex macromolecular machines, found within all living cells, that serves as the site of biological protein synthesis. Ribosomes link amino acids together in the order specified by messenger RNA (RNA) molecules. Ribosomes consist of two major components: the small ribosomal subunits, which read the mRNA, and the large subunits, which join amino acids to form a polypeptide chain. Each subunit consists of one or more ribosomal RNA (rRNA) molecules and a variety of ribosomal proteins. The ribosomes and associated molecules are also known as the translational apparatus.</p>
<p>The sequence of DNA, which encodes the sequence of the amino acids in a protein, is copied into a messenger RNA chain. It may be copied many times into RNA chains. Ribosomes can bind to a messenger RNA chain and use its sequence for determining the correct sequence of amino acids for generating a given protein. Amino acids are selected and collected and carried to the ribosome by transfer RNA (tRNA) molecules, which enter one part of the ribosome and bind to the messenger RNA chain. It is during this binding that the correct translation of nucleic acid sequence to amino acid sequence occurs. For each coding triplet in the messenger RNA there is a distinct transfer RNA that matches and which carries the correct amino acid for that coding triplet. The attached amino acids are then linked together by another part of the ribosome. Once the protein is produced, it can then fold to produce a specific functional three-dimensional structure although during synthesis some proteins start folding into their correct form.</p>
<p>A ribosome is made from complexes of RNAs and proteins and is therefore a ribonucleoprotein. Each ribosome is divided into two subunits:</p>
<ol style="list-style-type: decimal">
<li>a smaller subunit which binds to a larger subunit and the mRNA pattern, and</li>
<li>a larger subunit which binds to the tRNA, the amino acids, and the smaller subunit.</li>
</ol>
<p>When a ribosome finishes reading an mRNA molecule, these two subunits split apart. Ribosomes are ribozymes, because the catalytic peptidyl transferase activity that links amino acids together is performed by the ribosomal RNA. Ribosomes are often associated with the intracellular membranes that make up the rough endoplasmic reticulum.</p>
<p>Ribosomes from bacteria, archaea and eukaryotes in the three-domain system, resemble each other to a remarkable degree, evidence of a common origin. They differ in their size, sequence, structure, and the ratio of protein to RNA. The differences in structure allow some antibiotics to kill bacteria by inhibiting their ribosomes, while leaving human ribosomes unaffected. In bacteria and archaea, more than one ribosome may move along a single mRNA chain at one time, each “reading” its sequence and producing a corresponding protein molecule.</p>
<p>The mitochondrial ribosomes of eukaryotic cells, are produced from mitochondrial genes, and functionally resemble many features of those in bacteria, reflecting the likely evolutionary origin of mitochondria.</p>
<p>Ribosomes were first observed in the mid-1950s by Romanian-American cell biologist <a href="https://en.wikipedia.org/wiki/George_Emil_Palade">George Emil Palade</a>, using an electron microscope, as dense particles or granules. The term “ribosome” was proposed by scientist Richard B. Roberts in the end of 1950s.</p>
<p><a href="https://en.wikipedia.org/wiki/Albert_Claude">Albert Claude</a>, <a href="https://en.wikipedia.org/wiki/Christian_de_Duve">Christian de Duve</a>, and George Emil Palade were jointly awarded the Nobel Prize in Physiology or Medicine, in 1974, for the discovery of the ribosome. The Nobel Prize in Chemistry 2009 was awarded to <a href="https://en.wikipedia.org/wiki/Venki_Ramakrishnan">Venkatraman Ramakrishnan</a>, <a href="https://en.wikipedia.org/wiki/Thomas_A._Steitz">Thomas A. Steitz</a> and <a href="https://en.wikipedia.org/wiki/Ada_Yonath">Ada E. Yonath</a> for determining the detailed structure and mechanism of the ribosome.</p>
</div>
<div id="bacterial-ribosomes" class="section level3" number="9.6.26">
<h3><span class="header-section-number">9.6.26</span> Bacterial Ribosomes</h3>
<p>Prokaryotic ribosomes are around 20 nm (200 Å) in diameter and are composed of 65% rRNA and 35% ribosomal proteins. Eukaryotic ribosomes are between 25 and 30 nm (250–300 Å) in diameter with an rRNA-to-protein ratio that is close to 1. Crystallographic work has shown that there are no ribosomal proteins close to the reaction site for polypeptide synthesis. This suggests that the protein components of ribosomes do not directly participate in peptide bond formation catalysis, but rather that these proteins act as a scaffold that may enhance the ability of rRNA to synthesize protein</p>

<div class="figure" style="text-align: center"><span id="fig:ttribosome"></span>
<img src="figures/genetics/ttribosome.png" alt="Crystal structure of the bacterial 70S ribosome of the bacterium Thermus thermophilus. The 30S (small) ribosomal subunit proteins are colored in green, the 50S (large) subunit proteins are colored in blue, the ribosomal RNA is colored orange. The 30S subunits contains 3 tRNA molecules (based on atomic coordinates of PDB 1JGQ and PDB 1GIY rendered with open source molecular visualization tool PyMol.)" width="70%" />
<p class="caption">
Figure 9.15: Crystal structure of the bacterial 70S ribosome of the bacterium <em>Thermus thermophilus</em>. The 30S (small) ribosomal subunit proteins are colored in green, the 50S (large) subunit proteins are colored in blue, the ribosomal RNA is colored orange. The 30S subunits contains 3 tRNA molecules (based on atomic coordinates of <a href="https://www.rcsb.org/structure/1jgq">PDB 1JGQ</a> and <a href="(https://www.rcsb.org/structure/1jgq)">PDB 1GIY</a> rendered with open source molecular visualization tool PyMol.)
</p>
</div>
<p>The unit of measurement used to describe the ribosomal subunits and the rRNA fragments is the Svedberg unit, a measure of the rate of sedimentation in centrifugation rather than size. This accounts for why fragment names do not add up: for example, bacterial 70S ribosomes are made of 50S and 30S subunits.</p>
<p>Bacteria have 70S ribosomes, each consisting of a small (30S) and a large (50S) subunit. <em>Escherichia coli</em>, for example, has a 16S RNA subunit (consisting of 1540 nucleotides) that is bound to 21 proteins. The large subunit is composed of a 5S RNA subunit (120 nucleotides), a 23S RNA subunit (2900 nucleotides) and 31 proteins.</p>
</div>
<div id="eukaryotic-ribosomes" class="section level3" number="9.6.27">
<h3><span class="header-section-number">9.6.27</span> Eukaryotic Ribosomes</h3>
<p>Eukaryotes have 80S ribosomes located in their cytosol, each consisting of a small (40S) and large (60S) subunit. Their 40S subunit has an 18S RNA (1900 nucleotides) and 33 proteins. The large subunit is composed of a 5S RNA (120 nucleotides), 28S RNA (4700 nucleotides), a 5.8S RNA (160 nucleotides) subunits and 46 proteins.</p>

<div class="figure" style="text-align: center"><span id="fig:ribosome"></span>
<img src="figures/genetics/human_ribosome.png" alt="Crystal structure of the human 80S ribosome (based on atomic coordinates of PDB 4V6X rendered with open source molecular visualization tool PyMol). The 40S (small) ribosomal subunit proteins are shown in lightblue, the 60S (large) subunit proteins in palegreen, the ribosomal RNA in orange." width="70%" />
<p class="caption">
Figure 9.16: Crystal structure of the human 80S ribosome (based on atomic coordinates of <a href="https://www.rcsb.org/structure/4v6x">PDB 4V6X</a> rendered with open source molecular visualization tool PyMol). The 40S (small) ribosomal subunit proteins are shown in lightblue, the 60S (large) subunit proteins in palegreen, the ribosomal RNA in orange.
</p>
</div>
<p>The differences between the bacterial and eukaryotic ribosomes are exploited by pharmaceutical chemists to create antibiotics that can kill bacteria without harming eukaryotic cells. Due to the differences in their structures, the bacterial 70S ribosomes are vulnerable to these antibiotics while the eukaryotic 80S ribosomes are not.</p>
<p>The various ribosomes share a core structure, which is quite similar despite the large differences in size. Much of the RNA is highly organized into various tertiary structural motifs, for example pseudoknots that exhibit coaxial stacking. The extra RNA in the larger ribosomes is in several long continuous insertions, such that they form loops out of the core structure without disrupting or changing it. All of the catalytic activity of the ribosome is carried out by the RNA; the proteins reside on the surface and seem to stabilize the structure.</p>

<div class="figure" style="text-align: center"><span id="fig:trnastruc"></span>
<img src="figures/genetics/trna.png" alt="Tertiary structure of tRNA (based on atomic coordinates of PDB 1ehz rendered with open source molecular visualization tool PyMol.)" width="70%" />
<p class="caption">
Figure 9.17: Tertiary structure of tRNA (based on atomic coordinates of <a href="https://www.rcsb.org/structure/1ehz">PDB 1ehz</a> rendered with open source molecular visualization tool PyMol.)
</p>
</div>
<p>Aminoacyl tRNA synthetases (enzymes) catalyze the bonding between specific tRNAs and the amino acids that their anticodon sequences call for. The product of this reaction is an aminoacyl-tRNA. In prokaryotes, this aminoacyl-tRNA is carried to the ribosome by EF-Tu, where mRNA codons are matched through complementary base pairing to specific tRNA anticodons.</p>
<p>The ribosome has three sites for tRNA to bind. They are the aminoacyl site (abbreviated A), the peptidyl site (abbreviated P) and the exit site (abbreviated E). With respect to the mRNA, the three sites are oriented 5’ to 3’ E-P-A, because ribosomes move toward the 3’ end of mRNA. The A-site binds the incoming tRNA with the complementary codon on the mRNA. The P-site holds the tRNA with the growing polypeptide chain. The E-site holds the tRNA without its amino acid, and the tRNA is then released. When an aminoacyl-tRNA initially binds to its corresponding codon on the mRNA, it is in the A site. Then, a peptide bond forms between the amino acid of the tRNA in the A site and the amino acid of the charged tRNA in the P site. The growing polypeptide chain is transferred to the tRNA in the A site. Translocation occurs, moving the tRNA in the P site, now without an amino acid, to the E site; the tRNA that was in the A site, now charged with the polypeptide chain, is moved to the P site. The tRNA in the E site leaves and another aminoacyl-tRNA enters the A site to repeat the process.</p>
<p>After the new amino acid is added to the chain, and after the mRNA is released out of the nucleus and into the ribosome’s core, the energy provided by the hydrolysis of an ATP bound to the translocase EF-G (in prokaryotes) and eEF-2 (in eukaryotes) moves the ribosome down one codon towards the 3’ end. The energy required for translation of proteins is significant. For a protein containing n amino acids, the number of high-energy phosphate bonds required to translate it is 4n+1. The rate of translation varies; it is significantly higher in prokaryotic cells (up to 17-21 amino acid residues per second) than in eukaryotic cells (up to 6-9 amino acid residues per second).</p>
<p>Even though the ribosomes are usually considered accurate, processive machines, the translation process is subject to errors that can lead either to the synthesis of erroneous proteins or to the premature abandonment of translation. The rate of error in synthesizing proteins has been estimated to be between 1/10<sup>5</sup> and 1/10<sup>3</sup> misincorporated amino acids, depending on the experimental conditions. The rate of premature translation abandonment, instead, has been estimated to be of the order of magnitude of 10<sup>−4</sup> events per translated codon. The correct amino acid is covalently bonded to the correct transfer RNA (tRNA) by amino acyl transferases. The amino acid is joined by its carboxyl group to the 3’ OH of the tRNA by an ester bond. When the tRNA has an amino acid linked to it, the tRNA is termed “charged”. Initiation involves the small subunit of the ribosome binding to the 5’ end of mRNA with the help of initiation factors (IF). In prokaryotes, initiation of protein synthesis involves the recognition of a purine-rich initiation sequence on the mRNA called the Shine-Dalgarno sequence. The Shine-Dalgarno sequence binds to a complementary pyrimidine-rich sequence on the 3’ end of the 16S rRNA part of the 30S ribosomal subunit. The binding of these complementary sequences ensures that the 30S ribosomal subunit is bound to the mRNA and is aligned such that the initiation codon is placed in the 30S portion of the P-site. Once the mRNA and 30S subunit are properly bound, an initiation factor brings the initiator tRNA-amino acid complex, f-Met-tRNA, to the 30S P site. The initiation phase is completed once a 50S subunit joins the 30 subunit, forming an active 70S ribosome. Termination of the polypeptide occurs when the A site of the ribosome is occupied by a stop codon (UAA, UAG, or UGA) on the mRNA. mRNA usually cannot recognize or bind to stop codons. Instead, the stop codon induces the binding of a release factor protein (RF1 &amp; RF2) that prompts the disassembly of the entire ribosome/mRNA complex by the hydrolysis of the polypeptide chain from the peptidyl transferase center of the ribosome. Drugs or special sequence motifs on the mRNA can change the ribosomal structure so that near-cognate tRNAs are bound to the stop codon instead of the release factors. In such cases of ‘translational readthrough’, translation continues until the ribosome encounters the next stop codon.</p>
<p>The process of translation is highly regulated in prokaryotic and eukaryotic organisms. Regulation of translation can impact the global rate of protein synthesis which is closely coupled to the metabolic and proliferative state of a cell. In addition, recent work has revealed that genetic differences and their subsequent expression as mRNAs can also impact translation rate in an RNA-specific manner.</p>
</div>
<div id="translation" class="section level3" number="9.6.28">
<h3><span class="header-section-number">9.6.28</span> Translation</h3>
<p>In translation, messenger RNA (mRNA) is decoded in the ribosome decoding center to produce a specific amino acid chain, or polypeptide. The polypeptide later folds into an active protein and performs its functions in the cell. The ribosome facilitates decoding by inducing the binding of complementary tRNA anticodon sequences to mRNA codons. The tRNAs carry specific amino acids that are chained together into a polypeptide a the mRNA passes through and is read by the ribosome.</p>
<p>Translation proceeds in three phases:</p>
<ol style="list-style-type: decimal">
<li><strong>Initiation:</strong> The ribosome assembles around the target mRNA. The first tRNA is attached at the start codon.</li>
<li><strong>Elongation:</strong> The tRNA transfers an amino acid to the tRNA corresponding to the next codon. The ribosome then moves (translocates) to the next mRNA codon to continue the process, creating an amino acid chain.</li>
<li><strong>Termination:</strong> When a peptidyl tRNA encounters a stop codon, the ribosome detaches.</li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:translation"></span>
<img src="figures/genetics/Ribosome_mRNA_translation_en.svg" alt="Diagram showing the translation of mRNA and the synthesis of proteins by a ribosome." width="70%" />
<p class="caption">
Figure 9.18: <a href="https://commons.wikimedia.org/wiki/File:Ribosome_mRNA_translation_en.svg">Diagram showing the translation of mRNA and the synthesis of proteins by a ribosome.</a>
</p>
</div>
<p>In prokaryotes, translation occurs in the cytosol, where the medium and small subunits of the ribosome bind to the tRNA. In eukaryotes, translation occurs in the cytosol or across the membrane of the endoplasmic reticulum in a process called co-translational translocation. In co-translational translocation, the entire ribosome/mRNA complex binds to the outer membrane of the rough endoplasmic reticulum (ER) and the new protein is synthesized and released into the ER.</p>
<p>Many types of transcribed RNA, such as transfer RNA, ribosomal RNA, and small nuclear RNA, do not undergo translation into proteins.</p>
<p>A number of antibiotics act by inhibiting translation. These include clindamycin, anisomycin, cycloheximide, chloramphenicol, tetracycline, streptomycin, erythromycin, and puromycin. Prokaryotic ribosomes have a different structure from that of eukaryotic ribosomes, and thus antibiotics can specifically target bacterial infections without any harm to a eukaryotic host’s cells.</p>
<p>In 1954, Zamecnik and Hoagland discovered tRNA. In 1955, George E. Palade discovered ribosomes.</p>
</div>
<div id="eukaryotic-translation" class="section level3" number="9.6.29">
<h3><span class="header-section-number">9.6.29</span> Eukaryotic Translation</h3>
</div>
<div id="initiation-3" class="section level3" number="9.6.30">
<h3><span class="header-section-number">9.6.30</span> Initiation</h3>
<p>Initiation of translation usually involves the interaction of certain key proteins, the initiation factors, with a special tag bound to the 5’-end of an mRNA molecule, the 5’ cap, as well as with the 5’ UTR. These proteins bind the small (40S) ribosomal subunit and hold the mRNA in place. eIF3 is associated with the 40S ribosomal subunit and plays a role in keeping the large (60S) ribosomal subunit from prematurely binding. eIF3 also interacts with the eIF4F complex, which consists of three other initiation factors: eIF4A, eIF4E, and eIF4G. eIF4G is a scaffolding protein that directly associates with both eIF3 and the other two components. eIF4E is the cap-binding protein. Binding of the cap by eIF4E is often considered the rate-limiting step of cap-dependent initiation, and the concentration of eIF4E is a regulatory nexus of translational control. Certain viruses cleave a portion of eIF4G that binds eIF4E, thus preventing cap-dependent translation to hijack the host machinery in favor of the viral (cap-independent) messages. eIF4A is an ATP-dependent RNA helicase that aids the ribosome by resolving certain secondary structures formed along the mRNA transcript. The poly(A)-binding protein (PABP) also associates with the eIF4F complex via eIF4G, and binds the poly-A tail of most eukaryotic mRNA molecules. This 43S preinitiation complex (43S PIC) accompanied by the protein factors moves along the mRNA chain toward its 3’-end, in a process known as ‘scanning’, to reach the start codon (typically AUG). In eukaryotes and archaea, the amino acid encoded by the start codon is methionine. The Met-charged initiator tRNA (Met-tRNAiMet) is brought to the P-site of the small ribosomal subunit by eukaryotic initiation factor 2 (eIF2). It hydrolyzes GTP, and signals for the dissociation of several factors from the small ribosomal subunit, eventually leading to the association of the large subunit (or the 60S subunit). The complete ribosome (80S) then commences translation elongation. Regulation of protein synthesis is partly influenced by phosphorylation of eIF2 (via the α subunit), which is a part of the eIF2-GTP-Met-tRNAiMet ternary complex (eIF2-TC). When large numbers of eIF2 are phosphorylated, protein synthesis is inhibited. This occurs under amino acid starvation or after viral infection. However, a small fraction of this initiation factor is naturally phosphorylated. Another regulator is 4EBP, which binds to the initiation factor eIF4E and inhibits its interactions with eIF4G, thus preventing cap-dependent initiation. To oppose the effects of 4EBP, growth factors phosphorylate 4EBP, reducing its affinity for eIF4E and permitting protein synthesis. While protein synthesis is globally regulated by modulating the expression of key initiation factors as well as the number of ribosomes, individual mRNAs can have different translation rates due to the presence of regulatory sequence elements. This has been shown to be important in a variety of settings including yeast meiosis and ethylene response in plants. In addition, recent work in yeast and humans suggest that evolutionary divergence in cis-regulatory sequences can impact translation regulation. Additionally, RNA helicases such as DHX29 and Ded1/DDX3 may participate in the process of translation initiation, especially for mRNAs with structured 5’UTRs.</p>
<p>Cap-independent initiation
The best-studied example of cap-independent translation initiation in eukaryotes is that by the Internal ribosome entry site (IRES). What differentiates cap-independent translation from cap-dependent translation is that cap-independent translation does not require the 5’ cap to initiate scanning from the 5’ end of the mRNA until the start codon. The ribosome can be trafficked to the start site by direct binding, initiation factors, and/or ITAFs (IRES trans-acting factors) bypassing the need to scan the entire 5’ UTR. This method of translation has been found important in conditions that require the translation of specific mRNAs during cellular stress, when overall translation is reduced. Examples include factors responding to apoptosis and stress-induced responses.</p>
</div>
<div id="elongation-2" class="section level3" number="9.6.31">
<h3><span class="header-section-number">9.6.31</span> Elongation</h3>
<p>Elongation depends on eukaryotic elongation factors. At the end of the initiation step, the mRNA is positioned so that the next codon can be translated during the elongation stage of protein synthesis. The initiator tRNA occupies the P site in the ribosome, and the A site is ready to receive an aminoacyl-tRNA. During chain elongation, each additional amino acid is added to the nascent polypeptide chain in a three-step microcycle. The steps in this microcycle are (1) positioning the correct aminoacyl-tRNA in the A site of the ribosome, (2) forming the peptide bond and (3) shifting the mRNA by one codon relative to the ribosome. Unlike bacteria, in which translation initiation occurs as soon as the 5’ end of an mRNA is synthesized, in eukaryotes such tight coupling between transcription and translation is not possible because transcription and translation are carried out in separate compartments of the cell (the nucleus and cytoplasm). Eukaryotic mRNA precursors must be processed in the nucleus (e.g., capping, polyadenylation, splicing) before they are exported to the cytoplasm for translation.</p>
</div>
<div id="termination-2" class="section level3" number="9.6.32">
<h3><span class="header-section-number">9.6.32</span> Termination</h3>
<p>Termination of elongation depends on eukaryotic release factors. The process is similar to that of prokaryotic termination, but unlike prokaryotic termination, there is a universal release factor, eRF1, that recognizes all three stop codons. Upon termination, the ribosome is disassembled and the completed polypeptide is released. eRF3 is a ribosome-dependent GTPase that helps eRF1 release the completed polypeptide.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="an-introduction-to-microbial-metabolism.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="genetic-engineering-1.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/microbiology-text/edit/master/microbial_genetics.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
