<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1 The Main Themes Of Microbiology | Text For SCI204 Microbiology at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Microbiology." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="1 The Main Themes Of Microbiology | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/microbiology_text_cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Microbiology." />
  <meta name="github-repo" content="nikolaussucher/microbiology-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1 The Main Themes Of Microbiology | Text For SCI204 Microbiology at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Microbiology." />
  <meta name="twitter:image" content="/microbiology_text_cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="acknowledgements.html"/>
<link rel="next" href="the-chemistry-of-biology.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />




<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">SCI204 Microbiology at RCC</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html"><i class="fa fa-check"></i><b>1</b> The Main Themes Of Microbiology</a>
<ul>
<li class="chapter" data-level="1.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#general-characteristics-and-evolution-of-microorganisms"><i class="fa fa-check"></i><b>1.1</b> General Characteristics And Evolution Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#archaea"><i class="fa fa-check"></i><b>1.1.1</b> Archaea</a></li>
<li class="chapter" data-level="1.1.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#bacteria"><i class="fa fa-check"></i><b>1.1.2</b> Bacteria</a></li>
<li class="chapter" data-level="1.1.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#eukaryotes"><i class="fa fa-check"></i><b>1.1.3</b> Eukaryotes</a></li>
<li class="chapter" data-level="1.1.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#protists"><i class="fa fa-check"></i><b>1.1.4</b> Protists</a></li>
<li class="chapter" data-level="1.1.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#fungi"><i class="fa fa-check"></i><b>1.1.5</b> Fungi</a></li>
<li class="chapter" data-level="1.1.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#plants"><i class="fa fa-check"></i><b>1.1.6</b> Plants</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#ecology-of-microorganisms"><i class="fa fa-check"></i><b>1.2</b> Ecology Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbiome"><i class="fa fa-check"></i><b>1.2.1</b> Microbiome</a></li>
<li class="chapter" data-level="1.2.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbiota-members-of-the-microbiome"><i class="fa fa-check"></i><b>1.2.2</b> Microbiota – members of the microbiome</a></li>
<li class="chapter" data-level="1.2.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#microbial-networks-and-interaction"><i class="fa fa-check"></i><b>1.2.3</b> Microbial Networks And Interaction</a></li>
<li class="chapter" data-level="1.2.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-human-microbiome"><i class="fa fa-check"></i><b>1.2.4</b> The Human Microbiome</a></li>
<li class="chapter" data-level="1.2.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#extremophiles"><i class="fa fa-check"></i><b>1.2.5</b> Extremophiles</a></li>
<li class="chapter" data-level="1.2.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#soil-microorganisms"><i class="fa fa-check"></i><b>1.2.6</b> Soil Microorganisms</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#human-interactions-with-microbes"><i class="fa fa-check"></i><b>1.3</b> Human Interactions With Microbes</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#food-production"><i class="fa fa-check"></i><b>1.3.1</b> Food production</a></li>
<li class="chapter" data-level="1.3.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#water-treatment"><i class="fa fa-check"></i><b>1.3.2</b> Water Treatment</a></li>
<li class="chapter" data-level="1.3.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#fuel-production"><i class="fa fa-check"></i><b>1.3.3</b> Fuel Production</a></li>
<li class="chapter" data-level="1.3.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#production-of-chemicals-and-enzymes"><i class="fa fa-check"></i><b>1.3.4</b> Production Of Chemicals And Enzymes</a></li>
<li class="chapter" data-level="1.3.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#use-of-microorganisms-in-science"><i class="fa fa-check"></i><b>1.3.5</b> Use Of Microorganisms In Science</a></li>
<li class="chapter" data-level="1.3.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#use-of-microorganisms-for-warfare"><i class="fa fa-check"></i><b>1.3.6</b> Use of Microorganisms For Warfare</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#role-of-microorganisms-in-disease"><i class="fa fa-check"></i><b>1.4</b> Role Of Microorganisms In Disease</a></li>
<li class="chapter" data-level="1.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#role-of-microorganisms-in-agriculture-and-horticulture"><i class="fa fa-check"></i><b>1.5</b> Role Of Microorganisms In Agriculture And Horticulture</a></li>
<li class="chapter" data-level="1.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#history-of-microbiology"><i class="fa fa-check"></i><b>1.6</b> History Of Microbiology</a></li>
<li class="chapter" data-level="1.7" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#taxonomy-classification-and-naming-of-microorganisms"><i class="fa fa-check"></i><b>1.7</b> Taxonomy: Classification And Naming Of Microorganisms</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-concept-of-species-in-biology"><i class="fa fa-check"></i><b>1.7.1</b> The Concept of Species in Biology</a></li>
<li class="chapter" data-level="1.7.2" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#binomial-nomenclature"><i class="fa fa-check"></i><b>1.7.2</b> Binomial Nomenclature</a></li>
<li class="chapter" data-level="1.7.3" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#taxonomy-and-systematics"><i class="fa fa-check"></i><b>1.7.3</b> Taxonomy And Systematics</a></li>
<li class="chapter" data-level="1.7.4" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#kingdoms-and-domains"><i class="fa fa-check"></i><b>1.7.4</b> Kingdoms And Domains</a></li>
<li class="chapter" data-level="1.7.5" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#the-three-domain-system"><i class="fa fa-check"></i><b>1.7.5</b> The Three-Domain System</a></li>
<li class="chapter" data-level="1.7.6" data-path="the-main-themes-of-microbiology.html"><a href="the-main-themes-of-microbiology.html#bergeys-manual-of-systematic-bacteriology"><i class="fa fa-check"></i><b>1.7.6</b> Bergey’s Manual of Systematic Bacteriology</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html"><i class="fa fa-check"></i><b>2</b> The Chemistry Of Biology</a>
<ul>
<li class="chapter" data-level="2.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#matter"><i class="fa fa-check"></i><b>2.1</b> Matter</a></li>
<li class="chapter" data-level="2.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-atom"><i class="fa fa-check"></i><b>2.2</b> The Atom</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structure-of-the-atom"><i class="fa fa-check"></i><b>2.2.1</b> Structure of The Atom</a></li>
<li class="chapter" data-level="2.2.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-atomic-nucleus"><i class="fa fa-check"></i><b>2.2.2</b> The Atomic Nucleus</a></li>
<li class="chapter" data-level="2.2.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-electron-cloud"><i class="fa fa-check"></i><b>2.2.3</b> The Electron Cloud</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-periodic-table-of-the-elements"><i class="fa fa-check"></i><b>2.3</b> The Periodic Table of The Elements</a></li>
<li class="chapter" data-level="2.4" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#chemical-bonds-molecules-and-compounds"><i class="fa fa-check"></i><b>2.4</b> Chemical Bonds, Molecules, And Compounds</a></li>
<li class="chapter" data-level="2.5" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#energy"><i class="fa fa-check"></i><b>2.5</b> Energy</a></li>
<li class="chapter" data-level="2.6" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#chemical-reactions"><i class="fa fa-check"></i><b>2.6</b> Chemical Reactions</a></li>
<li class="chapter" data-level="2.7" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#radioactive-decay-of-atoms"><i class="fa fa-check"></i><b>2.7</b> Radioactive Decay of Atoms</a></li>
<li class="chapter" data-level="2.8" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#ions-and-salts"><i class="fa fa-check"></i><b>2.8</b> Ions And Salts</a></li>
<li class="chapter" data-level="2.9" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#solutions"><i class="fa fa-check"></i><b>2.9</b> Solutions</a></li>
<li class="chapter" data-level="2.10" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#acids-and-bases"><i class="fa fa-check"></i><b>2.10</b> Acids And Bases</a></li>
<li class="chapter" data-level="2.11" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#reduction-oxidation-redox-reactions"><i class="fa fa-check"></i><b>2.11</b> Reduction-oxidation (Redox) Reactions</a></li>
<li class="chapter" data-level="2.12" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#the-chemistry-of-water"><i class="fa fa-check"></i><b>2.12</b> The Chemistry Of Water</a></li>
<li class="chapter" data-level="2.13" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#basic-organic-chemistry-for-biology"><i class="fa fa-check"></i><b>2.13</b> Basic Organic Chemistry For Biology</a>
<ul>
<li class="chapter" data-level="2.13.1" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#functional-groups"><i class="fa fa-check"></i><b>2.13.1</b> Functional Groups</a></li>
<li class="chapter" data-level="2.13.2" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#biomolecules"><i class="fa fa-check"></i><b>2.13.2</b> Biomolecules</a></li>
<li class="chapter" data-level="2.13.3" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#proteins"><i class="fa fa-check"></i><b>2.13.3</b> Proteins</a></li>
<li class="chapter" data-level="2.13.4" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structure"><i class="fa fa-check"></i><b>2.13.4</b> Structure</a></li>
<li class="chapter" data-level="2.13.5" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#cellular-functions-of-proteins"><i class="fa fa-check"></i><b>2.13.5</b> Cellular Functions of Proteins</a></li>
<li class="chapter" data-level="2.13.6" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#cell-signaling-and-ligand-binding"><i class="fa fa-check"></i><b>2.13.6</b> Cell Signaling And Ligand Binding</a></li>
<li class="chapter" data-level="2.13.7" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#structural-proteins"><i class="fa fa-check"></i><b>2.13.7</b> Structural Proteins</a></li>
<li class="chapter" data-level="2.13.8" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#carbohydrates"><i class="fa fa-check"></i><b>2.13.8</b> Carbohydrates</a></li>
<li class="chapter" data-level="2.13.9" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#lipids"><i class="fa fa-check"></i><b>2.13.9</b> Lipids</a></li>
<li class="chapter" data-level="2.13.10" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#nucleic-acids"><i class="fa fa-check"></i><b>2.13.10</b> Nucleic Acids</a></li>
<li class="chapter" data-level="2.13.11" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#deoxyribonucleic-acid-dna"><i class="fa fa-check"></i><b>2.13.11</b> Deoxyribonucleic Acid (DNA)</a></li>
<li class="chapter" data-level="2.13.12" data-path="the-chemistry-of-biology.html"><a href="the-chemistry-of-biology.html#ribonucleic-acid-rna"><i class="fa fa-check"></i><b>2.13.12</b> Ribonucleic Acid (RNA)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html"><i class="fa fa-check"></i><b>3</b> An Introduction To Methods Of Studying Microorganisms</a>
<ul>
<li class="chapter" data-level="3.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#microscopy"><i class="fa fa-check"></i><b>3.1</b> Microscopy</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#light-microscopy"><i class="fa fa-check"></i><b>3.1.1</b> Light Microscopy</a></li>
<li class="chapter" data-level="3.1.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-optical-light-microscope"><i class="fa fa-check"></i><b>3.1.2</b> The Optical (Light) Microscope</a></li>
<li class="chapter" data-level="3.1.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#simple-microscope"><i class="fa fa-check"></i><b>3.1.3</b> Simple Microscope</a></li>
<li class="chapter" data-level="3.1.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#compound-microscope"><i class="fa fa-check"></i><b>3.1.4</b> Compound Microscope</a></li>
<li class="chapter" data-level="3.1.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-main-components-of-the-light-microscope"><i class="fa fa-check"></i><b>3.1.5</b> The Main Components Of The Light Microscope</a></li>
<li class="chapter" data-level="3.1.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#eyepiece-ocular-lens"><i class="fa fa-check"></i><b>3.1.6</b> Eyepiece (Ocular Lens)</a></li>
<li class="chapter" data-level="3.1.7" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#objective-lens"><i class="fa fa-check"></i><b>3.1.7</b> Objective Lens</a></li>
<li class="chapter" data-level="3.1.8" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#oil-immersion-objective"><i class="fa fa-check"></i><b>3.1.8</b> Oil Immersion Objective</a></li>
<li class="chapter" data-level="3.1.9" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#focus-knobs"><i class="fa fa-check"></i><b>3.1.9</b> Focus Knobs</a></li>
<li class="chapter" data-level="3.1.10" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-frame"><i class="fa fa-check"></i><b>3.1.10</b> The Frame</a></li>
<li class="chapter" data-level="3.1.11" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-stage"><i class="fa fa-check"></i><b>3.1.11</b> The Stage</a></li>
<li class="chapter" data-level="3.1.12" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-light-source"><i class="fa fa-check"></i><b>3.1.12</b> The Light Source</a></li>
<li class="chapter" data-level="3.1.13" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-condenser"><i class="fa fa-check"></i><b>3.1.13</b> The Condenser</a></li>
<li class="chapter" data-level="3.1.14" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#magnification"><i class="fa fa-check"></i><b>3.1.14</b> Magnification</a></li>
<li class="chapter" data-level="3.1.15" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-microscope-slide"><i class="fa fa-check"></i><b>3.1.15</b> The Microscope Slide</a></li>
<li class="chapter" data-level="3.1.16" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dry-mount"><i class="fa fa-check"></i><b>3.1.16</b> Dry Mount</a></li>
<li class="chapter" data-level="3.1.17" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#wet-mount-or-temporary-mount"><i class="fa fa-check"></i><b>3.1.17</b> Wet Mount Or Temporary Mount</a></li>
<li class="chapter" data-level="3.1.18" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#prepared-mount-or-permanent-mount"><i class="fa fa-check"></i><b>3.1.18</b> Prepared Mount Or Permanent Mount</a></li>
<li class="chapter" data-level="3.1.19" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#techniques-of-optical-microscopy"><i class="fa fa-check"></i><b>3.1.19</b> Techniques Of Optical Microscopy</a></li>
<li class="chapter" data-level="3.1.20" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#bright-field"><i class="fa fa-check"></i><b>3.1.20</b> Bright Field</a></li>
<li class="chapter" data-level="3.1.21" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#oblique-illumination"><i class="fa fa-check"></i><b>3.1.21</b> Oblique Illumination</a></li>
<li class="chapter" data-level="3.1.22" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dark-field"><i class="fa fa-check"></i><b>3.1.22</b> Dark Field</a></li>
<li class="chapter" data-level="3.1.23" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#dispersion-staining"><i class="fa fa-check"></i><b>3.1.23</b> Dispersion Staining</a></li>
<li class="chapter" data-level="3.1.24" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#phase-contrast"><i class="fa fa-check"></i><b>3.1.24</b> Phase Contrast</a></li>
<li class="chapter" data-level="3.1.25" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#differential-interference-contrast"><i class="fa fa-check"></i><b>3.1.25</b> Differential Interference Contrast</a></li>
<li class="chapter" data-level="3.1.26" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#interference-reflection"><i class="fa fa-check"></i><b>3.1.26</b> Interference Reflection</a></li>
<li class="chapter" data-level="3.1.27" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#fluorescence-microscopy"><i class="fa fa-check"></i><b>3.1.27</b> Fluorescence Microscopy</a></li>
<li class="chapter" data-level="3.1.28" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#confoca-microscopy"><i class="fa fa-check"></i><b>3.1.28</b> Confoca Microscopy</a></li>
<li class="chapter" data-level="3.1.29" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#two-photon-microscopy"><i class="fa fa-check"></i><b>3.1.29</b> Two-Photon Microscopy</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-gram-stain"><i class="fa fa-check"></i><b>3.2</b> The Gram Stain</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-positive-bacteria"><i class="fa fa-check"></i><b>3.2.1</b> Gram-Positive Bacteria</a></li>
<li class="chapter" data-level="3.2.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-negative-bacteria"><i class="fa fa-check"></i><b>3.2.2</b> Gram-Negative Bacteria</a></li>
<li class="chapter" data-level="3.2.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#gram-variable-and-gram-indeterminate-bacteria"><i class="fa fa-check"></i><b>3.2.3</b> Gram-Variable and Gram-Indeterminate Bacteria</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#electron-microscopy"><i class="fa fa-check"></i><b>3.3</b> Electron Microscopy</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-transmission-electron-microscope-tem"><i class="fa fa-check"></i><b>3.3.1</b> The Transmission Electron Microscope (TEM)</a></li>
<li class="chapter" data-level="3.3.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#the-scanning-electron-microscope-sem"><i class="fa fa-check"></i><b>3.3.2</b> The Scanning Electron Microscope (SEM)</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#microbiological-culture"><i class="fa fa-check"></i><b>3.4</b> Microbiological Culture</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#broth-cultures"><i class="fa fa-check"></i><b>3.4.1</b> Broth cultures</a></li>
<li class="chapter" data-level="3.4.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#agar-plates"><i class="fa fa-check"></i><b>3.4.2</b> Agar Plates</a></li>
<li class="chapter" data-level="3.4.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#stab-cultures"><i class="fa fa-check"></i><b>3.4.3</b> Stab Cultures</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#culture-collections"><i class="fa fa-check"></i><b>3.5</b> Culture Collections</a></li>
<li class="chapter" data-level="3.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#growth-medium"><i class="fa fa-check"></i><b>3.6</b> Growth Medium</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#culture-media"><i class="fa fa-check"></i><b>3.6.1</b> Culture Media</a></li>
<li class="chapter" data-level="3.6.2" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#minimal-media"><i class="fa fa-check"></i><b>3.6.2</b> Minimal Media</a></li>
<li class="chapter" data-level="3.6.3" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#selective-media"><i class="fa fa-check"></i><b>3.6.3</b> Selective Media</a></li>
<li class="chapter" data-level="3.6.4" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#differential-media"><i class="fa fa-check"></i><b>3.6.4</b> Differential Media</a></li>
<li class="chapter" data-level="3.6.5" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#transport-media"><i class="fa fa-check"></i><b>3.6.5</b> Transport Media</a></li>
<li class="chapter" data-level="3.6.6" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#enriched-media"><i class="fa fa-check"></i><b>3.6.6</b> Enriched Media</a></li>
<li class="chapter" data-level="3.6.7" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#agar-plate"><i class="fa fa-check"></i><b>3.6.7</b> Agar Plate</a></li>
<li class="chapter" data-level="3.6.8" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#blood-agar-plate"><i class="fa fa-check"></i><b>3.6.8</b> Blood Agar Plate</a></li>
<li class="chapter" data-level="3.6.9" data-path="an-introduction-to-methods-of-studying-microorganisms.html"><a href="an-introduction-to-methods-of-studying-microorganisms.html#chocolate-agar"><i class="fa fa-check"></i><b>3.6.9</b> Chocolate Agar</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><i class="fa fa-check"></i><b>4</b> An Introduction To Prokaryotic Cells And Microorganisms</a>
<ul>
<li class="chapter" data-level="4.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#basic-characteristics-of-life"><i class="fa fa-check"></i><b>4.1</b> Basic Characteristics Of Life</a></li>
<li class="chapter" data-level="4.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#origin-of-life"><i class="fa fa-check"></i><b>4.2</b> Origin Of Life</a></li>
<li class="chapter" data-level="4.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-foundations-of-modern-biology"><i class="fa fa-check"></i><b>4.3</b> The Foundations Of Modern Biology</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-cell-theory"><i class="fa fa-check"></i><b>4.3.1</b> The Cell Theory</a></li>
<li class="chapter" data-level="4.3.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#the-theory-of-evolution"><i class="fa fa-check"></i><b>4.3.2</b> The Theory Of Evolution</a></li>
<li class="chapter" data-level="4.3.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#genetics"><i class="fa fa-check"></i><b>4.3.3</b> Genetics</a></li>
<li class="chapter" data-level="4.3.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#homeostasis"><i class="fa fa-check"></i><b>4.3.4</b> Homeostasis</a></li>
<li class="chapter" data-level="4.3.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#energy-1"><i class="fa fa-check"></i><b>4.3.5</b> Energy</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#basic-characteristics-of-cells"><i class="fa fa-check"></i><b>4.4</b> Basic Characteristics Of Cells</a></li>
<li class="chapter" data-level="4.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#prokaryotic-cells"><i class="fa fa-check"></i><b>4.5</b> Prokaryotic Cells</a></li>
<li class="chapter" data-level="4.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacteria-1"><i class="fa fa-check"></i><b>4.6</b> Bacteria</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#intracellular-structure"><i class="fa fa-check"></i><b>4.6.1</b> Intracellular Structure</a></li>
<li class="chapter" data-level="4.6.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#extracellular-structures"><i class="fa fa-check"></i><b>4.6.2</b> Extracellular Structures</a></li>
<li class="chapter" data-level="4.6.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#endospores"><i class="fa fa-check"></i><b>4.6.3</b> Endospores</a></li>
<li class="chapter" data-level="4.6.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#metabolism"><i class="fa fa-check"></i><b>4.6.4</b> Metabolism</a></li>
<li class="chapter" data-level="4.6.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#growth-and-reproduction"><i class="fa fa-check"></i><b>4.6.5</b> Growth And Reproduction</a></li>
<li class="chapter" data-level="4.6.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacterial-genetics"><i class="fa fa-check"></i><b>4.6.6</b> Bacterial Genetics</a></li>
<li class="chapter" data-level="4.6.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#movement-of-bacteria"><i class="fa fa-check"></i><b>4.6.7</b> Movement Of Bacteria</a></li>
<li class="chapter" data-level="4.6.8" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacterial-communication"><i class="fa fa-check"></i><b>4.6.8</b> Bacterial Communication</a></li>
<li class="chapter" data-level="4.6.9" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#classification-and-identification"><i class="fa fa-check"></i><b>4.6.9</b> Classification And identification</a></li>
<li class="chapter" data-level="4.6.10" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#interactions-with-other-organisms"><i class="fa fa-check"></i><b>4.6.10</b> Interactions With Other Organisms</a></li>
<li class="chapter" data-level="4.6.11" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#bacteria-as-pathogens"><i class="fa fa-check"></i><b>4.6.11</b> Bacteria As Pathogens</a></li>
<li class="chapter" data-level="4.6.12" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#significance-in-technology-and-industry"><i class="fa fa-check"></i><b>4.6.12</b> Significance In Technology And Industry</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaea-1"><i class="fa fa-check"></i><b>4.7</b> Archaea</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#origin-and-evolution"><i class="fa fa-check"></i><b>4.7.1</b> Origin And evolution</a></li>
<li class="chapter" data-level="4.7.2" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#relationship-to-bacteria"><i class="fa fa-check"></i><b>4.7.2</b> Relationship To Bacteria</a></li>
<li class="chapter" data-level="4.7.3" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#relation-to-eukaryotes"><i class="fa fa-check"></i><b>4.7.3</b> Relation To Eukaryotes</a></li>
<li class="chapter" data-level="4.7.4" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#morphology"><i class="fa fa-check"></i><b>4.7.4</b> Morphology</a></li>
<li class="chapter" data-level="4.7.5" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#structure-composition-development-and-operation"><i class="fa fa-check"></i><b>4.7.5</b> Structure, Composition Development, And Operation</a></li>
<li class="chapter" data-level="4.7.6" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#cell-wall-and-flagella"><i class="fa fa-check"></i><b>4.7.6</b> Cell Wall And Flagella</a></li>
<li class="chapter" data-level="4.7.7" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#membranes"><i class="fa fa-check"></i><b>4.7.7</b> Membranes</a></li>
<li class="chapter" data-level="4.7.8" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-metabolism"><i class="fa fa-check"></i><b>4.7.8</b> Archaeal Metabolism</a></li>
<li class="chapter" data-level="4.7.9" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-genetics"><i class="fa fa-check"></i><b>4.7.9</b> Archaeal Genetics</a></li>
<li class="chapter" data-level="4.7.10" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#archaeal-ecology"><i class="fa fa-check"></i><b>4.7.10</b> Archaeal Ecology</a></li>
<li class="chapter" data-level="4.7.11" data-path="an-introduction-to-prokaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-prokaryotic-cells-and-microorganisms.html#significance-in-technology-and-industry-1"><i class="fa fa-check"></i><b>4.7.11</b> Significance In Technology And Industry</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><i class="fa fa-check"></i><b>5</b> An Introduction To Eukaryotic Cells And Microorganisms</a>
<ul>
<li class="chapter" data-level="5.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#characteristic-features-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.1</b> Characteristic Features Of Eukaryotic Cells</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cell-membrane"><i class="fa fa-check"></i><b>5.1.1</b> The Cell Membrane</a></li>
<li class="chapter" data-level="5.1.2" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cytoskeleton"><i class="fa fa-check"></i><b>5.1.2</b> The Cytoskeleton</a></li>
<li class="chapter" data-level="5.1.3" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-genetic-material"><i class="fa fa-check"></i><b>5.1.3</b> The Genetic Material</a></li>
<li class="chapter" data-level="5.1.4" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#organelles-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.1.4</b> Organelles of Eukaryotic Cells</a></li>
<li class="chapter" data-level="5.1.5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-cell-nucleus"><i class="fa fa-check"></i><b>5.1.5</b> The Cell Nucleus</a></li>
<li class="chapter" data-level="5.1.6" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-mitochondria"><i class="fa fa-check"></i><b>5.1.6</b> The Mitochondria</a></li>
<li class="chapter" data-level="5.1.7" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-chloroplasts"><i class="fa fa-check"></i><b>5.1.7</b> The Chloroplasts</a></li>
<li class="chapter" data-level="5.1.8" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-endoplasmic-reticulum"><i class="fa fa-check"></i><b>5.1.8</b> The Endoplasmic Reticulum</a></li>
<li class="chapter" data-level="5.1.9" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-golgi-apparatus"><i class="fa fa-check"></i><b>5.1.9</b> The Golgi Apparatus</a></li>
<li class="chapter" data-level="5.1.10" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#the-ribosomes"><i class="fa fa-check"></i><b>5.1.10</b> The Ribosomes</a></li>
<li class="chapter" data-level="5.1.11" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#structures-outside-the-cell-membrane"><i class="fa fa-check"></i><b>5.1.11</b> Structures Outside The Cell Membrane</a></li>
<li class="chapter" data-level="5.1.12" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cellular-replication"><i class="fa fa-check"></i><b>5.1.12</b> Cellular Replication</a></li>
<li class="chapter" data-level="5.1.13" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#dna-repair"><i class="fa fa-check"></i><b>5.1.13</b> DNA Repair</a></li>
<li class="chapter" data-level="5.1.14" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cellular-growth-and-metabolism"><i class="fa fa-check"></i><b>5.1.14</b> Cellular Growth And Metabolism</a></li>
<li class="chapter" data-level="5.1.15" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protein-synthesis"><i class="fa fa-check"></i><b>5.1.15</b> Protein Synthesis</a></li>
<li class="chapter" data-level="5.1.16" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#cell-motility"><i class="fa fa-check"></i><b>5.1.16</b> Cell Motility</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#origin-of-eukaryotic-cells"><i class="fa fa-check"></i><b>5.2</b> Origin Of Eukaryotic Cells</a></li>
<li class="chapter" data-level="5.3" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protists-1"><i class="fa fa-check"></i><b>5.3</b> Protists</a></li>
<li class="chapter" data-level="5.4" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#protozoa"><i class="fa fa-check"></i><b>5.4</b> Protozoa</a></li>
<li class="chapter" data-level="5.5" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#flagellates"><i class="fa fa-check"></i><b>5.5</b> Flagellates</a></li>
<li class="chapter" data-level="5.6" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#ciliates"><i class="fa fa-check"></i><b>5.6</b> Ciliates</a></li>
<li class="chapter" data-level="5.7" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#amoebozoa"><i class="fa fa-check"></i><b>5.7</b> Amoebozoa</a></li>
<li class="chapter" data-level="5.8" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#algae"><i class="fa fa-check"></i><b>5.8</b> Algae</a></li>
<li class="chapter" data-level="5.9" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#slime-molds"><i class="fa fa-check"></i><b>5.9</b> Slime Molds</a>
<ul>
<li class="chapter" data-level="5.9.1" data-path="an-introduction-to-eukaryotic-cells-and-microorganisms.html"><a href="an-introduction-to-eukaryotic-cells-and-microorganisms.html#reproduction-of-protists"><i class="fa fa-check"></i><b>5.9.1</b> Reproduction of Protists</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html"><i class="fa fa-check"></i><b>6</b> An Introduction To Viruses</a>
<ul>
<li class="chapter" data-level="6.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#theories-of-the-origin-of-viruses"><i class="fa fa-check"></i><b>6.1</b> Theories Of The Origin Of Viruses</a></li>
<li class="chapter" data-level="6.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#structure-and-morphology-of-viruses"><i class="fa fa-check"></i><b>6.2</b> Structure And Morphology Of Viruses</a></li>
<li class="chapter" data-level="6.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#replication-of-viruses"><i class="fa fa-check"></i><b>6.3</b> Replication Of Viruses</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#role-of-viruses-in-human-disease"><i class="fa fa-check"></i><b>6.3.1</b> Role Of Viruses In Human Disease</a></li>
<li class="chapter" data-level="6.3.2" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#animal-viruses"><i class="fa fa-check"></i><b>6.3.2</b> Animal Viruses</a></li>
<li class="chapter" data-level="6.3.3" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#plant-viruses"><i class="fa fa-check"></i><b>6.3.3</b> Plant Viruses</a></li>
<li class="chapter" data-level="6.3.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#bacterial-viruses"><i class="fa fa-check"></i><b>6.3.4</b> Bacterial Viruses</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#classification-of-viruses"><i class="fa fa-check"></i><b>6.4</b> Classification Of Viruses</a></li>
<li class="chapter" data-level="6.5" data-path="an-introduction-to-viruses.html"><a href="an-introduction-to-viruses.html#the-baltimore-classification-of-viruses"><i class="fa fa-check"></i><b>6.5</b> The Baltimore Classification Of Viruses</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html"><i class="fa fa-check"></i><b>7</b> Microbial Nutrition, Ecology And Growth</a>
<ul>
<li class="chapter" data-level="7.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-energy"><i class="fa fa-check"></i><b>7.1</b> Primary Sources Of Energy</a></li>
<li class="chapter" data-level="7.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-reducing-equivalents"><i class="fa fa-check"></i><b>7.2</b> Primary Sources Of Reducing Equivalents</a></li>
<li class="chapter" data-level="7.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#primary-sources-of-carbon"><i class="fa fa-check"></i><b>7.3</b> Primary Sources of Carbon</a></li>
<li class="chapter" data-level="7.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#nutrients"><i class="fa fa-check"></i><b>7.4</b> Nutrients</a></li>
<li class="chapter" data-level="7.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#the-nutrient-cycle"><i class="fa fa-check"></i><b>7.5</b> The Nutrient Cycle</a></li>
<li class="chapter" data-level="7.6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#transport-and-movement-of-substances-across-the-cell-membrane"><i class="fa fa-check"></i><b>7.6</b> Transport And Movement Of Substances Across The Cell Membrane</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#diffusion"><i class="fa fa-check"></i><b>7.6.1</b> Diffusion</a></li>
<li class="chapter" data-level="7.6.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#facilitated-diffusion"><i class="fa fa-check"></i><b>7.6.2</b> Facilitated Diffusion</a></li>
<li class="chapter" data-level="7.6.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#osmosis"><i class="fa fa-check"></i><b>7.6.3</b> Osmosis</a></li>
<li class="chapter" data-level="7.6.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#active-transport"><i class="fa fa-check"></i><b>7.6.4</b> Active Transport</a></li>
<li class="chapter" data-level="7.6.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#bulk-transport"><i class="fa fa-check"></i><b>7.6.5</b> Bulk transport</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#microbial-growth"><i class="fa fa-check"></i><b>7.7</b> Microbial Growth</a></li>
<li class="chapter" data-level="7.8" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#cell-counting"><i class="fa fa-check"></i><b>7.8</b> Cell Counting</a>
<ul>
<li class="chapter" data-level="7.8.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#counting-chamber"><i class="fa fa-check"></i><b>7.8.1</b> Counting Chamber</a></li>
<li class="chapter" data-level="7.8.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#plating-and-the-colony-forming-unit-counting"><i class="fa fa-check"></i><b>7.8.2</b> Plating And The Colony-Forming Unit Counting</a></li>
<li class="chapter" data-level="7.8.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#electrical-resistance"><i class="fa fa-check"></i><b>7.8.3</b> Electrical Resistance</a></li>
<li class="chapter" data-level="7.8.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#flow-cytometry"><i class="fa fa-check"></i><b>7.8.4</b> Flow Cytometry</a></li>
<li class="chapter" data-level="7.8.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#spectrophotometry"><i class="fa fa-check"></i><b>7.8.5</b> Spectrophotometry</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#environmental-factors-influencing-growth"><i class="fa fa-check"></i><b>7.9</b> Environmental Factors Influencing Growth</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#nutrient-availability"><i class="fa fa-check"></i><b>7.9.1</b> Nutrient Availability</a></li>
<li class="chapter" data-level="7.9.2" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#temperature"><i class="fa fa-check"></i><b>7.9.2</b> Temperature</a></li>
<li class="chapter" data-level="7.9.3" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#acidity"><i class="fa fa-check"></i><b>7.9.3</b> Acidity</a></li>
<li class="chapter" data-level="7.9.4" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#water-activity"><i class="fa fa-check"></i><b>7.9.4</b> Water activity</a></li>
<li class="chapter" data-level="7.9.5" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#oxygen"><i class="fa fa-check"></i><b>7.9.5</b> Oxygen</a></li>
<li class="chapter" data-level="7.9.6" data-path="microbial-nutrition-ecology-and-growth.html"><a href="microbial-nutrition-ecology-and-growth.html#toxic-compounds"><i class="fa fa-check"></i><b>7.9.6</b> Toxic compounds</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html"><i class="fa fa-check"></i><b>8</b> An Introduction To Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="8.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#thermodynamics-of-living-organisms"><i class="fa fa-check"></i><b>8.1</b> Thermodynamics of Living Organisms</a></li>
<li class="chapter" data-level="8.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#metabolism-1"><i class="fa fa-check"></i><b>8.2</b> Metabolism</a></li>
<li class="chapter" data-level="8.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#types-of-microbial-metabolism"><i class="fa fa-check"></i><b>8.3</b> Types Of Microbial Metabolism</a></li>
<li class="chapter" data-level="8.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#phototrophy"><i class="fa fa-check"></i><b>8.4</b> Phototrophy</a></li>
<li class="chapter" data-level="8.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#chemolithotrophy"><i class="fa fa-check"></i><b>8.5</b> Chemolithotrophy</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#hydrogen-oxidation"><i class="fa fa-check"></i><b>8.5.1</b> Hydrogen oxidation</a></li>
<li class="chapter" data-level="8.5.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#sulfur-oxidation"><i class="fa fa-check"></i><b>8.5.2</b> Sulfur oxidation</a></li>
<li class="chapter" data-level="8.5.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#ferrous-iron-fe2-oxidation"><i class="fa fa-check"></i><b>8.5.3</b> Ferrous Iron (Fe<sup>2</sup>) Oxidation</a></li>
<li class="chapter" data-level="8.5.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nitrification"><i class="fa fa-check"></i><b>8.5.4</b> Nitrification</a></li>
<li class="chapter" data-level="8.5.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nitrogen-fixation"><i class="fa fa-check"></i><b>8.5.5</b> Nitrogen Fixation</a></li>
<li class="chapter" data-level="8.5.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#catabolism"><i class="fa fa-check"></i><b>8.5.6</b> Catabolism</a></li>
<li class="chapter" data-level="8.5.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#digestion"><i class="fa fa-check"></i><b>8.5.7</b> Digestion</a></li>
<li class="chapter" data-level="8.5.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-organic-compounds"><i class="fa fa-check"></i><b>8.5.8</b> Energy From Organic Compounds</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-transformations"><i class="fa fa-check"></i><b>8.6</b> Energy Transformations</a>
<ul>
<li class="chapter" data-level="8.6.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-phosphorylation"><i class="fa fa-check"></i><b>8.6.1</b> Oxidative Phosphorylation</a></li>
<li class="chapter" data-level="8.6.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-inorganic-compounds"><i class="fa fa-check"></i><b>8.6.2</b> Energy From Inorganic Compounds</a></li>
<li class="chapter" data-level="8.6.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#energy-from-light"><i class="fa fa-check"></i><b>8.6.3</b> Energy From Light</a></li>
<li class="chapter" data-level="8.6.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#anabolism"><i class="fa fa-check"></i><b>8.6.4</b> Anabolism</a></li>
</ul></li>
<li class="chapter" data-level="8.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#enzymes"><i class="fa fa-check"></i><b>8.7</b> Enzymes</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#classification-and-nomenclature-of-enzymes"><i class="fa fa-check"></i><b>8.7.1</b> Classification And Nomenclature Of Enzymes</a></li>
<li class="chapter" data-level="8.7.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#stucture-of-enzymes"><i class="fa fa-check"></i><b>8.7.2</b> Stucture Of Enzymes</a></li>
<li class="chapter" data-level="8.7.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#mechanism-of-action-of-enzymes"><i class="fa fa-check"></i><b>8.7.3</b> Mechanism Of Action Of Enzymes</a></li>
<li class="chapter" data-level="8.7.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#allosteric-modulation"><i class="fa fa-check"></i><b>8.7.4</b> Allosteric Modulation</a></li>
<li class="chapter" data-level="8.7.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cofactors-of-enzymes-and-coenzymes"><i class="fa fa-check"></i><b>8.7.5</b> Cofactors Of Enzymes And Coenzymes</a></li>
<li class="chapter" data-level="8.7.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#inhibition-of-enzymes"><i class="fa fa-check"></i><b>8.7.6</b> Inhibition Of Enzymes</a></li>
<li class="chapter" data-level="8.7.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#factors-affecting-enzyme-activity"><i class="fa fa-check"></i><b>8.7.7</b> Factors Affecting Enzyme Activity</a></li>
<li class="chapter" data-level="8.7.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#control-of-activity"><i class="fa fa-check"></i><b>8.7.8</b> Control Of Activity</a></li>
<li class="chapter" data-level="8.7.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#post-translational-modification"><i class="fa fa-check"></i><b>8.7.9</b> Post-translational Modification</a></li>
<li class="chapter" data-level="8.7.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#quantity"><i class="fa fa-check"></i><b>8.7.10</b> Quantity</a></li>
<li class="chapter" data-level="8.7.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#subcellular-distribution"><i class="fa fa-check"></i><b>8.7.11</b> Subcellular Distribution</a></li>
<li class="chapter" data-level="8.7.12" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#organ-specialization"><i class="fa fa-check"></i><b>8.7.12</b> Organ Specialization</a></li>
<li class="chapter" data-level="8.7.13" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#coenzymes"><i class="fa fa-check"></i><b>8.7.13</b> Coenzymes</a></li>
<li class="chapter" data-level="8.7.14" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#mineral-cofactors"><i class="fa fa-check"></i><b>8.7.14</b> Mineral Cofactors</a></li>
</ul></li>
<li class="chapter" data-level="8.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#heterotrophic-microbial-metabolism"><i class="fa fa-check"></i><b>8.8</b> Heterotrophic Microbial Metabolism</a>
<ul>
<li class="chapter" data-level="8.8.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#carbon-fixation"><i class="fa fa-check"></i><b>8.8.1</b> Carbon Fixation</a></li>
<li class="chapter" data-level="8.8.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#carbohydrates-and-glycans"><i class="fa fa-check"></i><b>8.8.2</b> Carbohydrates And Glycans</a></li>
<li class="chapter" data-level="8.8.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#fatty-acids-isoprenoids-and-sterol"><i class="fa fa-check"></i><b>8.8.3</b> Fatty Acids, Isoprenoids and Sterol</a></li>
<li class="chapter" data-level="8.8.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#proteins-1"><i class="fa fa-check"></i><b>8.8.4</b> Proteins</a></li>
<li class="chapter" data-level="8.8.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#nucleotide-synthesis-and-salvage"><i class="fa fa-check"></i><b>8.8.5</b> Nucleotide Synthesis And Salvage</a></li>
</ul></li>
<li class="chapter" data-level="8.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthesis-and-cellular-respiration"><i class="fa fa-check"></i><b>8.9</b> Photosynthesis And Cellular Respiration</a>
<ul>
<li class="chapter" data-level="8.9.1" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#the-electron-transport-chain-and-chemiosmosis"><i class="fa fa-check"></i><b>8.9.1</b> The Electron Transport Chain And Chemiosmosis</a></li>
<li class="chapter" data-level="8.9.2" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthesis"><i class="fa fa-check"></i><b>8.9.2</b> Photosynthesis</a></li>
<li class="chapter" data-level="8.9.3" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#photosynthetic-membranes-and-organelles"><i class="fa fa-check"></i><b>8.9.3</b> Photosynthetic Membranes And Organelles</a></li>
<li class="chapter" data-level="8.9.4" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#z-scheme"><i class="fa fa-check"></i><b>8.9.4</b> Z Scheme</a></li>
<li class="chapter" data-level="8.9.5" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#water-photolysis"><i class="fa fa-check"></i><b>8.9.5</b> Water Photolysis</a></li>
<li class="chapter" data-level="8.9.6" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#calvin-cycle"><i class="fa fa-check"></i><b>8.9.6</b> Calvin Cycle</a></li>
<li class="chapter" data-level="8.9.7" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#symbiosis-and-the-origin-of-chloroplasts"><i class="fa fa-check"></i><b>8.9.7</b> Symbiosis And The Origin of Chloroplasts</a></li>
<li class="chapter" data-level="8.9.8" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cyanobacteria-and-the-evolution-of-photosynthesis"><i class="fa fa-check"></i><b>8.9.8</b> Cyanobacteria And The Evolution of Photosynthesis</a></li>
<li class="chapter" data-level="8.9.9" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#cellular-respiration"><i class="fa fa-check"></i><b>8.9.9</b> Cellular Respiration</a></li>
<li class="chapter" data-level="8.9.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#aerobic-respiration"><i class="fa fa-check"></i><b>8.9.10</b> Aerobic Respiration</a></li>
<li class="chapter" data-level="8.9.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#glycolysis"><i class="fa fa-check"></i><b>8.9.11</b> Glycolysis</a></li>
<li class="chapter" data-level="8.9.12" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-decarboxylation-of-pyruvate"><i class="fa fa-check"></i><b>8.9.12</b> Oxidative Decarboxylation of Pyruvate</a></li>
<li class="chapter" data-level="8.9.13" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#the-citric-acid-cycle"><i class="fa fa-check"></i><b>8.9.13</b> The Citric Acid Cycle</a></li>
<li class="chapter" data-level="8.9.14" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#oxidative-phosphorylation-1"><i class="fa fa-check"></i><b>8.9.14</b> Oxidative Phosphorylation</a></li>
</ul></li>
<li class="chapter" data-level="8.10" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#fermentation"><i class="fa fa-check"></i><b>8.10</b> Fermentation</a></li>
<li class="chapter" data-level="8.11" data-path="an-introduction-to-microbial-metabolism.html"><a href="an-introduction-to-microbial-metabolism.html#anaerobic-respiration"><i class="fa fa-check"></i><b>8.11</b> Anaerobic Respiration</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="microbial-genetics.html"><a href="microbial-genetics.html"><i class="fa fa-check"></i><b>9</b> Microbial Genetics</a>
<ul>
<li class="chapter" data-level="9.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#introduction-to-genetics-and-genes"><i class="fa fa-check"></i><b>9.1</b> Introduction To Genetics And Genes</a>
<ul>
<li class="chapter" data-level="9.1.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mendelian-classical-genetics"><i class="fa fa-check"></i><b>9.1.1</b> Mendelian (Classical) Genetics</a></li>
<li class="chapter" data-level="9.1.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#molecular-genetics"><i class="fa fa-check"></i><b>9.1.2</b> Molecular Genetics</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#the-genetic-code"><i class="fa fa-check"></i><b>9.2</b> The Genetic Code</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-expression"><i class="fa fa-check"></i><b>9.2.1</b> Gene Expression</a></li>
<li class="chapter" data-level="9.2.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#nature-and-nurture"><i class="fa fa-check"></i><b>9.2.2</b> Nature And Nurture</a></li>
<li class="chapter" data-level="9.2.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-regulation"><i class="fa fa-check"></i><b>9.2.3</b> Gene Regulation</a></li>
<li class="chapter" data-level="9.2.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genetic-change"><i class="fa fa-check"></i><b>9.2.4</b> Genetic Change</a></li>
<li class="chapter" data-level="9.2.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#natural-selection-and-evolution"><i class="fa fa-check"></i><b>9.2.5</b> Natural Selection And Evolution</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-genetics-1"><i class="fa fa-check"></i><b>9.3</b> Bacterial Genetics</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-transformation"><i class="fa fa-check"></i><b>9.3.1</b> Bacterial Transformation</a></li>
<li class="chapter" data-level="9.3.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#practical-aspects-of-transformation-in-molecular-biology"><i class="fa fa-check"></i><b>9.3.2</b> Practical aspects of transformation in molecular biology</a></li>
<li class="chapter" data-level="9.3.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#selection-and-screening-in-plasmid-transformation"><i class="fa fa-check"></i><b>9.3.3</b> Selection and screening in plasmid transformation</a></li>
<li class="chapter" data-level="9.3.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-conjugation"><i class="fa fa-check"></i><b>9.3.4</b> Bacterial Conjugation</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-and-function-of-deoxyribonucleic-acid"><i class="fa fa-check"></i><b>9.4</b> Structure And Function Of Deoxyribonucleic Acid</a>
<ul>
<li class="chapter" data-level="9.4.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-damage"><i class="fa fa-check"></i><b>9.4.1</b> DNA Damage</a></li>
<li class="chapter" data-level="9.4.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genes-and-genomes"><i class="fa fa-check"></i><b>9.4.2</b> Genes And Genomes</a></li>
<li class="chapter" data-level="9.4.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-translation"><i class="fa fa-check"></i><b>9.4.3</b> Transcription And Translation</a></li>
<li class="chapter" data-level="9.4.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-replication"><i class="fa fa-check"></i><b>9.4.4</b> DNA Replication</a></li>
<li class="chapter" data-level="9.4.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-dna-replication"><i class="fa fa-check"></i><b>9.4.5</b> Eukaryotic DNA Replication</a></li>
<li class="chapter" data-level="9.4.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation"><i class="fa fa-check"></i><b>9.4.6</b> Initiation</a></li>
<li class="chapter" data-level="9.4.7" data-path="microbial-genetics.html"><a href="microbial-genetics.html#pre-replication-complex"><i class="fa fa-check"></i><b>9.4.7</b> Pre-Replication Complex</a></li>
<li class="chapter" data-level="9.4.8" data-path="microbial-genetics.html"><a href="microbial-genetics.html#preinitiation-complex"><i class="fa fa-check"></i><b>9.4.8</b> Preinitiation Complex</a></li>
<li class="chapter" data-level="9.4.9" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation"><i class="fa fa-check"></i><b>9.4.9</b> Elongation</a></li>
<li class="chapter" data-level="9.4.10" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-fork"><i class="fa fa-check"></i><b>9.4.10</b> Replication Fork</a></li>
<li class="chapter" data-level="9.4.11" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-of-the-leading-strand"><i class="fa fa-check"></i><b>9.4.11</b> Replication Of The Leading Strand</a></li>
<li class="chapter" data-level="9.4.12" data-path="microbial-genetics.html"><a href="microbial-genetics.html#replication-of-the-lagging-strand"><i class="fa fa-check"></i><b>9.4.12</b> Replication Of The Lagging Strand</a></li>
<li class="chapter" data-level="9.4.13" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-replication-proteins"><i class="fa fa-check"></i><b>9.4.13</b> DNA Replication Proteins</a></li>
<li class="chapter" data-level="9.4.14" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination"><i class="fa fa-check"></i><b>9.4.14</b> Termination</a></li>
<li class="chapter" data-level="9.4.15" data-path="microbial-genetics.html"><a href="microbial-genetics.html#regulation-of-dna-replication"><i class="fa fa-check"></i><b>9.4.15</b> Regulation Of DNA Replication</a></li>
<li class="chapter" data-level="9.4.16" data-path="microbial-genetics.html"><a href="microbial-genetics.html#interactions-of-dna-with-proteins"><i class="fa fa-check"></i><b>9.4.16</b> Interactions Of DNA with Proteins</a></li>
<li class="chapter" data-level="9.4.17" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-binding-proteins"><i class="fa fa-check"></i><b>9.4.17</b> DNA-Binding Proteins</a></li>
<li class="chapter" data-level="9.4.18" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>9.4.18</b> DNA-modifying Enzymes</a></li>
<li class="chapter" data-level="9.4.19" data-path="microbial-genetics.html"><a href="microbial-genetics.html#nucleases-and-ligases"><i class="fa fa-check"></i><b>9.4.19</b> Nucleases And Ligases</a></li>
<li class="chapter" data-level="9.4.20" data-path="microbial-genetics.html"><a href="microbial-genetics.html#topoisomerases-and-helicases"><i class="fa fa-check"></i><b>9.4.20</b> Topoisomerases And Helicases</a></li>
<li class="chapter" data-level="9.4.21" data-path="microbial-genetics.html"><a href="microbial-genetics.html#polymerases"><i class="fa fa-check"></i><b>9.4.21</b> Polymerases</a></li>
<li class="chapter" data-level="9.4.22" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-recombination"><i class="fa fa-check"></i><b>9.4.22</b> DNA Recombination</a></li>
<li class="chapter" data-level="9.4.23" data-path="microbial-genetics.html"><a href="microbial-genetics.html#evolutionary-history-of-dna"><i class="fa fa-check"></i><b>9.4.23</b> Evolutionary History Of DNA</a></li>
<li class="chapter" data-level="9.4.24" data-path="microbial-genetics.html"><a href="microbial-genetics.html#genetic-engineering"><i class="fa fa-check"></i><b>9.4.24</b> Genetic Engineering</a></li>
<li class="chapter" data-level="9.4.25" data-path="microbial-genetics.html"><a href="microbial-genetics.html#dna-profiling"><i class="fa fa-check"></i><b>9.4.25</b> Dna Profiling</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-and-function-of-ribonucleic-acid"><i class="fa fa-check"></i><b>9.5</b> Structure And Function Of Ribonucleic Acid</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#comparison-with-dna"><i class="fa fa-check"></i><b>9.5.1</b> Comparison With DNA</a></li>
<li class="chapter" data-level="9.5.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#structure-of-rna"><i class="fa fa-check"></i><b>9.5.2</b> Structure Of RNA</a></li>
<li class="chapter" data-level="9.5.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#synthesis-of-rna"><i class="fa fa-check"></i><b>9.5.3</b> Synthesis Of RNA</a></li>
<li class="chapter" data-level="9.5.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#coding-and-non-coding-rna"><i class="fa fa-check"></i><b>9.5.4</b> Coding And Non-coding RNA</a></li>
<li class="chapter" data-level="9.5.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#regulatory-rna"><i class="fa fa-check"></i><b>9.5.5</b> Regulatory RNA</a></li>
<li class="chapter" data-level="9.5.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-interference-by-mirnas"><i class="fa fa-check"></i><b>9.5.6</b> RNA Interference By miRNAs</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#gene-expression-1"><i class="fa fa-check"></i><b>9.6</b> Gene Expression</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-rna-processing"><i class="fa fa-check"></i><b>9.6.1</b> Transcription And RNA Processing</a></li>
<li class="chapter" data-level="9.6.2" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-1"><i class="fa fa-check"></i><b>9.6.2</b> Initiation</a></li>
<li class="chapter" data-level="9.6.3" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation-1"><i class="fa fa-check"></i><b>9.6.3</b> Elongation</a></li>
<li class="chapter" data-level="9.6.4" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination-1"><i class="fa fa-check"></i><b>9.6.4</b> Termination</a></li>
<li class="chapter" data-level="9.6.5" data-path="microbial-genetics.html"><a href="microbial-genetics.html#inhibitors-of-transcription"><i class="fa fa-check"></i><b>9.6.5</b> Inhibitors Of Transcription</a></li>
<li class="chapter" data-level="9.6.6" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-factors"><i class="fa fa-check"></i><b>9.6.6</b> Transcription Factors</a></li>
<li class="chapter" data-level="9.6.7" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-processing"><i class="fa fa-check"></i><b>9.6.7</b> RNA Processing</a></li>
<li class="chapter" data-level="9.6.8" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mrna-processing"><i class="fa fa-check"></i><b>9.6.8</b> mRNA Processing</a></li>
<li class="chapter" data-level="9.6.9" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing"><i class="fa fa-check"></i><b>9.6.9</b> 5’ Processing</a></li>
<li class="chapter" data-level="9.6.10" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-1"><i class="fa fa-check"></i><b>9.6.10</b> 3’ Processing</a></li>
<li class="chapter" data-level="9.6.11" data-path="microbial-genetics.html"><a href="microbial-genetics.html#intron-splicing"><i class="fa fa-check"></i><b>9.6.11</b> Intron Splicing</a></li>
<li class="chapter" data-level="9.6.12" data-path="microbial-genetics.html"><a href="microbial-genetics.html#reverse-transcription"><i class="fa fa-check"></i><b>9.6.12</b> Reverse Transcription</a></li>
<li class="chapter" data-level="9.6.13" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-export"><i class="fa fa-check"></i><b>9.6.13</b> RNA Export</a></li>
<li class="chapter" data-level="9.6.14" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-and-rna-processing-1"><i class="fa fa-check"></i><b>9.6.14</b> Transcription And RNA Processing</a></li>
<li class="chapter" data-level="9.6.15" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-2"><i class="fa fa-check"></i><b>9.6.15</b> Initiation</a></li>
<li class="chapter" data-level="9.6.16" data-path="microbial-genetics.html"><a href="microbial-genetics.html#inhibitors-of-transcription-1"><i class="fa fa-check"></i><b>9.6.16</b> Inhibitors of transcription</a></li>
<li class="chapter" data-level="9.6.17" data-path="microbial-genetics.html"><a href="microbial-genetics.html#transcription-factors-1"><i class="fa fa-check"></i><b>9.6.17</b> Transcription Factors</a></li>
<li class="chapter" data-level="9.6.18" data-path="microbial-genetics.html"><a href="microbial-genetics.html#rna-processing-1"><i class="fa fa-check"></i><b>9.6.18</b> RNA Processing</a></li>
<li class="chapter" data-level="9.6.19" data-path="microbial-genetics.html"><a href="microbial-genetics.html#mrna-processing-1"><i class="fa fa-check"></i><b>9.6.19</b> mRNA Processing</a></li>
<li class="chapter" data-level="9.6.20" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-2"><i class="fa fa-check"></i><b>9.6.20</b> 5’ Processing</a></li>
<li class="chapter" data-level="9.6.21" data-path="microbial-genetics.html"><a href="microbial-genetics.html#processing-3"><i class="fa fa-check"></i><b>9.6.21</b> 3’ Processing</a></li>
<li class="chapter" data-level="9.6.22" data-path="microbial-genetics.html"><a href="microbial-genetics.html#intron-splicing-1"><i class="fa fa-check"></i><b>9.6.22</b> Intron Splicing</a></li>
<li class="chapter" data-level="9.6.23" data-path="microbial-genetics.html"><a href="microbial-genetics.html#reverse-transcription-1"><i class="fa fa-check"></i><b>9.6.23</b> Reverse Transcription</a></li>
<li class="chapter" data-level="9.6.24" data-path="microbial-genetics.html"><a href="microbial-genetics.html#translation-and-the-genetic-code"><i class="fa fa-check"></i><b>9.6.24</b> Translation And The Genetic Code</a></li>
<li class="chapter" data-level="9.6.25" data-path="microbial-genetics.html"><a href="microbial-genetics.html#ribosomes"><i class="fa fa-check"></i><b>9.6.25</b> Ribosomes</a></li>
<li class="chapter" data-level="9.6.26" data-path="microbial-genetics.html"><a href="microbial-genetics.html#bacterial-ribosomes"><i class="fa fa-check"></i><b>9.6.26</b> Bacterial Ribosomes</a></li>
<li class="chapter" data-level="9.6.27" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-ribosomes"><i class="fa fa-check"></i><b>9.6.27</b> Eukaryotic Ribosomes</a></li>
<li class="chapter" data-level="9.6.28" data-path="microbial-genetics.html"><a href="microbial-genetics.html#translation"><i class="fa fa-check"></i><b>9.6.28</b> Translation</a></li>
<li class="chapter" data-level="9.6.29" data-path="microbial-genetics.html"><a href="microbial-genetics.html#eukaryotic-translation"><i class="fa fa-check"></i><b>9.6.29</b> Eukaryotic Translation</a></li>
<li class="chapter" data-level="9.6.30" data-path="microbial-genetics.html"><a href="microbial-genetics.html#initiation-3"><i class="fa fa-check"></i><b>9.6.30</b> Initiation</a></li>
<li class="chapter" data-level="9.6.31" data-path="microbial-genetics.html"><a href="microbial-genetics.html#elongation-2"><i class="fa fa-check"></i><b>9.6.31</b> Elongation</a></li>
<li class="chapter" data-level="9.6.32" data-path="microbial-genetics.html"><a href="microbial-genetics.html#termination-2"><i class="fa fa-check"></i><b>9.6.32</b> Termination</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html"><i class="fa fa-check"></i><b>10</b> Genetic Engineering</a>
<ul>
<li class="chapter" data-level="10.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genetically-modified-organisms"><i class="fa fa-check"></i><b>10.1</b> Genetically Modified Organisms</a>
<ul>
<li class="chapter" data-level="10.1.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#production"><i class="fa fa-check"></i><b>10.1.1</b> Production</a></li>
<li class="chapter" data-level="10.1.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#bacteria-2"><i class="fa fa-check"></i><b>10.1.2</b> Bacteria</a></li>
<li class="chapter" data-level="10.1.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#viruses"><i class="fa fa-check"></i><b>10.1.3</b> Viruses</a></li>
<li class="chapter" data-level="10.1.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#fungi-1"><i class="fa fa-check"></i><b>10.1.4</b> Fungi</a></li>
<li class="chapter" data-level="10.1.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#plants-1"><i class="fa fa-check"></i><b>10.1.5</b> Plants</a></li>
<li class="chapter" data-level="10.1.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#crops"><i class="fa fa-check"></i><b>10.1.6</b> Crops</a></li>
<li class="chapter" data-level="10.1.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#animals"><i class="fa fa-check"></i><b>10.1.7</b> Animals</a></li>
<li class="chapter" data-level="10.1.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#laws-and-regulations"><i class="fa fa-check"></i><b>10.1.8</b> Laws and regulations</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-therapy"><i class="fa fa-check"></i><b>10.2</b> Gene therapy</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#laws-and-regulations-1"><i class="fa fa-check"></i><b>10.2.1</b> Laws and regulations</a></li>
<li class="chapter" data-level="10.2.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-isolation-and-cloning"><i class="fa fa-check"></i><b>10.2.2</b> Gene Isolation And Cloning</a></li>
<li class="chapter" data-level="10.2.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#inserting-dna-into-the-host-genome"><i class="fa fa-check"></i><b>10.2.3</b> Inserting DNA Into The Host Genome</a></li>
<li class="chapter" data-level="10.2.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#applications-of-genetic-editing"><i class="fa fa-check"></i><b>10.2.4</b> Applications Of Genetic Editing</a></li>
<li class="chapter" data-level="10.2.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#regulation-of-genetic-engineering"><i class="fa fa-check"></i><b>10.2.5</b> Regulation Of Genetic Engineering</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#recombinant-dna-technology"><i class="fa fa-check"></i><b>10.3</b> Recombinant DNA Technology</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#molecular-cloning"><i class="fa fa-check"></i><b>10.3.1</b> Molecular Cloning</a></li>
<li class="chapter" data-level="10.3.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#procedures"><i class="fa fa-check"></i><b>10.3.2</b> Procedures</a></li>
<li class="chapter" data-level="10.3.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#mutagenesis"><i class="fa fa-check"></i><b>10.3.3</b> Mutagenesis</a></li>
<li class="chapter" data-level="10.3.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#random-mutagenesis"><i class="fa fa-check"></i><b>10.3.4</b> Random Mutagenesis</a></li>
<li class="chapter" data-level="10.3.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>10.3.5</b> Site-Directed Mutagenesis</a></li>
<li class="chapter" data-level="10.3.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-synthesis"><i class="fa fa-check"></i><b>10.3.6</b> Gene Synthesis</a></li>
<li class="chapter" data-level="10.3.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knockout"><i class="fa fa-check"></i><b>10.3.7</b> Gene Knockout</a></li>
<li class="chapter" data-level="10.3.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#conditional-gene-knockout"><i class="fa fa-check"></i><b>10.3.8</b> Conditional Gene Knockout</a></li>
<li class="chapter" data-level="10.3.9" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knock-in"><i class="fa fa-check"></i><b>10.3.9</b> Gene Knock-In</a></li>
<li class="chapter" data-level="10.3.10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#rna-interference"><i class="fa fa-check"></i><b>10.3.10</b> RNA interference</a></li>
<li class="chapter" data-level="10.3.11" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#microrna"><i class="fa fa-check"></i><b>10.3.11</b> MicroRNA</a></li>
<li class="chapter" data-level="10.3.12" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#transcriptional-silencing"><i class="fa fa-check"></i><b>10.3.12</b> Transcriptional Silencing</a></li>
<li class="chapter" data-level="10.3.13" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-knockdown"><i class="fa fa-check"></i><b>10.3.13</b> Gene Knockdown</a></li>
<li class="chapter" data-level="10.3.14" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#cre-lox-recombination"><i class="fa fa-check"></i><b>10.3.14</b> Cre-Lox Recombination</a></li>
<li class="chapter" data-level="10.3.15" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#crispr-gene-editing"><i class="fa fa-check"></i><b>10.3.15</b> CRISPR Gene Editing</a></li>
<li class="chapter" data-level="10.3.16" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gene-drive"><i class="fa fa-check"></i><b>10.3.16</b> Gene Drive</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#human-germline-modification"><i class="fa fa-check"></i><b>10.4</b> Human Germline Modification</a></li>
<li class="chapter" data-level="10.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#gain-of-function-research"><i class="fa fa-check"></i><b>10.5</b> Gain-of-function Research</a></li>
<li class="chapter" data-level="10.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genomics"><i class="fa fa-check"></i><b>10.6</b> Genomics</a>
<ul>
<li class="chapter" data-level="10.6.1" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#complete-genomes"><i class="fa fa-check"></i><b>10.6.1</b> Complete Genomes</a></li>
<li class="chapter" data-level="10.6.2" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#the-omics-revolution"><i class="fa fa-check"></i><b>10.6.2</b> The “Omics” Revolution</a></li>
<li class="chapter" data-level="10.6.3" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-analysis"><i class="fa fa-check"></i><b>10.6.3</b> Genome Analysis</a></li>
<li class="chapter" data-level="10.6.4" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-annotation"><i class="fa fa-check"></i><b>10.6.4</b> Genome Annotation</a></li>
<li class="chapter" data-level="10.6.5" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#functional-genomics"><i class="fa fa-check"></i><b>10.6.5</b> Functional Genomics</a></li>
<li class="chapter" data-level="10.6.6" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#structural-genomics"><i class="fa fa-check"></i><b>10.6.6</b> Structural Genomics</a></li>
<li class="chapter" data-level="10.6.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#epigenomics"><i class="fa fa-check"></i><b>10.6.7</b> Epigenomics</a></li>
<li class="chapter" data-level="10.6.8" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#metagenomics"><i class="fa fa-check"></i><b>10.6.8</b> Metagenomics</a></li>
<li class="chapter" data-level="10.6.9" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#applications-of-genomics"><i class="fa fa-check"></i><b>10.6.9</b> Applications Of Genomics</a></li>
<li class="chapter" data-level="10.6.10" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genomic-medicine"><i class="fa fa-check"></i><b>10.6.10</b> Genomic Medicine</a></li>
<li class="chapter" data-level="10.6.11" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#synthetic-biology-and-bioengineering"><i class="fa fa-check"></i><b>10.6.11</b> Synthetic Biology And Bioengineering</a></li>
<li class="chapter" data-level="10.6.12" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#conservation-genomics"><i class="fa fa-check"></i><b>10.6.12</b> Conservation Genomics</a></li>
<li class="chapter" data-level="10.6.13" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genome-wide-association-study"><i class="fa fa-check"></i><b>10.6.13</b> Genome-Wide Association Study</a></li>
</ul></li>
<li class="chapter" data-level="10.7" data-path="genetic-engineering-1.html"><a href="genetic-engineering-1.html#genetic-testing"><i class="fa fa-check"></i><b>10.7</b> Genetic Testing</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html"><i class="fa fa-check"></i><b>11</b> Physical And Chemical Agents for Microbial Control</a>
<ul>
<li class="chapter" data-level="11.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#sterilization"><i class="fa fa-check"></i><b>11.1</b> Sterilization</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#methods-of-sterilization"><i class="fa fa-check"></i><b>11.1.1</b> Methods of Sterilization</a></li>
<li class="chapter" data-level="11.1.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#steam"><i class="fa fa-check"></i><b>11.1.2</b> Steam</a></li>
<li class="chapter" data-level="11.1.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#dry-heat"><i class="fa fa-check"></i><b>11.1.3</b> Dry Heat</a></li>
<li class="chapter" data-level="11.1.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#flaming"><i class="fa fa-check"></i><b>11.1.4</b> Flaming</a></li>
<li class="chapter" data-level="11.1.5" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#incineration"><i class="fa fa-check"></i><b>11.1.5</b> Incineration</a></li>
<li class="chapter" data-level="11.1.6" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#tyndallization"><i class="fa fa-check"></i><b>11.1.6</b> Tyndallization</a></li>
<li class="chapter" data-level="11.1.7" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#chemical-sterilization"><i class="fa fa-check"></i><b>11.1.7</b> Chemical Sterilization</a></li>
<li class="chapter" data-level="11.1.8" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#ethylene-oxide"><i class="fa fa-check"></i><b>11.1.8</b> Ethylene oxide</a></li>
<li class="chapter" data-level="11.1.9" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#nitrogen-dioxide"><i class="fa fa-check"></i><b>11.1.9</b> Nitrogen Dioxide</a></li>
<li class="chapter" data-level="11.1.10" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#ozone"><i class="fa fa-check"></i><b>11.1.10</b> Ozone</a></li>
<li class="chapter" data-level="11.1.11" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#glutaraldehyde-and-formaldehyde"><i class="fa fa-check"></i><b>11.1.11</b> Glutaraldehyde And Formaldehyde</a></li>
<li class="chapter" data-level="11.1.12" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#hydrogen-peroxide"><i class="fa fa-check"></i><b>11.1.12</b> Hydrogen Peroxide</a></li>
<li class="chapter" data-level="11.1.13" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#peracetic-acid"><i class="fa fa-check"></i><b>11.1.13</b> Peracetic Acid</a></li>
<li class="chapter" data-level="11.1.14" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#potential-for-chemical-sterilization-of-prions"><i class="fa fa-check"></i><b>11.1.14</b> Potential For Chemical Sterilization Of Prions</a></li>
<li class="chapter" data-level="11.1.15" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#radiation-sterilization"><i class="fa fa-check"></i><b>11.1.15</b> Radiation Sterilization</a></li>
<li class="chapter" data-level="11.1.16" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#non-ionizing-radiation-sterilization"><i class="fa fa-check"></i><b>11.1.16</b> Non-Ionizing Radiation Sterilization</a></li>
<li class="chapter" data-level="11.1.17" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#ionizing-radiation-sterilization"><i class="fa fa-check"></i><b>11.1.17</b> Ionizing Radiation Sterilization</a></li>
<li class="chapter" data-level="11.1.18" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#sterile-filtration"><i class="fa fa-check"></i><b>11.1.18</b> Sterile filtration</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#pasteurization"><i class="fa fa-check"></i><b>11.2</b> Pasteurization</a></li>
<li class="chapter" data-level="11.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#disinfectants"><i class="fa fa-check"></i><b>11.3</b> Disinfectants</a>
<ul>
<li class="chapter" data-level="11.3.1" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#types-of-disinfectants"><i class="fa fa-check"></i><b>11.3.1</b> Types Of Disinfectants</a></li>
<li class="chapter" data-level="11.3.2" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#air-disinfectants"><i class="fa fa-check"></i><b>11.3.2</b> Air Disinfectants</a></li>
<li class="chapter" data-level="11.3.3" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#alcohols"><i class="fa fa-check"></i><b>11.3.3</b> Alcohols</a></li>
<li class="chapter" data-level="11.3.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#aldehydes"><i class="fa fa-check"></i><b>11.3.4</b> Aldehydes</a></li>
<li class="chapter" data-level="11.3.5" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#oxidizing-agents"><i class="fa fa-check"></i><b>11.3.5</b> Oxidizing agents</a></li>
<li class="chapter" data-level="11.3.6" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#peroxy-and-peroxo-acids"><i class="fa fa-check"></i><b>11.3.6</b> Peroxy And Peroxo Acids</a></li>
<li class="chapter" data-level="11.3.7" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#phenolics"><i class="fa fa-check"></i><b>11.3.7</b> Phenolics</a></li>
<li class="chapter" data-level="11.3.8" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#quaternary-ammonium-compounds"><i class="fa fa-check"></i><b>11.3.8</b> Quaternary ammonium compounds</a></li>
<li class="chapter" data-level="11.3.9" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#inorganic-compounds"><i class="fa fa-check"></i><b>11.3.9</b> Inorganic compounds</a></li>
<li class="chapter" data-level="11.3.10" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#chlorine"><i class="fa fa-check"></i><b>11.3.10</b> Chlorine</a></li>
<li class="chapter" data-level="11.3.11" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#iodine"><i class="fa fa-check"></i><b>11.3.11</b> Iodine</a></li>
<li class="chapter" data-level="11.3.12" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#acids-and-bases-1"><i class="fa fa-check"></i><b>11.3.12</b> Acids And Bases</a></li>
<li class="chapter" data-level="11.3.13" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#metals"><i class="fa fa-check"></i><b>11.3.13</b> Metals</a></li>
<li class="chapter" data-level="11.3.14" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#terpenes"><i class="fa fa-check"></i><b>11.3.14</b> Terpenes</a></li>
<li class="chapter" data-level="11.3.15" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#other"><i class="fa fa-check"></i><b>11.3.15</b> Other</a></li>
<li class="chapter" data-level="11.3.16" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#non-chemical"><i class="fa fa-check"></i><b>11.3.16</b> Non-chemical</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="physical-and-chemical-agents-for-microbial-control.html"><a href="physical-and-chemical-agents-for-microbial-control.html#antiseptics"><i class="fa fa-check"></i><b>11.4</b> Antiseptics</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html"><i class="fa fa-check"></i><b>12</b> Drugs, Microbes, And The Host</a>
<ul>
<li class="chapter" data-level="12.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antibacterials"><i class="fa fa-check"></i><b>12.1</b> Antibacterials</a></li>
<li class="chapter" data-level="12.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antifungals"><i class="fa fa-check"></i><b>12.2</b> Antifungals</a></li>
<li class="chapter" data-level="12.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antivirals"><i class="fa fa-check"></i><b>12.3</b> Antivirals</a></li>
<li class="chapter" data-level="12.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antiparasitics"><i class="fa fa-check"></i><b>12.4</b> Antiparasitics</a></li>
<li class="chapter" data-level="12.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#broad-spectrum-therapeutics"><i class="fa fa-check"></i><b>12.5</b> Broad-Spectrum Therapeutics</a></li>
<li class="chapter" data-level="12.6" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antibiotics"><i class="fa fa-check"></i><b>12.6</b> Antibiotics</a>
<ul>
<li class="chapter" data-level="12.6.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#classes-of-antibiotics"><i class="fa fa-check"></i><b>12.6.1</b> Classes of Antibiotics</a></li>
<li class="chapter" data-level="12.6.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#penicillin-and-other-natural-antibiotics"><i class="fa fa-check"></i><b>12.6.2</b> Penicillin And Other Natural Antibiotics</a></li>
<li class="chapter" data-level="12.6.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antistaphylococcal-antibiotics"><i class="fa fa-check"></i><b>12.6.3</b> Antistaphylococcal antibiotics</a></li>
<li class="chapter" data-level="12.6.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#broad-spectrum-antibiotics"><i class="fa fa-check"></i><b>12.6.4</b> Broad-spectrum antibiotics</a></li>
<li class="chapter" data-level="12.6.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antipseudomonal-antibiotics"><i class="fa fa-check"></i><b>12.6.5</b> Antipseudomonal antibiotics</a></li>
<li class="chapter" data-level="12.6.6" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#chemical-structure-of-penicillins"><i class="fa fa-check"></i><b>12.6.6</b> Chemical Structure Of Penicillins</a></li>
<li class="chapter" data-level="12.6.7" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#mechanism-of-action-of-penicillin"><i class="fa fa-check"></i><b>12.6.7</b> Mechanism Of Action Of Penicillin</a></li>
<li class="chapter" data-level="12.6.8" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#aminoglycoside-antibiotics"><i class="fa fa-check"></i><b>12.6.8</b> Aminoglycoside Antibiotics</a></li>
<li class="chapter" data-level="12.6.9" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#mechanism-of-action-of-amimoglycoside-antibiotics"><i class="fa fa-check"></i><b>12.6.9</b> Mechanism Of Action Of Amimoglycoside Antibiotics</a></li>
</ul></li>
<li class="chapter" data-level="12.7" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#drug-resistance"><i class="fa fa-check"></i><b>12.7</b> Drug Resistance</a>
<ul>
<li class="chapter" data-level="12.7.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#bacterial-resistence-to-penicillin"><i class="fa fa-check"></i><b>12.7.1</b> Bacterial Resistence To Penicillin</a></li>
</ul></li>
<li class="chapter" data-level="12.8" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#replenishing-the-antibiotic-pipeline-and-developing-other-new-therapies"><i class="fa fa-check"></i><b>12.8</b> Replenishing The Antibiotic Pipeline And Developing Other New Therapies</a>
<ul>
<li class="chapter" data-level="12.8.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#natural-product-based-antibiotic-discovery"><i class="fa fa-check"></i><b>12.8.1</b> Natural Product-Based Antibiotic Discovery</a></li>
<li class="chapter" data-level="12.8.2" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#immunoglobulin-therapy"><i class="fa fa-check"></i><b>12.8.2</b> Immunoglobulin Therapy</a></li>
<li class="chapter" data-level="12.8.3" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#phage-therapy"><i class="fa fa-check"></i><b>12.8.3</b> Phage Therapy</a></li>
<li class="chapter" data-level="12.8.4" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#fecal-microbiota-transplants"><i class="fa fa-check"></i><b>12.8.4</b> Fecal Microbiota Transplants</a></li>
<li class="chapter" data-level="12.8.5" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#antisense-rna-based-treatments"><i class="fa fa-check"></i><b>12.8.5</b> Antisense RNA-Based Treatments</a></li>
<li class="chapter" data-level="12.8.6" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#crispr-cas9-based-treatments"><i class="fa fa-check"></i><b>12.8.6</b> CRISPR-Cas9-Based Treatments</a></li>
</ul></li>
<li class="chapter" data-level="12.9" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#testing-for-antibiotic-sensitivity"><i class="fa fa-check"></i><b>12.9</b> Testing For Antibiotic Sensitivity</a>
<ul>
<li class="chapter" data-level="12.9.1" data-path="drugs-microbes-and-the-host.html"><a href="drugs-microbes-and-the-host.html#the-disk-diffusion-test"><i class="fa fa-check"></i><b>12.9.1</b> The Disk Diffusion Test</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="13" data-path="infection-and-disease.html"><a href="infection-and-disease.html"><i class="fa fa-check"></i><b>13</b> Infection And Disease</a>
<ul>
<li class="chapter" data-level="13.1" data-path="infection-and-disease.html"><a href="infection-and-disease.html#pathogenic-bacteria"><i class="fa fa-check"></i><b>13.1</b> Pathogenic Bacteria</a></li>
<li class="chapter" data-level="13.2" data-path="infection-and-disease.html"><a href="infection-and-disease.html#pathogenic-susceptibility"><i class="fa fa-check"></i><b>13.2</b> Pathogenic Susceptibility</a>
<ul>
<li class="chapter" data-level="13.2.1" data-path="infection-and-disease.html"><a href="infection-and-disease.html#list-of-genera-and-microscopy-features"><i class="fa fa-check"></i><b>13.2.1</b> List of genera and microscopy features</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="infection-and-disease.html"><a href="infection-and-disease.html#infection"><i class="fa fa-check"></i><b>13.3</b> Infection</a></li>
<li class="chapter" data-level="13.4" data-path="infection-and-disease.html"><a href="infection-and-disease.html#disease"><i class="fa fa-check"></i><b>13.4</b> Disease</a></li>
<li class="chapter" data-level="13.5" data-path="infection-and-disease.html"><a href="infection-and-disease.html#epidemiology"><i class="fa fa-check"></i><b>13.5</b> Epidemiology</a></li>
<li class="chapter" data-level="13.6" data-path="infection-and-disease.html"><a href="infection-and-disease.html#pandemics"><i class="fa fa-check"></i><b>13.6</b> Pandemics</a>
<ul>
<li class="chapter" data-level="13.6.1" data-path="infection-and-disease.html"><a href="infection-and-disease.html#covid-19-pandemic"><i class="fa fa-check"></i><b>13.6.1</b> COVID-19 Pandemic</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html"><i class="fa fa-check"></i><b>14</b> An Introduction To Host Defenses And Innate Immune Systems</a>
<ul>
<li class="chapter" data-level="14.1" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#layered-defense"><i class="fa fa-check"></i><b>14.1</b> Layered defense</a></li>
<li class="chapter" data-level="14.2" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#surface-barriers"><i class="fa fa-check"></i><b>14.2</b> Surface Barriers</a></li>
<li class="chapter" data-level="14.3" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#innate-immune-system"><i class="fa fa-check"></i><b>14.3</b> Innate immune system</a></li>
<li class="chapter" data-level="14.4" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#anatomical-barriers"><i class="fa fa-check"></i><b>14.4</b> Anatomical barriers</a></li>
<li class="chapter" data-level="14.5" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#inflammation"><i class="fa fa-check"></i><b>14.5</b> Inflammation</a></li>
<li class="chapter" data-level="14.6" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#complement-system"><i class="fa fa-check"></i><b>14.6</b> Complement system</a></li>
<li class="chapter" data-level="14.7" data-path="an-introduction-to-host-defenses-and-innate-immune-systems.html"><a href="an-introduction-to-host-defenses-and-innate-immune-systems.html#white-blood-cells"><i class="fa fa-check"></i><b>14.7</b> White blood cells</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html"><i class="fa fa-check"></i><b>15</b> The Adaptive Immune System And Immunization</a>
<ul>
<li class="chapter" data-level="15.1" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#lymphocytes"><i class="fa fa-check"></i><b>15.1</b> Lymphocytes</a></li>
<li class="chapter" data-level="15.2" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#antigen-presentation"><i class="fa fa-check"></i><b>15.2</b> Antigen presentation</a></li>
<li class="chapter" data-level="15.3" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#exogenous-antigens"><i class="fa fa-check"></i><b>15.3</b> Exogenous antigens</a></li>
<li class="chapter" data-level="15.4" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#endogenous-antigens"><i class="fa fa-check"></i><b>15.4</b> Endogenous antigens</a></li>
<li class="chapter" data-level="15.5" data-path="the-adaptive-immune-system-and-immunization.html"><a href="the-adaptive-immune-system-and-immunization.html#t-lymphocytes"><i class="fa fa-check"></i><b>15.5</b> T lymphocytes</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For SCI204 Microbiology at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="the-main-themes-of-microbiology" class="section level1" number="1">
<h1><span class="header-section-number">1</span> The Main Themes Of Microbiology</h1>
<p><a href="https://en.wikipedia.org/wiki/Microbiology">Microbiology</a> (from Greek μῑκρος, mīkros, “small”; βίος, bios, “life”; and -λογία, -logia) is the scientific study of microorganisms, those being unicellular (single cell), multicellular (cell colony), or acellular (lacking cells). A unicellular organism, also known as a single-celled organism, is an organism that consists of a single cell, unlike a multicellular organism that consists of multiple cells. Unicellular organisms fall into two general categories: prokaryotic organisms and eukaryotic organisms. All prokaryotes are unicellular and are classified into bacteria and archaea. Many eukaryotes are multicellular, but many are unicellular such as protozoa, unicellular algae, and unicellular fungi. Unicellular organisms are thought to be the oldest form of life, with early protocells possibly emerging 3.8–4.0 billion years ago. Multicellular organisms are organisms that consist of more than one cell, in contrast to unicellular organisms. All species of animals, land plants and most fungi are multicellular, as are many algae, whereas a few organisms are partially uni- and partially multicellular, like slime molds and social amoebae such as the genus Dictyostelium.</p>
<p>Microbiology encompasses numerous sub-disciplines including <a href="https://en.wikipedia.org/wiki/Virology">virology</a>, <a href="https://en.wikipedia.org/wiki/Virology">bacteriology</a>, <a href="https://en.wikipedia.org/wiki/Virology">protistology</a>, <a href="https://en.wikipedia.org/wiki/Virology">mycology</a>, <a href="https://en.wikipedia.org/wiki/Virology">immunology</a> and <a href="https://en.wikipedia.org/wiki/Virology">parasitology</a>.</p>
<div id="general-characteristics-and-evolution-of-microorganisms" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> General Characteristics And Evolution Of Microorganisms</h2>
<p>A microorganism, or microbe,[a] is a microscopic organism, which may exist in its single-celled form or a colony of cells.</p>
<p><a href="https://en.wikipedia.org/wiki/Virus">Viruses</a> have been variably classified as organisms, as they have been considered either as very simple microorganisms or very complex molecules. <a href="https://en.wikipedia.org/wiki/Prion">Prions</a>, never considered as microorganisms, have been investigated by virologists, however, as the clinical effects traced to them were originally presumed due to chronic viral infections, and virologists took search—discovering “infectious proteins”.</p>
<p>The possible existence of unseen microbial life was suspected from ancient times, such as in Jain scriptures from sixth century BC India. The scientific study of microorganisms began with their observation under the microscope in the 1670s by Antonie van Leeuwenhoek. In the 1850s, Louis Pasteur found that microorganisms caused food spoilage, debunking the theory of spontaneous generation. In the 1880s, Robert Koch discovered that microorganisms caused the diseases tuberculosis, cholera, diphtheria, and anthrax.</p>
<p>Microorganisms include all unicellular organisms and so are extremely diverse. Viruses are generally regarded as not living and therefore not considered as microorganisms, although a subfield of microbiology is virology, the study of viruses. Of the three domains of life identified by Carl Woese, all of the Archaea and Bacteria are microorganisms. These were previously grouped in the two domain system as Prokaryotes, the other being the eukaryotes. The third domain Eukaryota includes all multicellular organisms and many unicellular protists and protozoans. Some protists are related to animals and some to green plants. Many of the multicellular organisms are microscopic, namely micro-animals, some fungi, and some algae.</p>
<p>They live in almost every habitat from the poles to the equator, deserts, geysers, rocks, and the deep sea. Some are adapted to extremes such as very hot or very cold conditions, others to high pressure, and a few, such as Deinococcus radiodurans, to high radiation environments. Microorganisms also make up the microbiota found in and on all multicellular organisms. There is evidence that 3.45-billion-year-old Australian rocks once contained microorganisms, the earliest direct evidence of life on Earth.</p>
<p>Microbes are important in human culture and health in many ways, serving to ferment foods and treat sewage, and to produce fuel, enzymes, and other bioactive compounds. Microbes are essential tools in biology as model organisms and have been put to use in biological warfare and bioterrorism. Microbes are a vital component of fertile soil. In the human body, microorganisms make up the human microbiota, including the essential gut flora. The pathogens responsible for many infectious diseases are microbes and, as such, are the target of hygiene measures.</p>
<p>Single-celled microorganisms were the first forms of life to develop on Earth, approximately 3.5 billion years ago. Further evolution was slow, and for about 3 billion years in the Precambrian eon, (much of the history of life on Earth), all organisms were microorganisms. Bacteria, algae and fungi have been identified in amber that is 220 million years old, which shows that the morphology of microorganisms has changed little since at least the Triassic period. The newly discovered biological role played by nickel, however – especially that brought about by volcanic eruptions from the Siberian Traps – may have accelerated the evolution of methanogens towards the end of the Permian–Triassic extinction event.</p>
<p>Microorganisms tend to have a relatively fast rate of evolution. Most microorganisms can reproduce rapidly, and bacteria are also able to freely exchange genes through conjugation, transformation and transduction, even between widely divergent species. This horizontal gene transfer, coupled with a high mutation rate and other means of transformation, allows microorganisms to swiftly evolve (via natural selection) to survive in new environments and respond to environmental stresses. This rapid evolution is important in medicine, as it has led to the development of multidrug resistant pathogenic bacteria, superbugs, that are resistant to antibiotics.</p>
<p>A possible transitional form of microorganism between a prokaryote and a eukaryote was discovered in 2012 by Japanese scientists. Parakaryon myojinensis is a unique microorganism larger than a typical prokaryote, but with nuclear material enclosed in a membrane as in a eukaryote, and the presence of endosymbionts. This is seen to be the first plausible evolutionary form of microorganism, showing a stage of development from the prokaryote to the eukaryote.</p>
<div id="archaea" class="section level3" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> Archaea</h3>
<p>Archaea are prokaryotic unicellular organisms, and form the first domain of life, in Carl Woese’s three-domain system. A prokaryote is defined as having no cell nucleus or other membrane bound-organelle. Archaea share this defining feature with the bacteria with which they were once grouped. In 1990 the microbiologist Woese proposed the three-domain system that divided living things into bacteria, archaea and eukaryotes, and thereby split the prokaryote domain.</p>
<p>Archaea differ from bacteria in both their genetics and biochemistry. For example, while bacterial cell membranes are made from phosphoglycerides with ester bonds, archaean membranes are made of ether lipids. Archaea were originally described as extremophiles living in extreme environments, such as hot springs, but have since been found in all types of habitats. Only now are scientists beginning to realize how common archaea are in the environment, with Crenarchaeota being the most common form of life in the ocean, dominating ecosystems below 150 m in depth. These organisms are also common in soil and play a vital role in ammonia oxidation.</p>
<p>The combined domains of archaea and bacteria make up the most diverse and abundant group of organisms on Earth and inhabit practically all environments where the temperature is below +140 °C. They are found in water, soil, air, as the microbiome of an organism, hot springs and even deep beneath the Earth’s crust in rocks. The number of prokaryotes is estimated to be around five nonillion, or 5 × 1030, accounting for at least half the biomass on Earth.</p>
<p>The biodiversity of the prokaryotes is unknown, but may be very large. A May 2016 estimate, based on laws of scaling from known numbers of species against the size of organism, gives an estimate of perhaps 1 trillion species on the planet, of which most would be microorganisms. Currently, only one-thousandth of one percent of that total have been described. Archael cells of some species aggregate and transfer DNA from one cell to another through direct contact, particularly under stressful environmental conditions that cause DNA damage.</p>
</div>
<div id="bacteria" class="section level3" number="1.1.2">
<h3><span class="header-section-number">1.1.2</span> Bacteria</h3>
<p>Bacteria like archaea are prokaryotic – unicellular, and having no cell nucleus or other membrane-bound organelle. Bacteria are microscopic, with a few extremely rare exceptions, such as Thiomargarita namibiensis. Bacteria function and reproduce as individual cells, but they can often aggregate in multicellular colonies. Some species such as myxobacteria can aggregate into complex swarming structures, operating as multicellular groups as part of their life cycle, or form clusters in bacterial colonies such as E.coli.</p>
<p>Their genome is usually a circular bacterial chromosome – a single loop of DNA, although they can also harbor small pieces of DNA called plasmids. These plasmids can be transferred between cells through bacterial conjugation. Bacteria have an enclosing cell wall, which provides strength and rigidity to their cells. They reproduce by binary fission or sometimes by budding, but do not undergo meiotic sexual reproduction. However, many bacterial species can transfer DNA between individual cells by a horizontal gene transfer process referred to as natural transformation. Some species form extraordinarily resilient spores, but for bacteria this is a mechanism for survival, not reproduction. Under optimal conditions bacteria can grow extremely rapidly and their numbers can double as quickly as every 20 minutes.</p>
</div>
<div id="eukaryotes" class="section level3" number="1.1.3">
<h3><span class="header-section-number">1.1.3</span> Eukaryotes</h3>
<p>Most living things that are visible to the naked eye in their adult form are eukaryotes, including humans. However, many eukaryotes are also microorganisms. Unlike bacteria and archaea, eukaryotes contain organelles such as the cell nucleus, the Golgi apparatus and mitochondria in their cells. The nucleus is an organelle that houses the DNA that makes up a cell’s genome. DNA (Deoxyribonucleic acid) itself is arranged in complex chromosomes. Mitochondria are organelles vital in metabolism as they are the site of the citric acid cycle and oxidative phosphorylation. They evolved from symbiotic bacteria and retain a remnant genome. Like bacteria, plant cells have cell walls, and contain organelles such as chloroplasts in addition to the organelles in other eukaryotes. Chloroplasts produce energy from light by photosynthesis, and were also originally symbiotic bacteria.</p>
<p>Unicellular eukaryotes consist of a single cell throughout their life cycle. This qualification is significant since most multicellular eukaryotes consist of a single cell called a zygote only at the beginning of their life cycles. Microbial eukaryotes can be either haploid or diploid, and some organisms have multiple cell nuclei.</p>
<p>Unicellular eukaryotes usually reproduce asexually by mitosis under favorable conditions. However, under stressful conditions such as nutrient limitations and other conditions associated with DNA damage, they tend to reproduce sexually by meiosis and syngamy.</p>
</div>
<div id="protists" class="section level3" number="1.1.4">
<h3><span class="header-section-number">1.1.4</span> Protists</h3>
<p>Of eukaryotic groups, the protists are most commonly unicellular and microscopic. This is a highly diverse group of organisms that are not easy to classify. Several algae species are multicellular protists, and slime molds have unique life cycles that involve switching between unicellular, colonial, and multicellular forms. The number of species of protists is unknown since only a small proportion has been identified. Protist diversity is high in oceans, deep sea-vents, river sediment and an acidic river, suggesting that many eukaryotic microbial communities may yet be discovered.</p>
</div>
<div id="fungi" class="section level3" number="1.1.5">
<h3><span class="header-section-number">1.1.5</span> Fungi</h3>
<p>The fungi have several unicellular species, such as baker’s yeast (Saccharomyces cerevisiae) and fission yeast (Schizosaccharomyces pombe). Some fungi, such as the pathogenic yeast Candida albicans, can undergo phenotypic switching and grow as single cells in some environments, and filamentous hyphae in others.</p>
</div>
<div id="plants" class="section level3" number="1.1.6">
<h3><span class="header-section-number">1.1.6</span> Plants</h3>
<p>The green algae are a large group of photosynthetic eukaryotes that include many microscopic organisms. Although some green algae are classified as protists, others such as charophyta are classified with embryophyte plants, which are the most familiar group of land plants. Algae can grow as single cells, or in long chains of cells. The green algae include unicellular and colonial flagellates, usually but not always with two flagella per cell, as well as various colonial, coccoid, and filamentous forms. In the Charales, which are the algae most closely related to higher plants, cells differentiate into several distinct tissues within the organism. There are about 6000 species of green algae.</p>
</div>
</div>
<div id="ecology-of-microorganisms" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Ecology Of Microorganisms</h2>
<p>Microorganisms are found in almost every habitat present in nature, including hostile environments such as the North and South poles, deserts, geysers, and rocks. They also include all the marine microorganisms of the oceans and deep sea. Some types of microorganisms have adapted to extreme environments and sustained colonies; these organisms are known as extremophiles. Extremophiles have been isolated from rocks as much as 7 kilometres below the Earth’s surface, and it has been suggested that the amount of organisms living below the Earth’s surface is comparable with the amount of life on or above the surface. Extremophiles have been known to survive for a prolonged time in a vacuum, and can be highly resistant to radiation, which may even allow them to survive in space. Many types of microorganisms have intimate symbiotic relationships with other larger organisms; some of which are mutually beneficial (mutualism), while others can be damaging to the host organism (parasitism). If microorganisms can cause disease in a host they are known as pathogens and then they are sometimes referred to as microbes. Microorganisms play critical roles in Earth’s biogeochemical cycles as they are responsible for decomposition and nitrogen fixation.</p>
<p>Bacteria use regulatory networks that allow them to adapt to almost every environmental niche on earth. A network of interactions among diverse types of molecules including DNA, RNA, proteins and metabolites, is utilised by the bacteria to achieve regulation of gene expression. In bacteria, the principal function of regulatory networks is to control the response to environmental changes, for example nutritional status and environmental stress. A complex organization of networks permits the microorganism to coordinate and integrate multiple environmental signals.</p>
<div id="microbiome" class="section level3" number="1.2.1">
<h3><span class="header-section-number">1.2.1</span> Microbiome</h3>
<p>The word microbiome (from the Greek micro meaning “small” and bíos meaning “life”) was first used by J.L. Mohr in 1952 in The Scientific Monthly to mean the microorganisms found in a specific environment. It was defined in 1988 by Whipps et al. as “a characteristic microbial community occupying a reasonably well-defined habitat which has distinct physio-chemical properties. The term thus not only refers to the microorganisms involved but also encompasses their theatre of activity”.</p>
<p>In 2020, an international panel of experts published the outcome of their discussions on the definition of the microbiome. They proposed a definition of the microbiome based on a revival of the “compact, clear, and comprehensive description of the term” as originally provided by Whipps et al., but supplemented with two explanatory sentences.</p>
<p>The first explanatory sentence pronounces the dynamic character of the microbiome:</p>
<p>The microbiome is defined as a characteristic microbial community occupying a reasonably well-defined habitat which has distinct physio-chemical properties. The microbiome not only refers to the microorganisms involved but also encompass their theatre of activity, which results in the formation of specific ecological niches. The microbiome, which forms a dynamic and interactive micro-ecosystem prone to change in time and scale, is integrated in macro-ecosystems including eukaryotic hosts, and here crucial for their functioning and health.
The second explanatory sentence clearly separates the term microbiota from the term microbiome:</p>
<p>The microbiota consists of the assembly of microorganisms belonging to different kingdoms (Prokaryotes [Bacteria, Archaea], Eukaryotes [e.g., Protozoa, Fungi, and Algae]), while their theatre of activity includes microbial structures, metabolites, mobile genetic elements (such as transposons, phages, and viruses), and relic DNA embedded in the environmental conditions of the habitat.
Secondary metabolites play an essential role in mediating complex interspecies interactions and ensure survival in competitive environments. Quorum sensing induced by small molecules allows bacteria to control cooperative activities and adapts their phenotypes to the biotic environment, resulting, e.g., in cell-cell adhesion or biofilm formation. Direct interspecies electron transfer (DIET) is an important mechanism for communication in most anaerobic ecosystems. In addition, volatile compounds can act as long-term messengers for cross-kingdom communication over long distances.</p>
<p>Microbiome research originated in microbiology and started back in the seventeenth century. The development of new techniques and equipment has boosted microbiological research and caused paradigm shifts in understanding health and disease. Since infectious diseases have affected human populations throughout most of history, medical microbiology was the earliest focus of research and public interest. Additionally, food microbiology is an old field of empirical applications. The development of the first microscopes allowed the discovery of a new, unknown world and led to the identification of microorganisms.</p>
<p>Access to the previously invisible world opened the eyes and the minds of the researchers of the seventeenth century. Antonie van Leeuwenhoek investigated diverse bacteria of various shapes, fungi, and protozoa, which he called animalcules, mainly from water, mud, and dental plaque samples, and discovered biofilms as a first indication of microorganisms interacting within complex communities. Robert Koch’s explanation of the origin of human and animal diseases as a consequence of microbial infection and development of the concept of pathogenicity was an important milestone in microbiology. These findings shifted the focus of the research community and the public on the role of microorganisms as disease-forming agents that needed to be eliminated.</p>
<p>However, comprehensive research over the past century has shown only a small proportion of microorganisms are associated with disease or pathogenicity. The overwhelming majority of microbes are essential for ecosystem functioning and known for beneficial interactions with other microbes as well as macroorganisms. At the end of the nineteenth century, microbial ecology started with the pioneering work by Martinus W. Beijerinck and Sergei Winogradsky. The newly established science of environmental microbiology resulted in another paradigm shift: microorganisms are everywhere in natural environments, often associated with hosts and, for the first time, beneficial effects on their hosts were reported.</p>
<p>Subsequently, the concept that microorganisms exist as single cells began to change as it became increasingly obvious that microbes occur within complex assemblages in which species interactions and communication are critical to population dynamics and functional activities. Discovery of DNA, the development of sequencing technologies, PCR, and cloning techniques enabled the investigation of microbial communities using cultivation-independent, DNA and RNA-based approaches.</p>
<p>A further important step was the introduction of phylogenetic markers such as the 16S rRNA gene for microbial community analysis by Carl Woese and George E. Fox in 1977. Today, we are able to barcode bacteria, archaea, fungi, algae, and protists in their natural habitats, e.g., by targeting their 16S and 18S rRNA genes, internal transcribed spacer (ITS), or, alternatively, specific functional regions of genes coding for specific enzymes.</p>
<p>Another major paradigm shift was initiated at the beginning of this century and continues through today, as new sequencing technologies and accumulated sequence data have highlighted both the ubiquity of microbial communities in association within higher organisms and the critical roles of microbes in human, animal, and plant health. These new possibilities have revolutionized microbial ecology, because the analysis of genomes and metagenomes in a high-throughput manner provides efficient methods for addressing the functional potential of individual microorganisms as well as of whole communities in their natural habitats. Multiomics technologies including metatranscriptome, metaproteome and metabolome approaches now provide detailed information on microbial activities in the environment. Based on the rich foundation of data, the cultivation of microbes, which was often ignored or underestimated over the last thirty years, has gained new importance, and high throughput culturomics is now an important part of the toolbox to study microbiomes. The high potential and power of combining multiple “omics” techniques to analyze host-microbe interactions are highlighted in several reviews.</p>
<p>Microbial communities have commonly been defined as the collection of microorganisms living together. More specifically, microbial communities are defined as multi-species assemblages, in which (micro) organisms interact with each other in a contiguous environment. In 1988, Whipps and colleagues working on the ecology of rhizosphere microorganisms provided the first definition of the term microbiome. They described the microbiome as a combination of the words micro and biome, naming a “characteristic microbial community” in a “reasonably well-defined habitat which has distinct physio-chemical properties” as their “theatre of activity”. This definition represents a substantial advancement of the definition of a microbial community, as it defines a microbial community with distinct properties and functions and its interactions with its environment, resulting in the formation of specific ecological niches.</p>
<p>However, many other microbiome definitions have been published in the last few decades. The currently most cited definition by Lederberg describes microbiomes within an ecological context, as a community of commensal, symbiotic, and pathogenic microorganisms within a body space or other environment. Marchesi and Ravel focused in their definition on the genomes and microbial (and viral) gene expression patterns and proteomes in a given environment and its prevailing biotic and abiotic conditions. All these definitions imply that general concepts of macro-ecology could be easily applied to microbe-microbe as well as to microbe-host interactions. However, the extent to which these concepts, developed for macro-eukaryotes, can be applied to prokaryotes with their different lifestyles regarding dormancy, variation of phenotype, and horizontal gene transfer as well as to micro-eukaryotes that is not quite clear. This raises the challenge of considering an entirely novel body of conceptual ecology models and theory for microbiome ecology, particularly in relation to the diverse hierarchies of interactions of microbes with one another and with the host biotic and abiotic environments.</p>
<p>In 2020, a panel of international experts, organised by the EU-funded MicrobiomeSupport project, published the results of their deliberations on the definition of the microbiome. The panel was composed of about 40 leaders from diverse microbiome areas, and about one hundred further experts from around the world contributed through an online survey. They proposed a definition of the microbiome based on a revival of the compact, clear, and comprehensive description of the term as originally provided by Whipps et al. in 1988, amended with a set of recommendations considering subsequent technological developments and research findings. They clearly separate the terms microbiome and microbiota and provide a comprehensive discussion considering the composition of microbiota, the heterogeneity and dynamics of microbiomes in time and space, the stability and resilience of microbial networks, the definition of core microbiomes, and functionally relevant keystone species as well as co-evolutionary principles of microbe-host and inter-species interactions within the microbiome.</p>
<p>The panel extended the Whipps et al. definition, which contains all important points that are valid even 30 years after its publication in 1988, by two explanatory sentences differentiating the terms microbiome and microbiota and pronouncing its dynamic character, as follows:</p>
<ul>
<li>The microbiome is defined as a characteristic microbial community occupying a reasonable well-defined habitat which has distinct physio-chemical properties. The microbiome not only refers to the microorganisms involved but also encompass their theatre of activity, which results in the formation of specific ecological niches. The microbiome, which forms a dynamic and interactive micro-ecosystem prone to change in time and scale, is integrated in macro-ecosystems including eukaryotic hosts, and here crucial for their functioning and health.</li>
<li>The microbiota consists of the assembly of microorganisms belonging to different kingdoms (prokaryotes (bacteria, archaea), eukaryotes (algae, protozoa, fungi etc), while “their theatre of activity” includes microbial structures, metabolites, mobile genetic elements (such as transposons, phages, and viruses), and relic DNA embedded in the environmental conditions of the habitat.</li>
</ul>
</div>
<div id="microbiota-members-of-the-microbiome" class="section level3" number="1.2.2">
<h3><span class="header-section-number">1.2.2</span> Microbiota – members of the microbiome</h3>
<p>The microbiota comprises all living members forming the microbiome. Most microbiome researchers agree bacteria, archaea, fungi, algae, and small protists should be considered as members of the microbiome. The integration of phages, viruses, plasmids, and mobile genetic elements is a more controversial issue in the definition of the microbiome. There is also no clear consensus as to whether extracellular DNA derived from dead cells, so-called “relic DNA”, belongs to the microbiome. Relic DNA can be up to 40% of the sequenced DNA in soil, and was up to 33% of the total bacterial DNA on average in a broader analysis of habitats with the highest proportion of 80% in some samples. Despite its omnipresence and abundance, relic DNA had a minimal effect on estimates of taxonomic and phylogenetic diversity.</p>
<p>When it comes to the use of specific terms, a clear differentiation between microbiome and microbiota helps to avoid the controversy concerning the members of a microbiome. Microbiota is usually defined as the assemblage of living microorganisms present in a defined environment. As phages, viruses, plasmids, prions, viroids, and free DNA are usually not considered as living microorganisms, they do not belong to the microbiota.</p>
<p>The term microbiome, as it was originally postulated by Whipps and coworkers, includes not only the community of the microorganisms but also their “theatre of activity”. The latter involves the whole spectrum of molecules produced by the microorganisms, including their structural elements (nucleic acids, proteins, lipids, polysaccharides), metabolites (signalling molecules, toxins, organic, and inorganic molecules), and molecules produced by coexisting hosts and structured by the surrounding environmental conditions. Therefore, all mobile genetic elements, such as phages, viruses, and “relic” and extracellular DNA, should be included in the term microbiome, but are not a part of microbiota. The term microbiome is also sometimes confused with the metagenome. Metagenome is, however, clearly defined as a collection of genomes and genes from the members of a microbiota.</p>
<p>Microbiome studies sometimes focus on the behaviour of a specific group of microbiota, generally in relation to or justified by a clear hypothesis. More and more terms like bacteriome, archaeome, mycobiome, or virome have started appearing in the scientific literature, but these terms do not refer to biomes (a regional ecosystem with a distinct assemblage of (micro) organisms, and physical environment often reflecting a certain climate and soil) as the microbiome itself. Consequently, it would be better to use the original terms (bacterial, archaeal, or fungal community). In contrast to the microbiota, which can be studied separately, the microbiome is always composed by all members, which interact with each other, live in the same habitat, and form their ecological niche together. The well-established term virome is derived from virus and genome and is used to describe viral shotgun metagenomes consisting of a collection of nucleic acids associated with a particular ecosystem or holobiont. Viral metagenomes can be suggested as a semantically and scientifically better term.</p>
</div>
<div id="microbial-networks-and-interaction" class="section level3" number="1.2.3">
<h3><span class="header-section-number">1.2.3</span> Microbial Networks And Interaction</h3>
<p>Microbes interact with one another, and these symbiotic interactions have diverse consequences for microbial fitness, population dynamics, and functional capacities within the microbiome. These interactions can either be between microorganisms of the same species or between different species, genera, families, and domains of life. The interactive patterns within these webs may be positive (mutualism, synergism, or commensalism), negative (amensalism [including predation, parasitism, antagonism, or competition]), or neutral—where there is no (or no observed) effect on the functional capacities or fitness of interacting species (see diagram at right) Microbial life strategy concepts (i.e., copiotrophic and oligotrophic strategists and competitor–stress tolerator–ruderals framework) can influence outcomes of interactions. For example, microorganisms competing for the same source can also benefit from each other when competing for the same compound at different trophic levels. Stability of a complex microbial ecosystem depends on trophic interactions for the same substrate at different concentration levels. As of 2020 microbial social adaptations in nature have been understudied. Here molecular markers can provide insight into social adaptations by supporting the theories, e.g., of altruists and cheaters in native microbiomes.</p>
<p>Secondary metabolites play an essential role in mediating complex interspecies interactions and ensure survival in competitive environments. Quorum sensing induced by small molecules like n-acyl-homoserine lactones or peptides allows bacteria to control cooperative activities and adapts their phenotypes to the biotic environment, resulting, e.g., in cell-cell adhesion or biofilm formation. Direct interspecies electron transfer (DIET) is an important mechanism for communication in most anaerobic ecosystems. In addition, volatile compounds can act as long-term messengers for cross-kingdom communication over long distances. Moreover, the so-called “fungal highways” serve as transportation systems for bacteria  as well as for water and nutrients  and can therefore play an important role in structuring microbial networks. Despite these examples, communication and interaction within the microbiome remain understudied and would profit from more knowledge on the metabolic interplay of all microbiome members. Here, reductionist experimental models and model microbiomes can help to identify microbes and molecular mechanisms involved in complex interactions.</p>
<p>Currently available methods for studying microbiomes, so-called multi-omics, range from high throughput isolation (culturomics) and visualization (microscopy), to targeting the taxonomic composition (metabarcoding), or addressing the metabolic potential (metabarcoding of functional genes, metagenomics) to analyze microbial activity (metatranscriptomics, metaproteomics, metabolomics), as shown in the diagram on the right. Based on metagenome data, microbial genomes can be reconstructed. While first metagenome-assembled genomes were reconstructed from environmental samples, in recent years, several thousands of bacterial genomes were binned without culturing the organisms behind. For example, 154,723 microbial genomes of the global human microbiome were recently reconstructed from 9,428 metagenomes.</p>
<p>Computational modeling of microbiomes has been used to compliment experimental methods for investigating microbial function by utilizing multi-omic data to predict complex inter-species and host-species dynamics. A popular in silico method is to combine metabolic network models of microbial taxa present in a community and use a mathematical modeling strategy such as flux balance analysis to predict the metabolic function of the microbial community at a taxon and community-level.</p>
<p>As of 2020, understanding is limited due to the missing links between the massive availability of microbiome DNA sequence data on the one hand and limited availability of microbial isolates needed to confirm metagenomic predictions of gene function on the other hand. Metagenome data provides a playground for new predictions, yet much more data is needed to strengthen the links between sequence and rigorous functional predictions. This becomes obvious when considering that the replacement of one single amino acid residue by another may lead to a radical functional change, resulting in an incorrect functional assignment to a given gene sequence. Additionally, cultivation of new strains is needed to help identify the large fraction of unknown sequences obtained from metagenomics analyses, which for poorly studied ecosystems can be more than 70%. Depending on the applied method, even in well-studied microbiomes, 40–70% of the annotated genes in fully sequenced microbial genomes have no known or predicted function. Moreover, current estimates predict that domains with unknown functions will outnumber families of known function very soon. There is a clear need for more classical microbiology including the use of targeted mutants in combination with microbial biochemistry to cope with this challenge. Moreover, there is much more to gain from thorough functional characterization of already discovered protein families with unknown function(s) than from further extending the list of these families. Understanding prokaryotic functional diversity, as of 2019, is challenging as 85 out of the currently established 118 phyla have not had a single species described to this date.</p>
<p>The number of prokaryotic phyla may reach hundreds, and archaeal ones are among the least. The growing gap between the diversity of Bacteria and Archaea held in pure culture and those detected by molecular methods has led to the proposal to establish a formal nomenclature for not-yet cultured taxa, primarily based on sequence information. According to this proposal, the concept of Candidatus species would be extended to the groups of closely related genome sequences, and their names would be published following established rules of bacterial nomenclature.</p>
<p>In 1985 Staley and Konopka identified “the great plate count anomaly” which describes the fact that 90 to 99.9% of bacterial species cannot be grown under standard laboratory conditions. For some micro-habitats, especially those with high nutrient content and microbial activity, the proportion of representative strains available in culture relative to the molecular species detected by sequencing grew from 35 to 65%, as it was stated for the gut microbiota. Similar advances are needed for microbial populations from other natural habitats as well as for the eukaryotic members of the microbiome. Micro-eukaryotes, e.g., members of protozoa, fungi, and algae, can often be better cultivated and microscopically studied; however, their phylogeny and taxonomy are more complex and less studied. Interestingly, primer-free 16S and 18S rRNA gene sequencing from various environments has shown that among microeukaryotes there is a huge number of previously not detected taxa.</p>
</div>
<div id="the-human-microbiome" class="section level3" number="1.2.4">
<h3><span class="header-section-number">1.2.4</span> The Human Microbiome</h3>
<p>The human microbiome is the aggregate of all microbiota that reside on or within human tissues and biofluids along with the corresponding anatomical sites in which they reside, including the skin, mammary glands, seminal fluid, uterus, ovarian follicles, lung, saliva, oral mucosa, conjunctiva, biliary tract, and gastrointestinal tract. Types of human microbiota include bacteria, archaea, fungi, protists and viruses. Though micro-animals can also live on the human body, they are typically excluded from this definition. In the context of genomics, the term human microbiome is sometimes used to refer to the collective genomes of resident microorganisms; however, the term human metagenome has the same meaning.</p>
<p>Humans are colonized by many microorganisms, with approximately the same order of magnitude of non-human cells as human cells. Some microorganisms that colonize humans are commensal, meaning they co-exist without harming humans; others have a mutualistic relationship with their human hosts.:700 Conversely, some non-pathogenic microorganisms can harm human hosts via the metabolites they produce, like trimethylamine, which the human body converts to trimethylamine N-oxide via FMO3-mediated oxidation. Certain microorganisms perform tasks that are known to be useful to the human host but the role of most of them is not well understood. Those that are expected to be present, and that under normal circumstances do not cause disease, are sometimes deemed normal flora or normal microbiota.</p>
<p>The Human Microbiome Project took on the project of sequencing the genome of the human microbiota, focusing particularly on the microbiota that normally inhabit the skin, mouth, nose, digestive tract, and vagina. It reached a milestone in 2012 when it published its initial results.</p>
<p>As of 2014, it was often reported in popular media and in the scientific literature that there are about 10 times as many microbial cells in the human body as there are human cells; this figure was based on estimates that the human microbiome includes around 100 trillion bacterial cells and that an adult human typically has around 10 trillion human cells. In 2014, the American Academy of Microbiology published a FAQ that emphasized that the number of microbial cells and the number of human cells are both estimates, and noted that recent research had arrived at a new estimate of the number of human cells – approximately 37.2 trillion, meaning that the ratio of microbial-to-human cells, if the original estimate of 100 trillion bacterial cells is correct, is closer to 3:1. In 2016, another group published a new estimate of the ratio being roughly 1:1 (1.3:1, with “an uncertainty of 25% and a variation of 53% over the population of standard 70-kg males”).</p>
<p>A more recent estimate is a ratio of 1.3 bacterial cells for every one human cell whereas the number of phages and viruses outnumber bacterial cells by at least a order of magnitude more. The number of bacterial genes (assuming 1000 bacterial species in the gut with 2000 genes per species) is estimate to be 2,000,000 genes, 100 times the number of approximately 20,000 human genes.</p>
<p>The problem of elucidating the human microbiome is essentially identifying the members of a microbial community which includes bacteria, eukaryotes, and viruses. This is done primarily using DNA-based studies, though RNA, protein and metabolite based studies are also performed. DNA-based microbiome studies typically can be categorized as either targeted amplicon studies or more recently shotgun metagenomic studies. The former focuses on specific known marker genes and is primarily informative taxonomically, while the latter is an entire metagenomic approach which can also be used to study the functional potential of the community. One of the challenges that is present in human microbiome studies, but not in other metagenomic studies is to avoid including the host DNA in the study.</p>
<p>Aside from simply elucidating the composition of the human microbiome, one of the major questions involving the human microbiome is whether there is a “core”, that is, whether there is a subset of the community that is shared among most humans. If there is a core, then it would be possible to associate certain community compositions with disease states, which is one of the goals of the Human Microbiome Project. It is known that the human microbiome (such as the gut microbiota) is highly variable both within a single subject and among different individuals, a phenomenon which is also observed in mice.</p>
<p>On 13 June 2012, a major milestone of the Human Microbiome Project (HMP) was announced by the NIH director Francis Collins. The announcement was accompanied with a series of coordinated articles published in Nature and several journals in the Public Library of Science (PLoS) on the same day. By mapping the normal microbial make-up of healthy humans using genome sequencing techniques, the researchers of the HMP have created a reference database and the boundaries of normal microbial variation in humans. From 242 healthy U.S. volunteers, more than 5,000 samples were collected from tissues from 15 (men) to 18 (women) body sites such as mouth, nose, skin, lower intestine (stool), and vagina. All the DNA, human and microbial, were analyzed with DNA sequencing machines. The microbial genome data were extracted by identifying the bacterial specific ribosomal RNA, 16S rRNA. The researchers calculated that more than 10,000 microbial species occupy the human ecosystem and they have identified 81 – 99% of the genera.</p>
<p>Populations of microbes (such as bacteria and yeasts) inhabit the skin and mucosal surfaces in various parts of the body. Their role forms part of normal, healthy human physiology, however if microbe numbers grow beyond their typical ranges (often due to a compromised immune system) or if microbes populate (such as through poor hygiene or injury) areas of the body normally not colonized or sterile (such as the blood, or the lower respiratory tract, or the abdominal cavity), disease can result (causing, respectively, bacteremia/sepsis, pneumonia, and peritonitis).[medical citation needed]</p>
<p>The Human Microbiome Project found that individuals host thousands of bacterial types, different body sites having their own distinctive communities. Skin and vaginal sites showed smaller diversity than the mouth and gut, these showing the greatest richness. The bacterial makeup for a given site on a body varies from person to person, not only in type, but also in abundance. Bacteria of the same species found throughout the mouth are of multiple subtypes, preferring to inhabit distinctly different locations in the mouth. Even the enterotypes in the human gut, previously thought to be well understood, are from a broad spectrum of communities with blurred taxon boundaries.</p>
<p>It is estimated that 500 to 1,000 species of bacteria live in the human gut but belong to just a few phyla: Firmicutes and Bacteroidetes dominate but there are also Proteobacteria, Verrucomicrobia, Actinobacteria, Fusobacteria and Cyanobacteria.</p>
<p>A number of types of bacteria, such as Actinomyces viscosus and A. naeslundii, live in the mouth, where they are part of a sticky substance called plaque. If this is not removed by brushing, it hardens into calculus (also called tartar). The same bacteria also secrete acids that dissolve tooth enamel, causing tooth decay.</p>
<p>The vaginal microflora consist mostly of various lactobacillus species. It was long thought that the most common of these species was Lactobacillus acidophilus, but it has later been shown that L. iners is in fact most common, followed by L. crispatus. Other lactobacilli found in the vagina are L. jensenii, L. delbruekii and L. gasseri. Disturbance of the vaginal flora can lead to infections such as bacterial vaginosis or candidiasis (“yeast infection”).</p>
<p>Archaea are present in the human gut, but, in contrast to the enormous variety of bacteria in this organ, the numbers of archaeal species are much more limited. The dominant group are the methanogens, particularly Methanobrevibacter smithii and Methanosphaera stadtmanae. However, colonization by methanogens is variable, and only about 50% of humans have easily detectable populations of these organisms.</p>
<p>As of 2007, no clear examples of archaeal pathogens were known, although a relationship has been proposed between the presence of some methanogens and human periodontal disease.</p>
<p>Fungi, in particular yeasts, are present in the human gut. The best-studied of these are Candida species due to their ability to become pathogenic in immunocompromised and even in healthy hosts. Yeasts are also present on the skin, such as Malassezia species, where they consume oils secreted from the sebaceous glands.</p>
<p>Viruses, especially bacterial viruses (bacteriophages), colonize various body sites. These colonized sites include the skin, gut, lungs, and oral cavity. Virus communities have been associated with some diseases, and do not simply reflect the bacterial communities.</p>
<p>Human bodies rely on the innumerable bacterial genes as the source of essential nutrients. Both metagenomic and epidemiological studies indicate vital roles for the human microbiome in preventing a wide range of diseases, from type 2 diabetes and obesity to inflammatory bowel disease, Parkinson’s disease, and even mental health conditions like depression. A symbiotic relationship between the gut microbiota and different bacteria may influence an individual’s immune response. Although in its infancy, microbiome-based treatment is also showing promise, most notably for treating drug-resistant C. difficile infection and in diabetes treatment.</p>
<p>Although cancer is generally a disease of host genetics and environmental factors, microorganisms are implicated in some 20% of human cancers. Particularly for potential factors in colon cancer, bacterial density is one million times higher than in the small intestine, and approximately 12-fold more cancers occur in the colon compared to the small intestine, possibly establishing a pathogenic role for microbiota in colon and rectal cancers. Microbial density may be used as a prognostic tool in assessment of colorectal cancers.</p>
<p>The microbiota may affect carcinogenesis in three broad ways: (i) altering the balance of tumor cell proliferation and death, (ii) regulating immune system function, and (iii) influencing metabolism of host-produced factors, foods and pharmaceuticals. Tumors arising at boundary surfaces, such as the skin, oropharynx and respiratory, digestive and urogenital tracts, harbor a microbiota. Substantial microbe presence at a tumor site does not establish association or causal links. Instead, microbes may find tumor oxygen tension or nutrient profile supportive. Decreased populations of specific microbes or induced oxidative stress may also increase risks. Of the around 1030 microbes on earth, ten are designated by the International Agency for Research on Cancer as human carcinogens. Microbes may secrete proteins or other factors directly drive cell proliferation in the host, or may up- or down-regulate the host immune system including driving acute or chronic inflammation in ways that contribute to carcinogenesis.</p>
<p>Concerning the relationship of immune function and development of inflammation, mucosal surface barriers are subject to environmental risks and must rapidly repair to maintain homeostasis. Compromised host or microbiota resiliency also reduce resistance to malignancy, possibly inducing inflammation and cancer. Once barriers are breached, microbes can elicit proinflammatory or immunosuppressive programs through various pathways. For example, cancer-associated microbes appear to activate NF-κΒ signaling within the tumor microenviroment. Other pattern recognition receptors, such as nucleotide-binding oligomerization domain–like receptor (NLR) family members NOD-2, NLRP3, NLRP6 and NLRP12, may play a role in mediating colorectal cancer. Likewise Helicobacter pylori appears to increase the risk of gastric cancer, due to its driving a chronic inflammatory response in the stomach.</p>
<p>With death, the microbiome of the living body collapses and a different composition of microorganisms named necrobiome establishes itself as an important active constituent of the complex physical decomposition process. Its predictable changes over time are thought to be useful to help determine the time of death.</p>
</div>
<div id="extremophiles" class="section level3" number="1.2.5">
<h3><span class="header-section-number">1.2.5</span> Extremophiles</h3>
<p>Extremophiles are microorganisms that have adapted so that they can survive and even thrive in extreme environments that are normally fatal to most life-forms. Thermophiles and hyperthermophiles thrive in high temperatures. Psychrophiles thrive in extremely low temperatures. – Temperatures as high as 130 °C (266 °F), as low as −17 °C (1 °F) Halophiles such as Halobacterium salinarum (an archaean) thrive in high salt conditions, up to saturation. Alkaliphiles thrive in an alkaline pH of about 8.5–11. Acidophiles can thrive in a pH of 2.0 or less. Piezophiles thrive at very high pressures: up to 1,000–2,000 atm, down to 0 atm as in a vacuum of space. A few extremophiles such as Deinococcus radiodurans are radioresistant, resisting radiation exposure of up to 5k Gy. Extremophiles are significant in different ways. They extend terrestrial life into much of the Earth’s hydrosphere, crust and atmosphere, their specific evolutionary adaptation mechanisms to their extreme environment can be exploited in biotechnology, and their very existence under such extreme conditions increases the potential for extraterrestrial life.</p>
</div>
<div id="soil-microorganisms" class="section level3" number="1.2.6">
<h3><span class="header-section-number">1.2.6</span> Soil Microorganisms</h3>
<p>The nitrogen cycle in soils depends on the fixation of atmospheric nitrogen. This is achieved by a number of diazotrophs. One way this can occur is in the root nodules of legumes that contain symbiotic bacteria of the genera Rhizobium, Mesorhizobium, Sinorhizobium, Bradyrhizobium, and Azorhizobium.</p>
<p>The roots of plants create a narrow region known as the rhizosphere that supports many microorganisms known as the root microbiome.</p>
</div>
</div>
<div id="human-interactions-with-microbes" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Human Interactions With Microbes</h2>
<p>Human interactions with microbes include both practical and symbolic uses of microbes, and negative interactions in the form of human, domestic animal, and crop diseases.</p>
<p>Practical use of microbes began in ancient times with fermentation in food processing; bread, beer and wine have been produced by yeasts from the dawn of civilisation, such as in ancient Egypt. More recently, microbes have been used in activities from biological warfare to the production of chemicals by fermentation, as industrial chemists discover how to manufacture a widening variety of organic chemicals including enzymes and bioactive molecules such as hormones and competitive inhibitors for use as medicines. Fermentation is used, too, to produce substitutes for fossil fuels in forms such as ethanol and methane; fuels may also be produced by algae. Anaerobic microorganisms are important in sewage treatment. In scientific research, yeasts and the bacterium Escherichia coli serve as model organisms especially in genetics and related fields.</p>
<p>On the symbolic side, an early poem about brewing is the Sumerian “Hymn to Ninkasi”, from 1800 BC. In the Middle Ages, Giovanni Boccaccio’s The Decameron and Geoffrey Chaucer’s The Canterbury Tales: addressed people’s fear of deadly contagion and the moral decline that could result. Novelists have exploited the apocalyptic possibilities of pandemics from Mary Shelley’s 1826 The Last Man and Jack London’s 1912 The Scarlet Plague onwards. Hilaire Belloc wrote a humorous poem to “The Microbe” in 1912. Dramatic plagues and mass infection have formed the story lines of many Hollywood films, starting with Nosferatu in 1922. In 1971, The Andromeda Strain told the tale of an extraterrestrial microbe threatening life on Earth. Microbiologists since Alexander Fleming have used coloured or fluorescing colonies of bacteria to create miniature artworks.</p>
<p>Microorganisms such as bacteria and viruses are important as pathogens, causing disease to humans, crop plants, and domestic animals.</p>
<div id="food-production" class="section level3" number="1.3.1">
<h3><span class="header-section-number">1.3.1</span> Food production</h3>
<p>Controlled fermentation with microbes in brewing, wine making, baking, pickling and cultured dairy products such as yogurt and cheese, is used to modify ingredients to make foods with desirable properties. The principal microbes involved are yeasts, in the case of beer, wine, and ordinary bread; and bacteria, in the case of anaerobically fermented vegetables, dairy products, and sourdough bread. The cultures variously provide flavour and aroma, inhibit pathogens, increase digestibility and palatability, make bread rise, reduce cooking time, and create useful products including alcohol, organic acids, vitamins, amino acids, and carbon dioxide. Safety is maintained with the help of food microbiology.</p>
</div>
<div id="water-treatment" class="section level3" number="1.3.2">
<h3><span class="header-section-number">1.3.2</span> Water Treatment</h3>
<p>Oxidative sewage treatment processes rely on microorganisms to oxidise organic constituents. Anaerobic microorganisms reduce sludge solids producing methane gas and a sterile mineralised residue. In potable water treatment, one method, the slow sand filter, employs a complex gelatinous layer composed of a wide range of microorganisms to remove both dissolved and particulate material from raw water.</p>
</div>
<div id="fuel-production" class="section level3" number="1.3.3">
<h3><span class="header-section-number">1.3.3</span> Fuel Production</h3>
<p>Microorganisms are used in fermentation to produce ethanol, and in biogas reactors to produce methane. Scientists are researching the use of algae to produce liquid fuels, and bacteria to convert various forms of agricultural and urban waste into usable fuels.</p>
</div>
<div id="production-of-chemicals-and-enzymes" class="section level3" number="1.3.4">
<h3><span class="header-section-number">1.3.4</span> Production Of Chemicals And Enzymes</h3>
<p>Microorganisms are used for many commercial and industrial purposes, including the production of chemicals, enzymes and other bioactive molecules, often through protein engineering. For example, acetic acid is produced by the bacterium Acetobacter aceti, while citric acid is produced by the fungus Aspergillus niger. Microorganisms are used to prepare a widening range of bioactive molecules and enzymes. For example, Streptokinase produced by the bacterium Streptococcus and modified by genetic engineering is used to remove clots from the blood vessels of patients who have suffered a heart attack. Cyclosporin A is an immunosuppressive agent in organ transplantation, while statins produced by the yeast Monascus purpureus serve as blood cholesterol lowering agents, competitively inhibiting the enzyme that synthesizes cholesterol.</p>
</div>
<div id="use-of-microorganisms-in-science" class="section level3" number="1.3.5">
<h3><span class="header-section-number">1.3.5</span> Use Of Microorganisms In Science</h3>
<p>Microorganisms are essential tools in biotechnology, biochemistry, genetics, and molecular biology. The yeasts brewer’s yeast (Saccharomyces cerevisiae) and fission yeast (Schizosaccharomyces pombe) are important model organisms in science, since they are simple eukaryotes that can be grown rapidly in large numbers and are easily manipulated.</p>
</div>
<div id="use-of-microorganisms-for-warfare" class="section level3" number="1.3.6">
<h3><span class="header-section-number">1.3.6</span> Use of Microorganisms For Warfare</h3>
<p>Pathogenic microbes, and toxins that they produce, have been developed as possible agents of warfare. Crude forms of biological warfare have been practiced since antiquity. In the 6th century BC, the Assyrians poisoned enemy wells with a fungus said to render the enemy delirious. In 1346, the bodies of Mongol warriors of the Golden Horde who had died of plague were thrown over the walls of the besieged Crimean city of Kaffa, possibly assisting the spread of the Black Death into Europe. Advances in bacteriology in the 20th century increased the sophistication of possible bio-agents in war. Biological sabotage—in the form of anthrax and glanders—was undertaken on behalf of the Imperial German government during World War I, with indifferent results. In World War II, Britain weaponised tularemia, anthrax, brucellosis, and botulism toxins, but never used them. The USA similarly explored biological warfare agents, developing anthrax spores, brucellosis, and botulism toxins for possible military use. Japan developed biological warfare agents, with the use of experiments on human prisoners, and was about to use them when the war ended.</p>
</div>
</div>
<div id="role-of-microorganisms-in-disease" class="section level2" number="1.4">
<h2><span class="header-section-number">1.4</span> Role Of Microorganisms In Disease</h2>
<p>Microorganisms are the causative agents (pathogens) in many infectious diseases of humans and domestic animals. Pathogenic bacteria cause diseases such as plague, tuberculosis and anthrax. Protozoa cause diseases including malaria, sleeping sickness, dysentery and toxoplasmosis. Microscopic fungi cause diseases such as ringworm, candidiasis and histoplasmosis. Pathogenic viruses cause diseases such as influenza, yellow fever and AIDS.</p>
<p>The practice of hygiene was created to prevent infection or food spoiling by eliminating microbes, especially bacteria, from the surroundings.</p>
</div>
<div id="role-of-microorganisms-in-agriculture-and-horticulture" class="section level2" number="1.5">
<h2><span class="header-section-number">1.5</span> Role Of Microorganisms In Agriculture And Horticulture</h2>
<p>Microorganisms including bacteria, fungi, and viruses are important as plant pathogens, causing disease to crop plants. Fungi cause serious crop diseases such as maize leaf rust, wheat stem rust, and powdery mildew. Bacteria cause plant diseases including leaf spot and crown galls. Viruses cause plant diseases such as leaf mosaic. The oomycete Phytophthora infestans causes potato blight, contributing to the Great Irish Famine of the 1840s.</p>
<p>The tulip breaking virus played a role in the tulip mania of the Dutch Golden Age. The famous Semper Augustus tulip, in particular, owed its striking pattern to infection with the plant disease, a kind of mosaic virus, making it the most expensive of all the tulip bulbs sold.</p>
</div>
<div id="history-of-microbiology" class="section level2" number="1.6">
<h2><span class="header-section-number">1.6</span> History Of Microbiology</h2>
<p>The existence of microorganisms was predicted many centuries before they were first observed, for example by the Jains in India and by Marcus Terentius Varro in ancient Rome. The first recorded microscope observation was of the fruiting bodies of moulds, by Robert Hooke in 1666, but the Jesuit priest Athanasius Kircher was likely the first to see microbes, which he mentioned observing in milk and putrid material in 1658. Antonie van Leeuwenhoek is considered a father of microbiology as he observed and experimented with microscopic organisms in the 1670s, using simple microscopes of his own design. Scientific microbiology developed in the 19th century through the work of <a href="https://en.wikipedia.org/wiki/Louis_Pasteur">Louis Pasteur</a> and in medical microbiology <a href="https://en.wikipedia.org/wiki/Robert_Koch">Robert Koch</a>.</p>



<p>In 1676, <a href="https://en.wikipedia.org/wiki/Antonie_van_Leeuwenhoek">Antonie van Leeuwenhoek</a>, who lived most of his life in Delft, Netherlands, observed bacteria and other microorganisms using a single-lens microscope of his own design.</p>


<p>He is considered a father of microbiology as he pioneered the use of simple single-lensed microscopes of his own design. While Van Leeuwenhoek is often cited as the first to observe microbes, Robert Hooke made his first recorded microscopic observation, of the fruiting bodies of moulds, in 1665. It has, however, been suggested that a Jesuit priest called Athanasius Kircher was the first to observe microorganisms. Kircher was among the first to design magic lanterns for projection purposes, so he must have been well acquainted with the properties of lenses. He wrote “Concerning the wonderful structure of things in nature, investigated by Microscope” in 1646, stating “who would believe that vinegar and milk abound with an innumerable multitude of worms.” He also noted that putrid material is full of innumerable creeping animalcules. He published his Scrutinium Pestis (Examination of the Plague) in 1658, stating correctly that the disease was caused by microbes, though what he saw was most likely red or white blood cells rather than the plague agent itself. Innovative laboratory glassware and experimental methods developed by <a href="https://commons.wikimedia.org/wiki/File:Albert_Edelfelt_-_Louis_Pasteur_-_1885.jpg">Louis Pasteur</a> and other biologists contributed to the young field of bacteriology in the late 19th century.</p>
<p>The field of bacteriology (later a subdiscipline of microbiology) was founded in the 19th century by Ferdinand Cohn, a botanist whose studies on algae and photosynthetic bacteria led him to describe several bacteria including Bacillus and Beggiatoa. Cohn was also the first to formulate a scheme for the taxonomic classification of bacteria, and to discover endospores. Louis Pasteur and Robert Koch were contemporaries of Cohn, and are often considered to be the fathers of modern microbiology and medical microbiology, respectively. Pasteur is most famous for his series of experiments designed to disprove the then widely held theory of <a href="https://en.wikipedia.org/wiki/Spontaneous_generation">spontaneous generation</a>, thereby solidifying microbiology’s identity as a biological science. One of his students, Adrien Certes, is considered the founder of marine microbiology. Pasteur also designed methods for food preservation (pasteurization) and vaccines against several diseases such as anthrax, fowl cholera and rabies. Koch is best known for his contributions to the germ theory of disease, proving that specific diseases were caused by specific pathogenic microorganisms. He developed a series of criteria that have become known as the Koch’s postulates. Koch was one of the first scientists to focus on the isolation of bacteria in pure culture resulting in his description of several novel bacteria including Mycobacterium tuberculosis, the causative agent of tuberculosis. While Pasteur and Koch are often considered the founders of microbiology, their work did not accurately reflect the true diversity of the microbial world because of their exclusive focus on microorganisms having direct medical relevance. It was not until the late 19th century and the work of <a href="https://en.wikipedia.org/wiki/Martinus_Beijerinck">Martinus Beijerinck</a> and <a href="https://en.wikipedia.org/wiki/Sergei_Winogradsky">Sergei Winogradsky</a> that the true breadth of microbiology was revealed. Beijerinck made two major contributions to microbiology: the discovery of viruses and the development of enrichment culture techniques. While his work on the <a href="https://en.wikipedia.org/wiki/Tobacco_mosaic_virus">tobacco mosaic virus</a> established the basic principles of virology, it was his development of enrichment culturing that had the most immediate impact on microbiology by allowing for the cultivation of a wide range of microbes with wildly different physiologies. Winogradsky was the first to develop the concept of chemolithotrophy and to thereby reveal the essential role played by microorganisms in geochemical processes. He was responsible for the first isolation and description of both nitrifying and nitrogen-fixing bacteria. French-Canadian microbiologist <a href="https://en.wikipedia.org/wiki/Félix_d%27Hérelle">Felix d’Herelle</a> co-discovered bacteriophages in 1917 and was one of the earliest applied microbiologists. <a href="https://en.wikipedia.org/wiki/Joseph_Lister">Joseph Lister</a> was the first to use phenol disinfectant on the open wounds of patients. The branches of microbiology can be classified into applied sciences, or divided according to taxonomy, as is the case with bacteriology, mycology, protozoology, virology, phycology, and microbial ecology. There is considerable overlap between the specific branches of microbiology with each other and with other disciplines, and certain aspects of these branches can extend beyond the traditional scope of microbiology. A pure research branch of microbiology is termed cellular microbiology.</p>
<p>While some fear microbes due to the association of some microbes with various human diseases, many microbes are also responsible for numerous beneficial processes such as industrial fermentation (e.g. the production of alcohol, vinegar and dairy products), antibiotic production and act as molecular vehicles to transfer DNA to complex organisms such as plants and animals. Scientists have also exploited their knowledge of microbes to produce biotechnologically important enzymes such as Taq polymerase, reporter genes for use in other genetic systems and novel molecular biology techniques such as the yeast two-hybrid system.</p>
<p>Bacteria can be used for the industrial production of amino acids. Corynebacterium glutamicum is one of the most important bacterial species with an annual production of more than two million tons of amino acids, mainly L-glutamate and L-lysine. Since some bacteria have the ability to synthesize antibiotics, they are used for medicinal purposes, such as Streptomyces to make aminoglycoside antibiotics.</p>

<p>A variety of biopolymers, such as polysaccharides, polyesters, and polyamides, are produced by microorganisms. Microorganisms are used for the biotechnological production of biopolymers with tailored properties suitable for high-value medical application such as tissue engineering and drug delivery. Microorganisms are for example used for the biosynthesis of xanthan, alginate, cellulose, cyanophycin, poly(gamma-glutamic acid), levan, hyaluronic acid, organic acids, oligosaccharides polysaccharide and polyhydroxyalkanoates. Microorganisms are beneficial for microbial biodegradation or bioremediation of domestic, agricultural and industrial wastes and subsurface pollution in soils, sediments and marine environments. The ability of each microorganism to degrade toxic waste depends on the nature of each contaminant. Since sites typically have multiple pollutant types, the most effective approach to microbial biodegradation is to use a mixture of bacterial and fungal species and strains, each specific to the biodegradation of one or more types of contaminants. Symbiotic microbial communities confer benefits to their human and animal hosts health including aiding digestion, producing beneficial vitamins and amino acids, and suppressing pathogenic microbes. Some benefit may be conferred by eating fermented foods, probiotics (bacteria potentially beneficial to the digestive system) or prebiotics (substances consumed to promote the growth of probiotic microorganisms). The ways the microbiome influences human and animal health, as well as methods to influence the microbiome are active areas of research. Research has suggested that microorganisms could be useful in the treatment of cancer. Various strains of non-pathogenic clostridia can infiltrate and replicate within solid tumors. Clostridial vectors can be safely administered and their potential to deliver therapeutic proteins has been demonstrated in a variety of preclinical models. Some bacteria are used to study fundamental mechanism. An example of model bacteria used to study motility or the production of polysaccharides and devElopment Is Myxococcus Xanthus.</p>
</div>
<div id="taxonomy-classification-and-naming-of-microorganisms" class="section level2" number="1.7">
<h2><span class="header-section-number">1.7</span> Taxonomy: Classification And Naming Of Microorganisms</h2>
<p><a href="https://en.wikipedia.org/wiki/Taxonomy_(biology)">Taxonomy</a> is the identification, naming and classification of organisms. <a href="https://en.wikipedia.org/wiki/Binomial_nomenclature">Binomial nomenclature</a> is a formal system of naming species of living things by giving each a name composed of two parts. <a href="https://en.wikipedia.org/wiki/Systematics">Systematics</a> is the branch of science that deals with unique properties of species and groups to recognise, describe name and arrange the diverse organisms according to an organised plan. In biology, phylogenetics (Greek: φυλή, φῦλον – phylé, phylon = tribe, clan, race + γενετικός – genetikós = origin, source, birth) is a part of systematics that addresses the inference of the evolutionary history and relationships among or within groups of organisms (e.g. species, or more inclusive taxa). Classifications are now usually based on phylogenetic data. The degree to which classification depends on inferred evolutionary history differs depending on the school of taxonomy: phenetics ignores phylogenetic speculation altogether, trying to represent the similarity between organisms instead; cladistics (phylogenetic systematics) tries to reflect phylogeny in its classifications by only recognizing groups based on shared, derived characters (synapomorphies); evolutionary taxonomy tries to take into account both the branching pattern and “degree of difference” to find a compromise between them.</p>
<p>Biological classification is a critical component of the taxonomic process. As a result, it informs the user as to what the relatives of the taxon are hypothesized to be. Biological classification uses taxonomic ranks, including among others (in order from most inclusive to least inclusive): Domain, Kingdom, Phylum, Class, Order, Family, Genus, and Species</p>

<div class="figure" style="text-align: center"><span id="fig:classification"></span>
<img src="figures/main_themes/biological_classification.svg" alt="Biological classification." width="70%" />
<p class="caption">
Figure 1.1: <a href="https://commons.wikimedia.org/wiki/File:Biological_classification_L_Pengo_vflip.svg">Biological classification.</a>
</p>
</div>
<p>The “definition” of a taxon is encapsulated by its description or its diagnosis or by both combined. There are no set rules governing the definition of taxa, but the naming and publication of new taxa is governed by sets of rules. In zoology, the nomenclature for the more commonly used ranks (superfamily to subspecies), is regulated by the International Code of Zoological Nomenclature (ICZN Code). In the fields of botany, phycology, and mycology, the naming of taxa is governed by the International Code of Nomenclature for algae, fungi, and plants (ICN). Bacterial taxonomy is the taxonomy, i.e. the rank-based classification, of bacteria. Despite there being no official and complete classification of prokaryotes, the names (nomenclature) given to prokaryotes are regulated by the International Code of Nomenclature of Bacteria (Bacteriological Code), a book which contains general considerations, principles, rules, and various notes, and advises in a similar fashion to the nomenclature codes of other groups.</p>
<p>In the scientific classification established by Carl Linnaeus, each species has to be assigned to a genus (binary nomenclature), which in turn is a lower level of a hierarchy of ranks (family, suborder, order, subclass, class, division/phyla, kingdom and domain). In the currently accepted classification of life, there are three domains (Eukaryotes, Bacteria and Archaea), which, in terms of taxonomy, despite following the same principles have several different conventions between them and between their subdivisions as they are studied by different disciplines (botany, zoology, mycology and microbiology). For example, in zoology there are type specimens, whereas in microbiology there are type strains.</p>
<p>The initial description of a taxon involves five main requirements:</p>
<ol style="list-style-type: decimal">
<li>The taxon must be given a name based on the 26 letters of the Latin alphabet (a binomial for new species, or uninomial for other ranks).</li>
<li>The name must be unique (i.e. not a homonym).</li>
<li>The description must be based on at least one name-bearing type specimen.</li>
<li>It should include statements about appropriate attributes either to describe (define) the taxon or to differentiate it from other taxa (the diagnosis, ICZN Code, Article 13.1.1, ICN, Article 38). Both codes deliberately separate defining the content of a taxon (its circumscription) from defining its name.</li>
<li>These first four requirements must be published in a work that is obtainable in numerous identical copies, as a permanent scientific record.</li>
</ol>
<p>However, often much more information is included, like the geographic range of the taxon, ecological notes, chemistry, behavior, etc. How researchers arrive at their taxa varies: depending on the available data, and resources, methods vary from simple quantitative or qualitative comparisons of striking features, to elaborate computer analyses of large amounts of DNA sequence data.</p>
<p>An “authority” may be placed after a scientific name. The authority is the name of the scientist or scientists who first validly published the name. For example, in 1758 Linnaeus gave the Asian elephant the scientific name <em>Elephas maximus</em>, so the name is sometimes written as “<em>Elephas maximus</em> Linnaeus, 1758”. The names of authors are frequently abbreviated: the abbreviation L., for Linnaeus, is commonly used. In botany, there is, in fact, a regulated list of standard abbreviations (see list of botanists by author abbreviation). The system for assigning authorities differs slightly between botany and zoology. However, it is standard that if a species’ name or placement has been changed since the original description, the original authority’s name is placed in parentheses.</p>

<div class="figure" style="text-align: center"><span id="fig:tree"></span>
<img src="figures/main_themes/tree_of_life.svg" alt="The tree of life." width="70%" />
<p class="caption">
Figure 1.2: <a href="https://commons.wikimedia.org/wiki/File:CollapsedtreeLabels-simplified.svg">The tree of life.</a>
</p>
</div>
<div id="the-concept-of-species-in-biology" class="section level3" number="1.7.1">
<h3><span class="header-section-number">1.7.1</span> The Concept of Species in Biology</h3>
<p>In biology, a species is the basic unit of classification and a taxonomic rank of an organism, as well as a unit of biodiversity. A species is often defined as the largest group of organisms in which any two individuals of the appropriate sexes or mating types can produce fertile offspring, typically by sexual reproduction. Other ways of defining species include their karyotype, DNA sequence, morphology, behaviour or ecological niche. In addition, paleontologists use the concept of the chronospecies since fossil reproduction cannot be examined.</p>
<p>Species were seen from the time of Aristotle until the 18th century as fixed categories that could be arranged in a hierarchy, the great chain of being. In the 19th century, biologists grasped that species could evolve given sufficient time. Charles Darwin’s 1859 book On the Origin of Species explained how species could arise by natural selection. That understanding was greatly extended in the 20th century through genetics and population ecology. Genetic variability arises from mutations and recombination, while organisms themselves are mobile, leading to geographical isolation and genetic drift with varying selection pressures. Genes can sometimes be exchanged between species by horizontal gene transfer; new species can arise rapidly through hybridisation and polyploidy; and species may become extinct for a variety of reasons.</p>
<p>Biologists and taxonomists have made many attempts to define species, beginning from morphology and moving towards genetics. Early taxonomists such as Linnaeus had no option but to describe what they saw: this was later formalised as the typological or morphological species concept. <a href="https://en.wikipedia.org/wiki/Ernst_Mayr">Ernst Mayr</a> proposed the widely used Biological Species Concept of reproductive isolation in 1942. Later biologists have tried to refine Mayr’s definition. Many of the concepts are quite similar or overlap, so they are not easy to count: the biologist R. L. Mayden recorded about 24 concepts, and the philosopher of science John Wilkins counted 26. Wilkins further grouped the species concepts into seven basic kinds of concepts: (1) agamospecies for asexual organisms (2) biospecies for reproductively isolated sexual organisms (3) ecospecies based on ecological niches (4) evolutionary species based on lineage (5) genetic species based on gene pool (6) morphospecies based on form or phenotype and (7) taxonomic species, a species as determined by a taxonomist.</p>
<p>A typological species is a group of organisms in which individuals conform to certain fixed properties (a type), so that even pre-literate people often recognise the same taxon as do modern taxonomists. The clusters of variations or phenotypes within specimens (such as longer or shorter tails) would differentiate the species. This method was used as a “classical” method of determining species, such as with Linnaeus early in evolutionary theory. However, different phenotypes are not necessarily different species (e.g. a four-winged Drosophila born to a two-winged mother is not a different species). Species named in this manner are called morphospecies.</p>
<p>A species is given a taxonomic name when a type specimen is described formally, in a publication that assigns it a unique scientific name. The description typically provides means for identifying the new species, differentiating it from other previously described and related or confusable species and provides a validly published name (in botany) or an available name (in zoology) when the paper is accepted for publication. The type material is usually held in a permanent repository, often the research collection of a major museum or university, that allows independent verification and the means to compare specimens. Describers of new species are asked to choose names that, in the words of the International Code of Zoological Nomenclature, are “appropriate, compact, euphonious, memorable, and do not cause offence”.</p>
<p>The naming of a particular species, including which genus (and higher taxa) it is placed in, is a hypothesis about the evolutionary relationships and distinguishability of that group of organisms. As further information comes to hand, the hypothesis may be confirmed or refuted. Sometimes, especially in the past when communication was more difficult, taxonomists working in isolation have given two distinct names to individual organisms later identified as the same species. When two named species are discovered to be of the same species, the older species name is given priority and usually retained, and the newer name considered as a junior synonym, a process called synonymisation. Dividing a taxon into multiple, often new, taxa is called splitting. Taxonomists are often referred to as “lumpers” or “splitters” by their colleagues, depending on their personal approach to recognising differences or commonalities between organisms.</p>

<div class="figure" style="text-align: center"><span id="fig:species"></span>
<img src="figures/main_themes/Heliconius_mimicry.png" alt="The butterfly genus Heliconius contains many similar species." width="70%" />
<p class="caption">
Figure 1.3: <a href="https://commons.wikimedia.org/wiki/File:Heliconius_mimicry.png">The butterfly genus Heliconius contains many similar species.</a>
</p>
</div>
<p>It is difficult to define a species in a way that applies to all organisms. The debate about species delimitation is called the species problem. The problem was recognized even in 1859, when Darwin wrote in On the Origin of Species:</p>
<blockquote>
<p>No one definition has satisfied all naturalists; yet every naturalist knows vaguely what he means when he speaks of a species. Generally the term includes the unknown element of a distinct act of creation.</p>
</blockquote>
<p>The evolutionary process by which biological populations evolve to become distinct or reproductively isolated as species is called speciation. Charles Darwin was the first to describe the role of natural selection in speciation in his 1859 book The Origin of Species. Speciation depends on a measure of reproductive isolation, a reduced gene flow. This occurs most easily in allopatric speciation, where populations are separated geographically and can diverge gradually as mutations accumulate. Reproductive isolation is threatened by hybridisation, but this can be selected against once a pair of populations have incompatible alleles of the same gene, as described in the Bateson–Dobzhansky–Muller model. A different mechanism, phyletic speciation, involves one lineage gradually changing over time into a new and distinct form, without increasing the number of resultant species.</p>
<p>Bacteria divide asexually and for the most part do not show regionalisms (“Everything is everywhere”), therefore the concept of species, which works best for animals, becomes entirely a matter of judgement.</p>
<p>The number of named species of bacteria and archaea (approximately 13,000) is surprisingly small considering their early evolution, genetic diversity and residence in all ecosystems. The reason for this is the differences in species concepts between the bacteria and macro-organisms, the difficulties in growing/characterising in pure culture (a prerequisite to naming new species, vide supra) and extensive horizontal gene transfer blurring the distinction of species.</p>
<p>The most commonly accepted definition is the polyphasic species definition, which takes into account both phenotypic and genetic differences. However, a quicker diagnostic ad hoc threshold to separate species is less than 70% DNA–DNA hybridisation, which corresponds to less than 97% 16S DNA sequence identity. It has been noted that if this were applied to animal classification, the order primates would be a single species. For this reason, more stringent species definitions based on whole genome sequences have been proposed.</p>
</div>
<div id="binomial-nomenclature" class="section level3" number="1.7.2">
<h3><span class="header-section-number">1.7.2</span> Binomial Nomenclature</h3>
<p>The commonly used names for kinds of organisms are often ambiguous: “cat” could mean the domestic cat or the cat family. Another problem with common names is that they often vary from place to place, so that puma, cougar, catamount, panther, painter and mountain lion all mean <em>Puma concolor</em> in various parts of America, while “panther” may also mean the jaguar (<em>Panthera onca</em>) of Latin America or the leopard (<em>Panthera pardus</em>) of Africa and Asia. In contrast, the scientific names of species are chosen to be unique and universal; they are in two parts used together: the genus as in <em>Puma</em>, and the specific epithet as in <em>concolor</em>.</p>
<p>Binomial nomenclature (“two-term naming system”), also called binominal nomenclature (“two-name naming system”) or binary nomenclature, is a formal system of naming species of living things by giving each a name composed of two parts, both of which use Latin grammatical forms, although they can be based on words from other languages. Such a name is called a binomial name (which may be shortened to just “binomial”), a binomen, binominal name or a scientific name; more informally it is also called a Latin name.</p>
<p>The first part of the name – the generic name – identifies the genus to which the species belongs, while the second part – the specific name or specific epithet – identifies the species within the genus. For example, humans belong to the genus <em>Homo</em> and within this genus to the species <em>Homo sapiens</em>. <em>Tyrannosaurus rex</em> is probably the most widely known binomial. The formal introduction of this system of naming species is credited to Carl Linnaeus, effectively beginning with his work Species Plantarum in 1753. But Gaspard Bauhin, in as early as 1622, had introduced in his book Pinax theatri botanici (English, Illustrated exposition of plants) many names of genera that were later adopted by Linnaeus.</p>
<p>The application of binomial nomenclature is now governed by various internationally agreed codes of rules, of which the two most important are the International Code of Zoological Nomenclature (ICZN) for animals and the International Code of Nomenclature for algae, fungi, and plants (ICNafp). Although the general principles underlying binomial nomenclature are common to these two codes, there are some differences, both in the terminology they use and in their precise rules.</p>
<p>In modern usage, the first letter of the first part of the name, the genus, is always capitalized in writing, while that of the second part is not, even when derived from a proper noun such as the name of a person or place. Similarly, both parts are italicized when a binomial name occurs in normal text (or underlined in handwriting). Thus the binomial name of the annual phlox (named after botanist Thomas Drummond) is now written as <em>Phlox drummondii</em>.</p>
<p>In scientific works, the authority for a binomial name is usually given, at least when it is first mentioned, and the date of publication may be specified.</p>
<p>For Bacteria, valid names must have a Latin or Neolatin name and can only use basic latin letters (w and j inclusive, see History of the Latin alphabet for these), consequently hyphens, accents and other letters are not accepted and should be transliterated correctly (e.g. ß=ss). Ancient Greek being written in the Greek alphabet, needs to be transliterated into the Latin alphabet.</p>
</div>
<div id="taxonomy-and-systematics" class="section level3" number="1.7.3">
<h3><span class="header-section-number">1.7.3</span> Taxonomy And Systematics</h3>
<p><a href="https://en.wikipedia.org/wiki/Taxonomy_(biology)">Taxonomy</a> (from Ancient Greek taxis, meaning ‘arrangement’, and nomia, meaning ‘method’) is the science of defining and naming groups of biological organisms on the basis of shared characteristics. Organisms are grouped together into taxa (singular: taxon) and these groups are given a taxonomic rank; groups of a given rank can be aggregated to form a super-group of higher rank, thus creating a taxonomic hierarchy. The principal ranks in modern use are domain, kingdom, phylum (division is sometimes used in botany in place of phylum), class, order, family, genus and species. The Swedish botanist <a href="https://en.wikipedia.org/wiki/Carl_Linnaeus">Carl Linnaeus</a> (1707–1778) is regarded as the father of taxonomy, as he developed a system known as Linnaean taxonomy for categorization of organisms and binomial nomenclature for naming organisms. With the advent of such fields of study as phylogenetics, cladistics, and systematics, the Linnaean system has progressed to a system of modern biological classification based on the evolutionary relationships between organisms, both living and extinct.</p>
<p>Systematics is the branch of science that deals with unique properties of species and groups to recognise, describe name and arrange the diverse organisms according to an organised plan. The word systematics is derived from Latin word systema which means systematic arrangement of organisms.linneous used ‘systema naturae’ as the title of his book.</p>
<p>The term “taxonomy” was coined by <a href="https://en.wikipedia.org/wiki/Augustin_Pyramus_de_Candolle">Augustin Pyramus de Candolle</a> while the term “systematic” was coined by Carl Linnaeus the father of taxonomy.</p>
<p>Biological systematics is the study of the diversification of living forms, both past and present, and the relationships among living things through time. Relationships are visualized as evolutionary trees (synonyms: cladograms, phylogenetic trees, phylogenies). Phylogenies have two components: branching order (showing group relationships) and branch length (showing amount of evolution). Phylogenetic trees of species and higher taxa are used to study the evolution of traits (e.g., anatomical or molecular characteristics) and the distribution of organisms (biogeography). Systematics, in other words, is used to understand the evolutionary history of life on Earth.</p>
<p>Biological systematics classifies species by using three specific branches. Numerical systematics, or biometry, uses biological statistics to identify and classify animals. Biochemical systematics classifies and identifies animals based on the analysis of the material that makes up the living part of a cell—such as the nucleus, organelles, and cytoplasm. Experimental systematics identifies and classifies animals based on the evolutionary units that comprise a species, as well as their importance in evolution itself. Factors such as mutations, genetic divergence, and hybridization all are considered evolutionary units.</p>
<p>With the specific branches, researchers are able to determine the applications and uses for modern-day systematics. These applications include:</p>
<ul>
<li>Studying the diversity of organisms and the differentiation between extinct and living creatures. Biologists study the well-understood relationships by making many different diagrams and “trees” (cladograms, phylogenetic trees, phylogenies, etc.).</li>
<li>Including the scientific names of organisms, species descriptions and overviews, taxonomic orders, and classifications of evolutionary and organism histories.</li>
<li>Explaining the biodiversity of the planet and its organisms. The systematic study is that of conservation.</li>
<li>Manipulating and controlling the natural world. This includes the practice of ‘biological control’, the intentional introduction of natural predators and disease.</li>
</ul>
<p>Linnaeus ushered in a new era of taxonomy. With his major works Systema Naturae 1st Edition in 1735, Species Plantarum in 1753, and Systema Naturae 10th Edition, he revolutionized modern taxonomy. His works implemented a standardized binomial naming system for animal and plant species, which proved to be an elegant solution to a chaotic and disorganized taxonomic literature. He not only introduced the standard of class, order, genus, and species, but also made it possible to identify plants and animals from his book, by using the smaller parts of the flower. Thus, the Linnaean system was born, and is still used in essentially the same way today as it was in the 18th century. Currently, plant and animal taxonomists regard Linnaeus’ work as the “starting point” for valid names (at 1753 and 1758 respectively). Names published before these dates are referred to as “pre-Linnaean”, and not considered valid (with the exception of spiders published in Svenska Spindlar). Even taxonomic names published by Linnaeus himself before these dates are considered pre-Linnaean.</p>
<p>Whereas Linnaeus classified for ease of identification, the idea of the Linnaean taxonomy as translating into a sort of dendrogram of the Animal- and Plant Kingdoms was formulated toward the end of the 18th century, well before On the Origin of Species was published. Among early works exploring the idea of a transmutation of species were <a href="https://en.wikipedia.org/wiki/Erasmus_Darwin">Erasmus Darwin’s</a> 1796 Zoönomia and <a href="https://en.wikipedia.org/wiki/Jean-Baptiste_Lamarck">Jean-Baptiste Lamarck’s</a> Philosophie Zoologique of 1809. The idea was popularized in the Anglophone world by the speculative but widely read <a href="https://en.wikipedia.org/wiki/Vestiges_of_the_Natural_History_of_Creation">Vestiges of the Natural History of Creation</a>, published anonymously by <a href="https://en.wikipedia.org/wiki/Robert_Chambers_(publisher,_born_1802)">Robert Chambers</a> in 1844.</p>
<p>With Darwin’s theory, a general acceptance quickly appeared that a classification should reflect the Darwinian principle of common descent. Tree of life representations became popular in scientific works, with known fossil groups incorporated. One of the first modern groups tied to fossil ancestors was birds. Using the then newly discovered fossils of Archaeopteryx and Hesperornis, Thomas Henry Huxley pronounced that they had evolved from dinosaurs, a group formally named by <a href="https://en.wikipedia.org/wiki/Richard_Owen">Richard Owen</a> in 1842. The resulting description, that of dinosaurs “giving rise to” or being “the ancestors of” birds, is the essential hallmark of evolutionary taxonomic thinking. As more and more fossil groups were found and recognized in the late 19th and early 20th centuries, paleontologists worked to understand the history of animals through the ages by linking together known groups. With the modern evolutionary synthesis of the early 1940s, an essentially modern understanding of the evolution of the major groups was in place.</p>
<p>The cladistic method has emerged since the 1960s. In 1958, <a href="https://en.wikipedia.org/wiki/Julian_Huxley">Julian Huxley</a> used the term clade. Later, in 1960, Cain and Harrison introduced the term cladistic. The salient feature is arranging taxa in a hierarchical evolutionary tree, ignoring ranks. A taxon is called monophyletic, if it includes all the descendants of an ancestral form. Groups that have descendant groups removed from them (e.g. dinosaurs, with birds as offspring group) are termed paraphyletic, while groups representing more than one branch from the tree of life are called polyphyletic. The International Code of Phylogenetic Nomenclature or PhyloCode is intended to regulate the formal naming of clades. Linnaean ranks will be optional under the PhyloCode, which is intended to coexist with the current, rank-based codes.</p>
<p>Bacteria were at first classified based solely on their shape (vibrio, bacillus, coccus etc.), presence of endospores, gram stain, aerobic conditions and motility. This system changed with the study of metabolic phenotypes, where metabolic characteristics were used. Recently, with the advent of molecular phylogeny, several genes are used to identify species, the most important of which is the 16S rRNA gene, followed by 23S, ITS region, gyrB and others to confirm a better resolution. The quickest way to identify to match an isolated strain to a species or genus today is done by amplifying it’s 16S gene with universal primers and sequence the 1.4kb amplicon and submit it to a specialised web-based identification database, namely either Ribosomal Database Project, which align the sequence to other 16S sequences using infernal, a secondary structure bases global alignment, or ARB SILVA, which aligns sequences via SINA (SILVA incremental aligner), which does a local alignment of a seed and extends it .</p>
<p>Several identification methods exists:</p>
<ul>
<li>Phenotypic analyses
<ul>
<li>fatty acid analyses</li>
<li>Growth conditions (Agar plate, Biolog multiwell plates)</li>
</ul></li>
<li>Genetic analyses
<ul>
<li>DNA-DNA hybridization</li>
<li>DNA profiling</li>
<li>Sequence</li>
<li>GC ratios</li>
</ul></li>
<li>Phylogenetic analyses
<ul>
<li>16S-based phylogeny</li>
<li>phylogeny based on other genes</li>
<li>Multi-gene sequence analysis</li>
<li>Whole-genome sequence based analysis</li>
</ul></li>
</ul>
</div>
<div id="kingdoms-and-domains" class="section level3" number="1.7.4">
<h3><span class="header-section-number">1.7.4</span> Kingdoms And Domains</h3>
<p>Well before Linnaeus, plants and animals were considered separate Kingdoms. Linnaeus used this as the top rank, dividing the physical world into the plant, animal and mineral kingdoms. As advances in microscopy made classification of microorganisms possible, the number of kingdoms increased, five and six-kingdom systems being the most common.</p>
<p>When Carl Linnaeus introduced the rank-based system of nomenclature into biology in 1735, the highest rank was given the name “kingdom” and was followed by four other main or principal ranks: class, order, genus and species. Later two further main ranks were introduced, making the sequence kingdom, phylum or division, class, order, family, genus and species. In 1990, the rank of domain was introduced above kingdom.</p>
<p>Prefixes can be added so subkingdom (subregnum) and infrakingdom (also known as infraregnum) are the two ranks immediately below kingdom. Superkingdom may be considered as an equivalent of domain or empire or as an independent rank between kingdom and domain or subdomain. In some classification systems the additional rank branch (Latin: ramus) can be inserted between subkingdom and infrakingdom, e.g., Protostomia and Deuterostomia in the classification of Cavalier-Smith.</p>
<p>Some recent classifications based on modern cladistics have explicitly abandoned the term “kingdom”, noting that the traditional kingdoms are not monophyletic, i.e., do not consist of all the descendants of a common ancestor.</p>
<p>By tradition, the binomial names of species are usually typeset in italics; for example, <em>Homo sapiens</em>. Generally, the binomial should be printed in a font style different from that used in the normal text; for example, “Several more <em>Homo sapiens</em> fossils were discovered.” When handwritten, a binomial name should be underlined; for example, <em>Homo sapiens</em>.</p>
<p>The first part of the binomial, the genus name, is always written with an initial capital letter. In current usage, the second part is never written with an initial capital. Older sources, particularly botanical works published before the 1950s, use a different convention. If the second part of the name is derived from a proper noun, e.g. the name of a person or place, a capital letter was used. Thus the modern form Berberis darwinii was written as Berberis Darwinii. A capital was also used when the name is formed by two nouns in apposition, e.g. <em>Panthera Leo</em> or <em>Centaurea Cyanus</em>.</p>
<p>When used with a common name, the scientific name often follows in parentheses, although this varies with publication. For example, “The house sparrow (<em>Passer domesticus</em>) is decreasing in Europe.”</p>
<p>The binomial name should generally be written in full. The exception to this is when several species from the same genus are being listed or discussed in the same paper or report, or the same species is mentioned repeatedly; in which case the genus is written in full when it is first used, but may then be abbreviated to an initial (and a period/full stop). For example, a list of members of the genus Canis might be written as “Canis lupus, C. aureus, C. simensis”. In rare cases, this abbreviated form has spread to more general use; for example, the bacterium Escherichia coli is often referred to as just E. coli, and Tyrannosaurus rex is perhaps even better known simply as T. rex, these two both often appearing in this form in popular writing even where the full genus name has not already been given.</p>
</div>
<div id="the-three-domain-system" class="section level3" number="1.7.5">
<h3><span class="header-section-number">1.7.5</span> The Three-Domain System</h3>
<p>The three-domain system is a biological classification first proposed in 1977 by <a href="https://en.wikipedia.org/wiki/Carl_Woese">Carl Woese’s</a> that divides cellular life forms into <a href="https://en.wikipedia.org/wiki/Archaea">Archaea</a>, <a href="https://en.wikipedia.org/wiki/Bacteria">Bacteria</a> and <a href="https://en.wikipedia.org/wiki/Eukaryote">Eukaryota</a>. The key difference from earlier classifications is the splitting of archaea and bacteria, previously grouped into the single kingdom Bacteria (a kingdom also sometimes called Monera).</p>
<p>The three-domain system adds a level of classification (the domains) “above” the kingdoms present in the previously used five- or six-kingdom systems. This classification system recognizes the fundamental divide between the two prokaryotic groups, insofar as Archaea appear to be more closely related to Eukaryotes than they are to other prokaryotes – bacteria-like organisms with no cell nucleus. The current system sorts the previously known kingdoms into these three domains: Archaea, Bacteria, and Eukarya.</p>
<p>Woese argued, on the basis of differences in 16S rRNA genes, that bacteria, archaea, and eukaryotes each arose separately from an ancestor with poorly developed genetic machinery, often called a progenote. To reflect these primary lines of descent, he treated each as a domain, divided into several different kingdoms. Originally his split of the prokaryotes was into Eubacteria (now Bacteria) and Archaebacteria (now Archaea). Woese initially used the term “kingdom” to refer to the three primary phylogenic groupings, and this nomenclature was widely used until the term “domain” was adopted in 1990.</p>
<p>Acceptance of the validity of Woese’s phylogenetically valid classification was a slow process. Prominent biologists including Salvador Luria and Ernst Mayr objected to his division of the prokaryotes. Not all criticism of him was restricted to the scientific level. A decade of labor-intensive oligonucleotide cataloging left him with a reputation as “a crank,” and Woese would go on to be dubbed as “Microbiology’s Scarred Revolutionary” by a news article printed in the journal Science. The growing amount of supporting data led the scientific community to accept the Archaea by the mid-1980s. Today, few scientists cling to the idea of a unified Prokarya.</p>
</div>
<div id="bergeys-manual-of-systematic-bacteriology" class="section level3" number="1.7.6">
<h3><span class="header-section-number">1.7.6</span> Bergey’s Manual of Systematic Bacteriology</h3>
<p><a href="https://onlinelibrary.wiley.com/doi/book/10.1002/9781118960608">Bergey’s Manual of Systematics of Archaea and Bacteria (BMSAB)</a> is the main resource for determining the identity of prokaryotic organisms, emphasizing bacterial species, using every characterizing aspect.</p>
<p>The manual was published subsequent to the Bergey’s Manual of Determinative Bacteriology, though the latter is still published as a guide for identifying unknown bacteria. First published in 1923 by David Hendricks Bergey, it is used to classify bacteria based on their structural and functional attributes by arranging them into specific familial orders. However, this process has become more empirical in recent years.</p>
<p>The Taxonomic Outline of Bacteria and Archaea is a derived publication indexing taxon names from version two of the manual. It used to be available for free from the Bergey’s manual trust website until September 2018. Michigan State University provides an alternative version that indexes NamesforLife records.</p>
<p>The change in volume set to “Systematic Bacteriology” came in a new contract in 1980, whereupon the new style included “relationships between organisms” and had “expanded scope” overall. This new style was picked up for a four-volume set that first began publishing in 1984. The information in the volumes was separated as:</p>
<p>Volume 1 included information on all types of Gram-negative bacteria that were considered to have “medical and industrial importance.” Volume 2 included information on all types of Gram-positive bacteria. Volume 3 deals with all of the remaining, slightly different Gram-negative bacteria, along with the Archaea. Volume 4 has information on filamentous actinomycetes and other, similar bacteria.</p>
<p>The current volumes differ drastically from previous volumes in that many higher taxa are not defined in terms of phenotype, but solely on 16S phylogeny, as is the case of the classes within Proteobacteria.</p>
<p>The current grouping is:</p>
<ul>
<li>Volume 1 (2001): The Archaea and the deeply branching and phototrophic Bacteria</li>
<li>Volume 2 (2005): The Proteobacteria—divided into three books:
<ul>
<li>2A: Introductory essays</li>
<li>2B: The Gammaproteobacteria</li>
<li>2C: Other classes of Proteobacteria</li>
</ul></li>
<li>Volume 3 (2009): The Firmicutes</li>
<li>Volume 4 (2011): The Bacteroidetes, Spirochaetes, Tenericutes (Mollicutes), Acidobacteria, Fibrobacteres, Fusobacteria, Dictyoglomi, Gemmatimonadetes, Lentisphaerae, Verrucomicrobia, Chlamydiae, and Planctomycetes</li>
<li>Volume 5 (in two parts) (2012): The Actinobacteria
Bergey’s manual of systematics of archaea and bacteria (2015), an online book, replaces the five-volume set.</li>
</ul>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="acknowledgements.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="the-chemistry-of-biology.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/microbiology-text/edit/master/main_themes.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
